<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Mcs.ogmCellNo (cellNo-organism 0-1-0.2020-03-26 draft)</title>
  <meta name="keywords" content="cellNo-organism, ogmCellNo, ModelConceptSensorial, McsHitp, Synagonism">
  <link rel="stylesheet" href="https://synagonism.github.io/hitp/hitp.css">
</head>

<body>
<header id="idHeader">
  <p></p>
  <h1 id="idHeaderH1">cellNo-organism
    <br>sensorial-concept-Mcs (ogmCellNo)
    </h1>
  <p id="idHeadercrd">McsHitp-creation:: {2020-03-26}
    <a class="clsHide" href="#idHeadercrd"></a></p>
</header>

<section id="idOverview">
  <h1 id="idOverviewH1">overview of ogmCellNo
    <a class="clsHide" href="#idOverviewH1"></a></h1>
  <p id="idName">name::
    <br>* Mcs.filMcsOgmCllN.last.html!⇒ogmCellNo,
    <br>* Mcs.dirOgm/filMcsOgmCllN.last.html!⇒ogmCellNo,
    <br>* Mcs.ogmCellNo,
    <br>* Mcs.ogm.011-cellNo!⇒ogmCellNo,
    <br>* Mcs.ogm.cellNo-011!⇒ogmCellNo,
    <br>* Mcs.acellular-ogm!⇒ogmCellNo,
    <br>* Mcs.noncellular-ogm!⇒ogmCellNo,
    <br>* Mcs.cellNo-organism!⇒ogmCellNo,
    <br>* Mcs.ogmCellNo'(cellNo-organism)!⇒ogmCellNo,
    <a class="clsHide" href="#idName"></a></p>
  <p id="idDescription">description::
    <br>· acellular-ogm is an-organism without cells.
    <br>===
    <br>"Non-cellular life, or acellular life is life that exists without a cellular structure for at least part of its life cycle.[1] Historically, most (descriptive) definitions of life postulated that a living organism must be composed of one or more cells,[2] but this is no longer considered necessary, and modern criteria allow for forms of life based on other structural arrangements.[3][4][5][3]
    <br>The primary candidates for non-cellular life are viruses. A minority of biologists consider viruses to be living organisms, but most do not. Their primary objection is that no known viruses are capable of autonomous reproduction: they must rely on cells to copy them.[1][6][7][8][9] However, the recent discovery of giant viruses that possess genes for part of the required translation machinery has raised the prospect that they may have had extinct ancestors that could evolve and replicate independently. Most biologists agree that such an ancestor would be a true non-cellular lifeform, but its existence and characteristics are still uncertain.[1][10][11][12][13]
    <br>Engineers sometimes use the term "artificial life" to refer to software and robots inspired by biological processes, but these do not satisfy any biological definition of life."
    <br>[https://en.wikipedia.org/wiki/Non-cellular_life]
    <a class="clsHide" href="#idDescription"></a></p>
</section>

<section id="idOgmCllNrscF">
  <h1 id="idOgmCllNrscFH1">resource of ogmCellNo
    <a class="clsHide" href="#idOgmCllNrscFH1"></a></h1>
  <p id="idOgmCllNrscnam">name::
    <br>* Mcs.ogmCellNo'attResource,
    <br>* Mcs.ogmCellNo'resource,
    <a class="clsHide" href="#idOgmCllNrscnam"></a></p>
  <p id="idOgmCllNrscwpa">addressWpg::
    <br>*
    <a class="clsHide" href="#idOgmCllNrscwpa"></a></p>
</section>

<section id="idOgmCllNsrtrF">
  <h1 id="idOgmCllNsrtrFH1">structure of ogmCellNo
    <a class="clsHide" href="#idOgmCllNsrtrFH1"></a></h1>
  <p id="idOgmCllNsrtrFnam">name::
    <br>* Mcs.ogmCellNo'attStructure,
    <br>* Mcs.ogmCellNo'structure,
    <a class="clsHide" href="#idOgmCllNsrtrFnam"></a></p>
  <p id="idOgmCllNsrtrFdsn">description::
    <br>·
    <a class="clsHide" href="#idOgmCllNsrtrFdsn"></a></p>
</section>

<section id="idOgmCllNdngF">
  <h1 id="idOgmCllNdngFH1">DOING of ogmCellNo
    <a class="clsHide" href="#idOgmCllNdngFH1"></a></h1>
  <p id="idOgmCllNdngFnam">name::
    <br>* Mcs.ogmCellNo'attDoing,
    <br>* Mcs.ogmCellNo'doing,
    <a class="clsHide" href="#idOgmCllNdngFnam"></a></p>
  <p id="idOgmCllNdngFdsn">description::
    <br>·
    <a class="clsHide" href="#idOgmCllNdngFdsn"></a></p>
</section>

<section id="idOgmCllNevgF">
  <h1 id="idOgmCllNevgFH1">EVOLUTING of ogmCellNo
    <a class="clsHide" href="#idOgmCllNevgFH1"></a></h1>
  <p id="idOgmCllNevgnam">name::
    <br>* Mcs.ogmCellNo'attEvoluting,
    <br>* Mcs.ogmCellNo'evoluting,
    <a class="clsHide" href="#idOgmCllNevgnam"></a></p>
  <p id="idOgmCllNevg20200326">{time.2020-03-26}::
    <br>=== McsHitp-creation:
    <br>· creation of current <a class="clsPreview" href="../dirCor/filMcs.last.html#idOverview">concept</a>.
    <a class="clsHide" href="#idOgmCllNevg20200326"></a></p>
</section>

<section id="idOgmCllNwptF">
  <h1 id="idOgmCllNwptFH1">WHOLE-PART-TREE of ogmCellNo
    <a class="clsHide" href="#idOgmCllNwhlFH1"></a></h1>
  <p id="idOgmCllNwptnam">name::
    <br>* Mcs.ogmCellNo'whole-part-tree,
    <a class="clsHide" href="#idOgmCllNwptnam"></a></p>
  <p id="idOgmCllNwtr">whole-tree-of-ogmCellNo::
    <br>*
    <a class="clsHide" href="#idOgmCllNwtr"></a></p>
  <p id="idOgmCllNptr">part-tree-of-ogmCellNo::
    <br>*
    <a class="clsHide" href="#idOgmCllNptr"></a></p>
</section>

<section id="idOgmCllNgst">
  <h1 id="idOgmCllNgstH1">GENERIC-SPECIFIC-TREE of ogmCellNo
    <a class="clsHide" href="#idOgmCllNgstH1"></a></h1>
  <p id="idOgmCllNgstnam">name::
    <br>* Mcs.ogmCellNo'generic-specific-tree,
    <a class="clsHide" href="#idOgmCllNgstnam"></a></p>

  <section id="idOgmCllNgtr">
  <h2 id="idOgmCllNgtrH2">GENERIC-TREE of ogmCellNo
    <a class="clsHide" href="#idOgmCllNgtrH2"></a></h2>
  <p id="idOgmCllNgnc">generic-of-ogmCellNo::
    <br>* <a class="clsPreview" href="filMcsOgm.last.html#idOgmgtr">organism</a>,
    <a class="clsHide" href="#idOgmCllNgnc"></a></p>
  <p id="idOgmCllNatttr">attribute-tree-of-ogmCellNo::
    <br>* ,
    <a class="clsHide" href="#idOgmCllNatttr"></a></p>
  <p id="idOgmCllNattihf">att-tree-inherited-from::
    <br>· :
    <br>* ,
    <a class="clsHide" href="#idOgmCllNattihf"></a></p>
  <p id="idOgmCllNattown">att-tree-own-of-ogmCellNo::
    <br>* ,
    <a class="clsHide" href="#idOgmCllNattown"></a></p>
  </section>

  <section id="idOgmCllNstr">
  <h2 id="idOgmCllNstrH2">SPECIFIC-TREE of ogmCellNo
    <a class="clsHide" href="#idOgmCllNstrH2"></a></h2>
  <p id="idOgmCllNspc">specific-of-ogmCellNo::
    <br>* ,
    <a class="clsHide" href="#idOgmCllNspc"></a></p>
  </section>
</section>

<section id="idOgmVrs">
  <h1 id="idOgmVrsH1">ogmCellNo.virus-001
    <a class="clsHide" href="#idOgmVrsH1"></a></h1>
  <p id="idOgmVrsnam">name::
    <br>* Mcs.ogmCellNo.001-virus,
    <br>* Mcs.ogmCellNo.virus-001,
    <br>* Mcs.virusOgm-001,
    <a class="clsHide" href="#idOgmVrsnam"></a></p>
  <p id="idOgmVrsdsn">description::
    <br>"A virus is a microscopic infectious agent that replicates only inside the living cells of an organism. Viruses can infect all types of life forms, from animals and plants to microorganisms, including bacteria and archaea.[1] Since Dmitri Ivanovsky's 1892 article describing a non-bacterial pathogen infecting tobacco plants, and the discovery of the tobacco mosaic virus by Martinus Beijerinck in 1898,[2] about 5,000 virus species have been described in detail,[3] of the millions of types of viruses in the environment.[4] Viruses are found in almost every ecosystem on Earth and are the most numerous type of biological entity.[5][6] The study of viruses is known as virology, a sub-speciality of microbiology.
    <br>While not inside an infected cell or in the process of infecting a cell, viruses exist in the form of independent particles, or virions, consisting of: (i) the genetic material, i.e. long molecules of DNA or RNA that encode the structure of the proteins by which the virus acts; (ii) a protein coat, the capsid, which surrounds and protects the genetic material; and in some cases (iii) an outside envelope of lipids. The shapes of these virus particles range from simple helical and icosahedral forms to more complex structures. Most virus species have virions too small to be seen with an optical microscope, about one hundredth the size of most bacteria.
    <br>The origins of viruses in the evolutionary history of life are unclear: some may have evolved from plasmids—pieces of DNA that can move between cells—while others may have evolved from bacteria. In evolution, viruses are an important means of horizontal gene transfer, which increases genetic diversity in a way analogous to sexual reproduction.[7] Viruses are considered by some to be a life form, because they carry genetic material, reproduce, and evolve through natural selection, although they lack key characteristics (such as cell structure) that are generally considered necessary to count as life. Because they possess some but not all such qualities, viruses have been described as "organisms at the edge of life",[8] and as replicators.[9]
    <br>Viruses spread in many ways. One transmission pathway is through disease-bearing organisms known as vectors: for example, viruses are often transmitted from plant to plant by insects that feed on plant sap, such as aphids; and viruses in animals can be carried by blood-sucking insects. Influenza viruses are spread by coughing and sneezing. Norovirus and rotavirus, common causes of viral gastroenteritis, are transmitted by the faecal–oral route, passed by contact and entering the body in food or water. HIV is one of several viruses transmitted through sexual contact and by exposure to infected blood. The variety of host cells that a virus can infect is called its "host range". This can be narrow, meaning a virus is capable of infecting few species, or broad, meaning it is capable of infecting many.[10]
    <br>Viral infections in animals provoke an immune response that usually eliminates the infecting virus. Immune responses can also be produced by vaccines, which confer an artificially acquired immunity to the specific viral infection. Some viruses, including those that cause AIDS, HPV infection, and viral hepatitis, evade these immune responses and result in chronic infections. Several antiviral drugs have been developed."
    <br>[https://en.wikipedia.org/wiki/Virus {2020-03-26}]
    <a class="clsHide" href="#idOgmVrsdsn"></a></p>

  <section id="idOgmVrsgst">
  <h2 id="idOgmVrsgstH2">GENERIC-SPECIFIC-TREE of virusOgm
    <a class="clsHide" href="#idOgmVrsgstH2"></a></h2>
  <p id="idOgmVrsgstnam">name::
    <br>* Mcs.virusOgm'generic-specific-tree,
    <a class="clsHide" href="#idOgmVrsgstnam"></a></p>

  <section id="idOgmVrsgtr">
  <h3 id="idOgmVrsgtrH3">GENERIC-TREE of virusOgm
    <a class="clsHide" href="#idOgmVrsgtrH3"></a></h3>
  <p id="idOgmVrsgnc">generic-of-virusOgm::
    <br>*
    <a class="clsHide" href="#idOgmVrsgnc"></a></p>
  <p id="idOgmVrsatttr">attribute-tree-of-virusOgm::
    <br>* ,
    <a class="clsHide" href="#idOgmVrsatttr"></a></p>
  <p id="idOgmVrsattihf">att-tree-inherited-from::
    <br>· :
    <br>* ,
    <a class="clsHide" href="#idOgmVrsattihf"></a></p>
  <p id="idOgmVrsattown">att-tree-own-of-virusOgm::
    <br>* ,
    <a class="clsHide" href="#idOgmVrsattown"></a></p>
  </section>

  <section id="idOgmVrsstr">
  <h3 id="idOgmVrsstrH3">SPECIFIC-TREE of virusOgm
    <a class="clsHide" href="#idOgmVrsstrH3"></a></h3>
  <p id="idOgmVrsspc">specific-of-virusOgm::
    <br>"Virus classification is the process of naming viruses and placing them into a taxonomic system. Similar to the classification systems used for cellular organisms, virus classification is the subject of ongoing debate and proposals. This is mainly due to the pseudo-living nature of viruses, which is to say they are non-living particles with some chemical characteristics similar to those of life, or non-cellular life. As such, they do not fit neatly into the established biological classification system in place for cellular organisms.
    <br>Viruses are mainly classified by phenotypic characteristics, such as morphology, nucleic acid type, mode of replication, host organisms, and the type of disease they cause. The formal taxonomic classification of viruses is the responsibility of the International Committee on Taxonomy of Viruses (ICTV) system, although the Baltimore classification system can be used to place viruses into one of seven groups based on their manner of mRNA synthesis. Specific naming conventions and further classification guidelines are set out by the ICTV.
    <br>A catalogue of all the world's known viruses has been proposed; some related preliminary efforts have been accomplished.[1]"
    <br>[https://en.wikipedia.org/wiki/Virus_classification]
    <br>===
    <br>"Since Dmitri Ivanovsky's 1892 article describing a non-bacterial pathogen infecting tobacco plants, and the discovery of the tobacco mosaic virus by Martinus Beijerinck in 1898,[2] about 5,000 virus species have been described in detail,[3] of the millions of types of viruses in the environment."
    <a class="clsHide" href="#idOgmVrsspc"></a></p>
  </section>
  </section>

  <section id="idOgmVrs001">
  <h2 id="idOgmVrs001H2">virusOgm.enveloped-001
    <a class="clsHide" href="#idOgmVrs001H2"></a></h2>
  <p id="idOgmVrs001nam">name::
    <br>* Mcs.virusOgm.001-enveloped,
    <br>* Mcs.virusOgm.enveloped-001,
    <br>* Mcs.enveloped-virusOgm-001,
    <a class="clsHide" href="#idOgmVrs001nam"></a></p>
  <p id="idOgmVrs001dsn">description::
    <br>"Some viruses (e.g. HIV and many animal viruses) have viral envelopes as their outer layer[1] at the stage of their life-cycle when they are between host cells. Some enveloped viruses also have a protein layer called a capsid between the envelope and their genome [1]. The envelopes are typically derived from portions of the host cell membranes (phospholipids and proteins), but include some viral glycoproteins. They may help viruses avoid the host immune system. Glycoproteins on the surface of the envelope serve to identify and bind to receptor sites on the host's membrane. The viral envelope then fuses with the host's membrane, allowing the capsid and viral genome to enter and infect the host.
    <br>The cell from which the virus itself buds will often die or be weakened and shed more viral particles for an extended period. The lipid bilayer envelope of these viruses is relatively sensitive to desiccation, heat, and detergents, therefore these viruses are easier to sterilize than non-enveloped viruses, have limited survival outside host environments, and typically must transfer directly from host to host. Enveloped viruses possess great adaptability and can change in a short time in order to evade the immune system. Enveloped viruses can cause persistent infections.
    <br>Classes of enveloped viruses that contain human pathogens:
    <br>* DNA viruses:
    <br>** Herpesviruses,
    <br>** Poxviruses,
    <br>** Hepadnaviruses,
    <br>** Asfarviridae,
    <br>* RNA viruses:
    <br>** Flavivirus,
    <br>** Alphavirus,
    <br>** Togavirus,
    <br>** Coronavirus,
    <br>** Hepatitis D,
    <br>** Orthomyxovirus,
    <br>** Paramyxovirus,
    <br>** Rhabdovirus[2],
    <br>** Bunyavirus,
    <br>** Filovirus,
    <br>* Retroviruses",
    <br>[https://en.wikipedia.org/wiki/Viral_envelope {2020-03-26}]
    <a class="clsHide" href="#idOgmVrs001dsn"></a></p>
  </section>
</section>

<section id="idMeta">
  <h1 id="idMetaH1">meta-info
    <a class="clsHide" href="#idMetaH1"></a></h1>
  <p id="idMetaCounter" class="clsCenter">this page was-visited
    <span class="clsColorRed">
    <script src="../../dirPgm/dirCntr/counter.php?page=Mws-dirOgm.OgmCllN"></script>
    </span>
    times since {2020-03-26}</p>
  <!-- the content of page-path paragraph is displayed as it is on top of toc -->
  <p id="idMetaWebpage_path"><span class="clsB clsColorGreen">page-wholepath</span>:
    <a class="clsPreview" href="../../#idOverview">synagonism.net</a> /
    <a class="clsPreview" href="../filMcsWorld.last.html#idOverview">Mcs-worldview</a> /
    <a class="clsPreview" href="filMcsDirOgm.last.html#idOverview">dirOgm</a> /
    ogmCellNo
    </p>
  <p id="idMetaP1">SEARCH::
    <br>· this page uses '<span class="clsColorRed">locator-names</span>', names that when you find them, you find the-LOCATION of the-concept they denote.
    <br>⊛ <strong>GLOBAL-SEARCH</strong>:
    <br>· clicking on <span class="clsColorGreenBg">the-green-BAR of a-page</span> you have access to the-global--locator-names of my-site.
    <br>· use the-prefix '<span class="clsColorRed">ogmCellNo</span>' for <a class="clsPreview" href="../dirCor/filMcs.last.html#idOverview">sensorial-concepts</a> related to current concept 'cellNo-organism'.
    <br>⊛ <strong>LOCAL-SEARCH</strong>:
    <br>· TYPE <span class="clsColorRed">CTRL+F "Mcs.words-of-concept's-name"</span>, to go to the-LOCATION of the-concept.
    <br>· a-preview of the-description of a-global-name makes reading fast.
    <a class="clsHide" href="#idMetaP1"></a></p>
  <p id="idFooterP1">footer::
    <br>• author: <a class="clsPreview" href="../../#idAboutme">Kaseluris.Nikos.1959</a>
    <br>• email:
    <br> &nbsp;<img src="../../misc/img/mail.png" alt="imgMail">
    <br>• edit on github: https://github.com/synagonism/Mcsw/blob/master/dirOgm/filMcsOgmCllN.last.html,
    <br>• comments on <a class="clsPreview" href="filMcsDirOgm.last.html#idComment">Disqus</a>,
    <br>• twitter: <a href="https://twitter.com/synagonism">@synagonism</a>,
    <br>• steemit: <a href="https://steemit.com/synagonism">https://steemit.com/@synagonism</a>,
    <a class="clsHide" href="#idFooterP1"></a></p>
  <!--                              -->
  <p id="idMetaVersion">webpage-versions::
    <br>• <a href="filMcsOgmCllN.last.html">filMcsOgmCllN.last.html</a>: dynamic,
    <br>• filMcsOgmCllN.0-1-0.2020-03-26.last.html: draft creation,
    <a class="clsHide" href="#idMetaVersion"></a></p>
</section>

<section id="idSupport">
  <h1 id="idSupportH1">support (<a class="clsPreview" href="../../#idSupport">link</a>)</h1>
  <p></p>
</section>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-19285371-5', 'synagonism.net');
  ga('send', 'pageview');
</script>

<script src="https://synagonism.github.io/hitp/hitp.js"></script>
</body>
</html>