<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Mcs.molProtein (protein-of-organism 0-12-0.2020-05-14 draft)</title>
  <meta name="keywords" content="protein-of-organism, molProtein, ModelConceptSensorial, McsHitp, Synagonism">
  <link rel="stylesheet" href="https://synagonism.github.io/hitp/hitp.css">
</head>

<body>
<header id="idHeader">
  <p></p>
  <h1 id="idHeaderH1">protein-material--of-organism
    <br>sensorial-concept-Mcs (molProtein)
    </h1>
  <p id="idHeadercrd">McsHitp-creation:: {2020-03-25},
    <a class="clsHide" href="#idHeadercrd"></a></p>
</header>

<section id="idOverview">
  <h1 id="idOverviewH1">overview of molProtein
    <a class="clsHide" href="#idOverviewH1"></a></h1>
  <p id="idName">name::
    <br>* Mcs.filMcsOgm-prtn.last.html!⇒molProtein,
    <br>* Mcs.dirOgm/filMcsOgm-prtn.last.html!⇒molProtein,
    <br>* Mcs.protein!⇒molProtein,
    <br>* Mcs.proteinOgm!⇒molProtein,
    <br>* Mcs.molProtein,
    <br>* Mcs.molProtein'(organism-protein)!⇒molProtein,
    <br>* Mcs.protein-of-organism!⇒molProtein,
    <br>* Mcs.molProtein'(protein-of-organism)!⇒molProtein,
    <a class="clsHide" href="#idName"></a></p>
  <p id="idDescription">description::
    <br>"Proteins are large biomolecules, or macromolecules, consisting of one or more long chains of amino acid residues. Proteins perform a vast array of functions within organisms, including catalysing metabolic reactions, DNA replication, responding to stimuli, providing structure to cells, and organisms, and transporting molecules from one location to another. Proteins differ from one another primarily in their sequence of amino acids, which is dictated by the nucleotide sequence of their genes, and which usually results in protein folding into a specific three-dimensional structure that determines its activity.
    <br>A linear chain of amino acid residues is called a polypeptide. A protein contains at least one long polypeptide. Short polypeptides, containing less than 20–30 residues, are rarely considered to be proteins and are commonly called peptides, or sometimes oligopeptides. The individual amino acid residues are bonded together by peptide bonds and adjacent amino acid residues. The sequence of amino acid residues in a protein is defined by the sequence of a gene, which is encoded in the genetic code. In general, the genetic code specifies 20 standard amino acids; however, in certain organisms the genetic code can include selenocysteine and—in certain archaea—pyrrolysine. Shortly after or even during synthesis, the residues in a protein are often chemically modified by post-translational modification, which alters the physical and chemical properties, folding, stability, activity, and ultimately, the function of the proteins. Sometimes proteins have non-peptide groups attached, which can be called prosthetic groups or cofactors. Proteins can also work together to achieve a particular function, and they often associate to form stable protein complexes.
    <br>Once formed, proteins only exist for a certain period and are then degraded and recycled by the cell's machinery through the process of protein turnover. A protein's lifespan is measured in terms of its half-life and covers a wide range. They can exist for minutes or years with an average lifespan of 1–2 days in mammalian cells. Abnormal or misfolded proteins are degraded more rapidly either due to being targeted for destruction or due to being unstable.
    <br>Like other biological macromolecules such as polysaccharides and nucleic acids, proteins are essential parts of organisms and participate in virtually every process within cells. Many proteins are enzymes that catalyse biochemical reactions and are vital to metabolism. Proteins also have structural or mechanical functions, such as actin and myosin in muscle and the proteins in the cytoskeleton, which form a system of scaffolding that maintains cell shape. Other proteins are important in cell signaling, immune responses, cell adhesion, and the cell cycle. In animals, proteins are needed in the diet to provide the essential amino acids that cannot be synthesized. Digestion breaks the proteins down for use in the metabolism.
    <br>Proteins may be purified from other cellular components using a variety of techniques such as ultracentrifugation, precipitation, electrophoresis, and chromatography; the advent of genetic engineering has made possible a number of methods to facilitate purification. Methods commonly used to study protein structure and function include immunohistochemistry, site-directed mutagenesis, X-ray crystallography, nuclear magnetic resonance and mass spectrometry."
    <br>[https://en.wikipedia.org/wiki/Protein {2020-03-25}]
    <a class="clsHide" href="#idDescription"></a></p>
</section>

<section id="idPrtnOgmatt003">
  <h1 id="idPrtnOgmatt003H1">01_amino-acid of molProtein
    <a class="clsHide" href="#idPrtnOgmatt003H1"></a></h1>
  <p id="idPrtnOgmatt003nam">name::
    <br>* Mcs.molProtein'01_amino-acid!⇒acidAmino,
    <br>* Mcs.molProtein'att003-amino-acid!⇒acidAmino,
    <br>* Mcs.molProtein'amino-acid-att003!⇒acidAmino,
    <br>* Mcs.acidAmino, /amíno/,
    <br>* Mcs.amino-acidOgm!⇒acidAmino,
    <br>* Mcs.amino-acid--chmOgn!⇒acidAmino,
    <br>* Mcs.chmOgn.amino-acid!⇒acidAmino,
    <br>* Mcs.chmOgn.005-amino-acid!⇒acidAmino,
    <a class="clsHide" href="#idPrtnOgmatt003nam"></a></p>
  <p id="idPrtnOgmatt003dsn">description::
    <br>"DNA, RNA, and proteins all consist of a repeating structure of related building blocks (nucleotides in the case of DNA and RNA, amino acids in the case of proteins)."
    <br>[{2020-03-26} https://en.wikipedia.org/wiki/Macromolecule]
    <br>"Amino acids are organic compounds that contain amine (-NH2) and carboxyl (-COOH) functional groups, along with a side chain (R group) specific to each amino acid.[1][2] The key elements of an amino acid are carbon (C), hydrogen (H), oxygen (O), and nitrogen (N), although other elements are found in the side chains of certain amino acids. About 500 naturally occurring amino acids are known (though only 20 appear in the genetic code) and can be classified in many ways.[3] They can be classified according to the core structural functional groups' locations as alpha- (α-), beta- (β-), gamma- (γ-) or delta- (δ-) amino acids; other categories relate to polarity, pH level, and side chain group type (aliphatic, acyclic, aromatic, containing hydroxyl or sulfur, etc.). In the form of proteins, amino acid residues form the second-largest component (water is the largest) of human muscles and other tissues.[4] Beyond their role as residues in proteins, amino acids participate in a number of processes such as neurotransmitter transport and biosynthesis."
    <br>[{2020-05-06} https://en.wikipedia.org/wiki/Amino_acid]
    <a class="clsHide" href="#idPrtnOgmatt003dsn"></a></p>

  <section id="idPrtnOgmatt003gst">
  <h2 id="idPrtnOgmatt003gstH2">GENERIC-SPECIFIC-TREE of acidAmino
    <a class="clsHide" href="#idPrtnOgmatt003gstH2"></a></h2>
  <p id="idPrtnOgmatt003gstnam">name::
    <br>* Mcs.acidAmino'generic-specific-tree,
    <a class="clsHide" href="#idPrtnOgmatt003gstnam"></a></p>

  <section id="idPrtnOgmatt003gtr">
  <h3 id="idPrtnOgmatt003gtrH3">GENERIC-TREE of acidAmino
    <a class="clsHide" href="#idPrtnOgmatt003gtrH3"></a></h3>
  <p id="idPrtnOgmatt003gnc">generic-of-acidAmino::
    <br>*
    <a class="clsHide" href="#idPrtnOgmatt003gnc"></a></p>
  <p id="idPrtnOgmatt003atttr">attribute-tree-of-acidAmino::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmatt003atttr"></a></p>
  <p id="idPrtnOgmatt003attihf">att-tree-inherited-from::
    <br>· :
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmatt003attihf"></a></p>
  <p id="idPrtnOgmatt003attown">att-tree-own-of-acidAmino::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmatt003attown"></a></p>
  </section>

  <section id="idPrtnOgmatt003str">
  <h3 id="idPrtnOgmatt003strH3">SPECIFIC-TREE of acidAmino
    <a class="clsHide" href="#idPrtnOgmatt003strH3"></a></h3>
  <p id="idPrtnOgmatt003spc">specific-of-acidAmino::
    <br>* Amino-Acid, Abbreviation:
    <br>* Alanine, Ala,
    <br>* Arginine, Arg,
    <br>* Asparagine, Asn,
    <br>* Aspartic_acid, Asp,
    <br>* Cysteine, Cys,
    <br>* Glutamine, Gln,
    <br>* Glutamic_acid, Glu,
    <br>* Glycine, Gly,
    <br>* Histidine, His,
    <br>* Isoleucine, Ile,
    <br>* Leucine, Leu,
    <br>* Lysine, Lys,
    <br>* Methionine, Met,
    <br>* Phenylalanine, Phe,
    <br>* Proline, Pro,
    <br>* Serine, Ser,
    <br>* Threonine, Thr,
    <br>* Tryptophan, Trp,
    <br>* Tyrosine, Tyr,
    <br>* Valine, Val,
    <a class="clsHide" href="#idPrtnOgmatt003spc"></a></p>
  </section>
  </section>

  <section id="idPrtnOgmatt003001">
  <h2 id="idPrtnOgmatt003001H2">acidAmino.aggregate
    <a class="clsHide" href="#idPrtnOgmatt003001H2"></a></h2>
  <p id="idPrtnOgmatt003001nam">name::
    <br>* Mcs.acidAmino.aggregate,
    <br>* Mcs.acidAmino.001-aggregate,
    <a class="clsHide" href="#idPrtnOgmatt003001nam"></a></p>
  <p id="idPrtnOgmatt003001dsn">description::
    <br>"About 500 naturally occurring amino acids are known (though only 20 appear in the genetic code) and can be classified in many ways.["
    <br>[{2020-05-11} https://en.wikipedia.org/wiki/Amino_acid]
    <a class="clsHide" href="#idPrtnOgmatt003001dsn"></a></p>
  </section>

  <section id="idPrtnOgmatt003002">
  <h2 id="idPrtnOgmatt003002H2">acidAmino.asparagine
    <a class="clsHide" href="#idPrtnOgmatt003002H2"></a></h2>
  <p id="idPrtnOgmatt003002nam">name::
    <br>* Mcs.,
    <a class="clsHide" href="#idPrtnOgmatt003002nam"></a></p>
  <p id="idPrtnOgmatt003002dsn">description::
    <br>"Asparagine was first isolated in 1806 in a crystalline form by French chemists Louis Nicolas Vauquelin and Pierre Jean Robiquet (then a young assistant) from asparagus juice,[3][4] in which it is abundant, hence the chosen name. It was the first amino acid to be isolated.[5]
    <br>... Asparagine (symbol Asn or N[2]), is an α-amino acid that is used in the biosynthesis of proteins. It contains an α-amino group (which is in the protonated −NH+3 form under biological conditions), an α-carboxylic acid group (which is in the deprotonated −COO− form under biological conditions), and a side chain carboxamide, classifying it as a polar (at physiological pH), aliphatic amino acid. It is non-essential in humans, meaning the body can synthesize it. It is encoded by the codons AAU and AAC.
    <br>A reaction between asparagine and reducing sugars or other source of carbonyls produces acrylamide in food when heated to sufficient temperature. These products occur in baked goods such as French fries, potato chips, and toasted bread."
    <br>[{2020-05-11} https://en.wikipedia.org/wiki/Asparagine]
    <a class="clsHide" href="#idPrtnOgmatt003002dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgmatt004">
  <h1 id="idPrtnOgmatt004H1">02_peptide of molProtein
    <a class="clsHide" href="#idPrtnOgmatt004H1"></a></h1>
  <p id="idPrtnOgmatt004nam">name::
    <br>* Mcs.molProtein'02_peptide,
    <br>* Mcs.molProtein'att004-peptide,
    <br>* Mcs.molProtein'peptide-att004,
    <br>* Mcs.peptide-004,
    <a class="clsHide" href="#idPrtnOgmatt004nam"></a></p>
  <p id="idPrtnOgmatt004dsn">description::
    <br>"Peptides (from Greek language πεπτός, peptós "digested"; derived from πέσσειν, péssein "to digest") are short chains of amino acids linked by peptide (amide) bonds.[1] The simplest peptides are dipeptides, followed by tripeptides, tetrapeptides, etc. A polypeptide is a long, continuous, and unbranched peptide chain. Hence, peptides fall under the broad chemical classes of biological oligomers and polymers, alongside nucleic acids, oligosaccharides, polysaccharides, and others.
    <br>Peptides are distinguished from proteins on the basis of size, and as an arbitrary benchmark can be understood to contain approximately 50 or fewer amino acids.[2][3] Proteins consist of one or more polypeptides arranged in a biologically functional way, often bound to ligands such as coenzymes and cofactors, or to another protein or other macromolecule (DNA, RNA, etc.), or to complex macromolecular assemblies.[4] Finally, while aspects of the lab techniques applied to peptides versus polypeptides and proteins differ (e.g., the specifics of electrophoresis, chromatography, etc.), the size boundaries that distinguish peptides from polypeptides and proteins are not absolute: long peptides such as amyloid beta have been referred to as proteins, and smaller proteins like insulin have been considered peptides.
    <br>Amino acids that have been incorporated into peptides are termed "residues". A water molecule is released during formation of each amide bond.[5] All peptides except cyclic peptides have an N-terminal (amine group) and C-terminal (carboxyl group) residue at the end of the peptide (as shown for the tetrapeptide in the image)."
    <br>[https://en.wikipedia.org/wiki/Peptide {2020-03-26}]
    <a class="clsHide" href="#idPrtnOgmatt004dsn"></a></p>
</section>

<section id="idPrtnOgmatt006">
  <h1 id="idPrtnOgmatt006H1">03_peptide-bond of molProtein
    <a class="clsHide" href="#idPrtnOgmatt006H1"></a></h1>
  <p id="idPrtnOgmatt006nam">name::
    <br>* Mcs.molProtein'03_peptide-bond,
    <br>* Mcs.molProtein'att006-peptide-bond,
    <br>* Mcs.molProtein'peptide-bond-att006,
    <br>* Mcs.peptide-bond-of-molProtein-att006,
    <a class="clsHide" href="#idPrtnOgmatt006nam"></a></p>
  <p id="idPrtnOgmatt006dsn">description::
    <br>"A peptide bond is an amide type of covalent chemical bond linking two consecutive alpha-amino acids from C1 (carbon number one) of one alpha-amino acid and N2 (nitrogen number two) of another along a peptide or protein chain.[1]
    <br>It can also be called an eupeptide bond[1] to separate it from an isopeptide bond, a different type of amide bond between two amino acids."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Peptide_bond]
    <a class="clsHide" href="#idPrtnOgmatt006dsn"></a></p>
</section>

<section id="idPrtnOgmatt005">
  <h1 id="idPrtnOgmatt005H1">04_polypeptide of molProtein
    <a class="clsHide" href="#idPrtnOgmatt005H1"></a></h1>
  <p id="idPrtnOgmatt005nam">name::
    <br>* Mcs.molProtein'04_polypeptide,
    <br>* Mcs.molProtein'att005-polypeptide,
    <br>* Mcs.molProtein'polypeptide-005,
    <br>* Mcs.polypeptide-005,
    <a class="clsHide" href="#idPrtnOgmatt005nam"></a></p>
  <p id="idPrtnOgmatt005dsn">description::
    <br>"A polypeptide is a long, continuous, and unbranched peptide chain."
    <br>[https://en.wikipedia.org/wiki/Peptide {2020-03-26}]
    <a class="clsHide" href="#idPrtnOgmatt005dsn"></a></p>
</section>

<section id="idPrtnOgmatt002">
  <h1 id="idPrtnOgmatt002H1">05_pro-protein of molProtein
    <a class="clsHide" href="#idPrtnOgmatt002H1"></a></h1>
  <p id="idPrtnOgmatt002nam">name::
    <br>* Mcs.molProtein'05_proprotein,
    <br>* Mcs.molProtein'att002-proprotein,
    <br>* Mcs.molProtein'proprotein,
    <br>* Mcs.pro-peptide,
    <br>* Mcs.protein-precursor,
    <br>* Mcs.proprotein,
    <a class="clsHide" href="#idPrtnOgmatt002nam"></a></p>
  <p id="idPrtnOgmatt002dsn">description::
    <br>"A protein precursor, also called a pro-protein or pro-peptide, is an inactive protein (or peptide) that can be turned into an active form by post-translational modification, such as breaking off a piece of the molecule or adding on another molecule. The name of the precursor for a protein is often prefixed by pro-. Examples include proinsulin and proopiomelanocortin, which are both prohormones.
    <br>Protein precursors are often used by an organism when the subsequent protein is potentially harmful, but needs to be available on short notice and/or in large quantities. Enzyme precursors are called zymogens or proenzymes. Examples are enzymes of the digestive tract in humans.
    <br>Some protein precursors are secreted from the cell. Many of these are synthesized with an N-terminal signal peptide that targets them for secretion. Like other proteins that contain a signal peptide, their name is prefixed by pre. They are thus called pre-pro-proteins or pre-pro-peptides.[1] The signal peptide is cleaved off in the endoplasmic reticulum.[1] An example is preproinsulin.
    <br>Pro-sequences are areas in the protein that are essential for its correct folding, usually in the transition of a protein from an inactive to an active state. Pro-sequences may also be involved in pro-protein"
    <br>[https://en.wikipedia.org/wiki/Protein_precursor {2020-02-28}]
    <a class="clsHide" href="#idPrtnOgmatt002dsn"></a></p>
</section>

<section id="idPrtnOgmatt007">
  <h1 id="idPrtnOgmatt007H1">shape of molProtein
    <a class="clsHide" href="#idPrtnOgmatt007H1"></a></h1>
  <p id="idPrtnOgmatt007nam">name::
    <br>* Mcs.molProtein'att007-shape,
    <br>* Mcs.molProtein'shape-att007,
    <a class="clsHide" href="#idPrtnOgmatt007nam"></a></p>
  <p id="idPrtnOgmatt007dsn">description::
    <br>"Protein structure is the three-dimensional arrangement of atoms in an amino acid-chain molecule. Proteins are polymers – specifically polypeptides – formed from sequences of amino acids, the monomers of the polymer. A single amino acid monomer may also be called a residue indicating a repeating unit of a polymer. Proteins form by amino acids undergoing condensation reactions, in which the amino acids lose one water molecule per reaction in order to attach to one another with a peptide bond. By convention, a chain under 30 amino acids is often identified as a peptide, rather than a protein.[1] To be able to perform their biological function, proteins fold into one or more specific spatial conformations driven by a number of non-covalent interactions such as hydrogen bonding, ionic interactions, Van der Waals forces, and hydrophobic packing. To understand the functions of proteins at a molecular level, it is often necessary to determine their three-dimensional structure. This is the topic of the scientific field of structural biology, which employs techniques such as X-ray crystallography, NMR spectroscopy, and dual polarisation interferometry to determine the structure of proteins.
    <br>Protein structures range in size from tens to several thousand amino acids.[2] By physical size, proteins are classified as nanoparticles, between 1–100 nm. Very large aggregates can be formed from protein subunits. For example, many thousands of actin molecules assemble into a microfilament.
    <br>A protein generally undergoes reversible structural changes in performing its biological function. The alternative structures of the same protein are referred to as different conformational isomers, or simply, conformations, and transitions between them are called conformational changes."
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_structure]
    <a class="clsHide" href="#idPrtnOgmatt007dsn"></a></p>
</section>

<section id="idPrtnOgmatt008">
  <h1 id="idPrtnOgmatt008H1">size of molProtein
    <a class="clsHide" href="#idPrtnOgmatt008H1"></a></h1>
  <p id="idPrtnOgmatt008nam">name::
    <br>* Mcs.molProtein'att008-size,
    <br>* Mcs.molProtein'size-att008,
    <a class="clsHide" href="#idPrtnOgmatt008nam"></a></p>
  <p id="idPrtnOgmatt008dsn">description::
    <br>"By physical size, proteins are classified as nanoparticles, between 1–100 nm."
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_structure]
    <a class="clsHide" href="#idPrtnOgmatt008dsn"></a></p>
</section>

<section id="idPrtnOgmrscF">
  <h1 id="idPrtnOgmrscFH1">resource of molProtein
    <a class="clsHide" href="#idPrtnOgmrscFH1"></a></h1>
  <p id="idPrtnOgmrscnam">name::
    <br>* Mcs.molProtein'attResource,
    <br>* Mcs.molProtein'resource,
    <a class="clsHide" href="#idPrtnOgmrscnam"></a></p>
  <p id="idPrtnOgmrscwpa">addressWpg::
    <br>* https://en.wikipedia.org/wiki/List_of_topics_related_to_protein,
    <a class="clsHide" href="#idPrtnOgmrscwpa"></a></p>

  <section id="idPrtnOgmatt001">
  <h2 id="idPrtnOgmatt001H2">proteomics of molProtein
    <a class="clsHide" href="#idPrtnOgmatt001H2"></a></h2>
  <p id="idPrtnOgmatt001nam">name::
    <br>* Mcs.molProtein'att001-proteomics,
    <br>* Mcs.molProtein'proteomics,
    <br>* Mcs.proteomics-science,
    <br>* Mcs.science.proteomics,
    <a class="clsHide" href="#idPrtnOgmatt001nam"></a></p>
  <p id="idPrtnOgmatt001dsn">description::
    <br>"Proteomics is the large-scale study of proteins.[1][2] Proteins are vital parts of living organisms, with many functions. The word proteome is a portmanteau of protein and genome, and was coined by Marc Wilkins in 1994 while he was a Ph.D. student at Macquarie University.[3] Macquarie University also founded the first dedicated proteomics laboratory in 1995.[4][5]
    <br>The proteome is the entire set of proteins that is produced or modified by an organism or system. Proteomics has enabled the identification of ever increasing numbers of protein. This varies with time and distinct requirements, or stresses, that a cell or organism undergoes.[6] Proteomics is an interdisciplinary domain that has benefitted greatly from the genetic information of various genome projects, including the Human Genome Project.[7] It covers the exploration of proteomes from the overall level of protein composition, structure, and activity. It is an important component of functional genomics.
    <br>Proteomics generally refers to the large-scale experimental analysis of proteins and proteomes, but often is used specifically to refer to protein purification and mass spectrometry."
    <br>[https://en.wikipedia.org/wiki/Proteomics {2020-03-11}]
    <a class="clsHide" href="#idPrtnOgmatt001dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgmsrtrF">
  <h1 id="idPrtnOgmsrtrFH1">structure of molProtein
    <a class="clsHide" href="#idPrtnOgmsrtrFH1"></a></h1>
  <p id="idPrtnOgmsrtrFnam">name::
    <br>* Mcs.molProtein'attStructure,
    <br>* Mcs.molProtein'structure,
    <a class="clsHide" href="#idPrtnOgmsrtrFnam"></a></p>
  <p id="idPrtnOgmsrtrFdsn">description::
    <br>"Protein structure is the three-dimensional arrangement of atoms in an amino acid-chain molecule. Proteins are polymers – specifically polypeptides – formed from sequences of amino acids, the monomers of the polymer. A single amino acid monomer may also be called a residue indicating a repeating unit of a polymer. Proteins form by amino acids undergoing condensation reactions, in which the amino acids lose one water molecule per reaction in order to attach to one another with a peptide bond. By convention, a chain under 30 amino acids is often identified as a peptide, rather than a protein.[1] To be able to perform their biological function, proteins fold into one or more specific spatial conformations driven by a number of non-covalent interactions such as hydrogen bonding, ionic interactions, Van der Waals forces, and hydrophobic packing. To understand the functions of proteins at a molecular level, it is often necessary to determine their three-dimensional structure. This is the topic of the scientific field of structural biology, which employs techniques such as X-ray crystallography, NMR spectroscopy, and dual polarisation interferometry to determine the structure of proteins.
    <br>Protein structures range in size from tens to several thousand amino acids.[2] By physical size, proteins are classified as nanoparticles, between 1–100 nm. Very large aggregates can be formed from protein subunits. For example, many thousands of actin molecules assemble into a microfilament.
    <br>A protein generally undergoes reversible structural changes in performing its biological function. The alternative structures of the same protein are referred to as different conformational isomers, or simply, conformations, and transitions between them are called conformational changes."
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Protein_structure]
    <br>"Proteins are the product of a gene that are formed from translation of a mature mRNA molecule.
    <br>Proteins contain 4 elements in regards to their structure: primary, secondary, tertiary and quaternary.
    <br>The linear amino acid sequence is also known as the primary structure.
    <br>Hydrogen bonding between the amino acids of the primary structure results in the formation of alpha helices or beta sheets.[7] These stable foldings are the secondary structure.
    <br>The particular combination of the primary and secondary structures form the tertiary structure of a polypeptide.[7]
    <br>The quaternary structure refers to the way multiple chains of polypeptides fold together.[7]"
    <br>[https://en.wikipedia.org/wiki/Gene_product#Protein_products {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgmsrtrFdsn"></a></p>
</section>

<section id="idPrtnOgmdngF">
  <h1 id="idPrtnOgmdngFH1">DOING of molProtein
    <a class="clsHide" href="#idPrtnOgmdngFH1"></a></h1>
  <p id="idPrtnOgmdngFnam">name::
    <br>* Mcs.molProtein'attDoing,
    <br>* Mcs.molProtein'doing,
    <a class="clsHide" href="#idPrtnOgmdngFnam"></a></p>
  <p id="idPrtnOgmdngFdsn">description::
    <br>"Proteins perform a variety of critical functions as enzymes, structural proteins or hormones and therefore, are crucial biological components."
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Protein_biosynthesis]
    <br>===
    <br>"Proteins are optimised for catalysis
    <br>Proteins are functional macromolecules responsible for catalysing the biochemical reactions that sustain life.[1]:3 Proteins carry out all functions of an organism, for example photosynthesis, neural function, vision, and movement.[13]
    <br>The single-stranded nature of protein molecules, together with their composition of 20 or more different amino acid building blocks, allows them to fold in to a vast number of different three-dimensional shapes, while providing binding pockets through which they can specifically interact with all manner of molecules. In addition, the chemical diversity of the different amino acids, together with different chemical environments afforded by local 3D structure, enables many proteins to act as enzymes, catalyzing a wide range of specific biochemical transformations within cells. In addition, proteins have evolved the ability to bind a wide range of cofactors and coenzymes, smaller molecules that can endow the protein with specific activities beyond those associated with the polypeptide chain alone."
    <br>[https://en.wikipedia.org/wiki/Macromolecule#Proteins_are_optimised_for_catalysis {2020-03-26}]
    <br>===
    <br>"Proteins have many different functions in a cell and the function may vary based on the polypeptides they interact with and their cellular environment. Chaperone proteins work to stabilize newly synthesized proteins. They ensure the new protein folds into its correct functional conformation in addition to making sure products do not aggregate in areas where they should not.[8] Proteins can also function as enzymes, increasing the rate of various biochemical reactions and turning substrates into products.[7][9] Products can be modified by attaching groups such as phosphate via an enzyme to specific amino acids in the primary sequence.[9] Proteins can also be used to move molecules in the cell to where they are needed, these are called motor proteins.[9] The shape of the cell is supported by proteins. Proteins such as actin, microtubules and intermediate filaments provide structure to the cell.[7] Another class of proteins are found in plasma membranes. Membrane proteins can be associated with the plasma membrane in different ways, depending on their structure.[9] These proteins allow the cell to import or export cell products, nutrients or signals to and from the extracellular space.[7][9] Other proteins help the cell to perform regulatory functions. For example, transcription factors bind to DNA to help transcription of RNA.[10]"
    <br>[https://en.wikipedia.org/wiki/Gene_product#Protein_functions {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgmdngFdsn"></a></p>

  <section id="idPrtnOgmatt014">
  <h2 id="idPrtnOgmatt014H2">synthesis of molProtein
    <a class="clsHide" href="#idPrtnOgmatt014H2"></a></h2>
  <p id="idPrtnOgmatt014nam">name::
    <br>* Mcs.molProtein'synthesis,
    <br>* Mcs.molProtein'att014-synthesis,
    <a class="clsHide" href="#idPrtnOgmatt014nam"></a></p>
  <p id="idPrtnOgmatt014dsn">description::
    <br>"Protein biosynthesis (or protein synthesis) is a core biological process, occurring inside cells, balancing the loss of cellular proteins (via degradation or export) through the production of new proteins. Proteins perform a variety of critical functions as enzymes, structural proteins or hormones and therefore, are crucial biological components. Protein synthesis is a very similar process for both prokaryotes and eukaryotes but there are some distinct differences [1].
    <br>Protein synthesis can be divided broadly into two phases - transcription and translation. During transcription, a section of DNA encoding a protein, known as a gene, is converted into a template molecule called messenger RNA. This conversion is carried out by enzymes, known as RNA polymerases, in the nucleus of the cell [2]. In eukaryotes, this messenger RNA (mRNA) is initially produced in a premature form (pre-mRNA) which undergoes post-transcriptional modifications to produce mature mRNA. The mature mRNA is exported from the nucleus via nuclear pores to the cytoplasm of the cell for translation to occur. During translation, the mRNA is read by ribosomes which use the nucleotide sequence of the mRNA to determine the sequence of amino acids. The ribosomes catalyse the formation of covalent peptide bonds between the encoded amino acids to form a polypeptide chain.
    <br>Following translation the polypeptide chain must fold to form a functional protein, for example, to function as an enzyme the polypeptide chain must fold correctly to produce a functional active site. In order to adopt a functional three-dimensional (3D) shape, the polypeptide chain must first form a series of smaller underlying structures called secondary structures. The polypeptide chain in these secondary structures then folds to produce the overall 3D tertiary structure. Once correctly folded, the protein can undergo further maturation through different post-translational modifications. Post-translational modifications can alter the protein's ability to function, where it is located within the cell (e.g. cytoplasm or nucleus) and the protein's ability to interact with other proteins [3].
    <br>Protein biosynthesis has a key role in disease as changes and errors in this process, through underlying DNA mutations or protein misfolding, are often the underlying causes of a disease. DNA mutations change the subsequent mRNA sequence, which then alters the mRNA encoded amino acid sequence. Mutations can cause the polypeptide chain to be shorter by generating a stop sequence which causes early termination of translation. Alternatively, a mutation in the mRNA sequence changes the specific amino acid encoded at that position in the polypeptide chain. This amino acid change can impact the proteins ability to function or to fold correctly [4]. Misfolded proteins are often implicated in disease as improperly folded proteins have a tendency to stick together to form dense protein clumps. These clumps are linked to a range of diseases, often neurological, including Alzheimer's disease and Parkinson's disease [5]."
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Protein_biosynthesis]
    <a class="clsHide" href="#idPrtnOgmatt014dsn"></a></p>

  <section id="idPrtnOgmatt015">
  <h3 id="idPrtnOgmatt015H3">transcription of molProtein
    <a class="clsHide" href="#idPrtnOgmatt015H3"></a></h3>
  <p id="idPrtnOgmatt015nam">name::
    <br>* Mcs.molProtein'transcription,
    <br>* Mcs.molProtein'att015-transcription,
    <a class="clsHide" href="#idPrtnOgmatt015nam"></a></p>
  <p id="idPrtnOgmatt015dsn">description::
    <br>"Transcription occurs in the nucleus using DNA as a template to produce mRNA. In eukaryotes, this mRNA molecule is known as pre-mRNA as it undergoes post-transcriptional modifications in the nucleus to produce a mature mRNA molecule. However, in prokaryotes post-transcriptional modifications are not required so the mature mRNA molecule is immediately produced by transcription [1].
    <br>A pentagon shaped 5 carbon sugar with a base and a phosphate group attached, joined via a phosphodiester bond to another nucleotide's phosphate group
    <br>Initially, an enzyme known as a helicase acts on the molecule of DNA. DNA has an antiparallel, double helix structure composed of two, complementary polynucleotide strands, held together by hydrogen bonds between the base pairs. The helicase disrupts the hydrogen bonds causing a region of DNA - corresponding to a gene - to unwind, separating the two DNA strands and exposing a series of bases. Despite DNA being a double stranded molecule, only one of the strands acts as a template for pre-mRNA synthesis - this strand is known as the template strand. The other DNA strand (which is complementary to the template strand) is known as the coding strand [6].
    <br>Both DNA and RNA have intrinsic directionality, meaning there are two distinct ends of the molecule. This property of directionality is due to the asymmetrical underlying nucleotide subunits, with a phosphate group on one side of the pentose sugar and a base on the other. The five carbons in the pentose sugar are numbered from 1' (where ' means prime) to 5'. Therefore, the phosphodiester bonds connecting the nucleotides are formed by joining the hydroxyl group of on the 3' carbon of one nucleotide to the phosphate group on the 5' carbon of another nucleotide. Hence, the coding strand of DNA runs in a 5' to 3' direction and the complementary, template DNA strand runs in the opposite direction from 3' to 5' [1].
    <br>The enzyme RNA polymerase binds to the exposed template strand and reads from the gene in the 3' to 5' direction. Simultaneously, the RNA polymerase synthesises a single strand of pre-mRNA in the 5'-to-3' direction by catalysing the formation of phosphodiester bonds between activated nucleotides (free in the nucleus) that are capable of complementary base pairing with the template strand. Behind the moving RNA polymerase the two strands of DNA rejoin, so only 12 base pairs of DNA are exposed at one time [6]. RNA polymerase builds the pre-mRNA molecule at a rate of 20 nucleotides per second enabling the production of thousands of pre-mRNA molecules from the same gene in an hour. Despite the fast rate of synthesis, the RNA polymerase enzyme contains its own proofreading mechanism. The proofreading mechanisms allows the RNA polymerase to remove incorrect nucleotides (which are not complementary to the template strand of DNA) from the growing pre-mRNA molecule through an excision reaction [1]. When RNA polymerases reaches a specific DNA sequence which terminates transcription, RNA polymerase detaches and pre-mRNA synthesis is complete [6].
    <br>The pre-mRNA molecule synthesised is complementary to the template DNA strand and shares the same nucleotide sequence as the coding DNA strand. However, there is one crucial difference in the nucleotide composition of DNA and mRNA molecules. DNA is composed of the bases - guanine, cytosine, adenine and thymine (G, C, A and T) - RNA is also composed of four bases - guanine, cytosine, adenine and uracil. In RNA molecules, the DNA base thymine is replaced by uracil which is able to base pair with adenine. Therefore, in the pre-mRNA molecule, all complementary bases which would be thymine in the coding DNA strand are replaced by uracil [7]."
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Protein_biosynthesis]
    <a class="clsHide" href="#idPrtnOgmatt015dsn"></a></p>
  </section>

  <section id="idPrtnOgmatt017">
  <h3 id="idPrtnOgmatt017H3">post-transcription-modification of molProtein
    <a class="clsHide" href="#idPrtnOgmatt017H3"></a></h3>
  <p id="idPrtnOgmatt017nam">name::
    <br>* Mcs.molProtein'post-transcription-modification,
    <br>* Mcs.molProtein'att017-post-transcription-modification,
    <a class="clsHide" href="#idPrtnOgmatt017nam"></a></p>
  <p id="idPrtnOgmatt017dsn">description::
    <br>"Once transcription is complete, the pre-mRNA molecule undergoes post-transcriptional modifications to produce a mature mRNA molecule.
    <br>There are 3 key steps within post-transcriptional modifications:
    <br>- Addition of a 5' cap to the 5' end of the pre-mRNA molecule
    <br>- Addition of a 3' poly(A) tail is added to the 3' end pre-mRNA molecule
    <br>-Removal of introns via RNA splicing
    <br>The 5' cap is added to the 5' end of the pre-mRNA molecule and is composed of a guanine nucleotide modified through methylation. The purpose of the 5' cap is to prevent break down of mature mRNA molecules before translation, the cap also aids binding of the ribosome to the mRNA to start translation [8] and enables mRNA to be differentiated from other RNAs in the cell [1]. In contrast, the 3' Poly(A) tail is added to the 3' end of the mRNA molecule and is composed of 100-200 adenine bases [8]. These distinct mRNA modifications enable the cell to detect that the full mRNA message is intact if both the 5' cap and 3' tail are present [1].
    <br>This modified pre-mRNA molecule then undergoes the process of RNA splicing. Genes are composed of a series of introns and exons, introns are nucleotide sequences which do not encode a protein while, exons are nucleotide sequences that directly encode a protein. Introns and exons are present in both the underlying DNA sequence and the pre-mRNA molecule, therefore, in order to produce a mature mRNA molecule encoding a protein, splicing must occur [6]. During splicing, the intervening introns are removed from the pre-mRNA molecule by a multi-protein complex known as a spliceosome (composed of over 150 proteins and RNA) [9]. This mature mRNA molecule is then exported into the cytoplasm through nuclear pores in the envelope of the nucleus."
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Protein_biosynthesis]
    <a class="clsHide" href="#idPrtnOgmatt017dsn"></a></p>
  </section>

  <section id="idPrtnOgmatt016">
  <h3 id="idPrtnOgmatt016H3">translation of molProtein
    <a class="clsHide" href="#idPrtnOgmatt016H3"></a></h3>
  <p id="idPrtnOgmatt016nam">name::
    <br>* Mcs.molProtein'translation,
    <br>* Mcs.molProtein'att016-translation,
    <a class="clsHide" href="#idPrtnOgmatt016nam"></a></p>
  <p id="idPrtnOgmatt016dsn">description::
    <br>"During translation, ribosomes synthesise polypeptide chains from mRNA template molecules. In eukaryotes, translation occurs in the cytoplasm of the cell, where the ribosomes are located either free floating or attached to the endoplasmic reticulum. In prokaryotes, which lack a nucleus, the processes of both transcription and translation occur in the cytoplasm [10].
    <br>Ribosomes are complex molecular machines, made of a mixture of protein and ribosomal RNA, arranged into two subunits (a large and a small subunit), which surround the mRNA molecule. The ribosome reads the mRNA molecule in a 5'-3' direction and uses it as a template to determine the order of amino acids in the polypeptide chain [11]. In order to translate the mRNA molecule, the ribosome uses small molecules, known as transfer RNAs (tRNA), to deliver the correct amino acids to the ribosome. Each tRNA is composed of 70-80 nucleotides and adopts a characteristic cloverleaf structure due to the formation of hydrogen bonds between the nucleotides within the molecule. There are around 60 different types of tRNAs, each tRNA binds to a specific sequence of three nucleotides (known as a codon) within the mRNA molecule and delivers a specific amino acid [12].
    <br>The ribosome initially attaches to the mRNA at the start codon (AUG) and begins to translate the molecule. The mRNA nucleotide sequence is read in triplets - three adjacent nucleotides in the mRNA molecule correspond to a single codon. Each tRNA has an exposed sequence of three nucleotides, known as the anticodon, which are complementary in sequence to a specific codon that may be present in mRNA. For example, the first codon encountered is the start codon composed of the nucleotides AUG. The correct tRNA with the anticodon (complementary 3 nucleotide sequence UAC) binds to the mRNA using the ribosome. This tRNA delivers the correct amino acid corresponding to the mRNA codon, in the case of the start codon, this is the amino acid methionine. The next codon (adjacent to the start codon) is then bound by the correct tRNA with complementary anticodon, delivering the next amino acid to ribosome. The ribosome then uses its peptidyl transferase enzymatic activity to catalyse the formation of the covalent peptide bond between the two adjacent amino acids [6].
    <br>The ribosome then moves along the mRNA molecule to the third codon. The ribosome then releases the first tRNA molecule, as only two tRNA molecules can be brought together by a single ribosome at one time. The next complementary tRNA with the correct anticodon complementary to the third codon is selected, delivering the next amino acid to the ribosome which is covalently joined to the growing polypeptide chain. This process continues with the ribosome moving along the mRNA molecule adding up to 15 amino acids per second to the polypeptide chain. Behind the first ribosome, up to 50 additional ribosomes can bind to the mRNA molecule forming a polysome, this enables simultaneous synthesis of multiple identical polypeptide chains [6]. Termination of the growing polypeptide chain occurs when the ribosome encounters a stop codon (UAA, UAG, or UGA) in the mRNA molecule. When this occurs, no tRNA can recognise it and a release factor induces the release of the complete polypeptide chain from the ribosome [12]."
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Protein_biosynthesis]
    <a class="clsHide" href="#idPrtnOgmatt016dsn"></a></p>
  </section>

  <section id="idPrtnOgmatt013">
  <h3 id="idPrtnOgmatt013H3">post-translation-modification of molProtein
    <a class="clsHide" href="#idPrtnOgmatt013H3"></a></h3>
  <p id="idPrtnOgmatt013nam">name::
    <br>* Mcs.molProtein'post-translation-modification,
    <br>* Mcs.molProtein'att013-post-translation-modification,
    <br>* Mcs.PTM-post-translation-modification,
    <a class="clsHide" href="#idPrtnOgmatt013nam"></a></p>
  <p id="idPrtnOgmatt013dsn">description::
    <br>"Post-translational modification (PTM) refers to the covalent and generally enzymatic modification of proteins following protein biosynthesis. Proteins are synthesized by ribosomes translating mRNA into polypeptide chains, which may then undergo PTM to form the mature protein product. PTMs are important components in cell signaling, as for example when prohormones are converted to hormones.
    <br>Post-translational modifications can occur on the amino acid side chains or at the protein's C- or N- termini.[1] They can extend the chemical repertoire of the 20 standard amino acids by modifying an existing functional group or introducing a new one such as phosphate. Phosphorylation is a very common mechanism for regulating the activity of enzymes and is the most common post-translational modification.[2] Many eukaryotic and prokaryotic proteins also have carbohydrate molecules attached to them in a process called glycosylation, which can promote protein folding and improve stability as well as serving regulatory functions. Attachment of lipid molecules, known as lipidation, often targets a protein or part of a protein attached to the cell membrane.
    <br>Other forms of post-translational modification consist of cleaving peptide bonds, as in processing a propeptide to a mature form or removing the initiator methionine residue. The formation of disulfide bonds from cysteine residues may also be referred to as a post-translational modification.[3] For instance, the peptide hormone insulin is cut twice after disulfide bonds are formed, and a propeptide is removed from the middle of the chain; the resulting protein consists of two polypeptide chains connected by disulfide bonds.
    <br>Some types of post-translational modification are consequences of oxidative stress. Carbonylation is one example that targets the modified protein for degradation and can result in the formation of protein aggregates.[4][5] Specific amino acid modifications can be used as biomarkers indicating oxidative damage.[6]
    <br>Sites that often undergo post-translational modification are those that have a functional group that can serve as a nucleophile in the reaction: the hydroxyl groups of serine, threonine, and tyrosine; the amine forms of lysine, arginine, and histidine; the thiolate anion of cysteine; the carboxylates of aspartate and glutamate; and the N- and C-termini. In addition, although the amide of asparagine is a weak nucleophile, it can serve as an attachment point for glycans. Rarer modifications can occur at oxidized methionines and at some methylenes in side chains.[7]
    <br>Post-translational modification of proteins can be experimentally detected by a variety of techniques, including mass spectrometry, Eastern blotting, and Western blotting. Additional methods are provided in the external links sections."
    <br>[{2020-05-11} https://en.wikipedia.org/wiki/Post-translational_modification]
    <a class="clsHide" href="#idPrtnOgmatt013dsn"></a></p>
  <p id="idPrtnOgmatt013wpa">addressWpg::
    <br>* https://www.phosphosite.org/homeAction.action,
    <a class="clsHide" href="#idPrtnOgmatt013wpa"></a></p>
  <p id="idPrtnOgmatt013str">specific-tree-of-::
    <br>· The 10 most common experimentally found modifications were as follows:[26]
    <br>Frequency	Modification
    <br>* 58383	Phosphorylation
    <br>* 6751	Acetylation
    <br>* 5526	N-linked glycosylation
    <br>* 2844	Amidation
    <br>* 1619	Hydroxylation
    <br>* 1523	Methylation
    <br>* 1133	O-linked glycosylation
    <br>* 878	Ubiquitylation
    <br>* 826	Pyrrolidone carboxylic acid
    <br>* 504	Sulfation
    <br>[{2020-05-11} https://en.wikipedia.org/wiki/Post-translational_modification]
    <a class="clsHide" href="#idPrtnOgmatt013str"></a></p>
  </section>

  <section id="idPrtnOgmatt009">
  <h3 id="idPrtnOgmatt009H3">folding of molProtein
    <a class="clsHide" href="#idPrtnOgmatt009H3"></a></h3>
  <p id="idPrtnOgmatt009nam">name::
    <br>* Mcs.molProtein'att009-folding,
    <br>* Mcs.molProtein'folding-att009,
    <a class="clsHide" href="#idPrtnOgmatt009nam"></a></p>
  <p id="idPrtnOgmatt009dsn">description::
    <br>"Protein folding is the physical process by which a protein chain acquires its native 3-dimensional structure, a conformation that is usually biologically functional, in an expeditious and reproducible manner. It is the physical process by which a polypeptide folds into its characteristic and functional three-dimensional structure from a random coil.[1] Each protein exists as an unfolded polypeptide or random coil when translated from a sequence of mRNA to a linear chain of amino acids. This polypeptide lacks any stable (long-lasting) three-dimensional structure (the left hand side of the first figure). As the polypeptide chain is being synthesized by a ribosome, the linear chain begins to fold into its three-dimensional structure. Folding begins to occur even during translation of the polypeptide chain. Amino acids interact with each other to produce a well-defined three-dimensional structure, the folded protein (the right hand side of the figure), known as the native state. The resulting three-dimensional structure is determined by the amino acid sequence or primary structure (Anfinsen's dogma).[2]
    <br>The correct three-dimensional structure is essential to function, although some parts of functional proteins may remain unfolded,[3] so that protein dynamics is important. Failure to fold into native structure generally produces inactive proteins, but in some instances misfolded proteins have modified or toxic functionality. Several neurodegenerative and other diseases are believed to result from the accumulation of amyloid fibrils formed by misfolded proteins.[4] Many allergies are caused by incorrect folding of some proteins, because the immune system does not produce antibodies for certain protein structures.[5]
    <br>Denaturation of proteins is a process of transition from the folded to the unfolded state. It happens in cooking, in burns, in proteinopathies, and in other contexts.
    <br>The duration of the folding process varies dramatically depending on the protein of interest. When studied outside the cell, the slowest folding proteins require many minutes or hours to fold primarily due to proline isomerization, and must pass through a number of intermediate states, like checkpoints, before the process is complete.[6] On the other hand, very small single-domain proteins with lengths of up to a hundred amino acids typically fold in a single step.[7] Time scales of milliseconds are the norm and the very fastest known protein folding reactions are complete within a few microseconds.[8]"
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_folding]
    <a class="clsHide" href="#idPrtnOgmatt009dsn"></a></p>
  </section>
  </section>

  <section id="idPrtnOgmatt010">
  <h2 id="idPrtnOgmatt010H2">metabolism of molProtein
    <a class="clsHide" href="#idPrtnOgmatt010H2"></a></h2>
  <p id="idPrtnOgmatt010nam">name::
    <br>* Mcs.molProtein'att010-metabolism,
    <br>* Mcs.molProtein'metabolism-att010,
    <a class="clsHide" href="#idPrtnOgmatt010nam"></a></p>
  <p id="idPrtnOgmatt010dsn">description::
    <br>· protein-metabolism is the-process of synthesis and breaking-down of proteins.
    <br>===
    <br>"Protein metabolism denotes the various biochemical processes responsible for the synthesis of proteins and amino acids (anabolism), and the breakdown of proteins by catabolism.
    <br>The steps of protein synthesis include transcription, translation, and post translational modifications. During transcription, RNA polymerase transcribes a coding region of the DNA in a cell producing a sequence of RNA, specifically messenger RNA (mRNA). This mRNA sequence contains codons: 3 nucleotide long segments that code for a specific amino acid. Ribosomes translate the codons to their respective amino acids.[1] In humans, non-essential amino acids are synthesized from intermediates in major metabolic pathways such as the Citric Acid Cycle.[2] Essential amino acids must be consumed and are made in other organisms. The amino acids are joined by peptide bonds making a polypeptide chain. This polypeptide chain then goes through post translational modifications and is sometimes joined with other polypeptide chains to form a fully functional protein.
    <br>Dietary proteins are first broken down to individual amino acids by various enzymes and hydrochloric acid present in the gastrointestinal tract. These amino acids are further broken down to α-keto acids which can be recycled in the body for generation of energy, and production of glucose or fat or other amino acids. Proteins can be broken down by enzymes known as peptidases or can break down as a result of denaturation. Proteins can denature in environmental conditions the protein is not made for.[3]"
    <br>[{2020-04-22} https://en.wikipedia.org/wiki/Protein_metabolism]
    <a class="clsHide" href="#idPrtnOgmatt010dsn"></a></p>
  </section>

  <section id="idPrtnOgmatt011">
  <h2 id="idPrtnOgmatt011H2">anabolism of molProtein
    <a class="clsHide" href="#idPrtnOgmatt011H2"></a></h2>
  <p id="idPrtnOgmatt011nam">name::
    <br>* Mcs.molProtein'att011-anabolism,
    <br>* Mcs.molProtein'anabolism-att011,
    <a class="clsHide" href="#idPrtnOgmatt011nam"></a></p>
  <p id="idPrtnOgmatt011dsn">description::
    <br>·
    <a class="clsHide" href="#idPrtnOgmatt011dsn"></a></p>
  </section>

  <section id="idPrtnOgmatt012">
  <h2 id="idPrtnOgmatt012H2">katabolism of molProtein
    <a class="clsHide" href="#idPrtnOgmatt012H2"></a></h2>
  <p id="idPrtnOgmatt012nam">name::
    <br>* Mcs.molProtein'att012-katabolism,
    <br>* Mcs.molProtein'katabolism-att012,
    <a class="clsHide" href="#idPrtnOgmatt012nam"></a></p>
  <p id="idPrtnOgmatt012dsn">description::
    <br>·
    <a class="clsHide" href="#idPrtnOgmatt012dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgmevgF">
  <h1 id="idPrtnOgmevgFH1">evoluting of molProtein
    <a class="clsHide" href="#idPrtnOgmevgFH1"></a></h1>
  <p id="idPrtnOgmevgnam">name::
    <br>* Mcs.molProtein'attEvoluting,
    <br>* Mcs.molProtein'evoluting,
    <a class="clsHide" href="#idPrtnOgmevgnam"></a></p>
  <p id="idPrtnOgmevg20200325">{time.2020-03-25}::
    <br>=== McsHitp-creation:
    <br>· creation of current <a class="clsPreview" href="../dirCor/filMcs.last.html#idOverview">concept</a>.
    <a class="clsHide" href="#idPrtnOgmevg20200325"></a></p>
</section>

<section id="idPrtnOgmwptF">
  <h1 id="idPrtnOgmwptFH1">WHOLE-PART-TREE of molProtein
    <a class="clsHide" href="#idPrtnOgmwhlFH1"></a></h1>
  <p id="idPrtnOgmwptnam">name::
    <br>* Mcs.molProtein'whole-part-tree,
    <a class="clsHide" href="#idPrtnOgmwptnam"></a></p>
  <p id="idPrtnOgmwtr">whole-tree-of-molProtein::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmwtr"></a></p>
  <p id="idPrtnOgmptr">part-tree-of-molProtein::
    <br>*
    <a class="clsHide" href="#idPrtnOgmptr"></a></p>
</section>

<section id="idPrtnOgmgst">
  <h1 id="idPrtnOgmgstH1">GENERIC-SPECIFIC-TREE of molProtein
    <a class="clsHide" href="#idPrtnOgmgstH1"></a></h1>
  <p id="idPrtnOgmgstnam">name::
    <br>* Mcs.molProtein'generic-specific-tree,
    <a class="clsHide" href="#idPrtnOgmgstnam"></a></p>

  <section id="idPrtnOgmgtr">
  <h2 id="idPrtnOgmgtrH2">GENERIC-TREE of molProtein
    <a class="clsHide" href="#idPrtnOgmgtrH2"></a></h2>
  <p id="idPrtnOgmgnc">generic-of-molProtein::
    <br>* biomaterial,
    <br>===
    <br>* polymer,
    <br>"Proteins are polymers – specifically polypeptides – formed from sequences of amino acids, the monomers of the polymer."
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_structure]
    <a class="clsHide" href="#idPrtnOgmgnc"></a></p>
  <p id="idPrtnOgmatttr">attribute-tree-of-molProtein::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmatttr"></a></p>
  <p id="idPrtnOgmattihf">att-tree-inherited-from::
    <br>· :
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmattihf"></a></p>
  <p id="idPrtnOgmattown">att-tree-own-of-molProtein::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmattown"></a></p>
  </section>

  <section id="idPrtnOgmstr">
  <h2 id="idPrtnOgmstrH2">SPECIFIC-TREE of molProtein
    <a class="clsHide" href="#idPrtnOgmstrH2"></a></h2>
  <p id="idPrtnOgmspc">specific-of-molProtein::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmspc"></a></p>
  </section>
</section>

<section id="idPrtn010">
  <h1 id="idPrtn010H1">molProtein.simple-010
    <a class="clsHide" href="#idPrtn010H1"></a></h1>
  <p id="idPrtn010nam">name::
    <br>* Mcs.molProtein.010-simple,
    <br>* Mcs.molProtein.simple-010,
    <br>* Mcs.protein-simple,
    <br>* Mcs.protein-subunit,
    <a class="clsHide" href="#idPrtn010nam"></a></p>
  <p id="idPrtn010dsn">description::
    <br>"In structural biology, a protein subunit is a single protein molecule that assembles (or "coassembles") with other protein molecules to form a protein complex. Some naturally occurring proteins have a relatively small number of subunits and therefore described as oligomeric, for example hemoglobin or DNA polymerase. Others may consist of a very large number of subunits and therefore described as multimeric, for example microtubules and other cytoskeleton proteins. The subunits of a multimeric protein may be identical, homologous or totally dissimilar and dedicated to disparate tasks.
    <br>In some protein assemblies, one subunit may be a "catalytic subunit" that enzymatically catalyzes a reaction, whereas a "regulatory subunit" will facilitate or inhibit the activity. Although telomerase has telomerase reverse transcriptase as a catalytic subunit, regulation is accomplished by factors outside the protein.[1] An enzyme composed of both regulatory and catalytic subunits when assembled is often referred to as a holoenzyme. For example, class I phosphoinositide 3-kinase is composed of a p110 catalytic subunit and a p85 regulatory subunit.[2] One subunit is made of one polypeptide chain. A polypeptide chain has one gene coding for it – meaning that a protein must have one gene for each unique subunit.
    <br>A subunit is often named with a Greek or Roman letter, and the numbers of this type of subunit in a protein is indicated by a subscript. For example, ATP synthase has a type of subunit called α. Three of these are present in the ATP synthase molecule, and is therefore designated α3. Larger groups of subunits can also be specified, like α3β3-hexamer and c-ring."
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_subunit]
    <a class="clsHide" href="#idPrtn010dsn"></a></p>
</section>

<section id="idPrtn011">
  <h1 id="idPrtn011H1">molProtein.complex-011
    <a class="clsHide" href="#idPrtn011H1"></a></h1>
  <p id="idPrtn011nam">name::
    <br>* Mcs.molProtein.011-complex,
    <br>* Mcs.molProtein.complex-011,
    <br>* Mcs.protein-complex,
    <a class="clsHide" href="#idPrtn011nam"></a></p>
  <p id="idPrtn011dsn">description::
    <br>"A protein complex or multiprotein complex is a group of two or more associated polypeptide chains. Different polypeptide chains may have different functions. This is distinct from a multienzyme complex, in which multiple catalytic domains are found in a single polypeptide chain.[1]
    <br>Protein complexes are a form of quaternary structure. Proteins in a protein complex are linked by non-covalent protein–protein interactions, and different protein complexes have different degrees of stability over time. These complexes are a cornerstone of many (if not most) biological processes and together they form various types of molecular machinery that perform a vast array of biological functions. The cell is seen to be composed of modular supramolecular complexes, each of which performs an independent, discrete biological function.[2]
    <br>Through proximity, the speed and selectivity of binding interactions between enzymatic complex and substrates can be vastly improved, leading to higher cellular efficiency. Many of the techniques used to break open cells and isolate proteins are inherently disruptive to such large complexes, so it is often difficult to determine the components of a complex. Examples of protein complexes include the proteasome for molecular degradation and most RNA polymerases. In stable complexes, large hydrophobic interfaces between proteins typically bury surface areas larger than 2500 square Ås.[3]"
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_complex]
    <a class="clsHide" href="#idPrtn011dsn"></a></p>
</section>

<section id="idPrtnHmn">
  <h1 id="idPrtnHmnH1">molProtein.human-007 of bodyHmn
    <a class="clsHide" href="#idPrtnHmnH1"></a></h1>
  <p id="idPrtnHmnnam">name::
    <br>* Mcs.bodyMtrHmnPrt.014-protein!⇒proteinHmn,
    <br>* Mcs.bodyMtrHmnPrt.protein-014!⇒proteinHmn,
    <br>* Mcs.bodyHmn'protein!⇒proteinHmn,
    <br>* Mcs.proteinHmn, /prótiin/,
    <br>* Mcs.proteinHmn'(human-protein)!⇒proteinHmn,
    <br>* Mcs.protein-of-bodyHmn!⇒proteinHmn,
    <br>* Mcs.molProtein.007-human!⇒proteinHmn,
    <br>* Mcs.molProtein.human-007!⇒proteinHmn,
    <br>====== langoGreek:
    <br>* McsEll.πρωτεΐνη!=proteinHmn,
    <a class="clsHide" href="#idPrtnHmnnam"></a></p>
  <p id="idPrtnHmndsn">description::
    <br>"A protein is a complex, high molecular weight organic compound that consists of amino acids joined by peptide bonds."
    <br>[https://en.wikipedia.org/wiki/Category:Proteins]
    <br>"Proteins are large biomolecules, or macromolecules, consisting of one or more long chains of amino acid residues. Proteins perform a vast array of functions within organisms, including catalysing metabolic reactions, DNA replication, responding to stimuli, providing structure to cells, and organisms, and transporting molecules from one location to another. Proteins differ from one another primarily in their sequence of amino acids, which is dictated by the nucleotide sequence of their genes, and which usually results in protein folding into a specific three-dimensional structure that determines its activity."
    <br>[https://en.wikipedia.org/wiki/Protein {2020-02-20}]
    <a class="clsHide" href="#idPrtnHmndsn"></a></p>

  <section id="idPrtnHmnatt001">
  <h2 id="idPrtnHmnatt001H2">amino-acid of proteinHmn
    <a class="clsHide" href="#idPrtnHmnatt001H2"></a></h2>
  <p id="idPrtnHmnatt001nam">name::
    <br>* Mcs.proteinHmn'amino-acid!⇒acidAminoHmn,
    <br>* Mcs.proteinHmn'att001-amino-acid!⇒acidAminoHmn,
    <br>* Mcs.sysMolsHmn.010-amino-acid!⇒acidAminoHmn,
    <br>* Mcs.sysMolsHmn.amino-acid-010!⇒acidAminoHmn,
    <br>* Mcs.acidAminoHmn,
    <br>* Mcs.amino-acidHmn!⇒acidAminoHmn,
    <a class="clsHide" href="#idPrtnHmnatt001nam"></a></p>
  <p id="idPrtnHmnatt001dsn">description::
    <br>"Amino acids are organic compounds that contain amine (-NH2) and carboxyl (-COOH) functional groups, along with a side chain (R group) specific to each amino acid.[1][2] The key elements of an amino acid are carbon (C), hydrogen (H), oxygen (O), and nitrogen (N), although other elements are found in the side chains of certain amino acids. About 500 naturally occurring amino acids are known (though only 20 appear in the genetic code) and can be classified in many ways.[3] They can be classified according to the core structural functional groups' locations as alpha- (α-), beta- (β-), gamma- (γ-) or delta- (δ-) amino acids; other categories relate to polarity, pH level, and side chain group type (aliphatic, acyclic, aromatic, containing hydroxyl or sulfur, etc.). In the form of proteins, amino acid residues form the second-largest component (water is the largest) of human muscles and other tissues.[4] Beyond their role as residues in proteins, amino acids participate in a number of processes such as neurotransmitter transport and biosynthesis.
    <br>In biochemistry, amino acids having both the amine and the carboxylic acid groups attached to the first (alpha-) carbon atom have particular importance. They are known as 2-, alpha-, or α-amino acids (generic formula H2NCHRCOOH in most cases,[a] where R is an organic substituent known as a "side chain");[5] often the term "amino acid" is used to refer specifically to these. They include the 22 proteinogenic ("protein-building") amino acids,[6][7][8] which combine into peptide chains ("polypeptides") to form the building-blocks of a vast array of proteins.[9] These are all L-stereoisomers ("left-handed" isomers), although a few D-amino acids ("right-handed") occur in bacterial envelopes, as a neuromodulator (D-serine), and in some antibiotics.[10]
    <br>Twenty of the proteinogenic amino acids are encoded directly by triplet codons in the genetic code and are known as "standard" amino acids. The other two ("non-standard" or "non-canonical") are selenocysteine (present in many prokaryotes as well as most eukaryotes, but not coded directly by DNA), and pyrrolysine (found only in some archaea and one bacterium). Pyrrolysine and selenocysteine are encoded via variant codons; for example, selenocysteine is encoded by stop codon and SECIS element.[11][12][13] N-formylmethionine (which is often the initial amino acid of proteins in bacteria, mitochondria, and chloroplasts) is generally considered as a form of methionine rather than as a separate proteinogenic amino acid. Codon–tRNA combinations not found in nature can also be used to "expand" the genetic code and form novel proteins known as alloproteins incorporating non-proteinogenic amino acids.[14][15][16]
    <br>Many important proteinogenic and non-proteinogenic amino acids have biological functions. For example, in the human brain, glutamate (standard glutamic acid) and gamma-amino-butyric acid ("GABA", non-standard gamma-amino acid) are, respectively, the main excitatory and inhibitory neurotransmitters.[17] Hydroxyproline, a major component of the connective tissue collagen, is synthesised from proline. Glycine is a biosynthetic precursor to porphyrins used in red blood cells. Carnitine is used in lipid transport.
    <br>Nine proteinogenic amino acids are called "essential" for humans because they cannot be produced from other compounds by the human body and so must be taken in as food. Others may be conditionally essential for certain ages or medical conditions. Essential amino acids may also differ between species.[b]
    <br>Because of their biological significance, amino acids are important in nutrition and are commonly used in nutritional supplements, fertilizers, feed, and food technology. Industrial uses include the production of drugs, biodegradable plastics, and chiral catalysts."
    <br>[https://en.wikipedia.org/wiki/Amino_acid {2020-02-28}]
    <a class="clsHide" href="#idPrtnHmnatt001dsn"></a></p>
  <p id="idPrtnHmnatt001wtr">whole-tree-of-amino-acid::
    <br>* <a class="clsPreview" href="#idPrtnHmn">protein</a>,
    <br>===
    <br>"Amino acids are the structural units (monomers) that make up proteins. They join together to form short polymer chains called peptides or longer chains called either polypeptides or proteins."
    <br>[https://en.wikipedia.org/wiki/Amino_acid {2020-02-28}]
    <a class="clsHide" href="#idPrtnHmnatt001wtr"></a></p>

  <section id="idPrtnHmnatt001gst">
  <h3 id="idPrtnHmnatt001gstH3">GENERIC-SPECIFIC-TREE of amino-acid
    <a class="clsHide" href="#idPrtnHmnatt001gstH3"></a></h3>
  <p id="idPrtnHmnatt001gstnam">name::
    <br>* Mcs.amino-acid'generic-specific-tree,
    <a class="clsHide" href="#idPrtnHmnatt001gstnam"></a></p>
  <p id="idPrtnHmnatt001gtr">generic-tree-of-amino-acid::
    <br>*
    <a class="clsHide" href="#idPrtnHmnatt001gtr"></a></p>
  <p id="idPrtnHmnatt001str">specific-tree-of-amino-acid::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essensial</a>,
    <a class="clsHide" href="#idPrtnHmnatt001str"></a></p>
  </section>

  <section id="idBdyHmnatt015025">
  <h3 id="idBdyHmnatt015025H3">amino-acid.essential
    <a class="clsHide" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025H3"></a></h3>
  <p id="idBdyHmnatt015025nam">name::
    <br>* Mcs.nutrient.025-essential-amino-acid,
    <br>* Mcs.nutrient.essential-amino-acid-025,
    <br>* Mcs.amino-acid.essential,
    <br>* Mcs.essential-amino-acid,
    <a class="clsHide" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025nam"></a></p>
  <p id="idBdyHmnatt015025dsn">description::
    <br>"An essential amino acid, or indispensable amino acid, is an amino acid that cannot be synthesized de novo (from scratch) by the organism at a rate commensurate with its demand, and thus must be supplied in its diet. Of the 21 amino acids common to all life forms, the nine amino acids humans cannot synthesize are phenylalanine, valine, threonine, tryptophan, methionine, leucine, isoleucine, lysine, and histidine.[1][2]"
    <br>[https://en.wikipedia.org/wiki/Essential_amino_acid]
    <a class="clsHide" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025dsn"></a></p>
  <p id="idBdyHmnatt015025str">specific-tree-of-essential-amino-acid::
    <br>* <a class="clsPreview" href="#idPrtnHmnatt001Pnln">phenylalanine</a>,
    <br>* <a class="clsPreview" href="#idPrtnHmnatt001valn">valine</a>,
    <br>* <a class="clsPreview" href="#idPrtnHmnatt001Trnn">threonine</a>,
    <br>* <a class="clsPreview" href="#idPrtnHmnatt001Trpfn">tryptophan</a>,
    <br>* <a class="clsPreview" href="#idPrtnHmnatt001Mtnn">methionine</a>,
    <br>* <a class="clsPreview" href="#idPrtnHmnatt001Lucn">leucine</a>,
    <br>* <a class="clsPreview" href="#idPrtnHmnatt001Iscn">isoleucine</a>,
    <br>* <a class="clsPreview" href="#idPrtnHmnatt001Lysn">lysine</a>,
    <br>* <a class="clsPreview" href="#idPrtnHmnatt001Hsdn">histidine</a>,
    <a class="clsHide" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025str"></a></p>
  </section>

  <section id="idPrtnHmnatt001Pnln">
  <h3 id="idPrtnHmnatt001PnlnH3">amino-acid.phenylalanine
    <a class="clsHide" href="#idPrtnHmnatt001PnlnH3"></a></h3>
  <p id="idPrtnHmnatt001Pnlnnam">name::
    <br>* Mcs.amino-acid.phenylalanine,
    <br>* Mcs.phenylalanine-amino-acid, /fenoálanin/,
    <br>====== langoGreek:
    <br>* McsEll.Φαινυλαλανίνη!=phenylalanine,
    <a class="clsHide" href="#idPrtnHmnatt001Pnlnnam"></a></p>
  <p id="idPrtnHmnatt001Pnlndsn">description::
    <br>"Phenylalanine (symbol Phe or F)[2] is an essential α-amino acid with the formula C9H11NO2. It can be viewed as a benzyl group substituted for the methyl group of alanine, or a phenyl group in place of a terminal hydrogen of alanine. This essential amino acid is classified as neutral, and nonpolar because of the inert and hydrophobic nature of the benzyl side chain. The L-isomer is used to biochemically form proteins, coded for by DNA. Phenylalanine is a precursor for tyrosine, the monoamine neurotransmitters dopamine, norepinephrine (noradrenaline), and epinephrine (adrenaline), and the skin pigment melanin. It is encoded by the codons UUU and UUC.
    <br>Phenylalanine is found naturally in the breast milk of mammals. It is used in the manufacture of food and drink products and sold as a nutritional supplement for its reputed analgesic and antidepressant effects. It is a direct precursor to the neuromodulator phenethylamine, a commonly used dietary supplement. As an essential amino acid, phenylalanine is not synthesized de novo in humans and other animals, who must ingest phenylalanine or phenylalanine-containing proteins."
    <br>[https://en.wikipedia.org/wiki/Phenylalanine {2020-03-10}]
    <a class="clsHide" href="#idPrtnHmnatt001Pnlndsn"></a></p>
  <p id="idPrtnHmnatt001Pnlngtr">generic-tree-of-phenylanine::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essential-amino-acid</a>,
    <a class="clsHide" href="#idPrtnHmnatt001Pnlngtr"></a></p>
  </section>

  <section id="idPrtnHmnatt001valn">
  <h3 id="idPrtnHmnatt001valnH3">amino-acid.valine
    <a class="clsHide" href="#idPrtnHmnatt001valnH3"></a></h3>
  <p id="idPrtnHmnatt001valnnam">name::
    <br>* Mcs.amino-acid.valine,
    <br>* Mcs.valine-amino-acid, /válin/,
    <a class="clsHide" href="#idPrtnHmnatt001valnnam"></a></p>
  <p id="idPrtnHmnatt001valndsn">description::
    <br>"Valine (symbol Val or V)[3] is an α-amino acid that is used in the biosynthesis of proteins. It contains an α-amino group (which is in the protonated −NH3+ form under biological conditions), an α-carboxylic acid group (which is in the deprotonated −COO− form under biological conditions), and a side chain isopropyl group, making it a non-polar aliphatic amino acid. It is essential in humans, meaning the body cannot synthesize it: it must be obtained from the diet. Human dietary sources are foods that contain protein, such as meats, dairy products, soy products, beans and legumes. It is encoded by all codons starting with GU (GUU, GUC, GUA, and GUG).
    <br>Like leucine and isoleucine, valine is a branched-chain amino acid. In sickle-cell disease, a single glutamic acid in β-globin is replaced with valine. Because valine is hydrophobic, whereas glutamic acid is hydrophilic, this change makes the hemoglobin prone to abnormal aggregation."
    <br>[https://en.wikipedia.org/wiki/Valine {2020-03-10}]
    <a class="clsHide" href="#idPrtnHmnatt001valndsn"></a></p>
  <p id="idPrtnHmnatt001valngtr">generic-tree-of-valine::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essential-amino-acid</a>,
    <a class="clsHide" href="#idPrtnHmnatt001valngtr"></a></p>
  </section>

  <section id="idPrtnHmnatt001Trnn">
  <h3 id="idPrtnHmnatt001TrnnH3">amino-acid.threonine
    <a class="clsHide" href="#idPrtnHmnatt001TrnnH3"></a></h3>
  <p id="idPrtnHmnatt001Trnnnam">name::
    <br>* Mcs.amino-acid.threonine,
    <br>* Mcs.threonine-amino-acid, /thríonin/,
    <a class="clsHide" href="#idPrtnHmnatt001Trnnnam"></a></p>
  <p id="idPrtnHmnatt001Trnndsn">description::
    <br>"Threonine (symbol Thr or T)[2] is an amino acid that is used in the biosynthesis of proteins. It contains an α-amino group (which is in the protonated −NH+3 form under biological conditions), a carboxyl group (which is in the deprotonated −COO− form under biological conditions), and a side chain containing a hydroxyl group, making it a polar, uncharged amino acid. It is essential in humans, meaning the body cannot synthesize it: it must be obtained from the diet. Threonine is synthesized from aspartate in bacteria such as E. coli.[3] It is encoded by all the codons starting AC (ACU, ACC, ACA, and ACG).
    <br>Threonine sidechains are often hydrogen bonded; the most common small motifs formed are based on interactions with serine: ST turns, ST motifs (often at the beginning of alpha helices) and ST staples (usually at the middle of alpha helices)."
    <br>[https://en.wikipedia.org/wiki/Threonine {2020-03-10}]
    <a class="clsHide" href="#idPrtnHmnatt001Trnndsn"></a></p>
  <p id="idPrtnHmnatt001Trnngtr">generic-tree-of-threonine::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essential-amino-acid</a>,
    <a class="clsHide" href="#idPrtnHmnatt001Trnngtr"></a></p>
  </section>

  <section id="idPrtnHmnatt001Trpfn">
  <h3 id="idPrtnHmnatt001TrpfnH3">amino-acid.tryptophan
    <a class="clsHide" href="#idPrtnHmnatt001TrpfnH3"></a></h3>
  <p id="idPrtnHmnatt001Trpfnnam">name::
    <br>* Mcs.amino-acid.tryptophan,
    <br>* Mcs.tryptophan-amino-acid, /tríptofan/,
    <a class="clsHide" href="#idPrtnHmnatt001Trpfnnam"></a></p>
  <p id="idPrtnHmnatt001Trpfndsn">description::
    <br>"Tryptophan (symbol Trp or W)[2] is an α-amino acid that is used in the biosynthesis of proteins. Tryptophan contains an α-amino group, an α-carboxylic acid group, and a side chain indole, making it a non-polar aromatic amino acid. It is essential in humans, meaning the body cannot synthesize it; it must be obtained from the diet. Tryptophan is also a precursor to the neurotransmitter serotonin, the hormone melatonin and vitamin B3.[3] It is encoded by the codon UGG."
    <br>[https://en.wikipedia.org/wiki/Tryptophan {2020-03-10}]
    <a class="clsHide" href="#idPrtnHmnatt001Trpfndsn"></a></p>
  <p id="idPrtnHmnatt001Trpfngtr">generic-tree-of-tryptophan::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essential-amino-acid</a>,
    <a class="clsHide" href="#idPrtnHmnatt001Trpfngtr"></a></p>
  </section>

  <section id="idPrtnHmnatt001Mtnn">
  <h3 id="idPrtnHmnatt001MtnnH3">amino-acid.methionine
    <a class="clsHide" href="#idPrtnHmnatt001MtnnH3"></a></h3>
  <p id="idPrtnHmnatt001Mtnnnam">name::
    <br>* Mcs.amino-acid.methionine,
    <br>* Mcs.methionine-amino-acid, /metháionin/,
    <a class="clsHide" href="#idPrtnHmnatt001Mtnnnam"></a></p>
  <p id="idPrtnHmnatt001Mtnndsn">description::
    <br>"Methionine (symbol Met or M)[3] (/mɪˈθaɪəniːn/)[4] is an essential amino acid in humans. As the substrate for other amino acids such as cysteine and taurine, versatile compounds such as SAM-e, and the important antioxidant glutathione, methionine plays a critical role in the metabolism and health of many species, including humans. It is encoded by the codon AUG.
    <br>Methionine is also an important part of angiogenesis, the growth of new blood vessels. Supplementation may benefit those suffering from copper poisoning.[5] Overconsumption of methionine, the methyl group donor in DNA methylation, is related to cancer growth in a number of studies.[6][7] Methionine was first isolated in 1921 by John Howard Mueller.[8]"
    <br>[https://en.wikipedia.org/wiki/Methionine {2020-03-10}]
    <a class="clsHide" href="#idPrtnHmnatt001Mtnndsn"></a></p>
  <p id="idPrtnHmnatt001Mtnngtr">generic-tree-of-methionine::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essential-amino-acid</a>,
    <a class="clsHide" href="#idPrtnHmnatt001Mtnngtr"></a></p>
  </section>

  <section id="idPrtnHmnatt001Lucn">
  <h3 id="idPrtnHmnatt001LucnH3">amino-acid.leucine
    <a class="clsHide" href="#idPrtnHmnatt001LucnH3"></a></h3>
  <p id="idPrtnHmnatt001Lucnnam">name::
    <br>* Mcs.amino-acid.leucine,
    <br>* Mcs.leucine-amino-acid, /lúsin/,
    <br>====== langoGreek:
    <br>* McsEll.λευκίνη!=leucine,
    <a class="clsHide" href="#idPrtnHmnatt001Lucnnam"></a></p>
  <p id="idPrtnHmnatt001Lucndsn">description::
    <br>"Leucine (symbol Leu or L)[2] is an essential amino acid that is used in the biosynthesis of proteins. Leucine is an α-amino acid, meaning it contains an α-amino group (which is in the protonated −NH3+ form under biological conditions), an α-carboxylic acid group (which is in the deprotonated −COO− form under biological conditions), and a side chain isobutyl group, making it a non-polar aliphatic amino acid. It is essential in humans, meaning the body cannot synthesize it: it must be obtained from the diet. Human dietary sources are foods that contain protein, such as meats, dairy products, soy products, and beans and other legumes. It is encoded by the codons UUA, UUG, CUU, CUC, CUA, and CUG.
    <br>Like valine and isoleucine, leucine is a branched-chain amino acid. The primary metabolic end products of leucine metabolism are acetyl-CoA and acetoacetate; consequently, it is one of the two exclusively ketogenic amino acids, with lysine being the other.[3] It is the most important ketogenic amino acid in humans.[4]
    <br>Leucine and β-hydroxy β-methylbutyric acid, a minor leucine metabolite, exhibit pharmacological activity in humans and have been demonstrated to promote protein biosynthesis via the phosphorylation of the mechanistic target of rapamycin (mTOR).[5][6]"
    <br>[https://en.wikipedia.org/wiki/Leucine {2020-03-10}]
    <a class="clsHide" href="#idPrtnHmnatt001Lucndsn"></a></p>
  <p id="idPrtnHmnatt001Lucngtr">generic-tree-of-leucine::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essential-amino-acid</a>,
    <a class="clsHide" href="#idPrtnHmnatt001Lucngtr"></a></p>
  </section>

  <section id="idPrtnHmnatt001Iscn">
  <h3 id="idPrtnHmnatt001IscnH3">amino-acid.isoleucine
    <a class="clsHide" href="#idPrtnHmnatt001IscnH3"></a></h3>
  <p id="idPrtnHmnatt001Iscnnam">name::
    <br>* Mcs.amino-acid.isoleucine,
    <br>* Mcs.isoleucine-amino-acid, /aisolúsin/,
    <a class="clsHide" href="#idPrtnHmnatt001Iscnnam"></a></p>
  <p id="idPrtnHmnatt001Iscndsn">description::
    <br>"Isoleucine (symbol Ile or I)[1] is an α-amino acid that is used in the biosynthesis of proteins. It contains an α-amino group (which is in the protonated −NH+3 form under biological conditions), an α-carboxylic acid group (which is in the deprotonated −COO− form under biological conditions), and a hydrocarbon side chain with a branch (a central carbon atom bound to three other carbon atoms). It is classified as a non-polar, uncharged (at physiological pH), branched-chain, aliphatic amino acid. It is essential in humans, meaning the body cannot synthesize it, and must be ingested in our diet. Isoleucine is synthesized from pyruvate employing leucine biosynthesis enzymes in other organisms such as bacteria.[2] It is encoded by the codons AUU, AUC, and AUA.
    <br>Inability to break down isoleucine, along with other amino acids, is associated with maple syrup urine disease.[3]"
    <br>[https://en.wikipedia.org/wiki/Isoleucine {2020-03-10}]
    <a class="clsHide" href="#idPrtnHmnatt001Iscndsn"></a></p>
  <p id="idPrtnHmnatt001Iscngtr">generic-tree-of-isoleucine::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essential-amino-acid</a>,
    <a class="clsHide" href="#idPrtnHmnatt001Iscngtr"></a></p>
  </section>

  <section id="idPrtnHmnatt001Lysn">
  <h3 id="idPrtnHmnatt001LysnH3">amino-acid.lysine
    <a class="clsHide" href="#idPrtnHmnatt001LysnH3"></a></h3>
  <p id="idPrtnHmnatt001Lysnnam">name::
    <br>* Mcs.amino-acid.lysine,
    <br>* Mcs.lysine-amino-acid, /láisin/,
    <a class="clsHide" href="#idPrtnHmnatt001Lysnnam"></a></p>
  <p id="idPrtnHmnatt001Lysndsn">description::
    <br>"Lysine (symbol Lys or K)[1] is an α-amino acid that is used in the biosynthesis of proteins. It contains an α-amino group (which is in the protonated −NH3+ form under biological conditions), an α-carboxylic acid group (which is in the deprotonated −COO− form under biological conditions), and a side chain lysyl ((CH2)4NH2), classifying it as a basic, charged (at physiological pH), aliphatic amino acid. It is encoded by the codons, AAA and AAG. Like almost all other amino acids, the α-carbon is chiral and lysine may refer to either enantiomer or a racemic mixture of both. For the purpose of this article, lysine will refer to the biologically active enantiomer L-lysine, where the α-carbon is in the S configuration.
    <br>The human body cannot synthesize lysine. It is essential in humans and must be obtained from the diet. In organisms that synthesise lysine, it has two main biosynthetic pathways, the diaminopimelate and α-aminoadipate pathways, which employ different enzymes and substrates and are found in different organisms. Lysine catabolism occurs through one of several pathways, the most common of which is the saccharopine pathway.
    <br>Lysine plays several roles in humans, most importantly proteinogenesis, but also in the crosslinking of collagen polypeptides, uptake of essential mineral nutrients, and in the production of carnitine, which is key in fatty acid metabolism. Lysine is also often involved in histone modifications, and thus, impacts the epigenome. The ε-amino group often participates in hydrogen bonding and as a general base in catalysis. The ε-ammonium group (NH3+) is attached to the fourth carbon from the α-carbon, which is attached to the carboxyl (C=OOH) group.[2]
    <br>Due to its importance in several biological processes, a lack of lysine can lead to several disease states including defective connective tissues, impaired fatty acid metabolism, anaemia, and systemic protein-energy deficiency. In contrast, an overabundance of lysine, caused by ineffective catabolism, can cause severe neurological disorders.
    <br>Lysine was first isolated by the German biological chemist Ferdinand Heinrich Edmund Drechsel in 1889 from the protein casein in milk.[3] He named it "lysin".[4] In 1902, the German chemists Emil Fischer and Fritz Weigert determined lysine's chemical structure by synthesizing it.[5]"
    <br>[https://en.wikipedia.org/wiki/Lysine {2020-03-05}]
    <a class="clsHide" href="#idPrtnHmnatt001Lysndsn"></a></p>
  <p id="idPrtnHmnatt001Lysngtr">generic-tree-of-lysine::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essential-amino-acid</a>,
    <a class="clsHide" href="#idPrtnHmnatt001Lysngtr"></a></p>
  </section>

  <section id="idPrtnHmnatt001Hsdn">
  <h3 id="idPrtnHmnatt001HsdnH3">amino-acid.histidine
    <a class="clsHide" href="#idPrtnHmnatt001HsdnH3"></a></h3>
  <p id="idPrtnHmnatt001Hsdnnam">name::
    <br>* Mcs.amino-acid.histidine,
    <br>* Mcs.histidine-amino-acid, /hístidin/,
    <a class="clsHide" href="#idPrtnHmnatt001Hsdnnam"></a></p>
  <p id="idPrtnHmnatt001Hsdndsn">description::
    <br>"Histidine (symbol His or H)[2] is an α-amino acid that is used in the biosynthesis of proteins. It contains an α-amino group (which is in the protonated –NH3+ form under biological conditions), a carboxylic acid group (which is in the deprotonated –COO− form under biological conditions), and an imidazole side chain (which is partially protonated), classifying it as a positively charged amino acid at physiological pH. Initially thought essential only for infants, longer-term studies have shown it is essential for adults also.[3] It is encoded by the codons CAU and CAC.
    <br>Histidine was first isolated by German physician Albrecht Kossel and Sven Gustaf Hedin in 1896.[4] It is also a precursor to histamine, a vital inflammatory agent in immune responses. The acyl radical is histidyl."
    <br>[https://en.wikipedia.org/wiki/Histidine {2020-03-05}]
    <a class="clsHide" href="#idPrtnHmnatt001Hsdndsn"></a></p>
  <p id="idPrtnHmnatt001Hsdngtr">generic-tree-of-histidine::
    <br>* <a class="clsPreview" href="../dirHlth/filMcsBdyHmn.last.html#idBdyHmnatt015025">essential-amino-acid</a>,
    <a class="clsHide" href="#idPrtnHmnatt001Hsdngtr"></a></p>
  </section>
  </section>

  <section id="idPrtnHmnrsc">
  <h2 id="idPrtnHmnrscH2">resource of proteinHmn
    <a class="clsHide" href="#idPrtnHmnrscH2"></a></h2>
  <p id="idPrtnHmnrscnam">name::
    <br>* Mcs.proteinHmn'resource,
    <a class="clsHide" href="#idPrtnHmnrscnam"></a></p>
  <p id="idPrtnHmnrscdsn">description::
    <br>* https://en.wikipedia.org/wiki/List_of_topics_related_to_protein,
    <a class="clsHide" href="#idPrtnHmnrscdsn"></a></p>
  </section>

  <section id="idPrtnHmngst">
  <h2 id="idPrtnHmngstH2">GENERIC-SPECIFIC-TREE of proteinHmn
    <a class="clsHide" href="#idPrtnHmngstH2"></a></h2>
  <p id="idPrtnHmngstnam">name::
    <br>* Mcs.proteinHmn'generic-specific-tree,
    <a class="clsHide" href="#idPrtnHmngstnam"></a></p>
  <p id="idPrtnHmngtr">generic-tree-of-proteinHmn::
    <br>* molProtein,
    <a class="clsHide" href="#idPrtnHmngtr"></a></p>
  <p id="idPrtnHmnstr">specific-tree-of-proteinHmn::
    <br>* blood-proteinHmn,
    <a class="clsHide" href="#idPrtnHmnstr"></a></p>
  <p id="idPrtnHmngstwpa">addressWpg::
    <br>* https://en.wikipedia.org/wiki/Category:Proteins,
    <br>* https://en.wikipedia.org/wiki/List_of_types_of_proteins,
    <br>* https://en.wikipedia.org/wiki/List_of_proteins,
    <a class="clsHide" href="#idPrtnHmngstwpa"></a></p>
  </section>

  <section id="idPrtnHmn001">
  <h2 id="idPrtnHmn001H2">proteinHmn.blood-001
    <a class="clsHide" href="#idPrtnHmn001H2"></a></h2>
  <p id="idPrtnHmn001nam">name::
    <br>* Mcs.proteinHmn.001-blood,
    <br>* Mcs.proteinHmn.blood-001,
    <br>* Mcs.blood-protein-001,
    <br>* Mcs.plasma-protein-001,
    <a class="clsHide" href="#idPrtnHmn001nam"></a></p>
  <p id="idPrtnHmn001dsn">description::
    <br>"Blood proteins, also termed plasma proteins, are proteins present in blood plasma. They serve many different functions, including transport of lipids, hormones, vitamins and minerals in activity and functioning of the immune system. Other blood proteins act as enzymes, complement components, protease inhibitors or kinin precursors. Contrary to popular belief, haemoglobin is not a blood protein, as it is carried within red blood cells, rather than in the blood serum.
    <br>Serum albumin accounts for 55% of blood proteins, is a major contributor to maintaining the oncotic pressure of plasma and assists, as a carrier, in the transport of lipids and steroid hormones. Globulins make up 38% of blood proteins and transport ions, hormones, and lipids assisting in immune function. Fibrinogen comprises 7% of blood proteins; conversion of fibrinogen to insoluble fibrin is essential for blood clotting. The remainder of the plasma proteins (1%) are regulatory proteins, such as enzymes, proenzymes, and hormones. All blood proteins are synthesized in liver except for the gamma globulins.
    <br>Separating serum proteins by electrophoresis is a valuable diagnostic tool as well as a way to monitor clinical progress. Current research regarding blood plasma proteins is centered on performing proteomics analyses of serum/plasma in the search for biomarkers. These efforts started with two-dimensional gel electrophoresis[1] efforts in the 1970s and in more recent times this research has been performed using LC-tandem MS[2][3][4] based proteomics. The normal laboratory value of serum total protein is around 7 g/dL."
    <br>[https://en.wikipedia.org/wiki/Blood_proteins {2020-03-11}]
    <a class="clsHide" href="#idPrtnHmn001dsn"></a></p>
  </section>

  <section id="idPrtnHmn003">
  <h2 id="idPrtnHmn003H2">proteinHmn.metalloprotein-003
    <a class="clsHide" href="#idPrtnHmn003H2"></a></h2>
  <p id="idPrtnHmn003nam">name::
    <br>* Mcs.proteinHmn.003-metalloprotein,
    <br>* Mcs.proteinHmn.metalloprotein-003,
    <br>* Mcs.metalloprotein-003,
    <a class="clsHide" href="#idPrtnHmn003nam"></a></p>
  <p id="idPrtnHmn003dsn">description::
    <br>"Metalloprotein is a generic term for a protein that contains a metal ion cofactor.[1][2] A large proportion of all proteins are part of this category. For instance, at least 1000 human proteins (out of ~20,000) contain zinc-binding protein domains[3] although there may be up to 3000 human zinc metalloproteins.[4]"
    <br>[https://en.wikipedia.org/wiki/Metalloprotein {2020-03-19}]
    <a class="clsHide" href="#idPrtnHmn003dsn"></a></p>
  </section>

  <section id="idPrtnHmn004">
  <h2 id="idPrtnHmn004H2">proteinHmn.enzyme
    <a class="clsHide" href="#idPrtnHmn004H2"></a></h2>
  <p id="idPrtnHmn004nam">name::
    <br>* Mcs.proteinHmn.enzyme!⇒enzymeHmn,
    <br>* Mcs.proteinHmn.004-enzyme!⇒enzymeHmn,
    <br>* Mcs.sysMolsHmn.012-enzyme!⇒enzymeHmn,
    <br>* Mcs.sysMolsHmn.enzyme-012!⇒enzymeHmn,
    <br>* Mcs.bodyHmn'enzyme!⇒enzymeHmn, /énzaim/,
    <br>* Mcs.enzymeHmn,
    <a class="clsHide" href="#idPrtnHmn004nam"></a></p>
  <p id="idPrtnHmn004dsn">description::
    <br>"Enzymes /ˈɛnzaɪmz/ are both proteins and biological catalysts (biocatalysts). Catalysts accelerate chemical reactions. The molecules upon which enzymes may act are called substrates, and the enzyme converts the substrates into different molecules known as products. Almost all metabolic processes in the cell need enzyme catalysis in order to occur at rates fast enough to sustain life.[1]:8.1 Metabolic pathways depend upon enzymes to catalyze individual steps. The study of enzymes is called enzymology and a new field of pseudoenzyme analysis has recently grown up, recognising that during evolution, some enzymes have lost the ability to carry out biological catalysis, which is often reflected in their amino acid sequences and unusual 'pseudocatalytic' properties.[2][3]
    <br>Enzymes are known to catalyze more than 5,000 biochemical reaction types.[4] Other biocatalysts are catalytic RNA molecules, called ribozymes. Enzymes' specificity comes from their unique three-dimensional structures."
    <br>[https://en.wikipedia.org/wiki/Enzyme {2020-02-20}]
    <a class="clsHide" href="#idPrtnHmn004dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgm001">
  <h1 id="idPrtnOgm001H1">molProtein.motor-001
    <a class="clsHide" href="#idPrtnOgm001H1"></a></h1>
  <p id="idPrtnOgm001nam">name::
    <br>* Mcs.molProtein.001-motor,
    <br>* Mcs.molProtein.motor-001,
    <br>* Mcs.motor-molProtein-001,
    <a class="clsHide" href="#idPrtnOgm001nam"></a></p>
  <p id="idPrtnOgm001dsn">description::
    <br>"Motor proteins are a class of molecular motors that can move along the cytoplasm of animal cells. They convert chemical energy into mechanical work by the hydrolysis of ATP. Flagellar rotation, however, is powered by a proton pump.[citation needed]"
    <br>[https://en.wikipedia.org/wiki/Motor_protein {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm001dsn"></a></p>

  <section id="idPrtnOgm002">
  <h2 id="idPrtnOgm002H2">molProtein.myosin-002
    <a class="clsHide" href="#idPrtnOgm002H2"></a></h2>
  <p id="idPrtnOgm002nam">name::
    <br>* Mcs.molProtein.002-myosin,
    <br>* Mcs.molProtein.myosin-002,
    <br>* Mcs.myosin-molProtein-002,
    <a class="clsHide" href="#idPrtnOgm002nam"></a></p>
  <p id="idPrtnOgm002dsn">description::
    <br>"Myosins (/ˈmaɪəsɪn, -oʊ-/[1][2]) are a superfamily of motor proteins best known for their roles in muscle contraction and in a wide range of other motility processes in eukaryotes. They are ATP-dependent and responsible for actin-based motility. The term was originally used to describe a group of similar ATPases found in the cells of both striated muscle tissue and smooth muscle tissue.[3] Following the discovery by Pollard and Korn (1973) of enzymes with myosin-like function in Acanthamoeba castellanii, a global range of divergent myosin genes have been discovered throughout the realm of eukaryotes.
    <br>Although myosin was originally thought to be restricted to muscle cells (hence myo-(s) + -in), there is no single "myosin"; rather it is a very large superfamily of genes whose protein products share the basic properties of actin binding, ATP hydrolysis (ATPase enzyme activity), and force transduction. Virtually all eukaryotic cells contain myosin isoforms. Some isoforms have specialized functions in certain cell types (such as muscle), while other isoforms are ubiquitous. The structure and function of myosin is globally conserved across species, to the extent that rabbit muscle myosin II will bind to actin from an amoeba.[4]"
    <br>[https://en.wikipedia.org/wiki/Myosin {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm002dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgm003">
  <h1 id="idPrtnOgm003H1">molProtein.chaperone-003
    <a class="clsHide" href="#idPrtnOgm003H1"></a></h1>
  <p id="idPrtnOgm003nam">name::
    <br>* Mcs.molProtein.003-chaperone,
    <br>* Mcs.molProtein.chaperone-003,
    <br>* Mcs.chaperone-molProtein-003,
    <a class="clsHide" href="#idPrtnOgm003nam"></a></p>
  <p id="idPrtnOgm003dsn">description::
    <br>"In molecular biology, molecular chaperones are proteins that assist the conformational folding or unfolding and the assembly or disassembly of other macromolecular structures. Chaperones are present when the macromolecules perform their normal biological functions and have correctly completed the processes of folding and/or assembly. The chaperones are concerned primarily with protein folding. The first protein to be called a chaperone assists the assembly of nucleosomes from folded histones and DNA and such assembly chaperones, especially in the nucleus,[1][2] are concerned with the assembly of folded subunits into oligomeric structures.[3]
    <br>One major function of chaperones is to prevent both newly synthesised polypeptide chains and assembled subunits from aggregating into nonfunctional structures. It is for this reason that many chaperones, but by no means all, are heat shock proteins because the tendency to aggregate increases as proteins are denatured by stress. In this case, chaperones do not convey any additional steric information required for proteins to fold. However, some highly specific 'steric chaperones' do convey unique structural (steric) information onto proteins, which cannot be folded spontaneously. Such proteins violate Anfinsen's dogma,[4] requiring protein dynamics to fold correctly.
    <br>Various approaches have been applied to study the structure, dynamics and functioning of chaperones. Bulk biochemical measurements have informed us on the protein folding efficiency, and prevention of aggregation when chaperones are present during protein folding. Recent advances in single-molecule analysis[5] have brought insights into structural heterogeneity of chaperones, folding intermediates and affinity of chaperones for unstructured and structured protein chains."
    <br>[https://en.wikipedia.org/wiki/Chaperone_(protein) {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm003dsn"></a></p>
</section>

<section id="idPrtnOgm004">
  <h1 id="idPrtnOgm004H1">molProtein.membrane-004
    <a class="clsHide" href="#idPrtnOgm004H1"></a></h1>
  <p id="idPrtnOgm004nam">name::
    <br>* Mcs.molProtein.004-membrane!⇒prtnMembrane,
    <br>* Mcs.molProtein.membrane-004!⇒prtnMembrane,
    <br>* Mcs.membrane-protein-004!⇒prtnMembrane,
    <br>* Mcs.prtnMembrane,
    <a class="clsHide" href="#idPrtnOgm004nam"></a></p>
  <p id="idPrtnOgm004dsn">description::
    <br>"Membrane proteins are common proteins that are part of, or interact with, biological membranes. Membrane proteins fall into several broad categories depending on their location. Integral membrane proteins are a permanent part of a cell membrane and can either penetrate the membrane (transmembrane) or associate with one or the other side of a membrane (integral monotopic). Peripheral membrane proteins are transiently associated with the cell membrane.
    <br>Membrane proteins are common, and medically important—about a third of all human proteins are membrane proteins, and these are targets for more than half of all drugs. Nonetheless, compared to other classes of proteins, determining membrane protein structures remains a challenge in large part due to the difficulty in establishing experimental conditions that can preserve the correct conformation of the protein in isolation from its native environment."
    <br>[https://en.wikipedia.org/wiki/Membrane_protein {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm004dsn"></a></p>

  <section id="idPrtnOgm004001">
  <h2 id="idPrtnOgm004001H2">prtnMembrane.integral
    <a class="clsHide" href="#idPrtnOgm004001H2"></a></h2>
  <p id="idPrtnOgm004001nam">name::
    <br>* Mcs.prtnMembrane.integral,
    <br>* Mcs.prtnMembrane.001-integral,
    <br>* Mcs.IMP-integral-membrane-protein,
    <br>* Mcs.integral-membrane-protein,
    <a class="clsHide" href="#idPrtnOgm004001nam"></a></p>
  <p id="idPrtnOgm004001dsn">description::
    <br>"An integral membrane protein (IMP) is a type of membrane protein that is permanently attached to the biological membrane. All transmembrane proteins are IMPs, but not all IMPs are transmembrane proteins.[1] IMPs comprise a significant fraction of the proteins encoded in an organism's genome.[2] Proteins that cross the membrane are surrounded by annular lipids, which are defined as lipids that are in direct contact with a membrane protein. Such proteins can only be separated from the membranes by using detergents, nonpolar solvents, or sometimes denaturing agents."
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Integral_membrane_protein]
    <a class="clsHide" href="#idPrtnOgm004001dsn"></a></p>
  <p id="idPrtnOgm004001str">specific-tree-of-::
    <br>"IMPs include transporters, linkers, channels, receptors, enzymes, structural membrane-anchoring domains, proteins involved in accumulation and transduction of energy, and proteins responsible for cell adhesion. Classification of transporters can be found in Transporter Classification Database.[5]"
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Integral_membrane_protein]
    <a class="clsHide" href="#idPrtnOgm004001str"></a></p>
  </section>

  <section id="idPrtnOgm004003">
  <h2 id="idPrtnOgm004003H2">prtnMembrane.transmembrane
    <a class="clsHide" href="#idPrtnOgm004003H2"></a></h2>
  <p id="idPrtnOgm004003nam">name::
    <br>* Mcs.prtnMembrane.transmembrane,
    <br>* Mcs.prtnMembrane.003-transmembrane,
    <br>* Mcs.TP-transmembrane-protein,
    <br>* Mcs.transmembrane-protein,
    <a class="clsHide" href="#idPrtnOgm004003nam"></a></p>
  <p id="idPrtnOgm004003dsn">description::
    <br>"A transmembrane protein (TP) is a type of integral membrane protein that spans the entirety of the cell membrane. Many transmembrane proteins function as gateways to permit the transport of specific substances across the membrane. They frequently undergo significant conformational changes to move a substance through the membrane. They are usually highly hydrophobic and aggregate and precipitate in water. They require detergents or nonpolar solvents for extraction, although some of them (beta-barrels) can be also extracted using denaturing agents.
    <br>The peptide sequence that spans the membrane, or the transmembrane segment, is largely hydrophobic and can be visualized using the hydropathy plot.[1] Depending on the number of transmembrane segments, transmembrane proteins can be classified as single-span (or bitopic) or multi-span (polytopic). Some other integral membrane proteins are called monotopic, meaning that they are also permanently attached to the membrane, but do not pass through it.[2]"
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Transmembrane_protein]
    <a class="clsHide" href="#idPrtnOgm004003dsn"></a></p>
  </section>

  <section id="idPrtnOgm004002">
  <h2 id="idPrtnOgm004002H2">prtnMembrane.peripheral
    <a class="clsHide" href="#idPrtnOgm004002H2"></a></h2>
  <p id="idPrtnOgm004002nam">name::
    <br>* Mcs.prtnMembrane.peripheral,
    <br>* Mcs.prtnMembrane.002-peripheral,
    <br>* Mcs.peripheral-membrane-protein,
    <a class="clsHide" href="#idPrtnOgm004002nam"></a></p>
  <p id="idPrtnOgm004002dsn">description::
    <br>"Peripheral membrane proteins are membrane proteins that adhere only temporarily to the biological membrane with which they are associated. These proteins attach to integral membrane proteins, or penetrate the peripheral regions of the lipid bilayer. The regulatory protein subunits of many ion channels and transmembrane receptors, for example, may be defined as peripheral membrane proteins. In contrast to integral membrane proteins, peripheral membrane proteins tend to collect in the water-soluble component, or fraction, of all the proteins extracted during a protein purification procedure. Proteins with GPI anchors are an exception to this rule and can have purification properties similar to those of integral membrane proteins.
    <br>The reversible attachment of proteins to biological membranes has shown to regulate cell signaling and many other important cellular events, through a variety of mechanisms.[1] For example, the close association between many enzymes and biological membranes may bring them into close proximity with their lipid substrate(s).[2] Membrane binding may also promote rearrangement, dissociation, or conformational changes within many protein structural domains, resulting in an activation of their biological activity.[3][4] Additionally, the positioning of many proteins are localized to either the inner or outer surfaces or leaflets of their resident membrane.[5] This facilitates the assembly of multi-protein complexes by increasing the probability of any appropriate protein–protein interactions."
    <br>[{2020-05-14} https://en.wikipedia.org/wiki/Peripheral_membrane_protein]
    <a class="clsHide" href="#idPrtnOgm004002dsn"></a></p>
  </section>

  <section id="idPrtnHmn002">
  <h2 id="idPrtnHmn002H2">prtnMembrane.human
    <a class="clsHide" href="#idPrtnHmn002H2"></a></h2>
  <p id="idPrtnHmn002nam">name::
    <br>* Mcs.prtMembrane.human,
    <br>* Mcs.proteinHmn.002-membrane, /mémbrein/,
    <br>* Mcs.proteinHmn.membrane-002,
    <br>* Mcs.membrane-proteinHmn-002,
    <a class="clsHide" href="#idPrtnHmn002nam"></a></p>
  <p id="idPrtnHmn002dsn">description::
    <br>"Membrane proteins are common proteins that are part of, or interact with, biological membranes. Membrane proteins fall into several broad categories depending on their location. Integral membrane proteins are a permanent part of a cell membrane and can either penetrate the membrane (transmembrane) or associate with one or the other side of a membrane (integral monotopic). Peripheral membrane proteins are transiently associated with the cell membrane.
    <br>Membrane proteins are common, and medically important—about a third of all human proteins are membrane proteins, and these are targets for more than half of all drugs. Nonetheless, compared to other classes of proteins, determining membrane protein structures remains a challenge in large part due to the difficulty in establishing experimental conditions that can preserve the correct conformation of the protein in isolation from its native environment."
    <br>[https://en.wikipedia.org/wiki/Membrane_protein {2020-03-11}]
    <a class="clsHide" href="#idPrtnHmn002dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgm005">
  <h1 id="idPrtnOgm005H1">molProtein.transcription-factor-005
    <a class="clsHide" href="#idPrtnOgm005H1"></a></h1>
  <p id="idPrtnOgm005nam">name::
    <br>* Mcs.molProtein.005-transcription-factor,
    <br>* Mcs.molProtein.transcription-factor-005,
    <br>* Mcs.transcription-factor-molProtein-005,
    <a class="clsHide" href="#idPrtnOgm005nam"></a></p>
  <p id="idPrtnOgm005dsn">description::
    <br>"In molecular biology, a transcription factor (TF) (or sequence-specific DNA-binding factor) is a protein that controls the rate of transcription of genetic information from DNA to messenger RNA, by binding to a specific DNA sequence.[1][2] The function of TFs is to regulate—turn on and off—genes in order to make sure that they are expressed in the right cell at the right time and in the right amount throughout the life of the cell and the organism. Groups of TFs function in a coordinated fashion to direct cell division, cell growth, and cell death throughout life; cell migration and organization (body plan) during embryonic development; and intermittently in response to signals from outside the cell, such as a hormone. There are up to 1600 TFs in the human genome.[3]
    <br>TFs work alone or with other proteins in a complex, by promoting (as an activator), or blocking (as a repressor) the recruitment of RNA polymerase (the enzyme that performs the transcription of genetic information from DNA to RNA) to specific genes.[4][5][6]
    <br>A defining feature of TFs is that they contain at least one DNA-binding domain (DBD), which attaches to a specific sequence of DNA adjacent to the genes that they regulate.[7][8] TFs are grouped into classes based on their DBDs.[9][10] Other proteins such as coactivators, chromatin remodelers, histone acetyltransferases, histone deacetylases, kinases, and methylases are also essential to gene regulation, but lack DNA-binding domains, and therefore are not TFs.[11]
    <br>TFs are of interest in medicine because TF mutations can cause specific diseases, and medications can be potentially targeted toward them."
    <br>[https://en.wikipedia.org/wiki/Transcription_factor]
    <a class="clsHide" href="#idPrtnOgm005dsn"></a></p>
</section>

<section id="idPrtnOgm006">
  <h1 id="idPrtnOgm006H1">molProtein.ribosomal-006
    <a class="clsHide" href="#idPrtnOgm006H1"></a></h1>
  <p id="idPrtnOgm006nam">name::
    <br>* Mcs.molProtein.006-ribosomal,
    <br>* Mcs.molProtein.ribosomal-006,
    <br>* Mcs.ribosomal-molProtein-006,
    <a class="clsHide" href="#idPrtnOgm006nam"></a></p>
  <p id="idPrtnOgm006dsn">description::
    <br>"A ribosomal protein (r-protein or rProtein[1][2][3]) is any of the proteins that, in conjunction with rRNA, make up the ribosomal subunits involved in the cellular process of translation. A large part of the knowledge about these organic molecules has come from the study of E. coli ribosomes. All ribosomal proteins have been isolated and many specific antibodies have been produced. These, together with electronic microscopy and the use of certain reactives, have allowed for the determination of the topography of the proteins in the ribosome. E. coli, other bacteria and Archaea have a 30S small subunit and a 50S large subunit, whereas humans and yeasts have a 40S small subunit and a 60S large subunit.[4] Equivalent subunits are frequently numbered differently between bacteria, Archaea, yeasts and humans.[5] More recently, a near-complete (near)atomic picture of the ribosomal proteins is emerging from the latest high-resolution cryo-EM data (including PDB ID: 5AFI)."
    <br>[https://en.wikipedia.org/wiki/Ribosomal_protein {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm006dsn"></a></p>
</section>

<section id="idPrtnOgm008">
  <h1 id="idPrtnOgm008H1">molProtein.GPX4-008
    <a class="clsHide" href="#idPrtnOgm008H1"></a></h1>
  <p id="idPrtnOgm008nam">name::
    <br>* Mcs.molProtein.008-GPX4,
    <br>* Mcs.molProtein.GPX4-008,
    <br>* Mcs.GPX4-molProtein-008,
    <a class="clsHide" href="#idPrtnOgm008nam"></a></p>
  <p id="idPrtnOgm008dsn">description::
    <br>"GPX4, μια πρωτεΐνη η οποία προστατεύει κύτταρα από οξειδωτικό στρες.
    <br>[https://www.naftemporiki.gr/story/1585510/isxuro-antikarkiniko-oplo-xari-stin-epilusi-moriakou-mustiriou]"
    <a class="clsHide" href="#idPrtnOgm008dsn"></a></p>
</section>

<section id="idPrtnOgm009">
  <h1 id="idPrtnOgm009H1">molProtein.antibody-009
    <a class="clsHide" href="#idPrtnOgm009H1"></a></h1>
  <p id="idPrtnOgm009nam">name::
    <br>* Mcs.molProtein.009-antibody!⇒molProteinAntibody,
    <br>* Mcs.molProtein.antibody-009!⇒molProteinAntibody,
    <br>* Mcs.antibody-molProtein-009!⇒molProteinAntibody,
    <br>* Mcs.molProteinAntibody,
    <a class="clsHide" href="#idPrtnOgm009nam"></a></p>
  <p id="idPrtnOgm009dsn">description::
    <br>"An antibody (Ab), also known as an immunoglobulin (Ig),[1] is a large, Y-shaped protein produced mainly by plasma cells that is used by the immune system to neutralize pathogens such as pathogenic bacteria and viruses. The antibody recognizes a unique molecule of the pathogen, called an antigen, via the fragment antigen-binding (Fab) variable region.[2][3] Each tip of the "Y" of an antibody contains a paratope (analogous to a lock) that is specific for one particular epitope (analogous to a key) on an antigen, allowing these two structures to bind together with precision. Using this binding mechanism, an antibody can tag a microbe or an infected cell for attack by other parts of the immune system, or can neutralize its target directly (for example, by inhibiting a part of a microbe that is essential for its invasion and survival). Depending on the antigen, the binding may impede the biological process causing the disease or may activate macrophages to destroy the foreign substance. The ability of an antibody to communicate with the other components of the immune system is mediated via its Fc region (located at the base of the "Y"), which contains a conserved glycosylation site involved in these interactions.[4] The production of antibodies is the main function of the humoral immune system.[5]
    <br>Antibodies are secreted by B cells of the adaptive immune system, mostly by differentiated B cells called plasma cells. Antibodies can occur in two physical forms, a soluble form that is secreted from the cell to be free in the blood plasma, and a membrane-bound form that is attached to the surface of a B cell and is referred to as the B-cell receptor (BCR). The BCR is found only on the surface of B cells and facilitates the activation of these cells and their subsequent differentiation into either antibody factories called plasma cells or memory B cells that will survive in the body and remember that same antigen so the B cells can respond faster upon future exposure.[6] In most cases, interaction of the B cell with a T helper cell is necessary to produce full activation of the B cell and, therefore, antibody generation following antigen binding.[7] Soluble antibodies are released into the blood and tissue fluids, as well as many secretions to continue to survey for invading microorganisms.
    <br>Antibodies are glycoproteins belonging to the immunoglobulin superfamily.[4] They constitute most of the gamma globulin fraction of the blood proteins. They are typically made of basic structural units—each with two large heavy chains and two small light chains. There are several different types of antibody heavy chains that define the five different types of crystallisable fragments (Fc) that may be attached to the antigen-binding fragments. The five different types of Fc regions allow antibodies to be grouped into five isotypes. Each Fc region of a particular antibody isotype is able to bind to its specific Fc Receptor (FcR), except for IgD, which is essentially the BCR, thus allowing the antigen-antibody complex to mediate different roles depending on which FcR it binds. The ability of an antibody to bind to its corresponding FcR is further modulated by the structure of the glycan(s) present at conserved sites within its Fc region.[4] The ability of antibodies to bind to FcRs helps to direct the appropriate immune response for each different type of foreign object they encounter.[8] For example, IgE is responsible for an allergic response consisting of mast cell degranulation and histamine release. IgE's Fab paratope binds to allergic antigen, for example house dust mite particles, while its Fc region binds to Fc receptor ε. The allergen-IgE-FcRε interaction mediates allergic signal transduction to induce conditions such as asthma.[9]
    <br>Though the general structure of all antibodies is very similar, a small region at the tip of the protein is extremely variable, allowing millions of antibodies with slightly different tip structures, or antigen-binding sites, to exist. This region is known as the hypervariable region. Each of these variants can bind to a different antigen.[2] This enormous diversity of antibody paratopes on the antigen-binding fragments allows the immune system to recognize an equally wide variety of antigens.[1] The large and diverse population of antibody paratope is generated by random recombination events of a set of gene segments that encode different antigen-binding sites (or paratopes), followed by random mutations in this area of the antibody gene, which create further diversity.[8][10] This recombinational process that produces clonal antibody paratope diversity is called V(D)J or VJ recombination. The antibody paratope is polygenic, made up of three genes, V, D, and J. Each paratope locus is also polymorphic, such that during antibody production, one allele of V, one of D, and one of J is chosen. These gene segments are then joined together using random genetic recombination to produce the paratope. The regions where the genes are randomly recombined together is the hypervariable region used to recognise different antigens on a clonal basis.
    <br>Antibody genes also re-organize in a process called class switching that changes the one type of heavy chain Fc fragment to another, creating a different isotype of the antibody that retains the antigen-specific variable region. This allows a single antibody to be used by different types of Fc receptors, expressed on different parts of the immune system."
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Antibody]
    <a class="clsHide" href="#idPrtnOgm009dsn"></a></p>
</section>

<section id="idPrtnOgm012">
  <h1 id="idPrtnOgm012H1">molProtein.prion-012
    <a class="clsHide" href="#idPrtnOgm012H1"></a></h1>
  <p id="idPrtnOgm012nam">name::
    <br>* Mcs.molProtein.012-prion!⇒proteinPrion,
    <br>* Mcs.molProtein.prion-012!⇒proteinPrion,
    <br>* Mcs.prion-molProtein-012!⇒proteinPrion,
    <br>* Mcs.proteinPrion,
    <a class="clsHide" href="#idPrtnOgm012nam"></a></p>
  <p id="idPrtnOgm012dsn">description::
    <br>"Prions (/ˈpriːɒn/ (About this soundlisten), /ˈpraɪɒn/[1][2]) are misfolded proteins with the ability to transmit their misfolded shape onto normal variants of the same protein. They characterize several fatal and transmissible neurodegenerative diseases in humans and many other animals.[3] It is not known what causes the normal protein to misfold, but the abnormal three-dimensional structure is suspected of conferring infectious properties, collapsing nearby protein molecules into the same shape. The word prion derives from "proteinaceous infectious particle".[4][5][6] The hypothesized role of a protein as an infectious agent stands in contrast to all other known infectious agents such as viruses, bacteria, fungi and parasites, all of which contain nucleic acids (DNA, RNA or both).
    <br>Prion variants of the prion protein (PrP), whose specific function is uncertain, are hypothesized as the cause of transmissible spongiform encephalopathies (TSEs),[7] including scrapie in sheep, chronic wasting disease (CWD) in deer, bovine spongiform encephalopathy (BSE) in cattle (commonly known as "mad cow disease") and Creutzfeldt–Jakob disease (CJD) in humans. All known prion diseases in mammals affect the structure of the brain or other neural tissue; all are progressive, have no known effective treatment and are always fatal.[8] Until 2015, all known mammalian prion diseases were considered to be caused by the prion protein (PrP); however in 2015 multiple system atrophy (MSA) was found to be transmissible and was hypothesized to be caused by a prion form of alpha-synuclein.[9]
    <br>Prions form abnormal aggregates of proteins called amyloids, which accumulate in infected tissue and are associated with tissue damage and cell death.[10] Amyloids are also responsible for several other neurodegenerative diseases such as Alzheimer's disease and Parkinson's disease.[11] Prion aggregates are stable, and this structural stability means that prions are resistant to denaturation by chemical and physical agents: they cannot be destroyed by ordinary disinfection or cooking. This makes disposal and containment of these particles difficult.
    <br>A prion disease is a type of proteopathy, or disease of structurally abnormal proteins. In humans, prions are believed to be the cause of Creutzfeldt–Jakob disease (CJD), its variant (vCJD), Gerstmann–Sträussler–Scheinker syndrome (GSS), fatal familial insomnia (FFI) and kuru.[4] There is also evidence suggesting prions may play a part in the process of Alzheimer's disease, Parkinson's disease and amyotrophic lateral sclerosis (ALS), and these have been termed prion-like diseases.[12][13][14][15] Several yeast proteins have also been identified as having prionogenic properties.[16][17] Prion replication is subject to epimutation and natural selection just as for other forms of replication, and their structure varies slightly between species.[18]"
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Prion]
    <a class="clsHide" href="#idPrtnOgm012dsn"></a></p>
</section>

<section id="idPrtnOgm013">
  <h1 id="idPrtnOgm013H1">molProtein.transporter-013
    <a class="clsHide" href="#idPrtnOgm013H1"></a></h1>
  <p id="idPrtnOgm013nam">name::
    <br>* Mcs.molProtein.transport!⇒proteinTransporter,
    <br>* Mcs.molProtein.013-transport!⇒proteinTransporter,
    <br>* Mcs.proteinTransporter,
    <br>* Mcs.transport-molProtein!⇒proteinTransporter,
    <a class="clsHide" href="#idPrtnOgm013nam"></a></p>
  <p id="idPrtnOgm013dsn">description::
    <br>"A transport protein (variously referred to as a transmembrane pump, transporter, escort protein, acid transport protein, cation transport protein, or anion transport protein) is a protein that serves the function of moving other materials within an organism. Transport proteins are vital to the growth and life of all living things. There are several different kinds of transport proteins.
    <br>Carrier proteins are proteins involved in the movement of ions, small molecules, or macromolecules, such as another protein, across a biological membrane.[1] Carrier proteins are integral membrane proteins; that is, they exist within and span the membrane across which they transport substances. The proteins may assist in the movement of substances by facilitated diffusion (i.e., passive transport) or active transport. These mechanisms of movement are known as carrier-mediated transport.[2] Each carrier protein is designed to recognize only one substance or one group of very similar substances. Research has correlated defects in specific carrier proteins with specific diseases.[3] A membrane transport protein (or simply transporter) is a membrane protein[4] that acts as such a carrier.
    <br>A vesicular transport protein is a transmembrane or membrane associated protein. It regulates or facilitates the movement by vesicles of the contents of the cell.[5]"
    <br>[{2020-05-06} https://en.wikipedia.org/wiki/Transport_protein]
    <a class="clsHide" href="#idPrtnOgm013dsn"></a></p>
</section>

<section id="idPrtnOgm014">
  <h1 id="idPrtnOgm014H1">molProtein.enzyme-014
    <a class="clsHide" href="#idPrtnOgm014H1"></a></h1>
  <p id="idPrtnOgm014nam">name::
    <br>* Mcs.molProtein.enzyme!⇒prtnEnzyme,
    <br>* Mcs.molProtein.014-enzyme!⇒prtnEnzyme,
    <br>* Mcs.proteinEnzymeOgm!⇒prtnEnzyme,
    <br>* Mcs.prtnEnzyme,
    <a class="clsHide" href="#idPrtnOgm014nam"></a></p>
  <p id="idPrtnOgm014dsn">description::
    <br>"Enzymes /ˈɛnzaɪmz/ are proteins that act as biological catalysts (biocatalysts). Catalysts accelerate chemical reactions. The molecules upon which enzymes may act are called substrates, and the enzyme converts the substrates into different molecules known as products. Almost all metabolic processes in the cell need enzyme catalysis in order to occur at rates fast enough to sustain life.[1]:8.1 Metabolic pathways depend upon enzymes to catalyze individual steps. The study of enzymes is called enzymology and a new field of pseudoenzyme analysis has recently grown up, recognising that during evolution, some enzymes have lost the ability to carry out biological catalysis, which is often reflected in their amino acid sequences and unusual 'pseudocatalytic' properties.[2][3]"
    <br>[{2020-05-10} https://en.wikipedia.org/wiki/Enzyme]
    <a class="clsHide" href="#idPrtnOgm014dsn"></a></p>

  <section id="idPrtnOgm014gst">
  <h2 id="idPrtnOgm014gstH2">GENERIC-SPECIFIC-TREE of prtnEnzyme
    <a class="clsHide" href="#idPrtnOgm014gstH2"></a></h2>
  <p id="idPrtnOgm014gstnam">name::
    <br>* Mcs.prtnEnzyme'generic-specific-tree,
    <a class="clsHide" href="#idPrtnOgm014gstnam"></a></p>

  <section id="idPrtnOgm014gtr">
  <h3 id="idPrtnOgm014gtrH3">GENERIC-TREE of prtnEnzyme
    <a class="clsHide" href="#idPrtnOgm014gtrH3"></a></h3>
  <p id="idPrtnOgm014gnc">generic-of-prtnEnzyme::
    <br>*
    <a class="clsHide" href="#idPrtnOgm014gnc"></a></p>
  <p id="idPrtnOgm014atttr">attribute-tree-of-prtnEnzyme::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgm014atttr"></a></p>
  <p id="idPrtnOgm014attihf">att-tree-inherited-from::
    <br>· :
    <br>* ,
    <a class="clsHide" href="#idPrtnOgm014attihf"></a></p>
  <p id="idPrtnOgm014attown">att-tree-own-of-prtnEnzyme::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgm014attown"></a></p>
  </section>

  <section id="idPrtnOgm014str">
  <h3 id="idPrtnOgm014strH3">SPECIFIC-TREE of prtnEnzyme
    <a class="clsHide" href="#idPrtnOgm014strH3"></a></h3>
  <p id="idPrtnOgm014spc">specific-of-prtnEnzyme::
    <br>* https://en.wikipedia.org/wiki/List_of_enzymes,
    <br>===
    <br>* enzymeHmn,
    <a class="clsHide" href="#idPrtnOgm014spc"></a></p>
  </section>
  </section>
</section>

<section id="idPrtnOgm015">
  <h1 id="idPrtnOgm015H1">molProtein.globular-015
    <a class="clsHide" href="#idPrtnOgm015H1"></a></h1>
  <p id="idPrtnOgm015nam">name::
    <br>* Mcs.molProtein.globular!⇒proteinGlobular,
    <br>* Mcs.molProtein.015-globular!⇒proteinGlobular,
    <br>* Mcs.globular-molProtein!⇒proteinGlobular,
    <br>* Mcs.proteinGlobular,
    <a class="clsHide" href="#idPrtnOgm015nam"></a></p>
  <p id="idPrtnOgm015dsn">description::
    <br>"Globular proteins or spheroproteins are spherical ("globe-like") proteins and are one of the common protein types (the others being fibrous, disordered and membrane proteins). Globular proteins are somewhat water-soluble (forming colloids in water), unlike the fibrous or membrane proteins.[1] There are multiple fold classes of globular proteins, since there are many different architectures that can fold into a roughly spherical shape.
    <br>The term globin can refer more specifically to proteins including the globin fold.[2]"
    <br>[{2020-05-10} https://en.wikipedia.org/wiki/Globular_protein]
    <a class="clsHide" href="#idPrtnOgm015dsn"></a></p>
</section>

<section id="idPrtnOgm016">
  <h1 id="idPrtnOgm016H1">molProtein.fibrous-016
    <a class="clsHide" href="#idPrtnOgm016H1"></a></h1>
  <p id="idPrtnOgm016nam">name::
    <br>* Mcs.molProtein.fibrous-016!⇒proteinFibrous,
    <br>* Mcs.molProtein.016-fibrous!⇒proteinFibrous,
    <br>* Mcs.fibrous-molProtein!⇒proteinFibrous,
    <br>* Mcs.proteinFibrous,
    <br>* Mcs.scleroprotein!⇒proteinFibrous,
    <a class="clsHide" href="#idPrtnOgm016nam"></a></p>
  <p id="idPrtnOgm016dsn">description::
    <br>"Scleroproteins or fibrous proteins[1] constitute one of the three main types of proteins (alongside globular and membrane proteins).[2] There are many scleroprotein superfamilies including keratin, collagen, elastin, and fibroin. The roles of such proteins include protection and support, forming connective tissue, tendons, bone matrices, and muscle fiber."
    <br>[{2020-05-10} https://en.wikipedia.org/wiki/Scleroprotein]
    <a class="clsHide" href="#idPrtnOgm016dsn"></a></p>
</section>

<section id="idPrtnOgm017">
  <h1 id="idPrtnOgm017H1">molProtein.disordered-017
    <a class="clsHide" href="#idPrtnOgm017H1"></a></h1>
  <p id="idPrtnOgm017nam">name::
    <br>* Mcs.molProtein.disordered-017!⇒proteinDisordered,
    <br>* Mcs.IDP!⇒proteinDisordered,
    <br>* Mcs.molProtein.017-disordered!⇒proteinDisordered,
    <br>* Mcs.disordered-molProtein!⇒proteinDisordered,
    <br>* Mcs.intrinsically-disordered-molProtein!⇒proteinDisordered,
    <br>* Mcs.proteinDisordered,
    <a class="clsHide" href="#idPrtnOgm017nam"></a></p>
  <p id="idPrtnOgm017dsn">description::
    <br>"An intrinsically disordered protein (IDP) is a protein that lacks a fixed or ordered three-dimensional structure.[2][3][4] IDPs range from fully unstructured to partially structured and include random coils, (pre-)molten globules, and large multi-domain proteins connected by flexible linkers. They are one of the main types of protein along with globular, fibrous and membrane proteins.[5]
    <br>The discovery of IDPs has challenged the structural paradigm that protein function depends on a fixed three-dimensional structure. That dogma has been challenged in the 21st century by increasing evidence from structural biology, suggesting that protein dynamics are highly relevant. Despite their lack of stable structure, IDPs are a very large and functionally important class of proteins. Some IDPs can adopt a fixed three-dimensional structure after binding to other macromolecules. Overall, IDPs are different from structured proteins in many ways and tend to have distinctive function, structure, sequence, interactions, evolution and regulation.[6]"
    <br>[{2020-05-10} https://en.wikipedia.org/wiki/Intrinsically_disordered_proteins]
    <a class="clsHide" href="#idPrtnOgm017dsn"></a></p>
</section>

<section id="idPrtnOgm018">
  <h1 id="idPrtnOgm018H1">molProtein.globin-018
    <a class="clsHide" href="#idPrtnOgm018H1"></a></h1>
  <p id="idPrtnOgm018nam">name::
    <br>* Mcs.molProtein.globin-018!⇒proteinGlobin,
    <br>* Mcs.molProtein.018-globin!⇒proteinGlobin,
    <br>* Mcs.globin-molProtein!⇒proteinGlobin,
    <br>* Mcs.proteinGlobin,
    <a class="clsHide" href="#idPrtnOgm018nam"></a></p>
  <p id="idPrtnOgm018dsn">description::
    <br>"The globins are a superfamily of heme-containing globular proteins, involved in binding and/or transporting oxygen. These proteins all incorporate the globin fold, a series of eight alpha helical segments. Two prominent members include myoglobin and hemoglobin. Both of these proteins reversibly bind oxygen via a heme prosthetic group. They are widely distributed in many organisms.[2]"
    <br>[{2020-05-10} https://en.wikipedia.org/wiki/Globin]
    <a class="clsHide" href="#idPrtnOgm018dsn"></a></p>
</section>

<section id="idPrtnOgm019">
  <h1 id="idPrtnOgm019H1">molProtein.albumin-019
    <a class="clsHide" href="#idPrtnOgm019H1"></a></h1>
  <p id="idPrtnOgm019nam">name::
    <br>* Mcs.molProtein.albumin-019!⇒proteinAlbumin,
    <br>* Mcs.molProtein.019-albumin!⇒proteinAlbumin,
    <br>* Mcs.albumin-molProtein!⇒proteinAlbumin,
    <br>* Mcs.proteinAlbumin,
    <a class="clsHide" href="#idPrtnOgm019nam"></a></p>
  <p id="idPrtnOgm019dsn">description::
    <br>"Albumin is a family of globular proteins, the most common of which are the serum albumins. All the proteins of the albumin family are water-soluble, moderately soluble in concentrated salt solutions, and experience heat denaturation. Albumins are commonly found in blood plasma and differ from other blood proteins in that they are not glycosylated. Substances containing albumins are called albuminoids.
    <br>A number of blood transport proteins are evolutionarily related in the albumin family, including serum albumin, alpha-fetoprotein, vitamin D-binding protein and afamin.[3][4][5] This family is only found in vertebrates.[6]
    <br>Albumins in a less strict sense can mean other proteins that coagulate under certain conditions."
    <br>[{2020-05-10} https://en.wikipedia.org/wiki/Albumin]
    <a class="clsHide" href="#idPrtnOgm019dsn"></a></p>
</section>

<section id="idPrtnOgm020">
  <h1 id="idPrtnOgm020H1">molProtein.kinase-020
    <a class="clsHide" href="#idPrtnOgm020H1"></a></h1>
  <p id="idPrtnOgm020nam">name::
    <br>* Mcs.molProtein.kinase-020!⇒proteinKinase, /káinis/,
    <br>* Mcs.molProtein.020-kinase!⇒proteinKinase,
    <br>* Mcs.kinase-molProtein!⇒proteinKinase,
    <br>* Mcs.proteinKinase,
    <a class="clsHide" href="#idPrtnOgm020nam"></a></p>
  <p id="idPrtnOgm020dsn">description::
    <br>"A protein kinase is a kinase enzyme that modifies other proteins by chemically adding phosphate groups to them (phosphorylation). Phosphorylation usually results in a functional change of the target protein (substrate) by changing enzyme activity, cellular location, or association with other proteins. The human genome contains about 500 protein kinase genes and they constitute about 2% of all human genes.[1] Protein kinases are also found in bacteria and plants. Up to 30% of all human proteins may be modified by kinase activity, and kinases are known to regulate the majority of cellular pathways, especially those involved in signal transduction."
    <br>[{2020-05-10} https://en.wikipedia.org/wiki/Protein_kinase]
    <a class="clsHide" href="#idPrtnOgm020dsn"></a></p>
</section>

<section id="idPrtnOgm021">
  <h1 id="idPrtnOgm021H1">molProtein.ApoB-021
    <a class="clsHide" href="#idPrtnOgm021H1"></a></h1>
  <p id="idPrtnOgm021nam">name::
    <br>* Mcs.molProtein.ApoB-021,
    <br>* Mcs.molProtein.021-ApoB,
    <br>* Mcs.apolipoprotein-B,
    <a class="clsHide" href="#idPrtnOgm021nam"></a></p>
  <p id="idPrtnOgm021dsn">description::
    <br>"Apolipoprotein B (ApoB) is a protein that in humans is encoded by the APOB gene.
    <br>Function
    <br>Apolipoprotein B is the primary apolipoprotein of chylomicrons, VLDL, IDL, and LDL particles (LDL - known commonly by the misnomer "bad cholesterol" when in reference to both heart disease and vascular disease in general), which is responsible for carrying fat molecules (lipids), including cholesterol, around the body (within the water outside cells) to all cells within all tissues. While all the functional roles of ApoB within the LDL (and all larger) particles remain somewhat unclear, it is the primary organizing protein (of the entire complex shell enclosing/carrying fat molecules within) component of the particles and is absolutely required for the formation of these particles. What is also clear is that the ApoB on the LDL particle acts as a ligand for LDL receptors in various cells throughout the body (i.e., less formally, ApoB indicates fat carrying particles are ready to enter any cells with ApoB receptors and deliver fats carried within into the cells).
    <br>Through mechanisms only partially understood, high levels of ApoB, especially associated with the higher LDL particle concentrations, are the primary driver of plaques that cause vascular disease (atherosclerosis), commonly first becoming obviously symptomatic as heart disease, stroke & many other body wide complications after decades of progression. There is considerable evidence that concentrations of ApoB[5][6] and especially the NMR assay[7] (specific for LDL-particle concentrations) are superior indicators of vascular/heart disease driving physiology than either total cholesterol or LDL-cholesterol (as long promoted by the NIH starting in the early 1970s). However, primarily for historic cost/complexity reasons, cholesterol, and estimated LDL-cholesterol by calculation, remains the most commonly promoted lipid test for the risk factor of atherosclerosis. ApoB is routinely measured using immunoassays such as ELISA or nephelometry. Refined and automated NMR methods allow measurement distinctions between the many different ApoB particles."
    <br>[{2020-05-13} https://en.wikipedia.org/wiki/Apolipoprotein_B]
    <a class="clsHide" href="#idPrtnOgm021dsn"></a></p>
</section>

<section id="idMeta">
  <h1 id="idMetaH1">meta-info
    <a class="clsHide" href="#idMetaH1"></a></h1>
  <p id="idMetaCounter" class="clsCenter">this page was-visited
    <span class="clsColorRed">
    <script src="../../dirPgm/dirCntr/counter.php?page=Mws-dirOgm.Ogm-prtn"></script>
    </span>
    times since {2020-03-25}</p>
  <!-- the content of page-path paragraph is displayed as it is on top of toc -->
  <p id="idMetaWebpage_path"><span class="clsB clsColorGreen">page-wholepath</span>:
    <a class="clsPreview" href="../../#idOverview">synagonism.net</a> /
    <a class="clsPreview" href="../filMcsWorld.last.html#idOverview">Mcs-worldview</a> /
    <a class="clsPreview" href="filMcsDirOgm.last.html#idOverview">dirOgm</a> /
    molProtein
    </p>
  <p id="idMetaP1">SEARCH::
    <br>· this page uses '<span class="clsColorRed">locator-names</span>', names that when you find them, you find the-LOCATION of the-concept they denote.
    <br>⊛ <strong>GLOBAL-SEARCH</strong>:
    <br>· clicking on <span class="clsColorGreenBg">the-green-BAR of a-page</span> you have access to the-global--locator-names of my-site.
    <br>· use the-prefix '<span class="clsColorRed">molProtein</span>' for <a class="clsPreview" href="../dirCor/filMcs.last.html#idOverview">sensorial-concepts</a> related to current concept 'protein-of-organism'.
    <br>⊛ <strong>LOCAL-SEARCH</strong>:
    <br>· TYPE <span class="clsColorRed">CTRL+F "Mcs.words-of-concept's-name"</span>, to go to the-LOCATION of the-concept.
    <br>· a-preview of the-description of a-global-name makes reading fast.
    <a class="clsHide" href="#idMetaP1"></a></p>
  <p id="idFooterP1">footer::
    <br>• author: <a class="clsPreview" href="../../#idAboutme">Kaseluris.Nikos.1959</a>
    <br>• email:
    <br> &nbsp;<img src="../../misc/img/mail.png" alt="imgMail">
    <br>• edit on github: https://github.com/synagonism/Mcsw/blob/master/dirOgm/filMcsOgm-prtn.last.html,
    <br>• comments on <a class="clsPreview" href="filMcsDirOgm.last.html#idComment">Disqus</a>,
    <br>• twitter: <a href="https://twitter.com/synagonism">@synagonism</a>,
    <br>• steemit: <a href="https://steemit.com/synagonism">https://steemit.com/@synagonism</a>,
    <a class="clsHide" href="#idFooterP1"></a></p>
  <!--                              -->
  <p id="idMetaVersion">webpage-versions::
    <br>• <a href="filMcsOgm-prtn.last.html">filMcsOgm-prtn.last.html</a>: dynamic,
    <br>• filMcsOgm-prtn.0-1-0.2020-03-25.last.html: draft creation,
    <a class="clsHide" href="#idMetaVersion"></a></p>
</section>

<section id="idSupport">
  <h1 id="idSupportH1">support (<a class="clsPreview" href="../../#idSupport">link</a>)</h1>
  <p></p>
</section>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-19285371-5', 'synagonism.net');
  ga('send', 'pageview');
</script>

<script src="https://synagonism.github.io/hitp/hitp.js"></script>
</body>
</html>