<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Mcs.proteinOgm (protein-material--of-organism 0-7-0.2020-04-22 draft)</title>
  <meta name="keywords" content="protein-of-organism, proteinOgm, ModelConceptSensorial, McsHitp, Synagonism">
  <link rel="stylesheet" href="https://synagonism.github.io/hitp/hitp.css">
</head>

<body>
<header id="idHeader">
  <p></p>
  <h1 id="idHeaderH1">protein-material--of-organism
    <br>sensorial-concept-Mcs (proteinOgm)
    </h1>
  <p id="idHeadercrd">McsHitp-creation:: {2020-03-25},
    <a class="clsHide" href="#idHeadercrd"></a></p>
</header>

<section id="idOverview">
  <h1 id="idOverviewH1">overview of proteinOgm
    <a class="clsHide" href="#idOverviewH1"></a></h1>
  <p id="idName">name::
    <br>* Mcs.filMcsOgm-prtn.last.html!⇒proteinOgm,
    <br>* Mcs.dirOgm/filMcsOgm-prtn.last.html!⇒proteinOgm,
    <br>* Mcs.proteinOgm,
    <br>* Mcs.protein-of-organism!⇒proteinOgm,
    <br>* Mcs.proteinOgm'(protein-of-organism)!⇒proteinOgm,
    <a class="clsHide" href="#idName"></a></p>
  <p id="idDescription">description::
    <br>"Proteins are large biomolecules, or macromolecules, consisting of one or more long chains of amino acid residues. Proteins perform a vast array of functions within organisms, including catalysing metabolic reactions, DNA replication, responding to stimuli, providing structure to cells, and organisms, and transporting molecules from one location to another. Proteins differ from one another primarily in their sequence of amino acids, which is dictated by the nucleotide sequence of their genes, and which usually results in protein folding into a specific three-dimensional structure that determines its activity.
    <br>A linear chain of amino acid residues is called a polypeptide. A protein contains at least one long polypeptide. Short polypeptides, containing less than 20–30 residues, are rarely considered to be proteins and are commonly called peptides, or sometimes oligopeptides. The individual amino acid residues are bonded together by peptide bonds and adjacent amino acid residues. The sequence of amino acid residues in a protein is defined by the sequence of a gene, which is encoded in the genetic code. In general, the genetic code specifies 20 standard amino acids; however, in certain organisms the genetic code can include selenocysteine and—in certain archaea—pyrrolysine. Shortly after or even during synthesis, the residues in a protein are often chemically modified by post-translational modification, which alters the physical and chemical properties, folding, stability, activity, and ultimately, the function of the proteins. Sometimes proteins have non-peptide groups attached, which can be called prosthetic groups or cofactors. Proteins can also work together to achieve a particular function, and they often associate to form stable protein complexes.
    <br>Once formed, proteins only exist for a certain period and are then degraded and recycled by the cell's machinery through the process of protein turnover. A protein's lifespan is measured in terms of its half-life and covers a wide range. They can exist for minutes or years with an average lifespan of 1–2 days in mammalian cells. Abnormal or misfolded proteins are degraded more rapidly either due to being targeted for destruction or due to being unstable.
    <br>Like other biological macromolecules such as polysaccharides and nucleic acids, proteins are essential parts of organisms and participate in virtually every process within cells. Many proteins are enzymes that catalyse biochemical reactions and are vital to metabolism. Proteins also have structural or mechanical functions, such as actin and myosin in muscle and the proteins in the cytoskeleton, which form a system of scaffolding that maintains cell shape. Other proteins are important in cell signaling, immune responses, cell adhesion, and the cell cycle. In animals, proteins are needed in the diet to provide the essential amino acids that cannot be synthesized. Digestion breaks the proteins down for use in the metabolism.
    <br>Proteins may be purified from other cellular components using a variety of techniques such as ultracentrifugation, precipitation, electrophoresis, and chromatography; the advent of genetic engineering has made possible a number of methods to facilitate purification. Methods commonly used to study protein structure and function include immunohistochemistry, site-directed mutagenesis, X-ray crystallography, nuclear magnetic resonance and mass spectrometry."
    <br>[https://en.wikipedia.org/wiki/Protein {2020-03-25}]
    <a class="clsHide" href="#idDescription"></a></p>
</section>

<section id="idPrtnOgmatt003">
  <h1 id="idPrtnOgmatt003H1">01_amino-acid of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt003H1"></a></h1>
  <p id="idPrtnOgmatt003nam">name::
    <br>* Mcs.proteinOgm'01_amino-acid,
    <br>* Mcs.proteinOgm'att003-amino-acid,
    <br>* Mcs.proteinOgm'amino-acid-att003,
    <br>* Mcs.amino-acidOgm,
    <a class="clsHide" href="#idPrtnOgmatt003nam"></a></p>
  <p id="idPrtnOgmatt003dsn">description::
    <br>"DNA, RNA, and proteins all consist of a repeating structure of related building blocks (nucleotides in the case of DNA and RNA, amino acids in the case of proteins)."
    <br>[https://en.wikipedia.org/wiki/Macromolecule {2020-03-26}]
    <a class="clsHide" href="#idPrtnOgmatt003dsn"></a></p>
</section>

<section id="idPrtnOgmatt004">
  <h1 id="idPrtnOgmatt004H1">02_peptide of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt004H1"></a></h1>
  <p id="idPrtnOgmatt004nam">name::
    <br>* Mcs.proteinOgm'02_peptide,
    <br>* Mcs.proteinOgm'att004-peptide,
    <br>* Mcs.proteinOgm'peptide-att004,
    <br>* Mcs.peptide-004,
    <a class="clsHide" href="#idPrtnOgmatt004nam"></a></p>
  <p id="idPrtnOgmatt004dsn">description::
    <br>"Peptides (from Greek language πεπτός, peptós "digested"; derived from πέσσειν, péssein "to digest") are short chains of amino acids linked by peptide (amide) bonds.[1] The simplest peptides are dipeptides, followed by tripeptides, tetrapeptides, etc. A polypeptide is a long, continuous, and unbranched peptide chain. Hence, peptides fall under the broad chemical classes of biological oligomers and polymers, alongside nucleic acids, oligosaccharides, polysaccharides, and others.
    <br>Peptides are distinguished from proteins on the basis of size, and as an arbitrary benchmark can be understood to contain approximately 50 or fewer amino acids.[2][3] Proteins consist of one or more polypeptides arranged in a biologically functional way, often bound to ligands such as coenzymes and cofactors, or to another protein or other macromolecule (DNA, RNA, etc.), or to complex macromolecular assemblies.[4] Finally, while aspects of the lab techniques applied to peptides versus polypeptides and proteins differ (e.g., the specifics of electrophoresis, chromatography, etc.), the size boundaries that distinguish peptides from polypeptides and proteins are not absolute: long peptides such as amyloid beta have been referred to as proteins, and smaller proteins like insulin have been considered peptides.
    <br>Amino acids that have been incorporated into peptides are termed "residues". A water molecule is released during formation of each amide bond.[5] All peptides except cyclic peptides have an N-terminal (amine group) and C-terminal (carboxyl group) residue at the end of the peptide (as shown for the tetrapeptide in the image)."
    <br>[https://en.wikipedia.org/wiki/Peptide {2020-03-26}]
    <a class="clsHide" href="#idPrtnOgmatt004dsn"></a></p>
</section>

<section id="idPrtnOgmatt006">
  <h1 id="idPrtnOgmatt006H1">03_peptide-bond of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt006H1"></a></h1>
  <p id="idPrtnOgmatt006nam">name::
    <br>* Mcs.proteinOgm'03_peptide-bond,
    <br>* Mcs.proteinOgm'att006-peptide-bond,
    <br>* Mcs.proteinOgm'peptide-bond-att006,
    <br>* Mcs.peptide-bond-of-proteinOgm-att006,
    <a class="clsHide" href="#idPrtnOgmatt006nam"></a></p>
  <p id="idPrtnOgmatt006dsn">description::
    <br>"A peptide bond is an amide type of covalent chemical bond linking two consecutive alpha-amino acids from C1 (carbon number one) of one alpha-amino acid and N2 (nitrogen number two) of another along a peptide or protein chain.[1]
    <br>It can also be called an eupeptide bond[1] to separate it from an isopeptide bond, a different type of amide bond between two amino acids."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Peptide_bond]
    <a class="clsHide" href="#idPrtnOgmatt006dsn"></a></p>
</section>

<section id="idPrtnOgmatt005">
  <h1 id="idPrtnOgmatt005H1">polypeptide of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt005H1"></a></h1>
  <p id="idPrtnOgmatt005nam">name::
    <br>* Mcs.proteinOgm'att005-polypeptide,
    <br>* Mcs.proteinOgm'polypeptide-005,
    <br>* Mcs.polypeptide-005,
    <a class="clsHide" href="#idPrtnOgmatt005nam"></a></p>
  <p id="idPrtnOgmatt005dsn">description::
    <br>"A polypeptide is a long, continuous, and unbranched peptide chain."
    <br>[https://en.wikipedia.org/wiki/Peptide {2020-03-26}]
    <a class="clsHide" href="#idPrtnOgmatt005dsn"></a></p>
</section>

<section id="idPrtnOgmatt002">
  <h1 id="idPrtnOgmatt002H1">pro-protein of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt002H1"></a></h1>
  <p id="idPrtnOgmatt002nam">name::
    <br>* Mcs.proteinOgm'att002-proprotein,
    <br>* Mcs.proteinOgm'proprotein,
    <br>* Mcs.pro-peptide,
    <br>* Mcs.protein-precursor,
    <br>* Mcs.proprotein,
    <a class="clsHide" href="#idPrtnOgmatt002nam"></a></p>
  <p id="idPrtnOgmatt002dsn">description::
    <br>"A protein precursor, also called a pro-protein or pro-peptide, is an inactive protein (or peptide) that can be turned into an active form by post-translational modification, such as breaking off a piece of the molecule or adding on another molecule. The name of the precursor for a protein is often prefixed by pro-. Examples include proinsulin and proopiomelanocortin, which are both prohormones.
    <br>Protein precursors are often used by an organism when the subsequent protein is potentially harmful, but needs to be available on short notice and/or in large quantities. Enzyme precursors are called zymogens or proenzymes. Examples are enzymes of the digestive tract in humans.
    <br>Some protein precursors are secreted from the cell. Many of these are synthesized with an N-terminal signal peptide that targets them for secretion. Like other proteins that contain a signal peptide, their name is prefixed by pre. They are thus called pre-pro-proteins or pre-pro-peptides.[1] The signal peptide is cleaved off in the endoplasmic reticulum.[1] An example is preproinsulin.
    <br>Pro-sequences are areas in the protein that are essential for its correct folding, usually in the transition of a protein from an inactive to an active state. Pro-sequences may also be involved in pro-protein"
    <br>[https://en.wikipedia.org/wiki/Protein_precursor {2020-02-28}]
    <a class="clsHide" href="#idPrtnOgmatt002dsn"></a></p>
</section>

<section id="idPrtnOgmatt007">
  <h1 id="idPrtnOgmatt007H1">shape of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt007H1"></a></h1>
  <p id="idPrtnOgmatt007nam">name::
    <br>* Mcs.proteinOgm'att007-shape,
    <br>* Mcs.proteinOgm'shape-att007,
    <a class="clsHide" href="#idPrtnOgmatt007nam"></a></p>
  <p id="idPrtnOgmatt007dsn">description::
    <br>"Protein structure is the three-dimensional arrangement of atoms in an amino acid-chain molecule. Proteins are polymers – specifically polypeptides – formed from sequences of amino acids, the monomers of the polymer. A single amino acid monomer may also be called a residue indicating a repeating unit of a polymer. Proteins form by amino acids undergoing condensation reactions, in which the amino acids lose one water molecule per reaction in order to attach to one another with a peptide bond. By convention, a chain under 30 amino acids is often identified as a peptide, rather than a protein.[1] To be able to perform their biological function, proteins fold into one or more specific spatial conformations driven by a number of non-covalent interactions such as hydrogen bonding, ionic interactions, Van der Waals forces, and hydrophobic packing. To understand the functions of proteins at a molecular level, it is often necessary to determine their three-dimensional structure. This is the topic of the scientific field of structural biology, which employs techniques such as X-ray crystallography, NMR spectroscopy, and dual polarisation interferometry to determine the structure of proteins.
    <br>Protein structures range in size from tens to several thousand amino acids.[2] By physical size, proteins are classified as nanoparticles, between 1–100 nm. Very large aggregates can be formed from protein subunits. For example, many thousands of actin molecules assemble into a microfilament.
    <br>A protein generally undergoes reversible structural changes in performing its biological function. The alternative structures of the same protein are referred to as different conformational isomers, or simply, conformations, and transitions between them are called conformational changes."
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_structure]
    <a class="clsHide" href="#idPrtnOgmatt007dsn"></a></p>
</section>

<section id="idPrtnOgmatt008">
  <h1 id="idPrtnOgmatt008H1">size of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt008H1"></a></h1>
  <p id="idPrtnOgmatt008nam">name::
    <br>* Mcs.proteinOgm'att008-size,
    <br>* Mcs.proteinOgm'size-att008,
    <a class="clsHide" href="#idPrtnOgmatt008nam"></a></p>
  <p id="idPrtnOgmatt008dsn">description::
    <br>"By physical size, proteins are classified as nanoparticles, between 1–100 nm."
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_structure]
    <a class="clsHide" href="#idPrtnOgmatt008dsn"></a></p>
</section>

<section id="idPrtnOgmrscF">
  <h1 id="idPrtnOgmrscFH1">resource of proteinOgm
    <a class="clsHide" href="#idPrtnOgmrscFH1"></a></h1>
  <p id="idPrtnOgmrscnam">name::
    <br>* Mcs.proteinOgm'attResource,
    <br>* Mcs.proteinOgm'resource,
    <a class="clsHide" href="#idPrtnOgmrscnam"></a></p>
  <p id="idPrtnOgmrscwpa">addressWpg::
    <br>* https://en.wikipedia.org/wiki/List_of_topics_related_to_protein,
    <a class="clsHide" href="#idPrtnOgmrscwpa"></a></p>

  <section id="idPrtnOgmatt001">
  <h2 id="idPrtnOgmatt001H2">proteomics of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt001H2"></a></h2>
  <p id="idPrtnOgmatt001nam">name::
    <br>* Mcs.proteinOgm'att001-proteomics,
    <br>* Mcs.proteinOgm'proteomics,
    <br>* Mcs.proteomics-science,
    <br>* Mcs.science.proteomics,
    <a class="clsHide" href="#idPrtnOgmatt001nam"></a></p>
  <p id="idPrtnOgmatt001dsn">description::
    <br>"Proteomics is the large-scale study of proteins.[1][2] Proteins are vital parts of living organisms, with many functions. The word proteome is a portmanteau of protein and genome, and was coined by Marc Wilkins in 1994 while he was a Ph.D. student at Macquarie University.[3] Macquarie University also founded the first dedicated proteomics laboratory in 1995.[4][5]
    <br>The proteome is the entire set of proteins that is produced or modified by an organism or system. Proteomics has enabled the identification of ever increasing numbers of protein. This varies with time and distinct requirements, or stresses, that a cell or organism undergoes.[6] Proteomics is an interdisciplinary domain that has benefitted greatly from the genetic information of various genome projects, including the Human Genome Project.[7] It covers the exploration of proteomes from the overall level of protein composition, structure, and activity. It is an important component of functional genomics.
    <br>Proteomics generally refers to the large-scale experimental analysis of proteins and proteomes, but often is used specifically to refer to protein purification and mass spectrometry."
    <br>[https://en.wikipedia.org/wiki/Proteomics {2020-03-11}]
    <a class="clsHide" href="#idPrtnOgmatt001dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgmsrtrF">
  <h1 id="idPrtnOgmsrtrFH1">structure of proteinOgm
    <a class="clsHide" href="#idPrtnOgmsrtrFH1"></a></h1>
  <p id="idPrtnOgmsrtrFnam">name::
    <br>* Mcs.proteinOgm'attStructure,
    <br>* Mcs.proteinOgm'structure,
    <a class="clsHide" href="#idPrtnOgmsrtrFnam"></a></p>
  <p id="idPrtnOgmsrtrFdsn">description::
    <br>"Proteins are the product of a gene that are formed from translation of a mature mRNA molecule.
    <br>Proteins contain 4 elements in regards to their structure: primary, secondary, tertiary and quaternary.
    <br>The linear amino acid sequence is also known as the primary structure.
    <br>Hydrogen bonding between the amino acids of the primary structure results in the formation of alpha helices or beta sheets.[7] These stable foldings are the secondary structure.
    <br>The particular combination of the primary and secondary structures form the tertiary structure of a polypeptide.[7]
    <br>The quaternary structure refers to the way multiple chains of polypeptides fold together.[7]"
    <br>[https://en.wikipedia.org/wiki/Gene_product#Protein_products {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgmsrtrFdsn"></a></p>
</section>

<section id="idPrtnOgmdngF">
  <h1 id="idPrtnOgmdngFH1">DOING of proteinOgm
    <a class="clsHide" href="#idPrtnOgmdngFH1"></a></h1>
  <p id="idPrtnOgmdngFnam">name::
    <br>* Mcs.proteinOgm'attDoing,
    <br>* Mcs.proteinOgm'doing,
    <a class="clsHide" href="#idPrtnOgmdngFnam"></a></p>
  <p id="idPrtnOgmdngFdsn">description::
    <br>"Proteins are optimised for catalysis
    <br>Proteins are functional macromolecules responsible for catalysing the biochemical reactions that sustain life.[1]:3 Proteins carry out all functions of an organism, for example photosynthesis, neural function, vision, and movement.[13]
    <br>The single-stranded nature of protein molecules, together with their composition of 20 or more different amino acid building blocks, allows them to fold in to a vast number of different three-dimensional shapes, while providing binding pockets through which they can specifically interact with all manner of molecules. In addition, the chemical diversity of the different amino acids, together with different chemical environments afforded by local 3D structure, enables many proteins to act as enzymes, catalyzing a wide range of specific biochemical transformations within cells. In addition, proteins have evolved the ability to bind a wide range of cofactors and coenzymes, smaller molecules that can endow the protein with specific activities beyond those associated with the polypeptide chain alone."
    <br>[https://en.wikipedia.org/wiki/Macromolecule#Proteins_are_optimised_for_catalysis {2020-03-26}]
    <br>===
    <br>"Proteins have many different functions in a cell and the function may vary based on the polypeptides they interact with and their cellular environment. Chaperone proteins work to stabilize newly synthesized proteins. They ensure the new protein folds into its correct functional conformation in addition to making sure products do not aggregate in areas where they should not.[8] Proteins can also function as enzymes, increasing the rate of various biochemical reactions and turning substrates into products.[7][9] Products can be modified by attaching groups such as phosphate via an enzyme to specific amino acids in the primary sequence.[9] Proteins can also be used to move molecules in the cell to where they are needed, these are called motor proteins.[9] The shape of the cell is supported by proteins. Proteins such as actin, microtubules and intermediate filaments provide structure to the cell.[7] Another class of proteins are found in plasma membranes. Membrane proteins can be associated with the plasma membrane in different ways, depending on their structure.[9] These proteins allow the cell to import or export cell products, nutrients or signals to and from the extracellular space.[7][9] Other proteins help the cell to perform regulatory functions. For example, transcription factors bind to DNA to help transcription of RNA.[10]"
    <br>[https://en.wikipedia.org/wiki/Gene_product#Protein_functions {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgmdngFdsn"></a></p>

  <section id="idPrtnOgmatt009">
  <h2 id="idPrtnOgmatt009H2">folding of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt009H2"></a></h2>
  <p id="idPrtnOgmatt009nam">name::
    <br>* Mcs.proteinOgm'att009-folding,
    <br>* Mcs.proteinOgm'folding-att009,
    <a class="clsHide" href="#idPrtnOgmatt009nam"></a></p>
  <p id="idPrtnOgmatt009dsn">description::
    <br>"Protein folding is the physical process by which a protein chain acquires its native 3-dimensional structure, a conformation that is usually biologically functional, in an expeditious and reproducible manner. It is the physical process by which a polypeptide folds into its characteristic and functional three-dimensional structure from a random coil.[1] Each protein exists as an unfolded polypeptide or random coil when translated from a sequence of mRNA to a linear chain of amino acids. This polypeptide lacks any stable (long-lasting) three-dimensional structure (the left hand side of the first figure). As the polypeptide chain is being synthesized by a ribosome, the linear chain begins to fold into its three-dimensional structure. Folding begins to occur even during translation of the polypeptide chain. Amino acids interact with each other to produce a well-defined three-dimensional structure, the folded protein (the right hand side of the figure), known as the native state. The resulting three-dimensional structure is determined by the amino acid sequence or primary structure (Anfinsen's dogma).[2]
    <br>The correct three-dimensional structure is essential to function, although some parts of functional proteins may remain unfolded,[3] so that protein dynamics is important. Failure to fold into native structure generally produces inactive proteins, but in some instances misfolded proteins have modified or toxic functionality. Several neurodegenerative and other diseases are believed to result from the accumulation of amyloid fibrils formed by misfolded proteins.[4] Many allergies are caused by incorrect folding of some proteins, because the immune system does not produce antibodies for certain protein structures.[5]
    <br>Denaturation of proteins is a process of transition from the folded to the unfolded state. It happens in cooking, in burns, in proteinopathies, and in other contexts.
    <br>The duration of the folding process varies dramatically depending on the protein of interest. When studied outside the cell, the slowest folding proteins require many minutes or hours to fold primarily due to proline isomerization, and must pass through a number of intermediate states, like checkpoints, before the process is complete.[6] On the other hand, very small single-domain proteins with lengths of up to a hundred amino acids typically fold in a single step.[7] Time scales of milliseconds are the norm and the very fastest known protein folding reactions are complete within a few microseconds.[8]"
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_folding]
    <a class="clsHide" href="#idPrtnOgmatt009dsn"></a></p>
  </section>

  <section id="idPrtnOgmatt010">
  <h2 id="idPrtnOgmatt010H2">metabolism of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt010H2"></a></h2>
  <p id="idPrtnOgmatt010nam">name::
    <br>* Mcs.proteinOgm'att010-metabolism,
    <br>* Mcs.proteinOgm'metabolism-att010,
    <a class="clsHide" href="#idPrtnOgmatt010nam"></a></p>
  <p id="idPrtnOgmatt010dsn">description::
    <br>· protein-metabolism is the-process of synthesis and breaking-down of proteins.
    <br>===
    <br>"Protein metabolism denotes the various biochemical processes responsible for the synthesis of proteins and amino acids (anabolism), and the breakdown of proteins by catabolism.
    <br>The steps of protein synthesis include transcription, translation, and post translational modifications. During transcription, RNA polymerase transcribes a coding region of the DNA in a cell producing a sequence of RNA, specifically messenger RNA (mRNA). This mRNA sequence contains codons: 3 nucleotide long segments that code for a specific amino acid. Ribosomes translate the codons to their respective amino acids.[1] In humans, non-essential amino acids are synthesized from intermediates in major metabolic pathways such as the Citric Acid Cycle.[2] Essential amino acids must be consumed and are made in other organisms. The amino acids are joined by peptide bonds making a polypeptide chain. This polypeptide chain then goes through post translational modifications and is sometimes joined with other polypeptide chains to form a fully functional protein.
    <br>Dietary proteins are first broken down to individual amino acids by various enzymes and hydrochloric acid present in the gastrointestinal tract. These amino acids are further broken down to α-keto acids which can be recycled in the body for generation of energy, and production of glucose or fat or other amino acids. Proteins can be broken down by enzymes known as peptidases or can break down as a result of denaturation. Proteins can denature in environmental conditions the protein is not made for.[3]"
    <br>[{2020-04-22} https://en.wikipedia.org/wiki/Protein_metabolism]
    <a class="clsHide" href="#idPrtnOgmatt010dsn"></a></p>
  </section>

  <section id="idPrtnOgmatt011">
  <h2 id="idPrtnOgmatt011H2">anabolism of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt011H2"></a></h2>
  <p id="idPrtnOgmatt011nam">name::
    <br>* Mcs.proteinOgm'att011-anabolism,
    <br>* Mcs.proteinOgm'anabolism-att011,
    <a class="clsHide" href="#idPrtnOgmatt011nam"></a></p>
  <p id="idPrtnOgmatt011dsn">description::
    <br>·
    <a class="clsHide" href="#idPrtnOgmatt011dsn"></a></p>
  </section>

  <section id="idPrtnOgmatt012">
  <h2 id="idPrtnOgmatt012H2">katabolism of proteinOgm
    <a class="clsHide" href="#idPrtnOgmatt012H2"></a></h2>
  <p id="idPrtnOgmatt012nam">name::
    <br>* Mcs.proteinOgm'att012-katabolism,
    <br>* Mcs.proteinOgm'katabolism-att012,
    <a class="clsHide" href="#idPrtnOgmatt012nam"></a></p>
  <p id="idPrtnOgmatt012dsn">description::
    <br>·
    <a class="clsHide" href="#idPrtnOgmatt012dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgmevgF">
  <h1 id="idPrtnOgmevgFH1">evoluting of proteinOgm
    <a class="clsHide" href="#idPrtnOgmevgFH1"></a></h1>
  <p id="idPrtnOgmevgnam">name::
    <br>* Mcs.proteinOgm'attEvoluting,
    <br>* Mcs.proteinOgm'evoluting,
    <a class="clsHide" href="#idPrtnOgmevgnam"></a></p>
  <p id="idPrtnOgmevg20200325">{time.2020-03-25}::
    <br>=== McsHitp-creation:
    <br>· creation of current <a class="clsPreview" href="../dirCor/filMcs.last.html#idOverview">concept</a>.
    <a class="clsHide" href="#idPrtnOgmevg20200325"></a></p>
</section>

<section id="idPrtnOgmwptF">
  <h1 id="idPrtnOgmwptFH1">WHOLE-PART-TREE of proteinOgm
    <a class="clsHide" href="#idPrtnOgmwhlFH1"></a></h1>
  <p id="idPrtnOgmwptnam">name::
    <br>* Mcs.proteinOgm'whole-part-tree,
    <a class="clsHide" href="#idPrtnOgmwptnam"></a></p>
  <p id="idPrtnOgmwtr">whole-tree-of-proteinOgm::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmwtr"></a></p>
  <p id="idPrtnOgmptr">part-tree-of-proteinOgm::
    <br>*
    <a class="clsHide" href="#idPrtnOgmptr"></a></p>
</section>

<section id="idPrtnOgmgst">
  <h1 id="idPrtnOgmgstH1">GENERIC-SPECIFIC-TREE of proteinOgm
    <a class="clsHide" href="#idPrtnOgmgstH1"></a></h1>
  <p id="idPrtnOgmgstnam">name::
    <br>* Mcs.proteinOgm'generic-specific-tree,
    <a class="clsHide" href="#idPrtnOgmgstnam"></a></p>

  <section id="idPrtnOgmgtr">
  <h2 id="idPrtnOgmgtrH2">GENERIC-TREE of proteinOgm
    <a class="clsHide" href="#idPrtnOgmgtrH2"></a></h2>
  <p id="idPrtnOgmgnc">generic-of-proteinOgm::
    <br>* biomaterial,
    <br>===
    <br>* polymer,
    <br>"Proteins are polymers – specifically polypeptides – formed from sequences of amino acids, the monomers of the polymer."
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_structure]
    <a class="clsHide" href="#idPrtnOgmgnc"></a></p>
  <p id="idPrtnOgmatttr">attribute-tree-of-proteinOgm::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmatttr"></a></p>
  <p id="idPrtnOgmattihf">att-tree-inherited-from::
    <br>· :
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmattihf"></a></p>
  <p id="idPrtnOgmattown">att-tree-own-of-proteinOgm::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmattown"></a></p>
  </section>

  <section id="idPrtnOgmstr">
  <h2 id="idPrtnOgmstrH2">SPECIFIC-TREE of proteinOgm
    <a class="clsHide" href="#idPrtnOgmstrH2"></a></h2>
  <p id="idPrtnOgmspc">specific-of-proteinOgm::
    <br>* ,
    <a class="clsHide" href="#idPrtnOgmspc"></a></p>
  </section>
</section>

<section id="idPrtn010">
  <h1 id="idPrtn010H1">proteinOgm.molecule-010
    <a class="clsHide" href="#idPrtn010H1"></a></h1>
  <p id="idPrtn010nam">name::
    <br>* Mcs.proteinOgm.010-molecule,
    <br>* Mcs.proteinOgm.molecule-010,
    <br>* Mcs.molProtein,
    <br>* Mcs.protein-subunit,
    <a class="clsHide" href="#idPrtn010nam"></a></p>
  <p id="idPrtn010dsn">description::
    <br>"In structural biology, a protein subunit is a single protein molecule that assembles (or "coassembles") with other protein molecules to form a protein complex. Some naturally occurring proteins have a relatively small number of subunits and therefore described as oligomeric, for example hemoglobin or DNA polymerase. Others may consist of a very large number of subunits and therefore described as multimeric, for example microtubules and other cytoskeleton proteins. The subunits of a multimeric protein may be identical, homologous or totally dissimilar and dedicated to disparate tasks.
    <br>In some protein assemblies, one subunit may be a "catalytic subunit" that enzymatically catalyzes a reaction, whereas a "regulatory subunit" will facilitate or inhibit the activity. Although telomerase has telomerase reverse transcriptase as a catalytic subunit, regulation is accomplished by factors outside the protein.[1] An enzyme composed of both regulatory and catalytic subunits when assembled is often referred to as a holoenzyme. For example, class I phosphoinositide 3-kinase is composed of a p110 catalytic subunit and a p85 regulatory subunit.[2] One subunit is made of one polypeptide chain. A polypeptide chain has one gene coding for it – meaning that a protein must have one gene for each unique subunit.
    <br>A subunit is often named with a Greek or Roman letter, and the numbers of this type of subunit in a protein is indicated by a subscript. For example, ATP synthase has a type of subunit called α. Three of these are present in the ATP synthase molecule, and is therefore designated α3. Larger groups of subunits can also be specified, like α3β3-hexamer and c-ring."
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_subunit]
    <a class="clsHide" href="#idPrtn010dsn"></a></p>
</section>

<section id="idPrtn011">
  <h1 id="idPrtn011H1">proteinOgm.sysMolecules-011
    <a class="clsHide" href="#idPrtn011H1"></a></h1>
  <p id="idPrtn011nam">name::
    <br>* Mcs.proteinOgm.011-sysMolecules,
    <br>* Mcs.proteinOgm.sysMolecules-011,
    <br>* Mcs.sysMolsProtein,
    <br>* Mcs.protein-complex,
    <a class="clsHide" href="#idPrtn011nam"></a></p>
  <p id="idPrtn011dsn">description::
    <br>"A protein complex or multiprotein complex is a group of two or more associated polypeptide chains. Different polypeptide chains may have different functions. This is distinct from a multienzyme complex, in which multiple catalytic domains are found in a single polypeptide chain.[1]
    <br>Protein complexes are a form of quaternary structure. Proteins in a protein complex are linked by non-covalent protein–protein interactions, and different protein complexes have different degrees of stability over time. These complexes are a cornerstone of many (if not most) biological processes and together they form various types of molecular machinery that perform a vast array of biological functions. The cell is seen to be composed of modular supramolecular complexes, each of which performs an independent, discrete biological function.[2]
    <br>Through proximity, the speed and selectivity of binding interactions between enzymatic complex and substrates can be vastly improved, leading to higher cellular efficiency. Many of the techniques used to break open cells and isolate proteins are inherently disruptive to such large complexes, so it is often difficult to determine the components of a complex. Examples of protein complexes include the proteasome for molecular degradation and most RNA polymerases. In stable complexes, large hydrophobic interfaces between proteins typically bury surface areas larger than 2500 square Ås.[3]"
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Protein_complex]
    <a class="clsHide" href="#idPrtn011dsn"></a></p>
</section>

<section id="idPrtnHmn">
  <h1 id="idPrtnHmnH1">proteinOgm.human-007 of bodyHmn
    <a class="clsHide" href="#idPrtnHmnH1"></a></h1>
  <p id="idPrtnHmnnam">name::
    <br>* Mcs.bodyMtrHmnPrt.014-protein!⇒protein,
    <br>* Mcs.bodyMtrHmnPrt.protein-014!⇒protein,
    <br>* Mcs.bodyHmn'protein!⇒protein,
    <br>* Mcs.protein, /prótiin/,
    <br>* Mcs.protein'(human-protein)!⇒protein,
    <br>* Mcs.protein-of-bodyHmn!⇒protein,
    <br>* Mcs.proteinOgm.007-human!⇒protein,
    <br>* Mcs.proteinOgm.human-007!⇒protein,
    <a class="clsHide" href="#idPrtnHmnnam"></a></p>
  <p id="idPrtnHmndsn">description::
    <br>"A protein is a complex, high molecular weight organic compound that consists of amino acids joined by peptide bonds."
    <br>[https://en.wikipedia.org/wiki/Category:Proteins]
    <br>"Proteins are large biomolecules, or macromolecules, consisting of one or more long chains of amino acid residues. Proteins perform a vast array of functions within organisms, including catalysing metabolic reactions, DNA replication, responding to stimuli, providing structure to cells, and organisms, and transporting molecules from one location to another. Proteins differ from one another primarily in their sequence of amino acids, which is dictated by the nucleotide sequence of their genes, and which usually results in protein folding into a specific three-dimensional structure that determines its activity."
    <br>[https://en.wikipedia.org/wiki/Protein {2020-02-20}]
    <a class="clsHide" href="#idPrtnHmndsn"></a></p>

  <section id="idPrtnHmnrsc">
  <h2 id="idPrtnHmnrscH2">resource of protein
    <a class="clsHide" href="#idPrtnHmnrscH2"></a></h2>
  <p id="idPrtnHmnrscnam">name::
    <br>* Mcs.protein'resource,
    <a class="clsHide" href="#idPrtnHmnrscnam"></a></p>
  <p id="idPrtnHmnrscdsn">description::
    <br>* https://en.wikipedia.org/wiki/List_of_topics_related_to_protein,
    <a class="clsHide" href="#idPrtnHmnrscdsn"></a></p>
  </section>

  <section id="idPrtnHmngst">
  <h2 id="idPrtnHmngstH2">GENERIC-SPECIFIC-TREE of protein
    <a class="clsHide" href="#idPrtnHmngstH2"></a></h2>
  <p id="idPrtnHmngstnam">name::
    <br>* Mcs.protein'generic-specific-tree,
    <a class="clsHide" href="#idPrtnHmngstnam"></a></p>
  <p id="idPrtnHmngtr">generic-tree-of-protein::
    <br>* proteinOgm,
    <a class="clsHide" href="#idPrtnHmngtr"></a></p>
  <p id="idPrtnHmnstr">specific-tree-of-protein::
    <br>* blood-protein,
    <a class="clsHide" href="#idPrtnHmnstr"></a></p>
  <p id="idPrtnHmngstwpa">addressWpg::
    <br>* https://en.wikipedia.org/wiki/Category:Proteins,
    <br>* https://en.wikipedia.org/wiki/List_of_types_of_proteins,
    <br>* https://en.wikipedia.org/wiki/List_of_proteins,
    <a class="clsHide" href="#idPrtnHmngstwpa"></a></p>
  </section>

  <section id="idPrtnHmn001">
  <h2 id="idPrtnHmn001H2">protein.blood-001
    <a class="clsHide" href="#idPrtnHmn001H2"></a></h2>
  <p id="idPrtnHmn001nam">name::
    <br>* Mcs.protein.001-blood,
    <br>* Mcs.protein.blood-001,
    <br>* Mcs.blood-protein-001,
    <br>* Mcs.plasma-protein-001,
    <a class="clsHide" href="#idPrtnHmn001nam"></a></p>
  <p id="idPrtnHmn001dsn">description::
    <br>"Blood proteins, also termed plasma proteins, are proteins present in blood plasma. They serve many different functions, including transport of lipids, hormones, vitamins and minerals in activity and functioning of the immune system. Other blood proteins act as enzymes, complement components, protease inhibitors or kinin precursors. Contrary to popular belief, haemoglobin is not a blood protein, as it is carried within red blood cells, rather than in the blood serum.
    <br>Serum albumin accounts for 55% of blood proteins, is a major contributor to maintaining the oncotic pressure of plasma and assists, as a carrier, in the transport of lipids and steroid hormones. Globulins make up 38% of blood proteins and transport ions, hormones, and lipids assisting in immune function. Fibrinogen comprises 7% of blood proteins; conversion of fibrinogen to insoluble fibrin is essential for blood clotting. The remainder of the plasma proteins (1%) are regulatory proteins, such as enzymes, proenzymes, and hormones. All blood proteins are synthesized in liver except for the gamma globulins.
    <br>Separating serum proteins by electrophoresis is a valuable diagnostic tool as well as a way to monitor clinical progress. Current research regarding blood plasma proteins is centered on performing proteomics analyses of serum/plasma in the search for biomarkers. These efforts started with two-dimensional gel electrophoresis[1] efforts in the 1970s and in more recent times this research has been performed using LC-tandem MS[2][3][4] based proteomics. The normal laboratory value of serum total protein is around 7 g/dL."
    <br>[https://en.wikipedia.org/wiki/Blood_proteins {2020-03-11}]
    <a class="clsHide" href="#idPrtnHmn001dsn"></a></p>
  </section>

  <section id="idPrtnHmn002">
  <h2 id="idPrtnHmn002H2">protein.membrane-002
    <a class="clsHide" href="#idPrtnHmn002H2"></a></h2>
  <p id="idPrtnHmn002nam">name::
    <br>* Mcs.protein.002-membrane, /mémbrein/,
    <br>* Mcs.protein.membrane-002,
    <br>* Mcs.membrane-protein-002,
    <a class="clsHide" href="#idPrtnHmn002nam"></a></p>
  <p id="idPrtnHmn002dsn">description::
    <br>"Membrane proteins are common proteins that are part of, or interact with, biological membranes. Membrane proteins fall into several broad categories depending on their location. Integral membrane proteins are a permanent part of a cell membrane and can either penetrate the membrane (transmembrane) or associate with one or the other side of a membrane (integral monotopic). Peripheral membrane proteins are transiently associated with the cell membrane.
    <br>Membrane proteins are common, and medically important—about a third of all human proteins are membrane proteins, and these are targets for more than half of all drugs. Nonetheless, compared to other classes of proteins, determining membrane protein structures remains a challenge in large part due to the difficulty in establishing experimental conditions that can preserve the correct conformation of the protein in isolation from its native environment."
    <br>[https://en.wikipedia.org/wiki/Membrane_protein {2020-03-11}]
    <a class="clsHide" href="#idPrtnHmn002dsn"></a></p>
  </section>

  <section id="idPrtnHmn003">
  <h2 id="idPrtnHmn003H2">protein.metalloprotein-003
    <a class="clsHide" href="#idPrtnHmn003H2"></a></h2>
  <p id="idPrtnHmn003nam">name::
    <br>* Mcs.protein.003-metalloprotein,
    <br>* Mcs.protein.metalloprotein-003,
    <br>* Mcs.metalloprotein-003,
    <a class="clsHide" href="#idPrtnHmn003nam"></a></p>
  <p id="idPrtnHmn003dsn">description::
    <br>"Metalloprotein is a generic term for a protein that contains a metal ion cofactor.[1][2] A large proportion of all proteins are part of this category. For instance, at least 1000 human proteins (out of ~20,000) contain zinc-binding protein domains[3] although there may be up to 3000 human zinc metalloproteins.[4]"
    <br>[https://en.wikipedia.org/wiki/Metalloprotein {2020-03-19}]
    <a class="clsHide" href="#idPrtnHmn003dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgm001">
  <h1 id="idPrtnOgm001H1">proteinOgm.motor-001
    <a class="clsHide" href="#idPrtnOgm001H1"></a></h1>
  <p id="idPrtnOgm001nam">name::
    <br>* Mcs.proteinOgm.001-motor,
    <br>* Mcs.proteinOgm.motor-001,
    <br>* Mcs.motor-proteinOgm-001,
    <a class="clsHide" href="#idPrtnOgm001nam"></a></p>
  <p id="idPrtnOgm001dsn">description::
    <br>"Motor proteins are a class of molecular motors that can move along the cytoplasm of animal cells. They convert chemical energy into mechanical work by the hydrolysis of ATP. Flagellar rotation, however, is powered by a proton pump.[citation needed]"
    <br>[https://en.wikipedia.org/wiki/Motor_protein {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm001dsn"></a></p>

  <section id="idPrtnOgm002">
  <h2 id="idPrtnOgm002H2">proteinOgm.myosin-002
    <a class="clsHide" href="#idPrtnOgm002H2"></a></h2>
  <p id="idPrtnOgm002nam">name::
    <br>* Mcs.proteinOgm.002-myosin,
    <br>* Mcs.proteinOgm.myosin-002,
    <br>* Mcs.myosin-proteinOgm-002,
    <a class="clsHide" href="#idPrtnOgm002nam"></a></p>
  <p id="idPrtnOgm002dsn">description::
    <br>"Myosins (/ˈmaɪəsɪn, -oʊ-/[1][2]) are a superfamily of motor proteins best known for their roles in muscle contraction and in a wide range of other motility processes in eukaryotes. They are ATP-dependent and responsible for actin-based motility. The term was originally used to describe a group of similar ATPases found in the cells of both striated muscle tissue and smooth muscle tissue.[3] Following the discovery by Pollard and Korn (1973) of enzymes with myosin-like function in Acanthamoeba castellanii, a global range of divergent myosin genes have been discovered throughout the realm of eukaryotes.
    <br>Although myosin was originally thought to be restricted to muscle cells (hence myo-(s) + -in), there is no single "myosin"; rather it is a very large superfamily of genes whose protein products share the basic properties of actin binding, ATP hydrolysis (ATPase enzyme activity), and force transduction. Virtually all eukaryotic cells contain myosin isoforms. Some isoforms have specialized functions in certain cell types (such as muscle), while other isoforms are ubiquitous. The structure and function of myosin is globally conserved across species, to the extent that rabbit muscle myosin II will bind to actin from an amoeba.[4]"
    <br>[https://en.wikipedia.org/wiki/Myosin {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm002dsn"></a></p>
  </section>
</section>

<section id="idPrtnOgm003">
  <h1 id="idPrtnOgm003H1">proteinOgm.chaperone-003
    <a class="clsHide" href="#idPrtnOgm003H1"></a></h1>
  <p id="idPrtnOgm003nam">name::
    <br>* Mcs.proteinOgm.003-chaperone,
    <br>* Mcs.proteinOgm.chaperone-003,
    <br>* Mcs.chaperone-proteinOgm-003,
    <a class="clsHide" href="#idPrtnOgm003nam"></a></p>
  <p id="idPrtnOgm003dsn">description::
    <br>"In molecular biology, molecular chaperones are proteins that assist the conformational folding or unfolding and the assembly or disassembly of other macromolecular structures. Chaperones are present when the macromolecules perform their normal biological functions and have correctly completed the processes of folding and/or assembly. The chaperones are concerned primarily with protein folding. The first protein to be called a chaperone assists the assembly of nucleosomes from folded histones and DNA and such assembly chaperones, especially in the nucleus,[1][2] are concerned with the assembly of folded subunits into oligomeric structures.[3]
    <br>One major function of chaperones is to prevent both newly synthesised polypeptide chains and assembled subunits from aggregating into nonfunctional structures. It is for this reason that many chaperones, but by no means all, are heat shock proteins because the tendency to aggregate increases as proteins are denatured by stress. In this case, chaperones do not convey any additional steric information required for proteins to fold. However, some highly specific 'steric chaperones' do convey unique structural (steric) information onto proteins, which cannot be folded spontaneously. Such proteins violate Anfinsen's dogma,[4] requiring protein dynamics to fold correctly.
    <br>Various approaches have been applied to study the structure, dynamics and functioning of chaperones. Bulk biochemical measurements have informed us on the protein folding efficiency, and prevention of aggregation when chaperones are present during protein folding. Recent advances in single-molecule analysis[5] have brought insights into structural heterogeneity of chaperones, folding intermediates and affinity of chaperones for unstructured and structured protein chains."
    <br>[https://en.wikipedia.org/wiki/Chaperone_(protein) {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm003dsn"></a></p>
</section>

<section id="idPrtnOgm004">
  <h1 id="idPrtnOgm004H1">proteinOgm.membrane-004
    <a class="clsHide" href="#idPrtnOgm004H1"></a></h1>
  <p id="idPrtnOgm004nam">name::
    <br>* Mcs.proteinOgm.004-membrane,
    <br>* Mcs.proteinOgm.membrane-004,
    <br>* Mcs.membrane-proteinOgm-004,
    <a class="clsHide" href="#idPrtnOgm004nam"></a></p>
  <p id="idPrtnOgm004dsn">description::
    <br>"Membrane proteins are common proteins that are part of, or interact with, biological membranes. Membrane proteins fall into several broad categories depending on their location. Integral membrane proteins are a permanent part of a cell membrane and can either penetrate the membrane (transmembrane) or associate with one or the other side of a membrane (integral monotopic). Peripheral membrane proteins are transiently associated with the cell membrane.
    <br>Membrane proteins are common, and medically important—about a third of all human proteins are membrane proteins, and these are targets for more than half of all drugs. Nonetheless, compared to other classes of proteins, determining membrane protein structures remains a challenge in large part due to the difficulty in establishing experimental conditions that can preserve the correct conformation of the protein in isolation from its native environment."
    <br>[https://en.wikipedia.org/wiki/Membrane_protein {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm004dsn"></a></p>
</section>

<section id="idPrtnOgm005">
  <h1 id="idPrtnOgm005H1">proteinOgm.transcription-factor-005
    <a class="clsHide" href="#idPrtnOgm005H1"></a></h1>
  <p id="idPrtnOgm005nam">name::
    <br>* Mcs.proteinOgm.005-transcription-factor,
    <br>* Mcs.proteinOgm.transcription-factor-005,
    <br>* Mcs.transcription-factor-proteinOgm-005,
    <a class="clsHide" href="#idPrtnOgm005nam"></a></p>
  <p id="idPrtnOgm005dsn">description::
    <br>"In molecular biology, a transcription factor (TF) (or sequence-specific DNA-binding factor) is a protein that controls the rate of transcription of genetic information from DNA to messenger RNA, by binding to a specific DNA sequence.[1][2] The function of TFs is to regulate—turn on and off—genes in order to make sure that they are expressed in the right cell at the right time and in the right amount throughout the life of the cell and the organism. Groups of TFs function in a coordinated fashion to direct cell division, cell growth, and cell death throughout life; cell migration and organization (body plan) during embryonic development; and intermittently in response to signals from outside the cell, such as a hormone. There are up to 1600 TFs in the human genome.[3]
    <br>TFs work alone or with other proteins in a complex, by promoting (as an activator), or blocking (as a repressor) the recruitment of RNA polymerase (the enzyme that performs the transcription of genetic information from DNA to RNA) to specific genes.[4][5][6]
    <br>A defining feature of TFs is that they contain at least one DNA-binding domain (DBD), which attaches to a specific sequence of DNA adjacent to the genes that they regulate.[7][8] TFs are grouped into classes based on their DBDs.[9][10] Other proteins such as coactivators, chromatin remodelers, histone acetyltransferases, histone deacetylases, kinases, and methylases are also essential to gene regulation, but lack DNA-binding domains, and therefore are not TFs.[11]
    <br>TFs are of interest in medicine because TF mutations can cause specific diseases, and medications can be potentially targeted toward them."
    <br>[https://en.wikipedia.org/wiki/Transcription_factor]
    <a class="clsHide" href="#idPrtnOgm005dsn"></a></p>
</section>

<section id="idPrtnOgm006">
  <h1 id="idPrtnOgm006H1">proteinOgm.ribosomal-006
    <a class="clsHide" href="#idPrtnOgm006H1"></a></h1>
  <p id="idPrtnOgm006nam">name::
    <br>* Mcs.proteinOgm.006-ribosomal,
    <br>* Mcs.proteinOgm.ribosomal-006,
    <br>* Mcs.ribosomal-proteinOgm-006,
    <a class="clsHide" href="#idPrtnOgm006nam"></a></p>
  <p id="idPrtnOgm006dsn">description::
    <br>"A ribosomal protein (r-protein or rProtein[1][2][3]) is any of the proteins that, in conjunction with rRNA, make up the ribosomal subunits involved in the cellular process of translation. A large part of the knowledge about these organic molecules has come from the study of E. coli ribosomes. All ribosomal proteins have been isolated and many specific antibodies have been produced. These, together with electronic microscopy and the use of certain reactives, have allowed for the determination of the topography of the proteins in the ribosome. E. coli, other bacteria and Archaea have a 30S small subunit and a 50S large subunit, whereas humans and yeasts have a 40S small subunit and a 60S large subunit.[4] Equivalent subunits are frequently numbered differently between bacteria, Archaea, yeasts and humans.[5] More recently, a near-complete (near)atomic picture of the ribosomal proteins is emerging from the latest high-resolution cryo-EM data (including PDB ID: 5AFI)."
    <br>[https://en.wikipedia.org/wiki/Ribosomal_protein {2020-03-25}]
    <a class="clsHide" href="#idPrtnOgm006dsn"></a></p>
</section>

<section id="idPrtnOgm008">
  <h1 id="idPrtnOgm008H1">proteinOgm.GPX4-008
    <a class="clsHide" href="#idPrtnOgm008H1"></a></h1>
  <p id="idPrtnOgm008nam">name::
    <br>* Mcs.proteinOgm.008-GPX4,
    <br>* Mcs.proteinOgm.GPX4-008,
    <br>* Mcs.GPX4-proteinOgm-008,
    <a class="clsHide" href="#idPrtnOgm008nam"></a></p>
  <p id="idPrtnOgm008dsn">description::
    <br>"GPX4, μια πρωτεΐνη η οποία προστατεύει κύτταρα από οξειδωτικό στρες.
    <br>[https://www.naftemporiki.gr/story/1585510/isxuro-antikarkiniko-oplo-xari-stin-epilusi-moriakou-mustiriou]"
    <a class="clsHide" href="#idPrtnOgm008dsn"></a></p>
</section>

<section id="idPrtnOgm009">
  <h1 id="idPrtnOgm009H1">proteinOgm.antibody-009
    <a class="clsHide" href="#idPrtnOgm009H1"></a></h1>
  <p id="idPrtnOgm009nam">name::
    <br>* Mcs.proteinOgm.009-antibody!⇒proteinOgmAntibody,
    <br>* Mcs.proteinOgm.antibody-009!⇒proteinOgmAntibody,
    <br>* Mcs.antibody-proteinOgm-009!⇒proteinOgmAntibody,
    <br>* Mcs.proteinOgmAntibody,
    <a class="clsHide" href="#idPrtnOgm009nam"></a></p>
  <p id="idPrtnOgm009dsn">description::
    <br>"An antibody (Ab), also known as an immunoglobulin (Ig),[1] is a large, Y-shaped protein produced mainly by plasma cells that is used by the immune system to neutralize pathogens such as pathogenic bacteria and viruses. The antibody recognizes a unique molecule of the pathogen, called an antigen, via the fragment antigen-binding (Fab) variable region.[2][3] Each tip of the "Y" of an antibody contains a paratope (analogous to a lock) that is specific for one particular epitope (analogous to a key) on an antigen, allowing these two structures to bind together with precision. Using this binding mechanism, an antibody can tag a microbe or an infected cell for attack by other parts of the immune system, or can neutralize its target directly (for example, by inhibiting a part of a microbe that is essential for its invasion and survival). Depending on the antigen, the binding may impede the biological process causing the disease or may activate macrophages to destroy the foreign substance. The ability of an antibody to communicate with the other components of the immune system is mediated via its Fc region (located at the base of the "Y"), which contains a conserved glycosylation site involved in these interactions.[4] The production of antibodies is the main function of the humoral immune system.[5]
    <br>Antibodies are secreted by B cells of the adaptive immune system, mostly by differentiated B cells called plasma cells. Antibodies can occur in two physical forms, a soluble form that is secreted from the cell to be free in the blood plasma, and a membrane-bound form that is attached to the surface of a B cell and is referred to as the B-cell receptor (BCR). The BCR is found only on the surface of B cells and facilitates the activation of these cells and their subsequent differentiation into either antibody factories called plasma cells or memory B cells that will survive in the body and remember that same antigen so the B cells can respond faster upon future exposure.[6] In most cases, interaction of the B cell with a T helper cell is necessary to produce full activation of the B cell and, therefore, antibody generation following antigen binding.[7] Soluble antibodies are released into the blood and tissue fluids, as well as many secretions to continue to survey for invading microorganisms.
    <br>Antibodies are glycoproteins belonging to the immunoglobulin superfamily.[4] They constitute most of the gamma globulin fraction of the blood proteins. They are typically made of basic structural units—each with two large heavy chains and two small light chains. There are several different types of antibody heavy chains that define the five different types of crystallisable fragments (Fc) that may be attached to the antigen-binding fragments. The five different types of Fc regions allow antibodies to be grouped into five isotypes. Each Fc region of a particular antibody isotype is able to bind to its specific Fc Receptor (FcR), except for IgD, which is essentially the BCR, thus allowing the antigen-antibody complex to mediate different roles depending on which FcR it binds. The ability of an antibody to bind to its corresponding FcR is further modulated by the structure of the glycan(s) present at conserved sites within its Fc region.[4] The ability of antibodies to bind to FcRs helps to direct the appropriate immune response for each different type of foreign object they encounter.[8] For example, IgE is responsible for an allergic response consisting of mast cell degranulation and histamine release. IgE's Fab paratope binds to allergic antigen, for example house dust mite particles, while its Fc region binds to Fc receptor ε. The allergen-IgE-FcRε interaction mediates allergic signal transduction to induce conditions such as asthma.[9]
    <br>Though the general structure of all antibodies is very similar, a small region at the tip of the protein is extremely variable, allowing millions of antibodies with slightly different tip structures, or antigen-binding sites, to exist. This region is known as the hypervariable region. Each of these variants can bind to a different antigen.[2] This enormous diversity of antibody paratopes on the antigen-binding fragments allows the immune system to recognize an equally wide variety of antigens.[1] The large and diverse population of antibody paratope is generated by random recombination events of a set of gene segments that encode different antigen-binding sites (or paratopes), followed by random mutations in this area of the antibody gene, which create further diversity.[8][10] This recombinational process that produces clonal antibody paratope diversity is called V(D)J or VJ recombination. The antibody paratope is polygenic, made up of three genes, V, D, and J. Each paratope locus is also polymorphic, such that during antibody production, one allele of V, one of D, and one of J is chosen. These gene segments are then joined together using random genetic recombination to produce the paratope. The regions where the genes are randomly recombined together is the hypervariable region used to recognise different antigens on a clonal basis.
    <br>Antibody genes also re-organize in a process called class switching that changes the one type of heavy chain Fc fragment to another, creating a different isotype of the antibody that retains the antigen-specific variable region. This allows a single antibody to be used by different types of Fc receptors, expressed on different parts of the immune system."
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Antibody]
    <a class="clsHide" href="#idPrtnOgm009dsn"></a></p>
</section>

<section id="idPrtnOgm012">
  <h1 id="idPrtnOgm012H1">proteinOgm.prion-012
    <a class="clsHide" href="#idPrtnOgm012H1"></a></h1>
  <p id="idPrtnOgm012nam">name::
    <br>* Mcs.proteinOgm.012-prion!⇒proteinPrion,
    <br>* Mcs.proteinOgm.prion-012!⇒proteinPrion,
    <br>* Mcs.prion-proteinOgm-012!⇒proteinPrion,
    <br>* Mcs.proteinPrion,
    <a class="clsHide" href="#idPrtnOgm012nam"></a></p>
  <p id="idPrtnOgm012dsn">description::
    <br>"Prions (/ˈpriːɒn/ (About this soundlisten), /ˈpraɪɒn/[1][2]) are misfolded proteins with the ability to transmit their misfolded shape onto normal variants of the same protein. They characterize several fatal and transmissible neurodegenerative diseases in humans and many other animals.[3] It is not known what causes the normal protein to misfold, but the abnormal three-dimensional structure is suspected of conferring infectious properties, collapsing nearby protein molecules into the same shape. The word prion derives from "proteinaceous infectious particle".[4][5][6] The hypothesized role of a protein as an infectious agent stands in contrast to all other known infectious agents such as viruses, bacteria, fungi and parasites, all of which contain nucleic acids (DNA, RNA or both).
    <br>Prion variants of the prion protein (PrP), whose specific function is uncertain, are hypothesized as the cause of transmissible spongiform encephalopathies (TSEs),[7] including scrapie in sheep, chronic wasting disease (CWD) in deer, bovine spongiform encephalopathy (BSE) in cattle (commonly known as "mad cow disease") and Creutzfeldt–Jakob disease (CJD) in humans. All known prion diseases in mammals affect the structure of the brain or other neural tissue; all are progressive, have no known effective treatment and are always fatal.[8] Until 2015, all known mammalian prion diseases were considered to be caused by the prion protein (PrP); however in 2015 multiple system atrophy (MSA) was found to be transmissible and was hypothesized to be caused by a prion form of alpha-synuclein.[9]
    <br>Prions form abnormal aggregates of proteins called amyloids, which accumulate in infected tissue and are associated with tissue damage and cell death.[10] Amyloids are also responsible for several other neurodegenerative diseases such as Alzheimer's disease and Parkinson's disease.[11] Prion aggregates are stable, and this structural stability means that prions are resistant to denaturation by chemical and physical agents: they cannot be destroyed by ordinary disinfection or cooking. This makes disposal and containment of these particles difficult.
    <br>A prion disease is a type of proteopathy, or disease of structurally abnormal proteins. In humans, prions are believed to be the cause of Creutzfeldt–Jakob disease (CJD), its variant (vCJD), Gerstmann–Sträussler–Scheinker syndrome (GSS), fatal familial insomnia (FFI) and kuru.[4] There is also evidence suggesting prions may play a part in the process of Alzheimer's disease, Parkinson's disease and amyotrophic lateral sclerosis (ALS), and these have been termed prion-like diseases.[12][13][14][15] Several yeast proteins have also been identified as having prionogenic properties.[16][17] Prion replication is subject to epimutation and natural selection just as for other forms of replication, and their structure varies slightly between species.[18]"
    <br>[{2020-04-16} https://en.wikipedia.org/wiki/Prion]
    <a class="clsHide" href="#idPrtnOgm012dsn"></a></p>
</section>

<section id="idMeta">
  <h1 id="idMetaH1">meta-info
    <a class="clsHide" href="#idMetaH1"></a></h1>
  <p id="idMetaCounter" class="clsCenter">this page was-visited
    <span class="clsColorRed">
    <script src="../../dirPgm/dirCntr/counter.php?page=Mws-dirOgm.Ogm-prtn"></script>
    </span>
    times since {2020-03-25}</p>
  <!-- the content of page-path paragraph is displayed as it is on top of toc -->
  <p id="idMetaWebpage_path"><span class="clsB clsColorGreen">page-wholepath</span>:
    <a class="clsPreview" href="../../#idOverview">synagonism.net</a> /
    <a class="clsPreview" href="../filMcsWorld.last.html#idOverview">Mcs-worldview</a> /
    <a class="clsPreview" href="filMcsDirOgm.last.html#idOverview">dirOgm</a> /
    proteinOgm
    </p>
  <p id="idMetaP1">SEARCH::
    <br>· this page uses '<span class="clsColorRed">locator-names</span>', names that when you find them, you find the-LOCATION of the-concept they denote.
    <br>⊛ <strong>GLOBAL-SEARCH</strong>:
    <br>· clicking on <span class="clsColorGreenBg">the-green-BAR of a-page</span> you have access to the-global--locator-names of my-site.
    <br>· use the-prefix '<span class="clsColorRed">proteinOgm</span>' for <a class="clsPreview" href="../dirCor/filMcs.last.html#idOverview">sensorial-concepts</a> related to current concept 'protein-of-organism'.
    <br>⊛ <strong>LOCAL-SEARCH</strong>:
    <br>· TYPE <span class="clsColorRed">CTRL+F "Mcs.words-of-concept's-name"</span>, to go to the-LOCATION of the-concept.
    <br>· a-preview of the-description of a-global-name makes reading fast.
    <a class="clsHide" href="#idMetaP1"></a></p>
  <p id="idFooterP1">footer::
    <br>• author: <a class="clsPreview" href="../../#idAboutme">Kaseluris.Nikos.1959</a>
    <br>• email:
    <br> &nbsp;<img src="../../misc/img/mail.png" alt="imgMail">
    <br>• edit on github: https://github.com/synagonism/Mcsw/blob/master/dirOgm/filMcsOgm-prtn.last.html,
    <br>• comments on <a class="clsPreview" href="filMcsDirOgm.last.html#idComment">Disqus</a>,
    <br>• twitter: <a href="https://twitter.com/synagonism">@synagonism</a>,
    <br>• steemit: <a href="https://steemit.com/synagonism">https://steemit.com/@synagonism</a>,
    <a class="clsHide" href="#idFooterP1"></a></p>
  <!--                              -->
  <p id="idMetaVersion">webpage-versions::
    <br>• <a href="filMcsOgm-prtn.last.html">filMcsOgm-prtn.last.html</a>: dynamic,
    <br>• filMcsOgm-prtn.0-1-0.2020-03-25.last.html: draft creation,
    <a class="clsHide" href="#idMetaVersion"></a></p>
</section>

<section id="idSupport">
  <h1 id="idSupportH1">support (<a class="clsPreview" href="../../#idSupport">link</a>)</h1>
  <p></p>
</section>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-19285371-5', 'synagonism.net');
  ga('send', 'pageview');
</script>

<script src="https://synagonism.github.io/hitp/hitp.js"></script>
</body>
</html>