<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Mcs.ogm-(1-1-0.2021-12-29) bio.organism</title>
  <meta name="keywords" content="organism, ogm, ModelConceptSensorial, McsHitp, Synagonism">
  <link rel="stylesheet" href="../Mcsmgr/mHitp.css">
</head>

<body>
<header id="idHeader">
  <p></p>
  <h1 id="idHeaderH1">bio.organism
    <br>senso-concept-Mcs (ogm)
    </h1>
  <p id="idHeadercrd">McsHitp-creation:: {2019-12-19},
    <a class="clsHide" href="#idHeadercrd"></a></p>
</header>

<section id="idOverview">
  <h1 id="idOverviewH1">overview of ogm
    <a class="clsHide" href="#idOverviewH1"></a></h1>
  <p id="idDescription">description::
    <br>* organism is <a class="clsPreview" href="../dirCor/McsCor000011.last.html#idSys005">a-material-system</a> capable of reproduction.
    <br>· organism is <a class="clsPreview" href="../dirOgm/McsOgm000002.last.html#idOverview">a-bio-system</a> which is NOT a-system of biosystems.
    <a class="clsHide" href="#idDescription"></a></p>
  <p id="idName">name::
    <br>* McsEngl.McsOgm000003.last.html//dirOgm//dirMcs!⇒ogm,
    <br>* McsEngl.dirMcs/dirOgm/McsOgm000003.last.html!⇒ogm,
    <br>* McsEngl.bio.001-organism!⇒ogm,
    <br>* McsEngl.bio.organism-001!⇒ogm,
    <br>* McsEngl.ogm,
    <br>* McsEngl.ogm'(McsOgm000003)!⇒ogm,
    <br>* McsEngl.organism!⇒ogm,
    <br>* McsEngl.organismOne!⇒ogm,
    <br>* McsEngl.ogm'(organism)!⇒ogm,
    <br>====== langoSinago:
    <br>* McsSngo.zo-fo!=organism,
    <br>====== langoChinese:
    <br>* McsZhon.shēngwù-生物!=organism,
    <br>* McsZhon.生物-shēngwù!=organism,
    <br>====== langoEsperanto:
    <br>* McsEspo.organismo!=organism,
    <br>====== langoGreek:
    <br>* McsElln.οργανισμός!~ο!=organism,
    <a class="clsHide" href="#idName"></a></p>
</section>

<section id="idOgmatt008">
  <h1 id="idOgmatt008H1">01_disease (<a class="clsPreview" href="../McsHlth000010.last.html#idOverview">link</a>) of ogm
    <a class="clsHide" href="#idOgmatt008H1"></a></h1>
</section>

<section id="idOgmatt009">
  <h1 id="idOgmatt009H1">02_managing-sys (<a class="clsPreview" href="../dirCor/McsCor000011.last.html#idSysMng001">link</a>) of ogm
    <a class="clsHide" href="#idOgmatt009H1"></a></h1>
</section>

<section id="idOgmatt010">
  <h1 id="idOgmatt010H1">03_health (<a class="clsPreview" href="../dirHlth/McsHlth000007.last.html#idOverview">link</a>) of ogm
    <a class="clsHide" href="#idOgmatt010H1"></a></h1>
</section>

<section id="idSysMtrlOgm">
  <h1 id="idSysMtrlOgmH1">04_material-node of ogm
    <a class="clsHide" href="#idSysMtrlOgmH1"></a></h1>
  <p id="idSysMtrlOgmdsn">description::
    <br>· nodeOgmMtrl is a-material-system part of an-organism.
    <br>· it is a-system MORE COMPLEX than a-system-of-molecules.
    <a class="clsHide" href="#idSysMtrlOgmdsn"></a></p>
  <p id="idSysMtrlOgmnam">name::
    <br>* McsEngl.ogm'04_material-node!⇒nodeOgmMtrl,
    <br>* McsEngl.ogm'att004-material-system!⇒nodeOgmMtrl,
    <br>* McsEngl.ogm'material-node!⇒nodeOgmMtrl,
    <br>* McsEngl.ogm'material-system!⇒nodeOgmMtrl,
    <br>* McsEngl.ogm'nodeMtrl!⇒nodeOgmMtrl,
    <br>* McsEngl.ogm'sysMtrl!⇒nodeOgmMtrl,
    <br>* McsEngl.sysMaterialOgm!⇒nodeOgmMtrl,
    <br>* McsEngl.sysMaterial-of-organism!⇒nodeOgmMtrl,
    <br>* McsEngl.sysMtrlOgm!⇒nodeOgmMtrl, {2020-05-13},
    <br>* McsEngl.nodeOgmMtrl, {2020-10-22},
    <a class="clsHide" href="#idSysMtrlOgmnam"></a></p>

  <section id="idSysMtrlOgm001">
  <h2 id="idSysMtrlOgm001H2">nodeOgmMtrl.body
    <a class="clsHide" href="#idSysMtrlOgm001H2"></a></h2>
  <p id="idSysMtrlOgm001nam">name::
    <br>* McsEngl.nodeOgmMtrl.body,
    <br>* McsEngl.nodeOgmMtrl.001-body,
    <br>* McsEngl.ogm'att013-body,
    <br>* McsEngl.ogm'body-att013,
    <br>* McsEngl.body-of-ogm-att013,
    <a class="clsHide" href="#idSysMtrlOgm001nam"></a></p>
  <p id="idSysMtrlOgm001dsn">description::
    <br>· an-organism is a-whole-part-tree of material-systems.
    <br>· body-of-ogm is the-outermost material-system of it.
    <a class="clsHide" href="#idSysMtrlOgm001dsn"></a></p>
  </section>

  <section id="idSysMtrlOgm002">
  <h2 id="idSysMtrlOgm002H2">nodeOgmMtrl.lipoprotein
    <a class="clsHide" href="#idSysMtrlOgm002H2"></a></h2>
  <p id="idSysMtrlOgm002nam">name::
    <br>* McsEngl.nodeOgmMtrl.lipoprotein,
    <br>* McsEngl.nodeOgmMtrl.002-lipoprotein,
    <br>* McsEngl.lipoprotein-particle,
    <a class="clsHide" href="#idSysMtrlOgm002nam"></a></p>
  <p id="idSysMtrlOgm002dsn">description::
    <br>"A lipoprotein is a biochemical assembly whose primary purpose is to transport hydrophobic lipid (also known as fat) molecules in water, as in blood plasma or other extracellular fluids. They have a single-layer phospholipid and cholesterol outer shell, with the hydrophilic portions oriented outward toward the surrounding water and lipophilic portions oriented inward toward the lipids molecules within the particles. Thus, the complex serves to emulsify the fats in extracellular fluids. A special kind of protein, called apolipoprotein, is embedded in the outer shell, both stabilising the complex and giving it a functional identity that determines its fate.
    <br>Many enzymes, transporters, structural proteins, antigens, adhesins, and toxins are lipoproteins. Examples include plasma lipoprotein particles (HDL, LDL, IDL, VLDL and chylomicrons). Subgroups of these plasma particles are primary drivers or modulators of atherosclerosis.[1]"
    <br>[{2020-05-13} https://en.wikipedia.org/wiki/Lipoprotein]
    <a class="clsHide" href="#idSysMtrlOgm002dsn"></a></p>

  <section id="idSysMtrlOgm002001">
  <h3 id="idSysMtrlOgm002001H3">lipoprotein.HDL
    <a class="clsHide" href="#idSysMtrlOgm002001H3"></a></h3>
  <p id="idSysMtrlOgm002001nam">name::
    <br>* McsEngl.lipoprotein.HDL,
    <br>* McsEngl.lipoprotein.001-HDL,
    <br>* McsEngl.HDL-high-density-lipoprotein,
    <br>====== langoGreek:
    <br>* McsElln.καλή-xοληστερίνη,
    <a class="clsHide" href="#idSysMtrlOgm002001nam"></a></p>
  <p id="idSysMtrlOgm002001dsn">description::
    <br>"High-density lipoprotein (HDL) is one of the five major groups of lipoproteins.[1] Lipoproteins are complex particles composed of multiple proteins which transport all fat molecules (lipids) around the body within the water outside cells. They are typically composed of 80–100 proteins per particle (organized by one, two or three ApoA; more as the particles enlarge picking up and carrying more fat molecules) and transporting up to hundreds of fat molecules per particle.
    <br>Lipoproteins have long been divided into 5 subgroups, by density/size (an inverse relationship), which also correlates with function and incidence of cardiovascular events. Unlike the larger lipoprotein particles which deliver fat molecules to cells, HDL particles remove fat molecules from cells which need to export fat molecules. The lipids carried include cholesterol, phospholipids, and triglycerides; amounts of each are quite variable.
    <br>Increasing concentrations of HDL particles are strongly associated with decreasing accumulation of atherosclerosis within the walls of arteries. This is important because atherosclerosis eventually results in sudden plaque ruptures, cardiovascular disease, stroke and other vascular diseases. HDL particles are sometimes referred to as "good cholesterol" because they can transport fat molecules out of artery walls, reduce macrophage accumulation, and thus help prevent or even regress atherosclerosis, but studies have shown that HDL-lacking mice still have the ability to transport cholesterol to bile, suggesting that there are alternative mechanisms for cholesterol removal."
    <br>[{2020-05-13} https://en.wikipedia.org/wiki/High-density_lipoprotein]
    <br>===
    <br>HDL and cholesterol: life after the divorce?1
    <br>Kasey C. Vickers2,*,† and Alan T. Remaley†
    <br>For decades, HDL and HDL-cholesterol (HDL-C) levels were viewed as synonymous, and modulation of HDL-C levels by drug therapy held great promise for the prevention and treatment of cardiovascular disease. Nevertheless, recent failures of drugs that raise HDL-C to reduce cardiovascular risk and the now greater understanding of the complexity of HDL composition and biology have prompted researchers in the field to redefine HDL. As such, the focus of HDL has now started to shift away from a cholesterol-centric view toward HDL particle number, subclasses, and other alternative metrics of HDL. Many of the recently discovered functions of HDL are, in fact, not strictly conferred by its ability to promote cholesterol flux but by the other molecules it transports, including a diverse set of proteins, small RNAs, hormones, carotenoids, vitamins, and bioactive lipids. Based on HDL's ability to interact with almost all cells and transport and deliver fat-soluble cargo, HDL has the remarkable capacity to affect a wide variety of endocrine-like systems. In this review, we characterize HDL's unique cargo and address the functional relevance and consequences of HDL transport and delivery of noncholesterol molecules to recipient cells and tissues.
    <br>[http://www.jlr.org/content/55/1/4]
    <a class="clsHide" href="#idSysMtrlOgm002001dsn"></a></p>

  <section id="idSysMtrlOgm002001rtm">
  <h4 id="idSysMtrlOgm002001rtmH4">relation-to-memory of HDL
    <a class="clsHide" href="#idSysMtrlOgm002001rtmH4"></a></h4>
  <p id="idSysMtrlOgm002001rtmnam">name::
    <br>* McsEngl.HDL'relation-to-memory,
    <br>* McsEngl.memory'relation-to-HDL,
    <a class="clsHide" href="#idSysMtrlOgm002001rtmnam"></a></p>
  <p id="idSysMtrlOgm002001rtmdsn">description::
    <br>"Fasting serum lipids have been associated with short term verbal memory. In a large sample of middle aged adults, low HDL cholesterol was associated with poor memory and decreasing levels over a five-year follow-up period were associated with decline in memory.[28]"
    <br>[{2020-05-13} https://en.wikipedia.org/wiki/High-density_lipoprotein]
    <a class="clsHide" href="#idSysMtrlOgm002001rtmdsn"></a></p>
  </section>
  </section>

  <section id="idSysMtrlOgm002002">
  <h3 id="idSysMtrlOgm002002H3">lipoprotein.LDL
    <a class="clsHide" href="#idSysMtrlOgm002002H3"></a></h3>
  <p id="idSysMtrlOgm002002nam">name::
    <br>* McsEngl.lipoprotein.LDL,
    <br>* McsEngl.lipoprotein.002-LDL,
    <br>* McsEngl.LDL-low-density-lipoprotein,
    <br>====== langoGreek:
    <br>* McsElln.κακή-xοληστερίνη,
    <a class="clsHide" href="#idSysMtrlOgm002002nam"></a></p>
  <p id="idSysMtrlOgm002002dsn">description::
    <br>LDL = ALL - (TG/5) - HDL
    <br>===
    <br>"Low-density lipoprotein (LDL) is one of the five major groups of lipoprotein which transport all fat molecules around the body in the extracellular water.[1] These groups, from least dense to most dense, are chylomicrons (aka ULDL by the overall density naming convention), very low-density lipoprotein (VLDL), intermediate-density lipoprotein (IDL), low-density lipoprotein and high-density lipoprotein (HDL). LDL delivers fat molecules to cells. LDL can contribute to atherosclerosis if it is oxidized within the walls of arteries."
    <br>[{2020-05-13} https://en.wikipedia.org/wiki/Low-density_lipoprotein]
    <a class="clsHide" href="#idSysMtrlOgm002002dsn"></a></p>
  </section>

  <section id="idSysMtrlOgm002003">
  <h3 id="idSysMtrlOgm002003H3">lipoprotein.IDL
    <a class="clsHide" href="#idSysMtrlOgm002003H3"></a></h3>
  <p id="idSysMtrlOgm002003nam">name::
    <br>* McsEngl.lipoprotein.IDL,
    <br>* McsEngl.lipoprotein.003-IDL,
    <br>* McsEngl.IDL-intermediate-density-lipoprotein,
    <a class="clsHide" href="#idSysMtrlOgm002003nam"></a></p>
  <p id="idSysMtrlOgm002003dsn">description::
    <br>"Intermediate-density lipoproteins (IDLs) belong to the lipoprotein particle family and are formed from the degradation of very low-density lipoproteins as well as high-density lipoproteins.[1] IDL is one of the five major groups of lipoproteins (chylomicrons, VLDL, IDL, LDL, HDL) that enable fats and cholesterol to move within the water-based solution of the bloodstream. Each native IDL particle consists of protein that encircles various lipids, enabling, as a water-soluble particle, these lipids to travel in the aqueous blood environment as part of the fat transport system within the body. Their size is, in general, 25 to 35 nm in diameter, and they contain primarily a range of triacylglycerols and cholesterol esters. They are cleared from the plasma into the liver by receptor-mediated endocytosis, or further degraded by hepatic lipase to form LDL particles.
    <br>Although one might intuitively assume that "intermediate-density" refers to a density between that of high-density and low-density lipoproteins, it in fact refers to a density between that of low-density and very-low-density lipoproteins.
    <br>In general, IDL, somewhat similar to low-density lipoprotein (LDL), transports a variety of triglyceride fats and cholesterol and, like LDL, can also promote the growth of atheroma.[citation needed]
    <br>VLDL is a large, triglyceride-rich lipoprotein secreted by the liver that transports triglyceride to adipose tissue and muscle. The triglycerides in VLDL are removed in capillaries by the enzyme lipoprotein lipase, and the VLDL returns to the circulation as a smaller particle with a new name, intermediate-density lipoprotein (IDL). The IDL particles have lost most of their triglyceride, but they retain cholesteryl esters. Some of the IDL particles are rapidly taken up by the liver; others remain in circulation, where they undergo further triglyceride hydrolysis by hepatic lipase and are converted to LDL. A distinguishing feature of the IDL particle is their content of multiple copies of the receptor ligand ApoE in addition to a single copy of ApoB-100. The multiple copies of ApoE allow IDL to bind to the LDL receptor with a very high affinity. When IDL is converted to LDL, the ApoE leaves the particle and only the ApoB-100 remains. Thereafter, the affinity for the LDL receptor is much reduced.[2]"
    <br>[{2020-05-13} https://en.wikipedia.org/wiki/Intermediate-density_lipoprotein]
    <a class="clsHide" href="#idSysMtrlOgm002003dsn"></a></p>
  </section>

  <section id="idSysMtrlOgm002004">
  <h3 id="idSysMtrlOgm002004H3">lipoprotein.VLDL
    <a class="clsHide" href="#idSysMtrlOgm002004H3"></a></h3>
  <p id="idSysMtrlOgm002004nam">name::
    <br>* McsEngl.lipoprotein.VLDL,
    <br>* McsEngl.lipoprotein.004-VLDL,
    <br>* McsEngl.VLDL-very-low-density-lipoprotein,
    <a class="clsHide" href="#idSysMtrlOgm002004nam"></a></p>
  <p id="idSysMtrlOgm002004dsn">description::
    <br>"Very-low-density lipoprotein (VLDL), density relative to extracellular water, is a type of lipoprotein made by the liver.[1] VLDL is one of the five major groups of lipoproteins (chylomicrons, VLDL, intermediate-density lipoprotein, low-density lipoprotein, high-density lipoprotein) that enable fats and cholesterol to move within the water-based solution of the bloodstream. VLDL is assembled in the liver from triglycerides, cholesterol, and apolipoproteins. VLDL is converted in the bloodstream to low-density lipoprotein (LDL) and intermediate-density lipoprotein (IDL). VLDL particles have a diameter of 30–80 nm. VLDL transports endogenous products, whereas chylomicrons transport exogenous (dietary) products. In the early 2010s both the lipid composition [2] and protein composition [3] of this lipoprotein were characterised in great detail."
    <br>[{2020-05-13} https://en.wikipedia.org/wiki/Very_low-density_lipoprotein]
    <a class="clsHide" href="#idSysMtrlOgm002004dsn"></a></p>
  </section>

  <section id="idSysMtrlOgm002005">
  <h3 id="idSysMtrlOgm002005H3">lipoprotein.ULDL
    <a class="clsHide" href="#idSysMtrlOgm002005H3"></a></h3>
  <p id="idSysMtrlOgm002005nam">name::
    <br>* McsEngl.lipoprotein.ULDL,
    <br>* McsEngl.lipoprotein.005-ULDL,
    <br>* McsEngl.chylomicron, /kailomáikron/,
    <br>* McsEngl.ULDL-ultra-low-density-lipoprotein,
    <a class="clsHide" href="#idSysMtrlOgm002005nam"></a></p>
  <p id="idSysMtrlOgm002005dsn">description::
    <br>"Chylomicrons (from the Greek χυλός, chylos, meaning juice (of plants or animals), and micron, meaning small particle), also known as ultra low-density lipoproteins (ULDL), are lipoprotein particles that consist of triglycerides (85–92%), phospholipids (6–12%), cholesterol (1–3%), and proteins (1–2%). They transport dietary lipids from the intestines to other locations in the body. ULDLs are one of the five major groups of lipoproteins (sorted by density) that enable fats and cholesterol to move within the water-based solution of the bloodstream.[1] A protein specific to chylomicrons is ApoB48.
    <br>There is an inverse relationship in the density and size of lipoprotein particles: the larger particles, which have a higher ratio of internal fat molecules with respect to the outer emulsifying protein molecules in the shell, and fats, are always lower density than water or smaller protein molecules. ULDLs, if in the region of 1,000 nm or more, are the only lipoprotein particles that can be seen using a light microscope, at maximum magnification. All the other classes are submicroscopic."
    <br>[{2020-05-13} https://en.wikipedia.org/wiki/Chylomicron]
    <a class="clsHide" href="#idSysMtrlOgm002005dsn"></a></p>
  </section>
  </section>
</section>

<section id="idOgmatt005">
  <h1 id="idOgmatt005H1">05_material of ogm
    <a class="clsHide" href="#idOgmatt005H1"></a></h1>
  <p id="idOgmatt005nam">name::
    <br>* McsEngl.ogm'05_material,
    <br>* McsEngl.ogm'att005-material,
    <br>* McsEngl.ogm'material-att005,
    <br>* McsEngl.materialOgm,
    <a class="clsHide" href="#idOgmatt005nam"></a></p>
  <p id="idOgmatt005dsn">description::
    <br>· materialOgm is any <a class="clsPreview" href="../dirNtr/McsNtr000002.last.html#idOverview">material</a> part of an-organism.
    <a class="clsHide" href="#idOgmatt005dsn"></a></p>

  <section id="idOgmatt003">
  <h2 id="idOgmatt003H2">sysMolecules (<a class="clsPreview" href="McsOgm000005.last.html#idOverview">link</a>) of ogm
    <a class="clsHide" href="#idOgmatt003H2"></a></h2>
  </section>

  <section id="idOgmatt002">
  <h2 id="idOgmatt002H2">molecule (<a class="clsPreview" href="McsOgm000004.last.html#idOverview">link</a>) of ogm
    <a class="clsHide" href="#idOgmatt002H2"></a></h2>
  </section>

  <section id="idOgmatt001">
  <h2 id="idOgmatt001H2">atom (<a class="clsPreview" href="../dirNtr/McsNtr000003.last.html#idAtm004">link</a>) of ogm
    <a class="clsHide" href="#idOgmatt001H2"></a></h2>
  </section>

  <section id="idOgmatt020">
  <h2 id="idOgmatt020H2">waste-product of ogm
    <a class="clsHide" href="#idOgmatt020H2"></a></h2>
  <p id="idOgmatt020nam">name::
    <br>* McsEngl.ogm'att020-waste-product,
    <br>* McsEngl.ogm'waste-product-att020,
    <br>* McsEngl.excrement,
    <br>* McsEngl.metabolic-waste,
    <br>* McsEngl.waste-product-of-ogm-att020,
    <a class="clsHide" href="#idOgmatt020nam"></a></p>
  <p id="idOgmatt020dsn">description::
    <br>· waste-product-of-organism is any material excreted by an-organism as not needed.
    <br>===
    <br>"Metabolic wastes or excrements are substances left over from metabolic processes (such as cellular respiration) which cannot be used by the organism (they are surplus or toxic), and must therefore be excreted. This includes nitrogen compounds, water, CO2, phosphates, sulphates, etc. Animals treat these compounds as excretes. Plants have chemical "machinery" which transforms some of them (primarily the nitrogen compounds) into useful substances.
    <br>All the metabolic wastes are excreted in a form of water solutes through the excretory organs (nephridia, Malpighian tubules, kidneys), with the exception of CO2, which is excreted together with the water vapor throughout the lungs. The elimination of these compounds enables the chemical homeostasis of the organism."
    <br>[{2020-04-22} https://en.wikipedia.org/wiki/Metabolic_waste]
    <a class="clsHide" href="#idOgmatt020dsn"></a></p>
  </section>
</section>

<section id="idOgmatt018">
  <h1 id="idOgmatt018H1">06_genome of ogm
    <a class="clsHide" href="#idOgmatt018H1"></a></h1>
  <p id="idOgmatt018nam">name::
    <br>* McsEngl.ogm'06_genome,
    <br>* McsEngl.ogm'att018-genome,
    <br>* McsEngl.ogm'genome-att018,
    <br>* McsEngl.genome-ogm-att018,
    <a class="clsHide" href="#idOgmatt018nam"></a></p>
  <p id="idOgmatt018dsn">description::
    <br>"In the fields of molecular biology and genetics, a genome is the genetic material of an organism. It consists of DNA (or RNA in RNA viruses). The genome includes both the genes (the coding regions) and the noncoding DNA,[1] as well as mitochondrial DNA[2] and chloroplast DNA. The study of the genome is called genomics."
    <br>[{2020-04-14} https://en.wikipedia.org/wiki/Genome]
    <a class="clsHide" href="#idOgmatt018dsn"></a></p>
</section>

<section id="idOgmatt023">
  <h1 id="idOgmatt023H1">satisfier (<a class="clsPreview" href="../dirStn/McsStn000003.last.html#idOverview">link</a>) of ogm
    <a class="clsHide" href="#idOgmatt023H1"></a></h1>
</section>

<section id="idOgmatt007">
  <h1 id="idOgmatt007H1">09_shape of ogm
    <a class="clsHide" href="#idOgmatt007H1"></a></h1>
  <p id="idOgmatt007nam">name::
    <br>* McsEngl.ogm'09_shape,
    <br>* McsEngl.ogm'att007-shape,
    <br>* McsEngl.ogm'shape-att007,
    <br>* McsEngl.shape-of-ogm-att007,
    <a class="clsHide" href="#idOgmatt007nam"></a></p>
  <p id="idOgmatt007dsn">description::
    <br>· an-organism is a-material-object as its body has <a class="clsPreview" href="../dirNtr/McsNtr000005.last.html#idSysMols009shp">a-shape</a>.
    <a class="clsHide" href="#idOgmatt007dsn"></a></p>

  <section id="idOgmatt021">
  <h2 id="idOgmatt021H2">symmetry of ogm
    <a class="clsHide" href="#idOgmatt021H2"></a></h2>
  <p id="idOgmatt021nam">name::
    <br>* McsEngl.ogm'att021-symmetry,
    <br>* McsEngl.ogm'symmetry-att021,
    <br>* McsEngl.symmetry-of-ogm-att021,
    <a class="clsHide" href="#idOgmatt021nam"></a></p>
  <p id="idOgmatt021dsn">description::
    <br>"Symmetry in biology refers to the symmetry observed in organisms, including plants, animals, fungi, and bacteria. External symmetry can be easily seen by just looking at an organism. For example, take the face of a human being which has a plane of symmetry down its centre, or a pine cone with a clear symmetrical spiral pattern. Internal features can also show symmetry, for example the tubes in the human body (responsible for transporting gases, nutrients, and waste products) which are cylindrical and have several planes of symmetry.
    <br>Biological symmetry can be thought of as a balanced distribution of duplicate body parts or shapes within the body of an organism. Importantly, unlike in mathematics, symmetry in biology is always approximate. For example, plant leaves – while considered symmetrical – rarely match up exactly when folded in half. Symmetry is one class of patterns in nature whereby there is near-repetition of the pattern element, either by reflection or rotation.
    <br>While sponges and placozoans represent two groups of animals which don't show any symmetry (i.e. are asymmetrical), the body plans of most multicellular organisms exhibit, and are defined by, some form of symmetry. There are only a few types of symmetry which are possible in body plans. These are radial (cylindrical), bilateral, biradial and spherical symmetry.[1] While the classification of viruses as an 'organism' remains controversial, viruses also contain icosahedral symmetry.
    <br>The importance of symmetry is illustrated by the fact that groups of animals have traditionally been defined by this feature in taxonomic groupings. The Radiata, animals with radial symmetry, formed one of the four branches of Georges Cuvier's classification of the animal kingdom.[2][3][4] Meanwhile, Bilateria is a taxonomic grouping still used today to represent organisms with embryonic bilateral symmetry."
    <br>[{2020-04-25} https://en.wikipedia.org/wiki/Symmetry_in_biology]
    <a class="clsHide" href="#idOgmatt021dsn"></a></p>
  </section>
</section>

<section id="idOgmatt014">
  <h1 id="idOgmatt014H1">10_size of ogm
    <a class="clsHide" href="#idOgmatt014H1"></a></h1>
  <p id="idOgmatt014nam">name::
    <br>* McsEngl.ogm'10_size,
    <br>* McsEngl.ogm'att014-size,
    <br>* McsEngl.ogm'size-att014,
    <br>* McsEngl.size-of-ogm-att014,
    <a class="clsHide" href="#idOgmatt014nam"></a></p>
  <p id="idOgmatt014dsn">description::
    <br>· size-of-ogm is any measure of its body like length, width, height, or volume.
    <a class="clsHide" href="#idOgmatt014dsn"></a></p>
</section>

<section id="idOgmrscF">
  <h1 id="idOgmrscFH1">info-resource of ogm
    <a class="clsHide" href="#idOgmrscFH1"></a></h1>
  <p id="idOgmrscnam">name::
    <br>* McsEngl.ogm'Infrsc,
    <a class="clsHide" href="#idOgmrscnam"></a></p>
  <p id="idOgmrscwpa">addressWpg::
    <br>* https://species.wikimedia.org/wiki/Main_Page,
    <a class="clsHide" href="#idOgmrscwpa"></a></p>

  <section id="idOgmsci">
  <h2 id="idOgmsciH2">science of bio
    <a class="clsHide" href="#idOgmsciH2"></a></h2>
  <p id="idOgmscidsn">description::
    <br>"The study of living things is called biology (also called biological science). An expert in this field is called a biologist. Several areas of biological studies include morphology, anatomy, cytology, histology, physiology, ecology, evolution, taxonomy, and pathology."
    <br>[https://www.biology-online.org/dictionary/Living_thing]
    <a class="clsHide" href="#idOgmscidsn"></a></p>
  <p id="idOgmscinam">name::
    <br>* McsEngl.biological-science!⇒sciBio,
    <br>* McsEngl.biology!⇒sciBio,
    <br>* McsEngl.bio'science!⇒sciBio,
    <br>* McsEngl.sciBio,
    <br>* McsEngl.science.biology!⇒sciBio,
    <br>====== langoSinago:
    <br>* McsSngo.sio-zo-fo!=sciBio,
    <br>====== langoChinese:
    <br>* McsZhon.shēngwùxué-生物学!=sciBio,
    <br>* McsZhon.生物学-shēngwùxué!=sciBio,
    <br>====== langoEsperanto:
    <br>* McsEspo.biologio!=sciBio,
    <br>====== langoGreek:
    <br>* McsElln.βιολογία!~η!=sciBio,
    <a class="clsHide" href="#idOgmscinam"></a></p>

  <section id="idOgmscirscF">
  <h3 id="idOgmscirscFH3">info-resource of sciBio
    <a class="clsHide" href="#idOgmscirscFH3"></a></h3>
  <p id="idOgmscirscFnam">name::
    <br>* McsEngl.sciBio'Infrsc,
    <a class="clsHide" href="#idOgmscirscFnam"></a></p>
  <p id="idOgmscirscFdsn">description::
    <br>* https://en.wikipedia.org/wiki/History_of_biology,
    <a class="clsHide" href="#idOgmscirscFdsn"></a></p>
  </section>
  </section>
</section>

<section id="idOgmsrtrF">
  <h1 id="idOgmsrtrFH1">structure of ogm
    <a class="clsHide" href="#idOgmsrtrFH1"></a></h1>
  <p id="idOgmsrtrFnam">name::
    <br>* McsEngl.ogm'structure,
    <a class="clsHide" href="#idOgmsrtrFnam"></a></p>
  <p id="idOgmsrtrFdsn">description::
    <br>·
    <a class="clsHide" href="#idOgmsrtrFdsn"></a></p>
</section>

<section id="idOgmdngF">
  <h1 id="idOgmdngFH1">DOING of ogm
    <a class="clsHide" href="#idOgmdngFH1"></a></h1>
  <p id="idOgmdngFdsn">description::
    <br>· any doing of organism.
    <a class="clsHide" href="#idOgmdngFdsn"></a></p>
  <p id="idOgmdngFnam">name::
    <br>* McsEngl.behavior-of-organism,
    <br>* McsEngl.ogm'behavior,
    <br>* McsEngl.ogm'doing,
    <a class="clsHide" href="#idOgmdngFnam"></a></p>
  <p id="idOgmdngFstr">specific-tree-of-::
    <br>* function,
    <br>* action,
    <br>===
    <br>* communication,
    <br>* metabolism,
    <br>* reproduction,
    <a class="clsHide" href="#idOgmdngFstr"></a></p>

  <section id="idOgmatt015">
  <h2 id="idOgmatt015H2">function of ogm
    <a class="clsHide" href="#idOgmatt015H2"></a></h2>
  <p id="idOgmatt015dsn">description::
    <br>· <a class="clsPreview" href="../dirCor/McsCor000004.last.html#idDngFctg">a-functing-(internal-doing)</a> of an-organism.
    <a class="clsHide" href="#idOgmatt015dsn"></a></p>
  <p id="idOgmatt015nam">name::
    <br>* McsEngl.ogm'att015-functing,
    <br>* McsEngl.ogm'functing-att015,
    <br>* McsEngl.ogm'doing.functing,
    <br>* McsEngl.biological-process,
    <a class="clsHide" href="#idOgmatt015nam"></a></p>
  </section>

  <section id="idOgmatt024">
  <h2 id="idOgmatt024H2">action of ogm
    <a class="clsHide" href="#idOgmatt024H2"></a></h2>
  <p id="idOgmatt024dsn">description::
    <br>· external doing of ogm.
    <a class="clsHide" href="#idOgmatt024dsn"></a></p>
  <p id="idOgmatt024nam">name::
    <br>* McsEngl.action-of-ogm,
    <br>* McsEngl.ogm'att024-action,
    <br>* McsEngl.ogm'action,
    <a class="clsHide" href="#idOgmatt024nam"></a></p>
  </section>

  <section id="idOgmatt025">
  <h2 id="idOgmatt025H2">reproduction of ogm
    <a class="clsHide" href="#idOgmatt025H2"></a></h2>
  <p id="idOgmatt025dsn">description::
    <br>"Reproduction (or procreation or breeding) is the biological process by which new individual organisms – "offspring" – are produced from their "parents". Reproduction is a fundamental feature of all known life; each individual organism exists as the result of reproduction. There are two forms of reproduction: asexual and sexual.
    <br>In asexual reproduction, an organism can reproduce without the involvement of another organism. Asexual reproduction is not limited to single-celled organisms. The cloning of an organism is a form of asexual reproduction. By asexual reproduction, an organism creates a genetically similar or identical copy of itself. The evolution of sexual reproduction is a major puzzle for biologists. The two-fold cost of sexual reproduction is that only 50% of organisms reproduce[1] and organisms only pass on 50% of their genes.[2]
    <br>Sexual reproduction typically requires the sexual interaction of two specialized organisms, called gametes, which contain half the number of chromosomes of normal cells and are created by meiosis, with typically a male fertilizing a female of the same species to create a fertilized zygote. This produces offspring organisms whose genetic characteristics are derived from those of the two parental organisms."
    <br>[{2020-10-30} https://en.wikipedia.org/wiki/Reproduction]
    <a class="clsHide" href="#idOgmatt025dsn"></a></p>
  <p id="idOgmatt025nam">name::
    <br>* McsEngl.ogm'att025-reproduction,
    <br>* McsEngl.ogm'reproduction,
    <br>* McsEngl.reproduction-of-ogm,
    <a class="clsHide" href="#idOgmatt025nam"></a></p>
  </section>

  <section id="idOgmatt022">
  <h2 id="idOgmatt022H2">biosynthesis of ogm
    <a class="clsHide" href="#idOgmatt022H2"></a></h2>
  <p id="idOgmatt022dsn">description::
    <br>"Biosynthesis is a multi-step, enzyme-catalyzed process where substrates are converted into more complex products in living organisms. In biosynthesis, simple compounds are modified, converted into other compounds, or joined together to form macromolecules. This process often consists of metabolic pathways. Some of these biosynthetic pathways are located within a single cellular organelle, while others involve enzymes that are located within multiple cellular organelles. Examples of these biosynthetic pathways include the production of lipid membrane components and nucleotides. Biosynthesis is usually synonymous with anabolism.
    <br>The prerequisite elements for biosynthesis include: precursor compounds, chemical energy (e.g. ATP), and catalytic enzymes which may require coenzymes (e.g.NADH, NADPH). These elements create monomers, the building blocks for macromolecules. Some important biological macromolecules include: proteins, which are composed of amino acid monomers joined via peptide bonds, and DNA molecules, which are composed of nucleotides joined via phosphodiester bonds."
    <br>[{2020-08-18} https://en.wikipedia.org/wiki/Biosynthesis]
    <a class="clsHide" href="#idOgmatt022dsn"></a></p>
  <p id="idOgmatt022nam">name::
    <br>* McsEngl.biosythesis,
    <br>* McsEngl.ogm'att022-biosythesis,
    <br>* McsEngl.ogm'biosythesis,
    <a class="clsHide" href="#idOgmatt022nam"></a></p>
  </section>

  <section id="idOgmatt016">
  <h2 id="idOgmatt016H2">metabolism of ogm
    <a class="clsHide" href="#idOgmatt016H2"></a></h2>
  <p id="idOgmatt016nam">name::
    <br>* McsEngl.ogm'att016-metabolism,
    <br>* McsEngl.ogm'metabolism-att016,
    <br>* McsEngl.metabolism-of-ogm-att016,
    <a class="clsHide" href="#idOgmatt016nam"></a></p>
  <p id="idOgmatt016dsn">description::
    <br>"Viruses do not have their own metabolism, and require a host cell to make new products. They therefore cannot naturally reproduce outside a host cell[66]—although bacterial species such as rickettsia and chlamydia are considered living organisms despite the same limitation."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Virus#Life_properties]
    <br>===
    <br>"Metabolism (/məˈtæbəlɪzəm/, from Greek: μεταβολή metabolē, "change") is the set of life-sustaining chemical reactions in organisms. The three main purposes of metabolism are: the conversion of food to energy to run cellular processes; the conversion of food/fuel to building blocks for proteins, lipids, nucleic acids, and some carbohydrates; and the elimination of nitrogenous wastes. These enzyme-catalyzed reactions allow organisms to grow and reproduce, maintain their structures, and respond to their environments. (The word metabolism can also refer to the sum of all chemical reactions that occur in living organisms, including digestion and the transport of substances into and between different cells, in which case the above described set of reactions within the cells is called intermediary metabolism or intermediate metabolism).
    <br>Metabolic reactions may be categorized as catabolic – the breaking down of compounds (for example, the breaking down of glucose to pyruvate by cellular respiration); or anabolic – the building up (synthesis) of compounds (such as proteins, carbohydrates, lipids, and nucleic acids). Usually, catabolism releases energy, and anabolism consumes energy.
    <br>The chemical reactions of metabolism are organized into metabolic pathways, in which one chemical is transformed through a series of steps into another chemical, each step being facilitated by a specific enzyme. Enzymes are crucial to metabolism because they allow organisms to drive desirable reactions that require energy that will not occur by themselves, by coupling them to spontaneous reactions that release energy. Enzymes act as catalysts – they allow a reaction to proceed more rapidly – and they also allow the regulation of the rate of a metabolic reaction, for example in response to changes in the cell's environment or to signals from other cells.
    <br>The metabolic system of a particular organism determines which substances it will find nutritious and which poisonous. For example, some prokaryotes use hydrogen sulfide as a nutrient, yet this gas is poisonous to animals.[1] The basal metabolic rate of an organism is the measure of the amount of energy consumed by all of these chemical reactions.
    <br>A striking feature of metabolism is the similarity of the basic metabolic pathways among vastly different species.[2] For example, the set of carboxylic acids that are best known as the intermediates in the citric acid cycle are present in all known organisms, being found in species as diverse as the unicellular bacterium Escherichia coli and huge multicellular organisms like elephants.[3] These similarities in metabolic pathways are likely due to their early appearance in evolutionary history, and their retention because of their efficacy.[4][5]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Metabolism]
    <a class="clsHide" href="#idOgmatt016dsn"></a></p>
  </section>

  <section id="idOgmatt019">
  <h2 id="idOgmatt019H2">photosynthesis of ogm
    <a class="clsHide" href="#idOgmatt019H2"></a></h2>
  <p id="idOgmatt019nam">name::
    <br>* McsEngl.ogm'att019-photosynthesis,
    <br>* McsEngl.ogm'photosynthesis-att019,
    <br>* McsEngl.photosynthesis-of-ogm-att019,
    <a class="clsHide" href="#idOgmatt019nam"></a></p>
  <p id="idOgmatt019dsn">description::
    <br>""
    <a class="clsHide" href="#idOgmatt019dsn"></a></p>
  </section>
</section>

<section id="idOgmevgF">
  <h1 id="idOgmevgFH1">evoluting of ogm
    <a class="clsHide" href="#idOgmevgFH1"></a></h1>
  <p id="idOgmevgnam">name::
    <br>* McsEngl.ogm'evoluting,
    <a class="clsHide" href="#idOgmevgnam"></a></p>
  <p id="idOgmevg20191219">{2019-12-19}::
    <br>=== McsHitp-creation:
    <br>· creation of current <a class="clsPreview" href="../dirCor/McsCor000002.last.html#idOverview">concept</a>.
    <a class="clsHide" href="#idOgmevg20191219"></a></p>
  <p id="idOgmevgBpB3_8">{BpK3x3.8..K3x3.5}-ogm-first::
    <br>"The last universal common ancestor (LUCA) is the most recent organism from which all organisms now living on Earth descend.[32] Thus it is the most recent common ancestor of all current life on Earth. The LUCA is estimated to have lived some 3.5 to 3.8 billion years ago (sometime in the Paleoarchean era).[33][34] The earliest evidence for life on Earth is graphite found to be biogenic in 3.7 billion-year-old metasedimentary rocks discovered in Western Greenland[35] and microbial mat fossils found in 3.48 billion-year-old sandstone discovered in Western Australia.[36][37] Although more than 99 percent of all species that ever lived on the planet are estimated to be extinct,[6][7] there are currently 2 million to 1 trillion species of life on Earth.[3]"
    <br>[{2020-08-31} https://en.wikipedia.org/wiki/Organism#Evolution]
    <br>* McsEngl.{BpK3x3.8..K3x3.5}-ogm-first,
    <a class="clsHide" href="#idOgmevgBpB3_8"></a></p>

  <section id="idOgmatt017">
  <h2 id="idOgmatt017H2">life-cycle of ogm
    <a class="clsHide" href="#idOgmatt017H2"></a></h2>
  <p id="idOgmatt017nam">name::
    <br>* McsEngl.ogm'att017-life-cycle,
    <br>* McsEngl.ogm'life-cycle-att017,
    <br>* McsEngl.life-cycle-of-ogm-att017,
    <a class="clsHide" href="#idOgmatt017nam"></a></p>
  <p id="idOgmatt017dsn">description::
    <br>·
    <a class="clsHide" href="#idOgmatt017dsn"></a></p>
  </section>

  <section id="idOgmatt012">
  <h2 id="idOgmatt012H2">lifetime of ogm
    <a class="clsHide" href="#idOgmatt012H2"></a></h2>
  <p id="idOgmatt012nam">name::
    <br>* McsEngl.ogm'att012-lifetime!⇒ogm'lifetime,
    <br>* McsEngl.ogm'lifetime,
    <br>* McsEngl.lifespan-of-ogm-att012!⇒ogm'lifetime,
    <br>* McsEngl.lifetime-of-ogm-att012!⇒ogm'lifetime,
    <a class="clsHide" href="#idOgmatt012nam"></a></p>
  <p id="idOgmatt012dsn">description::
    <br>"noun lifespan: the length of time for which a person or animal lives or a thing functions."
    <br>[Google dict]
    <a class="clsHide" href="#idOgmatt012dsn"></a></p>
  </section>
</section>

<section id="idOgmwpt">
  <h1 id="idOgmwptH1">WHOLE-PART-TREE of ogm
    <a class="clsHide" href="#idOgmwptH1"></a></h1>
  <p id="idOgmwptnam">name::
    <br>* McsEngl.ogm'whole-part-tree,
    <a class="clsHide" href="#idOgmwptnam"></a></p>
</section>

<section id="idOgmgst">
  <h1 id="idOgmgstH1">GENERIC-SPECIFIC-TREE of ogm
    <a class="clsHide" href="#idOgmgstH1"></a></h1>
  <p id="idOgmgstnam">name::
    <br>* McsEngl.ogm'generic-specific-tree,
    <a class="clsHide" href="#idOgmgstnam"></a></p>
  <p id="idOgmgstgtr">generic-tree::
    <br>* <a class="clsPreview" href="../dirOgm/McsOgm000002.last.html#idOverview">biosystem</a>,
    <br>* <a class="clsPreview" href="../dirCor/McsCor000008.last.html#idSysDnmc">dynamic-system</a>,
    <br>* <a class="clsPreview" href="../dirCor/McsCor000008.last.html#idSys">system</a>,
    <br>* <a class="clsPreview" href="../dirCor/McsCor000008.last.html#idOverview">whole-entity</a>,
    <br>* <a class="clsPreview" href="../dirCor/McsCor000003.last.html#idEntBody">body</a>,
    <br>* <a class="clsPreview" href="../dirCor/McsCor000003.last.html#idOverview">entity</a>,
    <a class="clsHide" href="#idOgmgstgtr"></a></p>

  <section id="idOgmSpcF">
  <h2 id="idOgmSpcFH2">ogm.SPECIFIC
    <a class="clsHide" href="#idOgmSpcFH2"></a></h2>
  <p id="idOgmSpcnam">name::
    <br>* McsEngl.ogm.specific,
    <a class="clsHide" href="#idOgmSpcnam"></a></p>
  <p id="idOgmSpcP1">specific::
    <br>* generic-ogm,
    <br>* genericNo-ogm,
    <br>* cell-ogm,
    <br>* cellNo-ogm,
    <br>* extinct-ogm,
    <br>* extinctNo-ogm,
    <br>* free-ogm,
    <br>* freeNo-ogm,
    <br>* microscopic-ogm,
    <br>* microscopicNo-ogm,
    <a class="clsHide" href="#idOgmSpcP1"></a></p>
  </section>

  <section id="idOgmScdn">
  <h2 id="idOgmScdnH2">ogm.specifics-division
    <a class="clsHide" href="#idOgmScdnH2"></a></h2>
  <p id="idOgmScdndsn">description::
    <br>"In biology, taxonomy (from Ancient Greek τάξις (taxis), meaning 'arrangement', and -νομία (-nomia), meaning 'method') is the science of naming, defining (circumscribing) and classifying groups of biological organisms on the basis of shared characteristics. Organisms are grouped together into taxa (singular: taxon) and these groups are given a taxonomic rank; groups of a given rank can be aggregated to form a super-group of higher rank, thus creating a taxonomic hierarchy. The principal ranks in modern use are domain, kingdom, phylum (division is sometimes used in botany in place of phylum), class, order, family, genus, and species. The Swedish botanist Carl Linnaeus is regarded as the founder of the current system of taxonomy, as he developed a system known as Linnaean taxonomy for categorizing organisms and binomial nomenclature for naming organisms.
    <br>With the advent of such fields of study as phylogenetics, cladistics, and systematics, the Linnaean system has progressed to a system of modern biological classification based on the evolutionary relationships between organisms, both living and extinct."
    <br>[{2019-12-24} https://en.wikipedia.org/wiki/Taxonomy_(biology)]
    <a class="clsHide" href="#idOgmScdndsn"></a></p>
  <p id="idOgmScdnnam">name::
    <br>* McsEngl.ogm.specifics-division,
    <br>* McsEngl.ogm.taxonomy,
    <br>* McsEngl.taxonomy.biology,
    <a class="clsHide" href="#idOgmScdnnam"></a></p>
  </section>

  <section id="idOgmScdnEvlg">
  <h2 id="idOgmScdnEvlgH2">ogm.specs-div.evolution
    <a class="clsHide" href="#idOgmScdnEvlgH2"></a></h2>
  <p id="idOgmScdnEvlgdsn">description::
    <br>"Evolutionary taxonomy, evolutionary systematics or Darwinian classification is a branch of biological classification that seeks to classify organisms using a combination of phylogenetic relationship (shared descent), progenitor-descendant relationship (serial descent), and degree of evolutionary change. This type of taxonomy may consider whole taxa rather than single species, so that groups of species can be inferred as giving rise to new groups.[1] The concept found its most well-known form in the modern evolutionary synthesis of the early 1940s.
    <br>Evolutionary taxonomy differs from strict pre-Darwinian Linnaean taxonomy (producing orderly lists only), in that it builds evolutionary trees. While in phylogenetic nomenclature each taxon must consist of a single ancestral node and all its descendants, evolutionary taxonomy allows for groups to be excluded from their parent taxa (e.g. dinosaurs are not considered to include birds, but to have given rise to them), thus permitting paraphyletic taxa.[2][3]"
    <br>[{2019-12-24} https://en.wikipedia.org/wiki/Evolutionary_taxonomy]
    <a class="clsHide" href="#idOgmScdnEvlgdsn"></a></p>
  <p id="idOgmScdnEvlgnam">name::
    <br>* McsEngl.ogm.specifics-division.evolution,
    <a class="clsHide" href="#idOgmScdnEvlgnam"></a></p>

  <section id="idOgmCld">
  <h3 id="idOgmCldH3">ogm.clade
    <a class="clsHide" href="#idOgmCldH3"></a></h3>
  <p id="idOgmClddsn">description::
    <br>"A clade (from Ancient Greek: κλάδος, klados, "branch"), also known as monophyletic group, is a group of organisms that consists of a common ancestor and all its lineal descendants, and represents a single "branch" on the "tree of life".[1] Rather than the English term, the equivalent Latin term cladus (plural cladi) is often used in taxonomical literature.
    <br>The common ancestor may be an individual, a population, a species (extinct or extant), and so on right up to a kingdom and further. Clades are nested, one in another, as each branch in turn splits into smaller branches. These splits reflect evolutionary history as populations diverged and evolved independently. Clades are termed monophyletic (Greek: "one clan") groups.
    <br>Over the last few decades, the cladistic approach has revolutionized biological classification and revealed surprising evolutionary relationships among organisms.[2] Increasingly, taxonomists try to avoid naming taxa that are not clades; that is, taxa that are not monophyletic. Some of the relationships between organisms that the molecular biology arm of cladistics has revealed are that fungi are closer relatives to animals than they are to plants, archaea are now considered different from bacteria, and multicellular organisms may have evolved from archaea.[3]"
    <br>[{2019-12-24} https://en.wikipedia.org/wiki/Clade]
    <a class="clsHide" href="#idOgmClddsn"></a></p>
  <p id="idOgmCldnam">name::
    <br>* McsEngl.ogm.clade,
    <br>* McsEngl.ogm.monophyletic-group,
    <a class="clsHide" href="#idOgmCldnam"></a></p>
  </section>
  </section>

  <section id="idOgmScdn2empr">
  <h2 id="idOgmScdn2emprH2">ogm.specs-div.two-empire
    <a class="clsHide" href="#idOgmScdn2emprH2"></a></h2>
  <p id="idOgmScdn2emprnam">name::
    <br>* McsEngl.ogm.specifics-division.two-empire,
    <a class="clsHide" href="#idOgmScdn2emprnam"></a></p>
  <p id="idOgmScdn2emprdsn">description::
    <br>"The two-empire system (two-superkingdom system) was the top-level biological classification system in general use before the establishment of the three-domain system. It classified life into Prokaryota and Eukaryota. When the three-domain system was introduced, some biologists preferred the two-superkingdom system, claiming that the three-domain system overemphasized the division between Archaea and Bacteria. However, given the current state of knowledge and the rapid progress in biological scientific advancement, especially due to genetic analyses, that view has all but vanished.
    <br>Some prominent scientists, such as Thomas Cavalier-Smith, still hold to the two-empire system.[1] The late Ernst Mayr, one of the 20th century's leading evolutionary biologists, wrote dismissively of the three-domain system, "I cannot see any merit at all in a three empire cladification."[2] Additionally, the scientist Radhey Gupta argues for a return to the two-empire system, claiming that the primary division within prokaryotes should be among those surrounded by a single membrane (monoderm), including gram-positive bacteria and archaebacteria, and those with an inner and outer cell membrane (diderm), including gram-negative bacteria.[3]"
    <br>[{2020-02-16} https://en.wikipedia.org/wiki/Two-empire_system]
    <br>===
    <br>* life,
    <br>** acellular-life,
    <br>** cellular-life,
    <br>*** prokaryota,
    <br>**** bakteria,
    <br>**** archea,
    <br>*** eukaryota,
    <br>**** protista,
    <br>**** fungi,
    <br>**** plantae,
    <br>**** animalia,
    <a class="clsHide" href="#idOgmScdn2emprdsn"></a></p>
  </section>

  <section id="idOgmScdn3dmn">
  <h2 id="idOgmScdn3dmnH2">ogm.specs-div.three-domain {1990}
    <a class="clsHide" href="#idOgmScdn3dmnH2"></a></h2>
  <p id="idOgmScdn3dmnnam">name::
    <br>* McsEngl.3-domain-system--of-organisms,
    <br>* McsEngl.ogm.specifics-division.three-domain,
    <br>* McsEngl.three-domain-system--of-organisms,
    <a class="clsHide" href="#idOgmScdn3dmnnam"></a></p>
  <p id="idOgmScdn3dmndsn">description::
    <br>"The three-domain system is a biological classification introduced by Carl Woese et al. in 1990[1][2] that divides cellular life forms into archaea, bacteria, and eukaryote domains. In particular, it emphasizes the separation of prokaryotes into two groups, originally called Eubacteria (now Bacteria) and Archaebacteria (now Archaea). Woese argued that, on the basis of differences in 16S rRNA genes, these two groups and the eukaryotes each arose separately from an ancestor with poorly developed genetic machinery, often called a progenote. To reflect these primary lines of descent, he treated each as a domain, divided into several different kingdoms. Woese initially used the term "kingdom" to refer to the three primary phylogenic groupings, and this nomenclature was widely used until the term "domain" was adopted in 1990.[2]
    <br>Parts of the three-domain theory have been challenged by scientists such as Radhey Gupta, who argues that the primary division within prokaryotes should be between those surrounded by a single membrane, and those with two membranes.[citation needed]"
    <br>[{2019-12-24} https://en.wikipedia.org/wiki/Three-domain_system]
    <a class="clsHide" href="#idOgmScdn3dmndsn"></a></p>

  <section id="idOgmScdn3dmndomain">
  <h3 id="idOgmScdn3dmndomainH3">domain of three-domain-system
    <a class="clsHide" href="#idOgmScdn3dmndomainH3"></a></h3>
  <p id="idOgmScdn3dmndomainnam">name::
    <br>* McsEngl.domain.3domain-sys,
    <a class="clsHide" href="#idOgmScdn3dmndomainnam"></a></p>
  <p id="idOgmScdn3dmndomaindsn">description::
    <br>·
    <a class="clsHide" href="#idOgmScdn3dmndomaindsn"></a></p>
  </section>

  <section id="idOgmScdn3dmnkngdm">
  <h3 id="idOgmScdn3dmnkngdmH3">kingdom of three-domain-system
    <a class="clsHide" href="#idOgmScdn3dmnkngdmH3"></a></h3>
  <p id="idOgmScdn3dmnkngdmnam">name::
    <br>* McsEngl.kingdom.3domain-sys,
    <a class="clsHide" href="#idOgmScdn3dmnkngdmnam"></a></p>
  <p id="idOgmScdn3dmnkngdmdsn">description::
    <br>"In biology, kingdom (Latin: regnum, plural regna) is the second highest taxonomic rank, just below domain. Kingdoms are divided into smaller groups called phyla.
    <br>Traditionally, some textbooks from the United States and Canada used a system of six kingdoms (Animalia, Plantae, Fungi, Protista, Archaea/Archaebacteria, and Bacteria/Eubacteria) while textbooks in countries like Great Britain, India, Greece, Brazil and other countries used five kingdoms (Animalia, Plantae, Fungi, Protista and Monera).
    <br>Some recent classifications based on modern cladistics have explicitly abandoned the term "kingdom", noting that the traditional kingdoms are not monophyletic, i.e., do not consist of all the descendants of a common ancestor."
    <br>[{2020-02-16} https://en.wikipedia.org/wiki/Kingdom_(biology)]
    <a class="clsHide" href="#idOgmScdn3dmnkngdmdsn"></a></p>
  </section>

  <section id="idOgmScdn3dmnphlm">
  <h3 id="idOgmScdn3dmnphlmH3">phylum of three-domain-system
    <a class="clsHide" href="#idOgmScdn3dmnphlmH3"></a></h3>
  <p id="idOgmScdn3dmnphlmnam">name::
    <br>* McsEngl.phylum.3domain-sys,
    <a class="clsHide" href="#idOgmScdn3dmnphlmnam"></a></p>
  <p id="idOgmScdn3dmnphlmdsn">description::
    <br>"In biology, a phylum (/ˈfaɪləm/; plural: phyla) is a level of classification or taxonomic rank below kingdom and above class. Traditionally, in botany the term division has been used instead of phylum, although the International Code of Nomenclature for algae, fungi, and plants accepts the terms as equivalent.[1][2][3] Depending on definitions, the animal kingdom Animalia or Metazoa contains approximately 35 phyla, the plant kingdom Plantae contains about 14, and the fungus kingdom Fungi contains about 8 phyla. Current research in phylogenetics is uncovering the relationships between phyla, which are contained in larger clades, like Ecdysozoa and Embryophyta."
    <br>[{2020-02-16} https://en.wikipedia.org/wiki/Phylum]
    <a class="clsHide" href="#idOgmScdn3dmnphlmdsn"></a></p>
  </section>

  <section id="idOgmScdn3dmnclss">
  <h3 id="idOgmScdn3dmnclssH3">class of three-domain-system
    <a class="clsHide" href="#idOgmScdn3dmnclssH3"></a></h3>
  <p id="idOgmScdn3dmnclssnam">name::
    <br>* McsEngl.class.3domain-sys,
    <a class="clsHide" href="#idOgmScdn3dmnclssnam"></a></p>
  <p id="idOgmScdn3dmnclssdsn">description::
    <br>"In biological classification, class (Latin: classis) is a taxonomic rank, as well as a taxonomic unit, a taxon, in that rank.[a] Other well-known ranks in descending order of size are life, domain, kingdom, phylum, order, family, genus, and species, with class fitting between phylum and order."
    <br>[{2020-02-16} https://en.wikipedia.org/wiki/Class_(biology)]
    <a class="clsHide" href="#idOgmScdn3dmnclssdsn"></a></p>
  </section>

  <section id="idOgmScdn3dmnordr">
  <h3 id="idOgmScdn3dmnordrH3">order of three-domain-system
    <a class="clsHide" href="#idOgmScdn3dmnordrH3"></a></h3>
  <p id="idOgmScdn3dmnordrnam">name::
    <br>* McsEngl.order.3domain-sys,
    <a class="clsHide" href="#idOgmScdn3dmnordrnam"></a></p>
  <p id="idOgmScdn3dmnordrdsn">description::
    <br>"In biological classification, the order (Latin: ordo) is
    <br>1. a taxonomic rank used in the classification of organisms and recognized by the nomenclature codes. Other well-known ranks are life, domain, kingdom, phylum, class, family, genus, and species, with order fitting in between class and family. An immediately higher rank, superorder, may be added directly above order, while suborder would be a lower rank.
    <br>2. a taxonomic unit, a taxon, in that rank. In that case the plural is orders (Latin ordines). Example: All owls belong to the order Strigiformes
    <br>What does and does not belong to each order is determined by a taxonomist, as is whether a particular order should be recognized at all. Often there is no exact agreement, with different taxonomists each taking a different position. There are no hard rules that a taxonomist needs to follow in describing or recognizing an order. Some taxa are accepted almost universally, while others are recognised only rarely.[1]
    <br>For some groups of organisms, consistent suffixes are used to denote that the rank is an order. The Latin suffix -(i)formes meaning "having the form of" is used for the scientific name of orders of birds and fishes, but not for those of mammals and invertebrates. The suffix -ales is for the name of orders of plants, fungi, and algae.[2]"
    <br>[{2020-02-16} https://en.wikipedia.org/wiki/Order_(biology)]
    <a class="clsHide" href="#idOgmScdn3dmnordrdsn"></a></p>
  </section>

  <section id="idOgmScdn3dmnfmly">
  <h3 id="idOgmScdn3dmnfmlyH3">family of three-domain-system
    <a class="clsHide" href="#idOgmScdn3dmnfmlyH3"></a></h3>
  <p id="idOgmScdn3dmnfmlynam">name::
    <br>* McsEngl.family.3domain-sys,
    <a class="clsHide" href="#idOgmScdn3dmnfmlynam"></a></p>
  <p id="idOgmScdn3dmnfmlydsn">description::
    <br>"Family (Latin: familia, plural familiae) is one of the eight major hierarchical taxonomic ranks in Linnaean taxonomy; it is classified between order and genus. A family may be divided into subfamilies, which are intermediate ranks between the ranks of family and genus. The official family names are Latin in origin; however, popular names are often used: for example, walnut trees and hickory trees belong to the family Juglandaceae, but that family is commonly referred to as being the "walnut family".
    <br>What does or does not belong to a family—or whether a described family should be recognized at all—are proposed and determined by practicing taxonomists. There are no hard rules for describing or recognizing a family. Taxonomists often take different positions about descriptions, and there may be no broad consensus across the scientific community for some time. The publishing of new data and opinions often enables adjustments and consensus."
    <br>[{2020-02-16} https://en.wikipedia.org/wiki/Family_(biology)]
    <a class="clsHide" href="#idOgmScdn3dmnfmlydsn"></a></p>
  </section>

  <section id="idOgmScdn3dmngns">
  <h3 id="idOgmScdn3dmngnsH3">genus of three-domain-system
    <a class="clsHide" href="#idOgmScdn3dmngnsH3"></a></h3>
  <p id="idOgmScdn3dmngnsnam">name::
    <br>* McsEngl.genus.3domain-sys,
    <a class="clsHide" href="#idOgmScdn3dmngnsnam"></a></p>
  <p id="idOgmScdn3dmngnsdsn">description::
    <br>"A genus (plural genera) is a taxonomic rank used in the biological classification of living and fossil organisms, as well as viruses,[1] in biology. In the hierarchy of biological classification, genus comes above species and below family."
    <br>[{2020-02-16} https://en.wikipedia.org/wiki/Genus]
    <a class="clsHide" href="#idOgmScdn3dmngnsdsn"></a></p>
  </section>

  <section id="idOgmScdn3dmnspcs">
  <h3 id="idOgmScdn3dmnspcsH3">species of three-domain-system
    <a class="clsHide" href="#idOgmScdn3dmnspcsH3"></a></h3>
  <p id="idOgmScdn3dmnspcsnam">name::
    <br>* McsEngl.species.3domain-sys,
    <a class="clsHide" href="#idOgmScdn3dmnspcsnam"></a></p>
  <p id="idOgmScdn3dmnspcsdsn">description::
    <br>"In biology, a species (/ˈspiːʃiːz/ (About this soundlisten)) is the basic unit of classification and a taxonomic rank of an organism, as well as a unit of biodiversity. A species is often defined as the largest group of organisms in which any two individuals of the appropriate sexes or mating types can produce fertile offspring, typically by sexual reproduction. Other ways of defining species include their karyotype, DNA sequence, morphology, behaviour or ecological niche. In addition, paleontologists use the concept of the chronospecies since fossil reproduction cannot be examined. The total number of species is estimated to be between 8 and 8.7 million.[1][2] However the vast majority of them are not studied or documented and it may take over 1000 years to fully catalog all of them.[3]
    <br>All species (except viruses) are given a two-part name, a "binomial". The first part of a binomial is the genus to which the species belongs. The second part is called the specific name or the specific epithet (in botanical nomenclature, also sometimes in zoological nomenclature). For example, Boa constrictor is one of four species of the genus Boa."
    <br>[{2020-02-16} https://en.wikipedia.org/wiki/Species]
    <a class="clsHide" href="#idOgmScdn3dmnspcsdsn"></a></p>
  </section>
  </section>

  <section id="idOgmScdnLns">
  <h2 id="idOgmScdnLnsH2">ogm.specs-div.Linnaeus {1735}
    <a class="clsHide" href="#idOgmScdnLnsH2"></a></h2>
  <p id="idOgmScdnLnsdsn">description::
    <br>"the particular form of biological classification (taxonomy) set up by Carl Linnaeus, as set forth in his Systema Naturae (1735) and subsequent works. In the taxonomy of Linnaeus there are three kingdoms, divided into classes, and they, in turn, into orders, genera (singular: genus), and species (singular: species)[1], with an additional rank lower than species."
    <br>[{2019-12-24} https://en.wikipedia.org/wiki/Linnaean_taxonomy]
    <a class="clsHide" href="#idOgmScdnLnsdsn"></a></p>
  <p id="idOgmScdnLnsnam">name::
    <br>* McsEngl.Linnaeus-taxonomy,
    <br>* McsEngl.ogm.specifics-division.Linnaeus,
    <a class="clsHide" href="#idOgmScdnLnsnam"></a></p>
  </section>

  <section id="idOgmScdnShp">
  <h2 id="idOgmScdnShpH2">ogm.specs-div.shape
    <a class="clsHide" href="#idOgmScdnShpH2"></a></h2>
  <p id="idOgmScdnShpnam">name::
    <br>* McsEngl.ogm.specifics-division.shape,
    <a class="clsHide" href="#idOgmScdnShpnam"></a></p>
  <p id="idOgmScdnShpdsn">description::
    <br>* symmetric-ogm:
    <br>* symmetricNo-ogm:
    <br>** bilateral-ogm,
    <br>** biradial-ogm,
    <br>** icosahedral-ogm,
    <br>** radial-ogm,
    <br>** spherical-ogm,
    <a class="clsHide" href="#idOgmScdnShpdsn"></a></p>
  </section>
</section>

<section id="idOgm021">
  <h1 id="idOgm021H1">ogm.aggregate-021
    <a class="clsHide" href="#idOgm021H1"></a></h1>
  <p id="idOgm021nam">name::
    <br>* McsEngl.ogm.021-aggregate,
    <br>* McsEngl.ogm.aggregate-021,
    <a class="clsHide" href="#idOgm021nam"></a></p>
  <p id="idOgm021dsn">description::
    <br>· all the-organisms past and present.
    <a class="clsHide" href="#idOgm021dsn"></a></p>
</section>

<section id="idOgm006">
  <h1 id="idOgm006H1">ogm.generic-006
    <a class="clsHide" href="#idOgm006H1"></a></h1>
  <p id="idOgm006dsn">description::
    <br>· generic-organism is an-organism with <a class="clsPreview" href="../dirCor/McsCor000002.last.html#idMcsattWhlNSpc">specifics</a>.
    <a class="clsHide" href="#idOgm006dsn"></a></p>
  <p id="idOgm006nam">name::
    <br>* McsEngl.ogm.006-generic!⇒ogmGnrc,
    <br>* McsEngl.ogm.generic-006!⇒ogmGnrc,
    <br>* McsEngl.generic-organism!⇒ogmGnrc,
    <br>* McsEngl.ogmGnrc,
    <a class="clsHide" href="#idOgm006nam"></a></p>

  <section id="idOgm006Agg">
  <h2 id="idOgm006AggH2">ogmGnrc.aggregate
    <a class="clsHide" href="#idOgm006AggH2"></a></h2>
  <p id="idOgm006Aggdsn">description::
    <br>"Some 1.9 million species have been identified and described, out of some 8.7 million that may actually exist.[1] Millions more have become extinct."
    <br>[{2019-12-20} https://en.wikipedia.org/wiki/Species_description]
    <a class="clsHide" href="#idOgm006Aggdsn"></a></p>
  <p id="idOgm006Aggnam">name::
    <br>* McsEngl.ogmGnrc.aggregate,
    <a class="clsHide" href="#idOgm006Aggnam"></a></p>
  </section>
</section>

<section id="idOgm007">
  <h1 id="idOgm007H1">ogm.genericNo-007
    <a class="clsHide" href="#idOgm007H1"></a></h1>
  <p id="idOgm007dsn">description::
    <br>· individual-ogm is an-organism without specifics.
    <a class="clsHide" href="#idOgm007dsn"></a></p>
  <p id="idOgm007nam">name::
    <br>* McsEngl.ogm.007-genericNo,
    <br>* McsEngl.ogm.genericNo-007,
    <br>* McsEngl.individual-ogm,
    <a class="clsHide" href="#idOgm007nam"></a></p>
</section>

<section id="idOgm008">
  <h1 id="idOgm008H1">ogm.solitary-008
    <a class="clsHide" href="#idOgm008H1"></a></h1>
  <p id="idOgm008dsn">description::
    <br>"a solitary organism is one in which all individuals live independently and have all of the functions needed to survive and reproduce."
    <br>[{2019-12-27} https://en.wikipedia.org/wiki/Colony_(biology)]
    <a class="clsHide" href="#idOgm008dsn"></a></p>
  <p id="idOgm008nam">name::
    <br>* McsEngl.ogm.008-solitary,
    <br>* McsEngl.ogm.solitary-008,
    <br>* McsEngl.solitary-ogm,
    <a class="clsHide" href="#idOgm008nam"></a></p>
</section>

<section id="idOgm009">
  <h1 id="idOgm009H1">ogm.solitaryNo-009
    <a class="clsHide" href="#idOgm009H1"></a></h1>
  <p id="idOgm009dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm009dsn"></a></p>
  <p id="idOgm009nam">name::
    <br>* McsEngl.ogm.009-solitaryNo,
    <br>* McsEngl.ogm.solitaryNo-009,
    <br>* McsEngl.social-ogm,
    <br>* McsEngl.solitaryNo-ogm,
    <a class="clsHide" href="#idOgm009nam"></a></p>
</section>

<section id="idOgm010">
  <h1 id="idOgm010H1">ogm.cell-010
    <a class="clsHide" href="#idOgm010H1"></a></h1>
  <p id="idOgm010dsn">description::
    <br>· cellular-ogm is an-organism with cells.
    <a class="clsHide" href="#idOgm010dsn"></a></p>
  <p id="idOgm010nam">name::
    <br>* McsEngl.ogm.010-cell,
    <br>* McsEngl.ogm.cell-010,
    <br>* McsEngl.cellular-ogm,
    <a class="clsHide" href="#idOgm010nam"></a></p>
</section>

<section id="idOgm011">
  <h1 id="idOgm011H1">ogm.cellNo-011 (<a class="clsPreview" href="McsOgm000008.last.html#idOverview">link</a>)
    <a class="clsHide" href="#idOgm011H1"></a></h1>
</section>

<section id="idOgm003">
  <h1 id="idOgm003H1">ogm.cellOne-003 (<a class="clsPreview" href="McsOgm000007.last.html#idOgm003">link</a>)
    <a class="clsHide" href="#idOgm003H1"></a></h1>
</section>

<section id="idOgm012">
  <h1 id="idOgm012H1">ogm.cellMany-012
    <a class="clsHide" href="#idOgm012H1"></a></h1>
  <p id="idOgm012nam">name::
    <br>* McsEngl.ogm.012-cellMany!⇒ogmCellMany,
    <br>* McsEngl.ogm.cellMany-012!⇒ogmCellMany,
    <br>* McsEngl.multicellular-organism!⇒ogmCellMany,
    <br>* McsEngl.ogmCellMany,
    <a class="clsHide" href="#idOgm012nam"></a></p>
  <p id="idOgm012dsn">description::
    <br>"Multicellular organisms are organisms that consist of more than one cell, in contrast to unicellular organisms.[1]
    <br>All species of animals, land plants and most fungi are multicellular, as are many algae, whereas a few organisms are partially uni- and partially multicellular, like slime molds and social amoebae such as the genus Dictyostelium.[2][3]
    <br>Multicellular organisms arise in various ways, for example by cell division or by aggregation of many single cells.[4][3] Colonial organisms are the result of many identical individuals joining together to form a colony. However, it can often be hard to separate colonial protists from true multicellular organisms, because the two concepts are not distinct; colonial protists have been dubbed "pluricellular" rather than "multicellular".[5][6]"
    <br>[{2020-03-12} https://en.wikipedia.org/wiki/Multicellular_organism]
    <a class="clsHide" href="#idOgm012dsn"></a></p>

  <section id="idOgm012att001">
  <h2 id="idOgm012att001H2">01_disease of ogmCellMany
    <a class="clsHide" href="#idOgm012att001H2"></a></h2>
  <p id="idOgm012att001nam">name::
    <br>* McsEngl.ogmCellMany'01_disease,
    <br>* McsEngl.ogmCellMany'att001-disease,
    <br>* McsEngl.ogmCellMany'disease,
    <a class="clsHide" href="#idOgm012att001nam"></a></p>
  <p id="idOgm012att001dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm012att001dsn"></a></p>
  </section>

  <section id="idOgm012att002">
  <h2 id="idOgm012att002H2">02_governance-system of ogmCellMany
    <a class="clsHide" href="#idOgm012att002H2"></a></h2>
  <p id="idOgm012att002nam">name::
    <br>* McsEngl.ogmCellMany'02_governance-system,
    <br>* McsEngl.ogmCellMany'att002-governance-system,
    <br>* McsEngl.ogmCellMany'governance-system,
    <a class="clsHide" href="#idOgm012att002nam"></a></p>
  <p id="idOgm012att002dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm012att002dsn"></a></p>
  </section>

  <section id="idOgm012att003">
  <h2 id="idOgm012att003H2">03_health of ogmCellMany
    <a class="clsHide" href="#idOgm012att003H2"></a></h2>
  <p id="idOgm012att003nam">name::
    <br>* McsEngl.ogmCellMany'03_health,
    <br>* McsEngl.ogmCellMany'att003-health,
    <br>* McsEngl.ogmCellMany'health,
    <a class="clsHide" href="#idOgm012att003nam"></a></p>
  <p id="idOgm012att003dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm012att003dsn"></a></p>
  </section>

  <section id="idOgm012att004">
  <h2 id="idOgm012att004H2">04_tissue of ogmCellMany
    <a class="clsHide" href="#idOgm012att004H2"></a></h2>
  <p id="idOgm012att004nam">name::
    <br>* McsEngl.ogmCellMany'04_tissue!⇒tissueOgm,
    <br>* McsEngl.ogmCellMany'att004-tissue!⇒tissueOgm,
    <br>* McsEngl.ogmCellMany'tissue!⇒tissueOgm,
    <br>* McsEngl.tissueOgm,
    <a class="clsHide" href="#idOgm012att004nam"></a></p>
  <p id="idOgm012att004dsn">description::
    <br>"In biology, tissue is a cellular organizational level between cells and a complete organ. A tissue is an ensemble of similar cells and their extracellular matrix from the same origin that together carry out a specific function. Organs are then formed by the functional grouping together of multiple tissues.
    <br>The English word "tissue" is derived from the French word "tissu", meaning that something that is "woven", from the verb tisser, "to weave".
    <br>The study of human and animal tissues is known as histology or, in connection with disease, histopathology also not forgetting to add, archeology. For plants, the discipline is called plant anatomy. The classical tools for studying tissues are the paraffin block in which tissue is embedded and then sectioned, the histological stain, and the optical microscope. Developments in electron microscopy, immunofluorescence, and the use of frozen tissue sections have enhanced the detail that can be observed in tissues. With these tools, the classical appearances of tissues can be examined in health and disease, enabling considerable refinement of medical diagnosis and prognosis."
    <br>[{2020-04-23} https://en.wikipedia.org/wiki/Tissue_(biology)]
    <a class="clsHide" href="#idOgm012att004dsn"></a></p>

  <section id="idOgm012att004tseng">
  <h3 id="idOgm012att004tsengH3">engineering of tissueOgm
    <a class="clsHide" href="#idOgm012att004tsengH3"></a></h3>
  <p id="idOgm012att004tsengnam">name::
    <br>* McsEngl.tissueOgm'engineering,
    <br>* McsEngl.tissue-engineering,
    <br>* McsEngl.regenerative-medicine,
    <a class="clsHide" href="#idOgm012att004tsengnam"></a></p>
  <p id="idOgm012att004tsengdsn">description::
    <br>"Tissue engineering is the use of a combination of cells, engineering, and materials methods, and suitable biochemical and physicochemical factors to improve or replace biological tissues. Tissue engineering involves the use of a tissue scaffold for the formation of new viable tissue for a medical purpose. While it was once categorized as a sub-field of biomaterials, having grown in scope and importance it can be considered as a field in its own.
    <br>While most definitions of tissue engineering cover a broad range of applications, in practice the term is closely associated with applications that repair or replace portions of or whole tissues (i.e., bone, cartilage,[1] blood vessels, bladder, skin, muscle etc.). Often, the tissues involved require certain mechanical and structural properties for proper functioning. The term has also been applied to efforts to perform specific biochemical functions using cells within an artificially-created support system (e.g. an artificial pancreas, or a bio artificial liver). The term regenerative medicine is often used synonymously with tissue engineering, although those involved in regenerative medicine place more emphasis on the use of stem cells or progenitor cells to produce tissues."
    <br>[{2020-05-06} https://en.wikipedia.org/wiki/Tissue_engineering]
    <a class="clsHide" href="#idOgm012att004tsengdsn"></a></p>
  </section>
  </section>
</section>

<section id="idOgm001">
  <h1 id="idOgm001H1">ogm.eukaryote-001
    <a class="clsHide" href="#idOgm001H1"></a></h1>
  <p id="idOgm001nam">name::
    <br>* McsEngl.ogm.001-eukaryote, /yyukáriot/,
    <br>* McsEngl.ogm.eukaryote-001,
    <br>* McsEngl.eukaryote-ogm-001,
    <a class="clsHide" href="#idOgm001nam"></a></p>
  <p id="idOgm001dsn">description::
    <br>"Eukaryotes (/juːˈkærioʊt, -ət/) are organisms whose cells have a nucleus enclosed within membranes, unlike prokaryotes (Bacteria and Archaea), which have no membrane-bound organelles.[3][4][5] Eukaryotes belong to the domain Eukaryota or Eukarya. Their name comes from the Greek εὖ (eu, "well" or "true") and κάρυον (karyon, "nut" or "kernel").[6] Eukaryotic cells typically contain other membrane-bound organelles such as mitochondria and the Golgi apparatus, and in addition, some cells of plants and algae contain chloroplasts. Unlike unicellular archaea and bacteria, eukaryotes may also be multicellular and include organisms consisting of many cell types forming different kinds of tissue. Animals and plants are the most familiar eukaryotes.
    <br>Eukaryotes can reproduce both asexually through mitosis and sexually through meiosis and gamete fusion. In mitosis, one cell divides to produce two genetically identical cells. In meiosis, DNA replication is followed by two rounds of cell division to produce four haploid daughter cells. These act as sex cells (gametes). Each gamete has just one set of chromosomes, each a unique mix of the corresponding pair of parental chromosomes resulting from genetic recombination during meiosis.[citation needed]
    <br>The domain Eukaryota is monophyletic and makes up one of the domains of life in the three-domain system. The two other domains, Bacteria and Archaea, are prokaryotes[7] and have none of the above features. Eukaryotes represent a tiny minority of all living things.[8] However, due to their generally much larger size, their collective worldwide biomass is estimated to be about equal to that of prokaryotes.[8] Eukaryotes evolved approximately 1.6–2.1 billion years ago, during the Proterozoic eon."
    <br>[{2020-03-12} https://en.wikipedia.org/wiki/Eukaryote]
    <a class="clsHide" href="#idOgm001dsn"></a></p>
</section>

<section id="idOgm002">
  <h1 id="idOgm002H1">ogm.prokaryote-002 (<a class="clsPreview" href="McsOgm000007.last.html#idCellOgm002">link</a>)
    <a class="clsHide" href="#idOgm002H1"></a></h1>
</section>

<section id="idOgm013">
  <h1 id="idOgm013H1">ogm.microscopic-013
    <a class="clsHide" href="#idOgm013H1"></a></h1>
  <p id="idOgm013dsn">description::
    <br>· microorganism is an-organism we need a-MICROSCOPE to see it.
    <a class="clsHide" href="#idOgm013dsn"></a></p>
  <p id="idOgm013nam">name::
    <br>* McsEngl.ogm.013-microscopic!⇒ogmMicro,
    <br>* McsEngl.ogm.microscopic-013!⇒ogmMicro,
    <br>* McsEngl.microorganism!⇒ogmMicro,
    <br>* McsEngl.microscopic-ogm!⇒ogmMicro,
    <br>* McsEngl.ogmMicro,
    <a class="clsHide" href="#idOgm013nam"></a></p>

  <section id="idOgm013gst">
  <h2 id="idOgm013gstH2">GENERIC-SPECIFIC-TREE of ogmMicro
    <a class="clsHide" href="#idOgm013gstH2"></a></h2>
  <p id="idOgm013gstnam">name::
    <br>* McsEngl.ogmMicro'generic-specific-tree,
    <a class="clsHide" href="#idOgm013gstnam"></a></p>

  <section id="idOgm013gtr">
  <h3 id="idOgm013gtrH3">GENERIC-TREE of ogmMicro
    <a class="clsHide" href="#idOgm013gtrH3"></a></h3>
  <p id="idOgm013gnc">generic-of-ogmMicro::
    <br>*
    <a class="clsHide" href="#idOgm013gnc"></a></p>
  <p id="idOgm013atttr">attribute-tree-of-ogmMicro::
    <br>* ,
    <a class="clsHide" href="#idOgm013atttr"></a></p>
  <p id="idOgm013attihf">att-tree-inherited-from::
    <br>· :
    <br>* ,
    <a class="clsHide" href="#idOgm013attihf"></a></p>
  <p id="idOgm013attown">att-tree-own-of-ogmMicro::
    <br>* ,
    <a class="clsHide" href="#idOgm013attown"></a></p>
  </section>

  <section id="idOgm013str">
  <h3 id="idOgm013strH3">SPECIFIC-TREE of ogmMicro
    <a class="clsHide" href="#idOgm013strH3"></a></h3>
  <p id="idOgm013spc">specific-of-ogmMicro::
    <br>* bacterium-ogmMicro,
    <br>* cellNo-ogmMicro,
    <br>* cellOne-ogmMicro,
    <br>* fungus-ogmMicro,
    <br>* strain-ogmMicro,
    <br>* virus-ogmMicro,
    <a class="clsHide" href="#idOgm013spc"></a></p>
  </section>
  </section>

  <section id="idOgm013Strn">
  <h2 id="idOgm013StrnH2">ogmMicro.strain
    <a class="clsHide" href="#idOgm013StrnH2"></a></h2>
  <p id="idOgm013Strnnam">name::
    <br>* McsEngl.ogmMicro.strain,
    <br>* McsEngl.strain-ogmMicro,
    <a class="clsHide" href="#idOgm013Strnnam"></a></p>
  <p id="idOgm013Strndsn">description::
    <br>"A strain is a genetic variant or subtype of a microorganism (e.g., virus or bacterium or fungus). For example, a "flu strain" is a certain biological form of the influenza or "flu" virus. These flu strains are characterized by their differing isoforms of surface proteins. New viral strains can be created due to mutation or swapping of genetic components when two or more viruses infect the same cell in nature.[2] These phenomena are known respectively as antigenic drift and antigenic shift. Microbial strains can also be differentiated by their genetic makeup using metagenomic methods to maximize resolution within species.[3] This has become a valuable tool to analyze the microbiome."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Strain_(biology)#Microbiology_or_virology]
    <a class="clsHide" href="#idOgm013Strndsn"></a></p>
  </section>
</section>

<section id="idOgm014">
  <h1 id="idOgm014H1">ogm.microscopicNo-014
    <a class="clsHide" href="#idOgm014H1"></a></h1>
  <p id="idOgm014dsn">description::
    <br>· microscopicNo-ogm is an-organism we do-not-need a-MICROSCOPE to see it.
    <a class="clsHide" href="#idOgm014dsn"></a></p>
  <p id="idOgm014nam">name::
    <br>* McsEngl.ogm.014-microscopicNo,
    <br>* McsEngl.ogm.microscopicNo-014,
    <br>* McsEngl.microscopicNo-ogm,
    <a class="clsHide" href="#idOgm014nam"></a></p>
</section>

<section id="idOgm015">
  <h1 id="idOgm015H1">ogm.free-015
    <a class="clsHide" href="#idOgm015H1"></a></h1>
  <p id="idOgm015dsn">description::
    <br>· free-ogm is an-organism which is-not-depends on another one to function.
    <a class="clsHide" href="#idOgm015dsn"></a></p>
  <p id="idOgm015nam">name::
    <br>* McsEngl.ogm.015-free,
    <br>* McsEngl.ogm.free-015,
    <br>* McsEngl.freeOgm-015,
    <a class="clsHide" href="#idOgm015nam"></a></p>
</section>

<section id="idOgm016">
  <h1 id="idOgm016H1">ogm.freeNo-016
    <a class="clsHide" href="#idOgm016H1"></a></h1>
  <p id="idOgm016dsn">description::
    <br>· freeNo-ogm is an-organism which depends on another one to function.
    <a class="clsHide" href="#idOgm016dsn"></a></p>
  <p id="idOgm016nam">name::
    <br>* McsEngl.ogm.016-freeNo,
    <br>* McsEngl.ogm.freeNo-016,
    <br>* McsEngl.freeNoOgm-016,
    <a class="clsHide" href="#idOgm016nam"></a></p>
</section>

<section id="idOgm017">
  <h1 id="idOgm017H1">ogm.time.present-017 (extant)
    <a class="clsHide" href="#idOgm017H1"></a></h1>
  <p id="idOgm017dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm017dsn"></a></p>
  <p id="idOgm017nam">name::
    <br>* McsEngl.ogm.017-extant,
    <br>* McsEngl.ogm.extant-017,
    <br>* McsEngl.extant-ogm,
    <br>* McsEngl.ogm.living,
    <br>* McsEngl.ogm.time.present,
    <a class="clsHide" href="#idOgm017nam"></a></p>
</section>

<section id="idOgm018">
  <h1 id="idOgm018H1">ogm.time.past-018 (extinct)
    <a class="clsHide" href="#idOgm018H1"></a></h1>
  <p id="idOgm018dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm018dsn"></a></p>
  <p id="idOgm018nam">name::
    <br>* McsEngl.ogm.018-extinct,
    <br>* McsEngl.ogm.extinct-018,
    <br>* McsEngl.extinct-ogm,
    <br>* McsEngl.ogm.livingNo,
    <br>* McsEngl.ogm.time.past,
    <a class="clsHide" href="#idOgm018nam"></a></p>
</section>

<section id="idOgm019">
  <h1 id="idOgm019H1">ogm.natural-019
    <a class="clsHide" href="#idOgm019H1"></a></h1>
  <p id="idOgm019nam">name::
    <br>* McsEngl.ogm.019-natural,
    <br>* McsEngl.ogm.natural-019,
    <br>* McsEngl.natural-ogm-019,
    <a class="clsHide" href="#idOgm019nam"></a></p>
  <p id="idOgm019dsn">description::
    <br>· natural-organism is an-organism created by Nature, evolutionarily.
    <a class="clsHide" href="#idOgm019dsn"></a></p>
</section>

<section id="idOgm020">
  <h1 id="idOgm020H1">ogm.naturalNo-020
    <a class="clsHide" href="#idOgm020H1"></a></h1>
  <p id="idOgm020nam">name::
    <br>* McsEngl.naturalNo-ogm-020!⇒ogmSynth,
    <br>* McsEngl.ogmSynth,
    <br>* McsEngl.ogm.020-naturalNo!⇒ogmSynth,
    <br>* McsEngl.ogm.naturalNo-020!⇒ogmSynth,
    <br>* McsEngl.synthetic-ogm-020!⇒ogmSynth,
    <a class="clsHide" href="#idOgm020nam"></a></p>
  <p id="idOgm020dsn">description::
    <br>· naturalNo-organism is an-organism created by bio-molecules, NOT machine.
    <a class="clsHide" href="#idOgm020dsn"></a></p>

  <section id="idOgm020001">
  <h2 id="idOgm020001H2">ogmSynth.
    <a class="clsHide" href="#idOgm020001H2"></a></h2>
  <p id="idOgm020001dsn">description::
    <br>"Xenobots, named after the African clawed frog (Xenopus laevis), are synthetic organisms that are automatically designed by computers to perform some desired function and built by combining together different biological tissues.[1][2][3][4][5][6]
    <br>Xenobots are less than a 1 millimeter (0.039 inches) wide and composed of just two things: skin cells and heart muscle cells, both of which are derived from stem cells harvested from early (blastula stage) frog embryos.[7] The skin cells provide rigid support and the heart cells act as small motors, contracting and expanding in volume to propel the xenobot forward. The shape of a xenobot's body, and its distribution of skin and heart cells, are automatically designed in simulation to perform a specific task, using a process of trial and error (an evolutionary algorithm). Xenobots have been designed to walk, swim, push pellets, carry payloads, and work together in a swarm to aggregate debris scattered along the surface of their dish into neat piles. They can survive for weeks without food and heal themselves after lacerations.[1]"
    <br>[{2020-08-10} https://en.wikipedia.org/wiki/Xenobot]
    <a class="clsHide" href="#idOgm020001dsn"></a></p>
  <p id="idOgm020001nam">name::
    <br>* McsEngl.ogmSynth.zenobot!⇒ogmXenobot,
    <br>* McsEngl.ogmXenobot,
    <br>* McsEngl.zenobot!⇒ogmXenobot,
    <a class="clsHide" href="#idOgm020001nam"></a></p>
  </section>
</section>

<section id="idOgm023">
  <h1 id="idOgm023H1">ogm.descendant-023
    <a class="clsHide" href="#idOgm023H1"></a></h1>
  <p id="idOgm023nam">name::
    <br>* McsEngl.ogm.023-descendant,
    <br>* McsEngl.ogm.descendant-023,
    <br>* McsEngl.descendant-ogm-023,
    <br>* McsEngl.offspring-ogm-023,
    <br>* McsEngl.progeny-ogm-023,
    <br>====== langoGreek:
    <br>* McsElln.απόγονος-οργανισμός!=ogmDescendant,
    <a class="clsHide" href="#idOgm023nam"></a></p>
  <p id="idOgm023dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm023dsn"></a></p>
</section>

<section id="idOgm024">
  <h1 id="idOgm024H1">ogm.antecedent-024
    <a class="clsHide" href="#idOgm024H1"></a></h1>
  <p id="idOgm024nam">name::
    <br>* McsEngl.ogm.024-ancestor,
    <br>* McsEngl.ogm.ancestor-024,
    <br>* McsEngl.ancestor-ogm-024,
    <br>* McsEngl.antecedent-ogm-024,
    <br>* McsEngl.forebear-ogm-023,
    <br>* McsEngl.forefather-ogm-023,
    <br>* McsEngl.predecessor-ogm-023,
    <br>* McsEngl.primogenitor-ogm-023,
    <br>* McsEngl.progenitor-ogm-024,
    <br>====== langoGreek:
    <br>* McsElln.πρόγονος-οργανισμός!=ogmAncestor,
    <a class="clsHide" href="#idOgm024nam"></a></p>
  <p id="idOgm024dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm024dsn"></a></p>
</section>

<section id="idOgm025">
  <h1 id="idOgm025H1">ogm.alga-025
    <a class="clsHide" href="#idOgm025H1"></a></h1>
  <p id="idOgm025nam">name::
    <br>* McsEngl.ogm.025-alga!⇒ogmAlga,
    <br>* McsEngl.ogm.alga-025!⇒ogmAlga,
    <br>* McsEngl.algae!⇒ogmAlga,
    <br>* McsEngl.ogmAlga,
    <a class="clsHide" href="#idOgm025nam"></a></p>
  <p id="idOgm025dsn">description::
    <br>"Algae vs. Plants
    <br>Plants and algae are both photosynthetic. Both are also considered eukaryotes, consisting of cells with specialized components. They both also have the same life cycle called alternation of generations. However, algae are not plants. So, what are they? They are merely members of the Kingdom Protista. Plants compose their own kingdom, Kingdom Plantae. While plants and algae may sometimes appear to be quite similar visually, they in fact have a number of differences between them. In terms of where they live, how they survive and reproduce, and what composes them, plants and algae are vastly different.
    <br>Did you know that seaweed is not a plant? First of all, algae may be unicellular, colonial, or multi-cellular. Plants, on the other hand, are only multi-cellular. Holdfasts, stapes and blades compose multi-cellular algae. In comparison, plants have roots, stems, leaves, flowers, fruits, seeds and cones. The roots of plants not only hold them in place, they nourish them. Plants possess vascular systems, which allow for the uptake and transport of water and nutrients. In contrast, each cell in algae must obtain its own nutrients from water for survival.
    <br>Clearly, plants cannot move, as they are rooted to the ground. On some algae, holdfasts, which are comparable to the roots of plants, hold them in place. Some algae drift with the water currents. Some algae are actually actively mobile. Dinoflagellates, for instance, whip themselves through the water with a tail-like structures called flagella. Other algae may move by pushing their bodies forward in a crawling motion.
    <br>Typically algae are found in water; although, they may be found on land or snow and, strangely enough, even growing in rocks or marine animals or on the fur of some rainforest animals such as sloth. Plants are generally found on land; however, they can also live in water, such as eelgrass in marine systems and water lilies in fresh water.
    <br>Reproduction could not be more different for plants and algae. Plants have complex, multi-cellular reproductive systems and some even require the assistance of wind, birds, or bugs for pollination. Algae, comparatively, can reproduce through tiny spores or even by replication or the growth of broken pieces.
    <br>Despite all of their differences, algae and plants can often appear deceptively similar. So, next time you’re on the beach and you come across what appears to be a plant, take a second glance because it may in fact be algae.
    <br>~Elizabeth Gooding"
    <br>[{2020-04-16} http://simply-science-nbep.blogspot.com/2011/06/algae-vs-plants.html]
    <a class="clsHide" href="#idOgm025dsn"></a></p>
</section>

<section id="idOgm028">
  <h1 id="idOgm028H1">ogm.autotroph-028
    <a class="clsHide" href="#idOgm028H1"></a></h1>
  <p id="idOgm028nam">name::
    <br>* McsEngl.ogm.028-autotroph,
    <br>* McsEngl.ogm.autotroph-028,
    <br>* McsEngl.autotroph-ogm-028,
    <a class="clsHide" href="#idOgm028nam"></a></p>
  <p id="idOgm028dsn">description::
    <br>"An autotroph or primary producer is an organism that produces complex organic compounds (such as carbohydrates, fats, and proteins) using carbon from simple substances such as carbon dioxide,[1] generally using energy from light (photosynthesis) or inorganic chemical reactions (chemosynthesis).[2] Autotrophs do not need a living source of carbon or energy and are the producers in a food chain, such as plants on land or algae in water (in contrast to heterotrophs as consumers of autotrophs or other heterotrophs). Autotrophs can reduce carbon dioxide to make organic compounds for biosynthesis and as stored chemical fuel. Most autotrophs use water as the reducing agent, but some can use other hydrogen compounds such as hydrogen sulfide.
    <br>Some autotrophs, such as green plants and algae, are phototrophs, meaning that they convert electromagnetic energy from sunlight into chemical energy in the form of glucose. Others, including methanogens, are chemotrophs, which use organic or inorganic chemical compounds as a source of energy. Most chemoautotrophs are lithotrophs, using inorganic electron donors such as hydrogen sulfide, hydrogen gas, elemental sulfur, ammonium and ferrous oxide as reducing agents and hydrogen sources for biosynthesis and chemical energy release. Autotrophs use a portion of the ATP produced during photosynthesis or the oxidation of chemical compounds to reduce NADP+ to NADPH to form organic compounds.[3]"
    <br>[{2020-04-17} https://en.wikipedia.org/wiki/Autotroph]
    <a class="clsHide" href="#idOgm028dsn"></a></p>
</section>

<section id="idOgm029">
  <h1 id="idOgm029H1">ogm.heterotroph-029
    <a class="clsHide" href="#idOgm029H1"></a></h1>
  <p id="idOgm029nam">name::
    <br>* McsEngl.ogm.029-heterotroph,
    <br>* McsEngl.ogm.heterotroph-029,
    <br>* McsEngl.heterotroph-ogm-029,
    <a class="clsHide" href="#idOgm029nam"></a></p>
  <p id="idOgm029dsn">description::
    <br>"A heterotroph (/ˈhɛtərəˌtroʊf, -ˌtrɒf/;[1] Ancient Greek ἕτερος héteros = "other" plus trophe = "nutrition") is an organism that cannot produce its own food, instead taking nutrition from other sources of organic carbon, mainly plant or animal matter. In the food chain, heterotrophs are primary, secondary and tertiary consumers, but not producers.[2][3] Living organisms that are heterotrophic include all animals and fungi, some bacteria and protists,[4] and many parasitic plants. The term heterotroph arose in microbiology in 1946 as part of a classification of microorganisms based on their type of nutrition.[5] The term is now used in many fields, such as ecology in describing the food chain.
    <br>Heterotrophs may be subdivided according to their energy source. If the heterotroph uses chemical energy, it is a chemoheterotroph (e.g., humans and mushrooms). If it uses light for energy, then it is a photoheterotroph (e.g., green non-sulfur bacteria).
    <br>Heterotrophs represent one of the two mechanisms of nutrition (trophic levels), the other being autotrophs (auto = self, troph = nutrition). Autotrophs use energy from sunlight (photoautotrophs) or oxidation of inorganic compounds (lithoautotrophs) to convert inorganic carbon dioxide to organic carbon compounds and energy to sustain their life. Comparing the two in basic terms, heterotrophs (such as animals) eat either autotrophs (such as plants) or other heterotrophs, or both.
    <br>Detritivores are heterotrophs which obtain nutrients by consuming detritus (decomposing plant and animal parts as well as feces).[6] Saprotrophs (also called lysotrophs) are chemoheterotrophs that use extracellular digestion in processing decayed organic matter; the term most often used to describe fungi. The process is most often facilitated through the active transport of such materials through endocytosis within the internal mycelium and its constituent hyphae.[7]"
    <br>[{2020-04-17} https://en.wikipedia.org/wiki/Heterotroph]
    <a class="clsHide" href="#idOgm029dsn"></a></p>
</section>

<section id="idOgm026">
  <h1 id="idOgm026H1">ogm.photoautotroph-026
    <a class="clsHide" href="#idOgm026H1"></a></h1>
  <p id="idOgm026nam">name::
    <br>* McsEngl.ogm.026-photoautotroph,
    <br>* McsEngl.ogm.photoautotroph-026,
    <br>* McsEngl.photoautotroph-ogm-026,
    <a class="clsHide" href="#idOgm026nam"></a></p>
  <p id="idOgm026dsn">description::
    <br>"Most of the well-recognized phototrophs are autotrophic, also known as photoautotrophs, and can fix carbon. They can be contrasted with chemotrophs that obtain their energy by the oxidation of electron donors in their environments. Photoautotrophs are capable of synthesizing their own food from inorganic substances using light as an energy source. Green plants and photosynthetic bacteria are photoautotrophs. Photoautotrophic organisms are sometimes referred to as holophytic.[3] Such organisms derive their energy for food synthesis from light and are capable of using carbon dioxide as their principal source of carbon.
    <br>Oxygenic photosynthetic organisms use chlorophyll for light-energy capture and oxidize water, "splitting" it into molecular oxygen. In contrast, anoxygenic photosynthetic bacteria have a substance called bacteriochlorophyll – which absorbs predominantly at non-optical wavelengths – for light-energy capture, live in aquatic environments, and will, using light, oxidize chemical substances such as hydrogen sulfide rather than water."
    <br>[{2020-04-17} https://en.wikipedia.org/wiki/Phototroph#Photoautotroph]
    <a class="clsHide" href="#idOgm026dsn"></a></p>
</section>

<section id="idOgm027">
  <h1 id="idOgm027H1">ogm.photoheterotroph-027
    <a class="clsHide" href="#idOgm027H1"></a></h1>
  <p id="idOgm027nam">name::
    <br>* McsEngl.ogm.027-photoheterotroph,
    <br>* McsEngl.ogm.photoheterotroph-027,
    <br>* McsEngl.photoheterotroph-ogm-027,
    <a class="clsHide" href="#idOgm027nam"></a></p>
  <p id="idOgm027dsn">description::
    <br>"Photoheterotrophs (Gk: photo = light, hetero = (an)other, troph = nourishment) are heterotrophic phototrophs – that is, they are organisms that use light for energy, but cannot use carbon dioxide as their sole carbon source. Consequently, they use organic compounds from the environment to satisfy their carbon requirements; these compounds include carbohydrates, fatty acids, and alcohols. Examples of photoheterotrophic organisms include purple non-sulfur bacteria, green non-sulfur bacteria, and heliobacteria.[1] Recent research has indicated that the oriental hornet and some aphids may be able to use light to supplement their energy supply.[2]"
    <br>[{2020-04-17} https://en.wikipedia.org/wiki/Photoheterotroph]
    <a class="clsHide" href="#idOgm027dsn"></a></p>
</section>

<section id="idOgm030">
  <h1 id="idOgm030H1">ogm.neural-030
    <a class="clsHide" href="#idOgm030H1"></a></h1>
  <p id="idOgm030nam">name::
    <br>* McsEngl.ogm.030-neural!⇒ogmNeural,
    <br>* McsEngl.ogm.neural-030!⇒ogmNeural,
    <br>* McsEngl.neural-ogm-030!⇒ogmNeural,
    <br>* McsEngl.ogmNeural,
    <a class="clsHide" href="#idOgm030nam"></a></p>
  <p id="idOgm030dsn">description::
    <br>· ogmNeural is an-organism with neurons.
    <br>===
    <br>"It[neuron] is the main component of nervous tissue in all animals except sponges and placozoa. Plants and fungi do not have nerve cells."
    <br>[{2020-04-24} https://en.wikipedia.org/wiki/Neuron]
    <a class="clsHide" href="#idOgm030dsn"></a></p>

  <section id="idOgm030att001">
  <h2 id="idOgm030att001H2">01_disease of ogmNeural
    <a class="clsHide" href="#idOgm030att001H2"></a></h2>
  <p id="idOgm030att001nam">name::
    <br>* McsEngl.ogmNeural'01_disease,
    <br>* McsEngl.ogmNeural'att001-disease,
    <br>* McsEngl.ogmNeural'disease-att001,
    <a class="clsHide" href="#idOgm030att001nam"></a></p>
  <p id="idOgm030att001dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att001dsn"></a></p>
  </section>

  <section id="idOgm030att002">
  <h2 id="idOgm030att002H2">02_governance-sys of ogmNeural
    <a class="clsHide" href="#idOgm030att002H2"></a></h2>
  <p id="idOgm030att002nam">name::
    <br>* McsEngl.ogmNeural'02_governance-sys,
    <br>* McsEngl.ogmNeural'att002-governance-sys,
    <br>* McsEngl.ogmNeural'governance-sys-att002,
    <a class="clsHide" href="#idOgm030att002nam"></a></p>
  <p id="idOgm030att002dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att002dsn"></a></p>
  </section>

  <section id="idOgm030att003">
  <h2 id="idOgm030att003H2">03_health of ogmNeural
    <a class="clsHide" href="#idOgm030att003H2"></a></h2>
  <p id="idOgm030att003nam">name::
    <br>* McsEngl.ogmNeural'03_health,
    <br>* McsEngl.ogmNeural'att003-health,
    <br>* McsEngl.ogmNeural'health-att003,
    <a class="clsHide" href="#idOgm030att003nam"></a></p>
  <p id="idOgm030att003dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att003dsn"></a></p>
  </section>

  <section id="idOgm030att004">
  <h2 id="idOgm030att004H2">04_organ-sys of ogmNeural
    <a class="clsHide" href="#idOgm030att004H2"></a></h2>
  <p id="idOgm030att004nam">name::
    <br>* McsEngl.ogmNeural'04_organ-sys,
    <br>* McsEngl.ogmNeural'att004-organ-sys,
    <br>* McsEngl.ogmNeural'organ-sys-att004,
    <a class="clsHide" href="#idOgm030att004nam"></a></p>
  <p id="idOgm030att004dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att004dsn"></a></p>
  </section>

  <section id="idOgm030att005">
  <h2 id="idOgm030att005H2">05_organ of ogmNeural
    <a class="clsHide" href="#idOgm030att005H2"></a></h2>
  <p id="idOgm030att005nam">name::
    <br>* McsEngl.ogmNeural'05_organ,
    <br>* McsEngl.ogmNeural'att005-organ,
    <br>* McsEngl.ogmNeural'organ-att005,
    <a class="clsHide" href="#idOgm030att005nam"></a></p>
  <p id="idOgm030att005dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att005dsn"></a></p>
  </section>

  <section id="idOgm030att006">
  <h2 id="idOgm030att006H2">06_tissue of ogmNeural
    <a class="clsHide" href="#idOgm030att006H2"></a></h2>
  <p id="idOgm030att006nam">name::
    <br>* McsEngl.ogmNeural'06_tissue,
    <br>* McsEngl.ogmNeural'att006-tissue,
    <br>* McsEngl.ogmNeural'tissue-att006,
    <a class="clsHide" href="#idOgm030att006nam"></a></p>
  <p id="idOgm030att006dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att006dsn"></a></p>
  </section>

  <section id="idOgm030att007">
  <h2 id="idOgm030att007H2">07_cell of ogmNeural
    <a class="clsHide" href="#idOgm030att007H2"></a></h2>
  <p id="idOgm030att007nam">name::
    <br>* McsEngl.ogmNeural'07_cell,
    <br>* McsEngl.ogmNeural'att007-cell,
    <br>* McsEngl.ogmNeural'cell-att007,
    <a class="clsHide" href="#idOgm030att007nam"></a></p>
  <p id="idOgm030att007dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att007dsn"></a></p>
  </section>

  <section id="idOgm030att008">
  <h2 id="idOgm030att008H2">08_material-sys of ogmNeural
    <a class="clsHide" href="#idOgm030att008H2"></a></h2>
  <p id="idOgm030att008nam">name::
    <br>* McsEngl.ogmNeural'08_material-sys,
    <br>* McsEngl.ogmNeural'att008-material-sys,
    <br>* McsEngl.ogmNeural'material-sys-att008,
    <a class="clsHide" href="#idOgm030att008nam"></a></p>
  <p id="idOgm030att008dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att008dsn"></a></p>
  </section>

  <section id="idOgm030att009">
  <h2 id="idOgm030att009H2">09_material of ogmNeural
    <a class="clsHide" href="#idOgm030att009H2"></a></h2>
  <p id="idOgm030att009nam">name::
    <br>* McsEngl.ogmNeural'09_material,
    <br>* McsEngl.ogmNeural'att009-material,
    <br>* McsEngl.ogmNeural'material-att009,
    <a class="clsHide" href="#idOgm030att009nam"></a></p>
  <p id="idOgm030att009dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att009dsn"></a></p>
  </section>

  <section id="idOgm030att010">
  <h2 id="idOgm030att010H2">10_nervous-sys of ogmNeural
    <a class="clsHide" href="#idOgm030att010H2"></a></h2>
  <p id="idOgm030att010nam">name::
    <br>* McsEngl.ogmNeural'10_nervous-sys!⇒sysNervousOgm,
    <br>* McsEngl.ogmNeural'att010-nervous-sys!⇒sysNervousOgm,
    <br>* McsEngl.ogmNeural'nervous-sys-att010!⇒sysNervousOgm,
    <br>* McsEngl.nervous-sys-of-ogmNeural!⇒sysNervousOgm,
    <br>* McsEngl.sysNervousOgm,
    <br>====== langoGreek:
    <br>* McsElln.νευρικό-σύστημα-οργανισμού!=sysNervousOgm,
    <a class="clsHide" href="#idOgm030att010nam"></a></p>
  <p id="idOgm030att010dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att010dsn"></a></p>

  <section id="idOgm030att010att001">
  <h3 id="idOgm030att010att001H3">cellNeuronOgm (<a class="clsPreview" href="McsOgm000007.last.html#idCellOgm009">link</a>) of sysNervousOgm
    <a class="clsHide" href="#idOgm030att010att001H3"></a></h3>
  </section>

  <section id="idOgm030att010att002">
  <h3 id="idOgm030att010att002H3">ephaptic-coupling of sysNervousOgm
    <a class="clsHide" href="#idOgm030att010att002H3"></a></h3>
  <p id="idOgm030att010att002nam">name::
    <br>* McsEngl.sysNervousOgm'ephaptic-coupling,
    <br>* McsEngl.sysNervousOgm'att002-ephaptic-coupling,
    <br>* McsEngl.ephaptic-coupling-of-sysNervousOgm,
    <a class="clsHide" href="#idOgm030att010att002nam"></a></p>
  <p id="idOgm030att010att002dsn">description::
    <br>"Ephaptic coupling is a form of communication within the nervous system and is distinct from direct communication systems like electrical synapses and chemical synapses. It may refer to the coupling of adjacent (touching) nerve fibers caused by the exchange of ions between the cells, or it may refer to coupling of nerve fibers as a result of local electric fields.[1] In either case ephaptic coupling can influence the synchronization and timing of action potential firing in neurons. Myelination is thought to inhibit ephaptic interactions.[2]"
    <br>[{2020-05-08} https://en.wikipedia.org/wiki/Ephaptic_coupling]
    <a class="clsHide" href="#idOgm030att010att002dsn"></a></p>
  </section>

  <section id="idOgm030att010001">
  <h3 id="idOgm030att010001H3">sysNervousOgm.brain-001
    <a class="clsHide" href="#idOgm030att010001H3"></a></h3>
  <p id="idOgm030att010001nam">name::
    <br>* McsEngl.sysNervousOgm.001-brain!⇒sysNervousOgmBrain,
    <br>* McsEngl.sysNervousOgm.brain-001!⇒sysNervousOgmBrain,
    <br>* McsEngl.sysNervousOgmBrain,
    <a class="clsHide" href="#idOgm030att010001nam"></a></p>
  <p id="idOgm030att010001dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att010001dsn"></a></p>
  </section>

  <section id="idOgm030att010002">
  <h3 id="idOgm030att010002H3">sysNervousOgm.brainNo-002
    <a class="clsHide" href="#idOgm030att010002H3"></a></h3>
  <p id="idOgm030att010002nam">name::
    <br>* McsEngl.sysNervousOgm.002-brainNo!⇒sysNervousOgmBrainNo,
    <br>* McsEngl.sysNervousOgm.brainNo-002!⇒sysNervousOgmBrainNo,
    <br>* McsEngl.sysNervousOgm.nerve-net-002!⇒sysNervousOgmBrainNo,
    <br>* McsEngl.nerve-net!⇒sysNervousOgmBrainNo,
    <br>* McsEngl.sysNervousOgmBrainNo,
    <a class="clsHide" href="#idOgm030att010002nam"></a></p>
  <p id="idOgm030att010002dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att010002dsn"></a></p>
  </section>
  </section>

  <section id="idOgm030att011">
  <h2 id="idOgm030att011H2">11_genome of ogmNeural
    <a class="clsHide" href="#idOgm030att011H2"></a></h2>
  <p id="idOgm030att011nam">name::
    <br>* McsEngl.ogmNeural'11_genome,
    <br>* McsEngl.ogmNeural'att011-genome,
    <br>* McsEngl.ogmNeural'genome-att011,
    <a class="clsHide" href="#idOgm030att011nam"></a></p>
  <p id="idOgm030att011dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att011dsn"></a></p>
  </section>

  <section id="idOgm030att012">
  <h2 id="idOgm030att012H2">12_food of ogmNeural
    <a class="clsHide" href="#idOgm030att012H2"></a></h2>
  <p id="idOgm030att012nam">name::
    <br>* McsEngl.ogmNeural'12_food,
    <br>* McsEngl.ogmNeural'att012-food,
    <br>* McsEngl.ogmNeural'food-att012,
    <a class="clsHide" href="#idOgm030att012nam"></a></p>
  <p id="idOgm030att012dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att012dsn"></a></p>
  </section>

  <section id="idOgm030att013">
  <h2 id="idOgm030att013H2">13_shape of ogmNeural
    <a class="clsHide" href="#idOgm030att013H2"></a></h2>
  <p id="idOgm030att013nam">name::
    <br>* McsEngl.ogmNeural'13_shape,
    <br>* McsEngl.ogmNeural'att013-shape,
    <br>* McsEngl.ogmNeural'shape-att013,
    <a class="clsHide" href="#idOgm030att013nam"></a></p>
  <p id="idOgm030att013dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att013dsn"></a></p>
  </section>

  <section id="idOgm030att014">
  <h2 id="idOgm030att014H2">14_size of ogmNeural
    <a class="clsHide" href="#idOgm030att014H2"></a></h2>
  <p id="idOgm030att014nam">name::
    <br>* McsEngl.ogmNeural'14_size,
    <br>* McsEngl.ogmNeural'att014-size,
    <br>* McsEngl.ogmNeural'size-att014,
    <a class="clsHide" href="#idOgm030att014nam"></a></p>
  <p id="idOgm030att014dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030att014dsn"></a></p>
  </section>

  <section id="idOgm030evgF">
  <h2 id="idOgm030evgFH2">evoluting of ogmNeural
    <a class="clsHide" href="#idOgm030evgFH2"></a></h2>
  <p id="idOgm030evgFdsn">description::
    <br>"So almost from the start, the cells within early animals had the potential to communicate with each other using electrical pulses and chemical signals. From there, it was not a big leap for some cells to become specialised for carrying messages.
    <br>These nerve cells evolved long, wire-like extensions – axons – for carrying electrical signals over long distances. They still pass signals on to other cells by releasing chemicals such as glutamate, but they do so where they meet them, at synapses. That means the chemicals only have to diffuse across a tiny gap, greatly speeding things up. And so, very early on, the nervous system was born.
    <br>The first neurons were probably connected in a diffuse network across the body (see diagram). This kind of structure, known as a nerve net, can still be seen in the quivering bodies of jellyfish and sea anemones"
    <br>[{2020-08-24} https://www.newscientist.com/article/mg21128311-800-a-brief-history-of-the-brain/]
    <a class="clsHide" href="#idOgm030evgFdsn"></a></p>
  <p id="idOgm030evgFnam">name::
    <br>* McsEngl.ogmNeural'evoluting,
    <a class="clsHide" href="#idOgm030evgFnam"></a></p>
  </section>

  <section id="idOgm030001">
  <h2 id="idOgm030001H2">ogmNeural.brain-001 (<a class="clsPreview" href="McsOgm000009.last.html#idOgmAnml009">link</a>)
    <a class="clsHide" href="#idOgm030001H2"></a></h2>
  </section>

  <section id="idOgm030002">
  <h2 id="idOgm030002H2">ogmNeural.brainNo-002
    <a class="clsHide" href="#idOgm030002H2"></a></h2>
  <p id="idOgm030002nam">name::
    <br>* McsEngl.ogmNeural.brainNo-002!⇒anmlBrainNo,
    <br>* McsEngl.ogmNeural.brainNo-002!⇒anmlBrainNo,
    <br>* McsEngl.anmlBrainNo,
    <a class="clsHide" href="#idOgm030002nam"></a></p>
  <p id="idOgm030002dsn">description::
    <br>·
    <a class="clsHide" href="#idOgm030002dsn"></a></p>
  </section>
</section>

<section id="idOgm031">
  <h1 id="idOgm031H1">ogm.neuralNo-031
    <a class="clsHide" href="#idOgm031H1"></a></h1>
  <p id="idOgm031nam">name::
    <br>* McsEngl.ogm.031-neuralNo!⇒ogmNeuralNo,
    <br>* McsEngl.ogm.neuralNo-031!⇒ogmNeuralNo,
    <br>* McsEngl.neuralNo-ogm-031!⇒ogmNeuralNo,
    <br>* McsEngl.ogmNeuralNo,
    <a class="clsHide" href="#idOgm031nam"></a></p>
  <p id="idOgm031dsn">description::
    <br>"It[neuron] is the main component of nervous tissue in all animals except sponges and placozoa. Plants and fungi do not have nerve cells."
    <br>[{2020-04-24} https://en.wikipedia.org/wiki/Neuron]
    <a class="clsHide" href="#idOgm031dsn"></a></p>
</section>

<section id="idOgm032">
  <h1 id="idOgm032H1">ogm.neural.brain-032
    <a class="clsHide" href="#idOgm032H1"></a></h1>
  <p id="idOgm032nam">name::
    <br>* McsEngl.ogm.032-brain!⇒anmlBrain,
    <br>* McsEngl.ogm.brain-032!⇒anmlBrain,
    <br>* McsEngl.ogm.neural.brain!⇒anmlBrain,
    <br>* McsEngl.ogmNeural.brain-001!⇒anmlBrain,
    <br>* McsEngl.ogmNeural.brain-001!⇒anmlBrain,
    <br>* McsEngl.anmlBrain,
    <a class="clsHide" href="#idOgm032nam"></a></p>
  <p id="idOgm032dsn">description::
    <br>"Cephalization is an evolutionary trend in which, over many generations, the mouth, sense organs, and nerve ganglia become concentrated at the front end of an animal, producing a head region. This is associated with movement and bilateral symmetry, such that the animal has a definite head end. This led to the formation of a highly sophisticated brain in three groups of animals, namely the arthropods, cephalopod molluscs, and vertebrates."
    <br>[{2020-04-25} https://en.wikipedia.org/wiki/Cephalization]
    <a class="clsHide" href="#idOgm032dsn"></a></p>
</section>

<section id="idOgm033">
  <h1 id="idOgm033H1">ogm.protist-033
    <a class="clsHide" href="#idOgm033H1"></a></h1>
  <p id="idOgm033nam">name::
    <br>* McsEngl.ogm.protist!⇒ogmProtist,
    <br>* McsEngl.ogm.033-protist!⇒ogmProtist,
    <br>* McsEngl.ogmProtist,
    <br>* McsEngl.protist-ogm!⇒ogmProtist,
    <a class="clsHide" href="#idOgm033nam"></a></p>
  <p id="idOgm033dsn">description::
    <br>"A protist (/ˈproʊtɪst/) is any eukaryotic organism (one with cells containing a nucleus) that is not an animal, plant, or fungus. The protists do not form a natural group, or clade, since they exclude certain eukaryotes with whom they share a common ancestor[a] i.e. some protists are more closely related to plants or animals than they are to other protists. However, like algae or invertebrates, the grouping is used for convenience. In some systems of biological classification, such as the popular five-kingdom scheme proposed by Robert Whittaker in 1969, the protists make up a kingdom called Protista, composed of eukaryotic "organisms which are unicellular or unicellular-colonial and which form no tissues".[2][3][b]
    <br>Besides their relatively simple levels of organization, protists do not necessarily have much in common.[6] When used, the term "protists" is now considered to mean a paraphyletic assemblage of similar-appearing but diverse taxa (biological groups); these taxa do not have an exclusive common ancestor beyond being composed of eukaryotes and have different life cycles, trophic levels, modes of locomotion and cellular structures.[7][8] In the classification system of Lynn Margulis, the term protist is reserved for microscopic organisms, while the more inclusive term Protoctista, or protoctist, is applied to a biological kingdom that includes certain large multicellular eukaryotes, such as kelp, red algae and slime molds.[9] Others use the term protist more broadly, to encompass both microbial eukaryotes and macroscopic organisms that do not fit into the other traditional kingdoms.
    <br>In cladistic systems (classifications based on common ancestry), there are no equivalents to the taxa Protista or Protoctista, both terms referring to a paraphyletic group that spans the entire eukaryotic tree of life. In cladistic classification, the contents of Protista are distributed among various supergroups (SAR, such as protozoa and some algae, Archaeplastida, such as land plants and some algae, Excavata, which are a group of unicellular organisms, and Opisthokonta, such as animals and fungi, etc.). "Protista", ''Protoctista'' and "Protozoa" are considered obsolete. However, the term "protist" continues to be used informally as a catch-all term for unicellular eukaryotic microorganisms. For example, the word "protist pathogen" may be used to denote any disease-causing microbe that is not bacteria, virus, viroid, prion, or metazoa.[10]
    <br>Examples of protists include:[11]
    <br>* Amoeba,
    <br>* Choanaflagellates,
    <br>* Ciliates,
    <br>* Diatoms,
    <br>* Dinoflagellates,
    <br>* Foraminifera,
    <br>* Giardia,
    <br>* Nucleariids,
    <br>* Oomycetes,
    <br>* Plasmodium (causes malaria),
    <br>* Phytophthora (cause of the Great Famine of Ireland),
    <br>* Slime molds",
    <br>[{2020-05-10} https://en.wikipedia.org/wiki/Protist]
    <a class="clsHide" href="#idOgm033dsn"></a></p>
</section>

<section id="idOgm034">
  <h1 id="idOgm034H1">ogm.vertebrate-034
    <a class="clsHide" href="#idOgm034H1"></a></h1>
  <p id="idOgm034nam">name::
    <br>* McsEngl.ogm.vertebrate!⇒ogmVertebrate,
    <br>* McsEngl.ogm.034-vertebrate!⇒ogmVertebrate,
    <br>* McsEngl.ogmVertebrate,
    <br>* McsEngl.vertebrate!⇒ogmVertebrate,
    <a class="clsHide" href="#idOgm034nam"></a></p>
  <p id="idOgm034dsn">description::
    <br>"There are about 50,000 species of animals that have a vertebral column.[2] The human vertebral column is one of the most-studied examples."
    <br>[{2020-05-28} https://en.wikipedia.org/wiki/Vertebral_column]
    <br>===
    <br>"Vertebrates /ˈvɜːrtəˌbrəts/ comprise all species of animals within the subphylum Vertebrata /-ə/ (chordates with backbones). Vertebrates represent the overwhelming majority of the phylum Chordata, with currently about 69,963 species described.[4] Vertebrates include such groups as the following:
    <br>* jawless fishes
    <br>* jawed vertebrates, which include the cartilaginous fishes (sharks, rays, and ratfish)
    <br>* tetrapods, which include amphibians, reptiles, birds and mammals
    <br>* bony fishes
    <br>Extant vertebrates range in size from the frog species Paedophryne amauensis, at as little as 7.7 mm (0.30 in), to the blue whale, at up to 33 m (108 ft). Vertebrates make up less than five percent of all described animal species; the rest are invertebrates, which lack vertebral columns.
    <br>The vertebrates traditionally include the hagfish, which do not have proper vertebrae due to their loss in evolution,[5] though their closest living relatives, the lampreys, do.[6] Hagfish do, however, possess a cranium. For this reason, the vertebrate subphylum is sometimes referred to as "Craniata" when discussing morphology. Molecular analysis since 1992 has suggested that hagfish are most closely related to lampreys,[7] and so also are vertebrates in a monophyletic sense. Others consider them a sister group of vertebrates in the common taxon of craniata.[8]
    <br>The populations of vertebrates have dropped in the past 50 years[9]."
    <br>[{2020-05-28} https://en.wikipedia.org/wiki/Vertebrate]
    <a class="clsHide" href="#idOgm034dsn"></a></p>
</section>

<section id="idOgm035">
  <h1 id="idOgm035H1">ogm.group-035
    <a class="clsHide" href="#idOgm035H1"></a></h1>
  <p id="idOgm035dsn">description::
    <br>· an-organism part of a-group.
    <a class="clsHide" href="#idOgm035dsn"></a></p>
  <p id="idOgm035nam">name::
    <br>* McsEngl.groupOgm'01_organism,
    <br>* McsEngl.groupOgm'att001-organism,
    <br>* McsEngl.groupOgm'member,
    <br>* McsEngl.groupOgm'organism,
    <br>* McsEngl.ogm.035-group,
    <br>* McsEngl.ogm.group,
    <a class="clsHide" href="#idOgm035nam"></a></p>
</section>

<section id="idOgm036">
  <h1 id="idOgm036H1">ogm.organization-036
    <a class="clsHide" href="#idOgm036H1"></a></h1>
  <p id="idOgm036dsn">description::
    <br>· an-organism part of an-organization.
    <a class="clsHide" href="#idOgm036dsn"></a></p>
  <p id="idOgm036nam">name::
    <br>* McsEngl.ogm.036-organization,
    <br>* McsEngl.ogm.organization,
    <a class="clsHide" href="#idOgm036nam"></a></p>
</section>

<section id="idOgm037">
  <h1 id="idOgm037H1">ogm.society-037
    <a class="clsHide" href="#idOgm037H1"></a></h1>
  <p id="idOgm037dsn">description::
    <br>· an-organism part of a-society.
    <a class="clsHide" href="#idOgm037dsn"></a></p>
  <p id="idOgm037nam">name::
    <br>* McsEngl.ogm.037-society,
    <br>* McsEngl.ogm.society,
    <a class="clsHide" href="#idOgm037nam"></a></p>
</section>

<section id="idMeta">
  <h1 id="idMetaH1">meta-info
    <a class="clsHide" href="#idMetaH1"></a></h1>
  <p id="idMetaCounter" class="clsCenter">this webpage was-visited
    <span class="clsColorRed">
    <script src="../../dirPgm/dirCntr/counter.php?page=McsOgm000003"></script>
    </span>
    times since {2019-12-19}</p>
  <!-- the content of page-path paragraph is displayed as it is on top of toc -->
  <p id="idMetaWebpage_path"><span class="clsB clsColorGreen">page-wholepath</span>:
    <a class="clsPreview" href="../../#idOverview">synagonism.net</a> /
    <a class="clsPreview" href="../Mcs000000.last.html#idOverview">worldviewSngo</a> /
    <a class="clsPreview" href="McsOgm000000.last.html#idOverview">dirOgm</a> /
    ogm
    </p>
  <p id="idMetaP1">SEARCH::
    <br>· this page uses '<span class="clsColorRed">locator-names</span>', names that when you find them, you find the-LOCATION of the-concept they denote.
    <br>⊛ <strong>GLOBAL-SEARCH</strong>:
    <br>· clicking on <span class="clsColorGreenBg">the-green-BAR of a-page</span> you have access to the-global--locator-names of my-site.
    <br>· use the-prefix '<span class="clsColorRed">ogm</span>' for <a class="clsPreview" href="../dirCor/McsCor000002.last.html#idOverview">sensorial-concepts</a> related to current concept 'organism'.
    <br>⊛ <strong>LOCAL-SEARCH</strong>:
    <br>· TYPE <span class="clsColorRed">CTRL+F "McsLang.words-of-concept's-name"</span>, to go to the-LOCATION of the-concept.
    <br>· a-preview of the-description of a-global-name makes reading fast.
    <a class="clsHide" href="#idMetaP1"></a></p>
  <p id="idFooterP1">footer::
    <br>• author: <a class="clsPreview" href="../dirHmn/McsHmn000003.last.html#idOverview">Kaseluris.Nikos.1959</a>
    <br>• email:
    <br> &nbsp;<img src="../../dirRsc/dirImg/mail.png">
    <br>• edit on github: https://github.com/synagonism/McsWorld/blob/master/dirOgm/McsOgm000003.last.html,
    <br>• comments on <a class="clsPreview" href="McsOgm000000.last.html#idComment">Disqus</a>,
    <br>• twitter: <a href="https://twitter.com/synagonism">@synagonism</a>,
    <a class="clsHide" href="#idFooterP1"></a></p>
  <!--                           -->
  <p id="idMetaVersion">webpage-versions::
    <br>• version.last.dynamic: <a class="clsPreview" href="McsOgm000003.last.html">McsOgm000003.last.html</a>,
    <br>• version.1-0-0.2021-04-15: (0-44) <a href="../../dirMiwMcs/dirOgm/filMcsOgm.1-0-0.2021-04-15.html">../../dirMiwMcs/dirOgm/filMcsOgm.1-0-0.2021-04-15.html</a>,
    <br>• version.0-1-0.2019-12-19 draft creation,
    <a class="clsHide" href="#idMetaVersion"></a></p>
</section>

<section id="idSupport">
  <h1 id="idSupportH1">support (<a class="clsPreview" href="../../#idSupport">link</a>)</h1>
  <p></p>
</section>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-19285371-5', 'synagonism.net');
  ga('send', 'pageview');
</script>

<script type="module">
  import * as oMcsh from '../Mcsmgr/mMcsh.js'
</script>
</body>
</html>