<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Mcs.ogmCellNo-(1-0-0.2021-04-15) cellNo-organism</title>
  <meta name="keywords" content="cellNo-organism, ogmCellNo, ModelConceptSensorial, McsHitp, Synagonism">
  <link rel="stylesheet" href="../Mcsmgr/hitp.css">
</head>

<body>
<header id="idHeader">
  <p></p>
  <h1 id="idHeaderH1">cellNo-organism
    <br>sensorial-concept-Mcs (ogmCellNo)
    </h1>
  <p id="idHeadercrd">McsHitp-creation:: {2020-04-02},
    <a class="clsHide" href="#idHeadercrd"></a></p>
</header>

<section id="idOverview">
  <h1 id="idOverviewH1">overview of ogmCellNo
    <a class="clsHide" href="#idOverviewH1"></a></h1>
  <p id="idName">name::
    <br>* McsEngl.McsOgm000008.last.html//dirOgm//dirMcs!⇒ogmCellNo,
    <br>* McsEngl.dirMcs/dirOgm/McsOgm000008.last.html!⇒ogmCellNo,
    <br>* McsEngl.cellNo-organism!⇒ogmCellNo,
    <br>* McsEngl.ogmCellNo,
    <br>* McsEngl.ogmCellNo'(McsOgm000008)!⇒ogmCellNo,
    <br>* McsEngl.ogmCellNo'(cellNo-organism)!⇒ogmCellNo,
    <br>* McsEngl.ogm.022-cellNo!⇒ogmCellNo,
    <br>* McsEngl.ogm.cellNo-022!⇒ogmCellNo,
    <a class="clsHide" href="#idName"></a></p>
  <p id="idDescription">description::
    <br>·
    <a class="clsHide" href="#idDescription"></a></p>
</section>

<section id="idOgmCllNatt001">
  <h1 id="idOgmCllNatt001H1">01_disease of ogmCellNo
    <a class="clsHide" href="#idOgmCllNatt001H1"></a></h1>
</section>

<section id="idOgmCllNatt002">
  <h1 id="idOgmCllNatt002H1">02_governance-system of ogmCellNo
    <a class="clsHide" href="#idOgmCllNatt002H1"></a></h1>
</section>

<section id="idOgmCllNatt003">
  <h1 id="idOgmCllNatt003H1">03_health of ogmCellNo
    <a class="clsHide" href="#idOgmCllNatt003H1"></a></h1>
</section>

<section id="idOgmCllNatt004">
  <h1 id="idOgmCllNatt004H1">04_material-system of ogmCellNo
    <a class="clsHide" href="#idOgmCllNatt004H1"></a></h1>
  <p id="idOgmCllNatt004nam">name::
    <br>* McsEngl.ogmCellNo'04_material-system,
    <br>* McsEngl.ogmCellNo'material-system,
    <a class="clsHide" href="#idOgmCllNatt004nam"></a></p>
  <p id="idOgmCllNatt004dsn">description::
    <br>·
    <a class="clsHide" href="#idOgmCllNatt004dsn"></a></p>
</section>

<section id="idOgmCllNatt005">
  <h1 id="idOgmCllNatt005H1">05_material of ogmCellNo
    <a class="clsHide" href="#idOgmCllNatt005H1"></a></h1>
  <p id="idOgmCllNatt005nam">name::
    <br>* McsEngl.ogmCellNo'05_material,
    <br>* McsEngl.ogmCellNo'material,
    <a class="clsHide" href="#idOgmCllNatt005nam"></a></p>
  <p id="idOgmCllNatt005dsn">description::
    <br>·
    <a class="clsHide" href="#idOgmCllNatt005dsn"></a></p>
</section>

<section id="idOgmCllNatt006">
  <h1 id="idOgmCllNatt006H1">06_nutrient of ogmCellNo
    <a class="clsHide" href="#idOgmCllNatt006H1"></a></h1>
  <p id="idOgmCllNatt006nam">name::
    <br>* McsEngl.ogmCellNo'06_nutrient,
    <br>* McsEngl.ogmCellNo'nutrient,
    <a class="clsHide" href="#idOgmCllNatt006nam"></a></p>
  <p id="idOgmCllNatt006dsn">description::
    <br>·
    <a class="clsHide" href="#idOgmCllNatt006dsn"></a></p>
</section>

<section id="idOgmCllNatt007">
  <h1 id="idOgmCllNatt007H1">07_shape of ogmCellNo
    <a class="clsHide" href="#idOgmCllNatt007H1"></a></h1>
  <p id="idOgmCllNatt007nam">name::
    <br>* McsEngl.ogmCellNo'07_shape,
    <br>* McsEngl.ogmCellNo'shape,
    <a class="clsHide" href="#idOgmCllNatt007nam"></a></p>
  <p id="idOgmCllNatt007dsn">description::
    <br>·
    <a class="clsHide" href="#idOgmCllNatt007dsn"></a></p>
</section>

<section id="idOgmCllNatt008">
  <h1 id="idOgmCllNatt008H1">08_size of ogmCellNo
    <a class="clsHide" href="#idOgmCllNatt008H1"></a></h1>
  <p id="idOgmCllNatt008nam">name::
    <br>* McsEngl.ogmCellNo'08_size,
    <br>* McsEngl.ogmCellNo'size,
    <a class="clsHide" href="#idOgmCllNatt008nam"></a></p>
  <p id="idOgmCllNatt008dsn">description::
    <br>·
    <a class="clsHide" href="#idOgmCllNatt008dsn"></a></p>
</section>

<section id="idOgmCllNrscF">
  <h1 id="idOgmCllNrscFH1">09_resource of ogmCellNo
    <a class="clsHide" href="#idOgmCllNrscFH1"></a></h1>
  <p id="idOgmCllNrscnam">name::
    <br>* McsEngl.ogmCellNo'09_resource,
    <br>* McsEngl.ogmCellNo'Infrsc,
    <a class="clsHide" href="#idOgmCllNrscnam"></a></p>
  <p id="idOgmCllNrscwpa">addressWpg::
    <br>*
    <a class="clsHide" href="#idOgmCllNrscwpa"></a></p>
</section>

<section id="idOgmCllNsrtr">
  <h1 id="idOgmCllNsrtrH1">10_structure of ogmCellNo
    <a class="clsHide" href="#idOgmCllNsrtrH1"></a></h1>
  <p id="idOgmCllNsrtrnam">name::
    <br>* McsEngl.ogmCellNo'10_structure,
    <br>* McsEngl.ogmCellNo'structure,
    <a class="clsHide" href="#idOgmCllNsrtrnam"></a></p>
  <p id="idOgmCllNsrtrdsn">description::
    <br>·
    <a class="clsHide" href="#idOgmCllNsrtrdsn"></a></p>
</section>

<section id="idOgmCllNdng">
  <h1 id="idOgmCllNdngH1">11_DOING of ogmCellNo
    <a class="clsHide" href="#idOgmCllNdngH1"></a></h1>
  <p id="idOgmCllNdngnam">name::
    <br>* McsEngl.ogmCellNo'11_doing,
    <br>* McsEngl.ogmCellNo'doing,
    <a class="clsHide" href="#idOgmCllNdngnam"></a></p>
  <p id="idOgmCllNdngdsn">description::
    <br>·
    <a class="clsHide" href="#idOgmCllNdngdsn"></a></p>
</section>

<section id="idOgmCllNevgF">
  <h1 id="idOgmCllNevgFH1">12_evoluting of ogmCellNo
    <a class="clsHide" href="#idOgmCllNevgFH1"></a></h1>
  <p id="idOgmCllNevgnam">name::
    <br>* McsEngl.ogmCellNo'12_evoluting,
    <br>* McsEngl.ogmCellNo'evoluting,
    <a class="clsHide" href="#idOgmCllNevgnam"></a></p>
  <p id="idOgmCllNevg20200402">{time.2020-04-02}::
    <br>=== McsHitp-creation:
    <br>· creation of current <a class="clsPreview" href="../dirCor/McsCor000002.last.html#idOverview">concept</a>.
    <a class="clsHide" href="#idOgmCllNevg20200402"></a></p>
</section>

<section id="idOgmCllNwptF">
  <h1 id="idOgmCllNwptFH1">WHOLE-PART-TREE of ogmCellNo
    <a class="clsHide" href="#idOgmCllNwhlFH1"></a></h1>
  <p id="idOgmCllNwptnam">name::
    <br>* McsEngl.ogmCellNo'whole-part-tree,
    <a class="clsHide" href="#idOgmCllNwptnam"></a></p>
  <p id="idOgmCllNwtr">whole-tree-of-ogmCellNo::
    <br>*
    <br>* ... <a class="clsPreview" href="../dirCor/McsCor000003.last.html#idOverview">Sympan</a>.
    <a class="clsHide" href="#idOgmCllNwtr"></a></p>
  <p id="idOgmCllNptr">part-tree-of-ogmCellNo::
    <br>*
    <a class="clsHide" href="#idOgmCllNptr"></a></p>
</section>

<section id="idOgmCllNgstF">
  <h1 id="idOgmCllNgstFH1">GENERIC-SPECIFIC-TREE of ogmCellNo
    <a class="clsHide" href="#idOgmCllNgstFH1"></a></h1>
  <p id="idOgmCllNgstnam">name::
    <br>* McsEngl.ogmCellNo'generic-specific-tree,
    <a class="clsHide" href="#idOgmCllNgstnam"></a></p>
  <p id="idOgmCllNgtr">generic-tree-of-ogmCellNo::
    <br>* ,
    <br>* ... <a class="clsPreview" href="../dirCor/McsCor000003.last.html#idOverview">entity</a>.
    <a class="clsHide" href="#idOgmCllNgtr"></a></p>
  <p id="idOgmCllNstr">specific-tree-of-ogmCellNo::
    <br>* ,
    <a class="clsHide" href="#idOgmCllNstr"></a></p>
</section>

<section id="idOgmVrs">
  <h1 id="idOgmVrsH1">ogmCellNo.virus-001
    <a class="clsHide" href="#idOgmVrsH1"></a></h1>
  <p id="idOgmVrsnam">name::
    <br>* McsEngl.ogmCellNo.001-virus!⇒ogmVirus,
    <br>* McsEngl.ogmCellNo.virus-001!⇒ogmVirus,
    <br>* McsEngl.ogmVirus,
    <br>* McsEngl.virus!⇒ogmVirus,
    <a class="clsHide" href="#idOgmVrsnam"></a></p>
  <p id="idOgmVrsdsn">description::
    <br>"A virus is a submicroscopic infectious agent that replicates only inside the living cells of an organism. Viruses can infect all types of life forms, from animals and plants to microorganisms, including bacteria and archaea.[1] Since Dmitri Ivanovsky's 1892 article describing a non-bacterial pathogen infecting tobacco plants, and the discovery of the tobacco mosaic virus by Martinus Beijerinck in 1898,[2] about 5,000 virus species have been described in detail,[3] of the millions of types of viruses in the environment.[4] Viruses are found in almost every ecosystem on Earth and are the most numerous type of biological entity.[5][6] The study of viruses is known as virology, a subspeciality of microbiology.
    <br>When not inside an infected cell or in the process of infecting a cell, viruses exist in the form of independent particles, or virions, consisting of: (i) the genetic material, i.e. long molecules of DNA or RNA that encode the structure of the proteins by which the virus acts; (ii) a protein coat, the capsid, which surrounds and protects the genetic material; and in some cases (iii) an outside envelope of lipids. The shapes of these virus particles range from simple helical and icosahedral forms to more complex structures. Most virus species have virions too small to be seen with an optical microscope as they are one hundredth the size of most bacteria.
    <br>The origins of viruses in the evolutionary history of life are unclear: some may have evolved from plasmids—pieces of DNA that can move between cells—while others may have evolved from bacteria. In evolution, viruses are an important means of horizontal gene transfer, which increases genetic diversity in a way analogous to sexual reproduction.[7] Viruses are considered by some biologists to be a life form, because they carry genetic material, reproduce, and evolve through natural selection, although they lack key characteristics (such as cell structure) that are generally considered necessary to count as life. Because they possess some but not all such qualities, viruses have been described as "organisms at the edge of life",[8] and as replicators.[9]
    <br>Viruses spread in many ways. One transmission pathway is through disease-bearing organisms known as vectors: for example, viruses are often transmitted from plant to plant by insects that feed on plant sap, such as aphids; and viruses in animals can be carried by blood-sucking insects. Influenza viruses are spread by coughing and sneezing. Norovirus and rotavirus, common causes of viral gastroenteritis, are transmitted by the faecal–oral route, passed by contact and entering the body in food or water. HIV is one of several viruses transmitted through sexual contact and by exposure to infected blood. The variety of host cells that a virus can infect is called its "host range". This can be narrow, meaning a virus is capable of infecting few species, or broad, meaning it is capable of infecting many.[10]
    <br>Viral infections in animals provoke an immune response that usually eliminates the infecting virus. Immune responses can also be produced by vaccines, which confer an artificially acquired immunity to the specific viral infection. Some viruses, including those that cause AIDS, HPV infection, and viral hepatitis, evade these immune responses and result in chronic infections. Several antiviral drugs have been developed."
    <br>[{2020-04-12} https://en.wikipedia.org/wiki/Virus]
    <a class="clsHide" href="#idOgmVrsdsn"></a></p>

  <section id="idOgmVrsatt002">
  <h2 id="idOgmVrsatt002H2">host of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt002H2"></a></h2>
  <p id="idOgmVrsatt002nam">name::
    <br>* McsEngl.ogmVirus'att002-host,
    <br>* McsEngl.ogmVirus'host-att002,
    <a class="clsHide" href="#idOgmVrsatt002nam"></a></p>
  <p id="idOgmVrsatt002dsn">description::
    <br>"Viruses are by far the most abundant biological entities on Earth and they outnumber all the others put together.[132] They infect all types of cellular life including animals, plants, bacteria and fungi.[3] Different types of viruses can infect only a limited range of hosts and many are species-specific. Some, such as smallpox virus for example, can infect only one species—in this case humans,[133] and are said to have a narrow host range. Other viruses, such as rabies virus, can infect different species of mammals and are said to have a broad range.[134] The viruses that infect plants are harmless to animals, and most viruses that infect other animals are harmless to humans.[135] The host range of some bacteriophages is limited to a single strain of bacteria and they can be used to trace the source of outbreaks of infections by a method called phage typing.[136]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Host_range]
    <a class="clsHide" href="#idOgmVrsatt002dsn"></a></p>

  <section id="idOgmVrsatt017">
  <h3 id="idOgmVrsatt017H3">cytopathic-effect of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt017H3"></a></h3>
  <p id="idOgmVrsatt017nam">name::
    <br>* McsEngl.ogmVirus'att017-cytopathic-effect,
    <br>* McsEngl.ogmVirus'cytopathic-effect-att017,
    <a class="clsHide" href="#idOgmVrsatt017nam"></a></p>
  <p id="idOgmVrsatt017dsn">description::
    <br>"Cytopathic effect or cytopathogenic effect (abbreviated CPE) refers to structural changes in host cells that are caused by viral invasion. The infecting virus causes lysis of the host cell or when the cell dies without lysis due to an inability to reproduce.[1] Both of these effects occur due to CPEs. If a virus causes these morphological changes in the host cell, it is said to be cytopathogenic.[2] Common examples of CPE include rounding of the infected cell, fusion with adjacent cells to form syncytia, and the appearance of nuclear or cytoplasmic inclusion bodies.[3]
    <br>CPEs and other changes in cell morphology are only a few of the many effects by cytocidal viruses. When a cytocidal virus infects a permissive cell, the viruses kill the host cell through changes in cell morphology, in cell physiology, and the biosynthetic events that follow. These changes are necessary for efficient virus replication but at the expense of the host cell.[3]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Cytopathic_effect]
    <br>"The range of structural and biochemical effects that viruses have on the host cell is extensive.[123] These are called 'cytopathic effects'.[124] Most virus infections eventually result in the death of the host cell. The causes of death include cell lysis, alterations to the cell's surface membrane and apoptosis.[125] Often cell death is caused by cessation of its normal activities because of suppression by virus-specific proteins, not all of which are components of the virus particle.[126] The distinction between cytopathic and harmless is gradual. Some viruses, such as Epstein–Barr virus, can cause cells to proliferate without causing malignancy,[127] while others, such as papillomaviruses, are established causes of cancer.[128]"
    <br>[https://en.wikipedia.org/wiki/Virus#Cytopathic_effects_on_the_host_cell]
    <a class="clsHide" href="#idOgmVrsatt017dsn"></a></p>
  </section>
  </section>

  <section id="idOgmVrsatt006">
  <h2 id="idOgmVrsatt006H2">genome of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt006H2"></a></h2>
  <p id="idOgmVrsatt006nam">name::
    <br>* McsEngl.ogmVirus'att006-genome,
    <br>* McsEngl.ogmVirus'genome-att006,
    <a class="clsHide" href="#idOgmVrsatt006nam"></a></p>
  <p id="idOgmVrsatt006dsn">description::
    <br>* Dna,
    <br>* Rna,
    <br>* single-stranded,
    <br>* double-stranded,
    <a class="clsHide" href="#idOgmVrsatt006dsn"></a></p>

  <section id="idOgmVrsatt010">
  <h3 id="idOgmVrsatt010H3">shape of genome of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt010H3"></a></h3>
  <p id="idOgmVrsatt010nam">name::
    <br>* McsEngl.ogmVirus'att010-genome'shape,
    <br>* McsEngl.ogmVirus'genome'shape-att010,
    <a class="clsHide" href="#idOgmVrsatt010nam"></a></p>
  <p id="idOgmVrsatt010dsn">description::
    <br>"Viral genomes are circular, as in the polyomaviruses, or linear, as in the adenoviruses. The type of nucleic acid is irrelevant to the shape of the genome. Among RNA viruses and certain DNA viruses, the genome is often divided up into separate parts, in which case it is called segmented. For RNA viruses, each segment often codes for only one protein and they are usually found together in one capsid. All segments are not required to be in the same virion for the virus to be infectious, as demonstrated by brome mosaic virus and several other plant viruses."
    <br>[{2020-04-14} https://en.wikipedia.org/wiki/Virus#Genome]
    <a class="clsHide" href="#idOgmVrsatt010dsn"></a></p>
  </section>

  <section id="idOgmVrsatt011">
  <h3 id="idOgmVrsatt011H3">size of genome of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt011H3"></a></h3>
  <p id="idOgmVrsatt011nam">name::
    <br>* McsEngl.ogmVirus'att011-genome'size,
    <br>* McsEngl.ogmVirus'genome'size-att011,
    <a class="clsHide" href="#idOgmVrsatt011nam"></a></p>
  <p id="idOgmVrsatt011dsn">description::
    <br>"Genome size varies greatly between species. The smallest—the ssDNA circoviruses, family Circoviridae—code for only two proteins and have a genome size of only two kilobases;[94] the largest—the pandoraviruses—have genome sizes of around two megabases which code for about 2500 proteins.[95] Virus genes rarely have introns and often are arranged in the genome so that they overlap.[96]
    <br>In general, RNA viruses have smaller genome sizes than DNA viruses because of a higher error-rate when replicating, and have a maximum upper size limit.[45] Beyond this, errors when replicating render the virus useless or uncompetitive. To compensate, RNA viruses often have segmented genomes—the genome is split into smaller molecules—thus reducing the chance that an error in a single-component genome will incapacitate the entire genome. In contrast, DNA viruses generally have larger genomes because of the high fidelity of their replication enzymes.[97] Single-strand DNA viruses are an exception to this rule, as mutation rates for these genomes can approach the extreme of the ssRNA virus case.[98]"
    <a class="clsHide" href="#idOgmVrsatt011dsn"></a></p>
  </section>

  <section id="idOgmVrsatt008">
  <h3 id="idOgmVrsatt008H3">genome.aggregate of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt008H3"></a></h3>
  <p id="idOgmVrsatt008nam">name::
    <br>* McsEngl.ogmVirus'att008-genome.aggregate,
    <br>* McsEngl.ogmVirus'genome.aggregate-att008,
    <a class="clsHide" href="#idOgmVrsatt008nam"></a></p>
  <p id="idOgmVrsatt008dsn">description::
    <br>"As of September 2015, the NCBI Virus genome database has more than 75,000 complete genome sequences,[89] but there are doubtlessly many more to be discovered."
    <br>[{2020-04-14} https://en.wikipedia.org/wiki/Virus#Genome]
    <br>===
    <br>"As of November 2017, the full-length genome sequences of 7454 different viruses, including smallpox, are publicly available in an online database maintained by the National Institutes of Health.[269]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Synthetic_viruses]
    <a class="clsHide" href="#idOgmVrsatt008dsn"></a></p>
  </section>
  </section>

  <section id="idOgmVrsatt009">
  <h2 id="idOgmVrsatt009H2">capsid of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt009H2"></a></h2>
  <p id="idOgmVrsatt009nam">name::
    <br>* McsEngl.ogmVirus'att009-capsid,
    <br>* McsEngl.ogmVirus'capsid-att009,
    <br>* McsEngl.capsid-of-ogmVirus-att009,
    <a class="clsHide" href="#idOgmVrsatt009nam"></a></p>
  <p id="idOgmVrsatt009dsn">description::
    <br>"A capsid is the protein shell of a virus, enclosing genetic material. It consists of several oligomeric structural subunits made of protein called protomers. The observable 3-dimensional morphological subunits, which may or may not correspond to individual proteins, are called capsomeres.
    <br>Capsids are broadly classified according to their structure. The majority of viruses have capsids with either helical or icosahedral[2][3] structure. Some viruses, such as bacteriophages, have developed more complicated structures due to constraints of elasticity and electrostatics.[4] The icosahedral shape, which has 20 equilateral triangular faces, approximates a sphere, while the helical shape resembles the shape of a spring, taking the space of a cylinder but not being a cylinder itself.[5] The capsid faces may consist of one or more proteins. For example, the foot-and-mouth disease virus capsid has faces consisting of three proteins named VP1–3.[6]
    <br>Some viruses are enveloped, meaning that the capsid is coated with a lipid membrane known as the viral envelope. The envelope is acquired by the capsid from an intracellular membrane in the virus' host; examples include the inner nuclear membrane, the Golgi membrane, and the cell's outer membrane.[7]
    <br>Once the virus has infected a cell and begins replicating itself, new capsid subunits are synthesized using the protein biosynthesis mechanism of the cell. In some viruses, including those with helical capsids and especially those with RNA genomes, the capsid proteins co-assemble with their genomes. In other viruses, especially more complex viruses with double-stranded DNA genomes, the capsid proteins assemble into empty precursor procapsids that includes a specialized portal structure at one vertex. Through this portal, viral DNA is translocated into the capsid.[8]
    <br>Structural analyses of major capsid protein (MCP) architectures have been used to categorise viruses into lineages. For example, the bacteriophage PRD1, the algal virus Paramecium bursaria Chlorella virus (PBCV-1), mimivirus and the mammalian adenovirus have been placed in the same lineage, whereas tailed, double-stranded DNA bacteriophages (Caudovirales) and herpesvirus belong to a second lineage.[9][10]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Capsid]
    <a class="clsHide" href="#idOgmVrsatt009dsn"></a></p>
  </section>

  <section id="idOgmVrsatt004">
  <h2 id="idOgmVrsatt004H2">shape of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt004H2"></a></h2>
  <p id="idOgmVrsatt004nam">name::
    <br>* McsEngl.ogmVirus'att004-shape,
    <br>* McsEngl.ogmVirus'shape-att004,
    <a class="clsHide" href="#idOgmVrsatt004nam"></a></p>
  <p id="idOgmVrsatt004dsn">description::
    <br>·
    <a class="clsHide" href="#idOgmVrsatt004dsn"></a></p>
  </section>

  <section id="idOgmVrsatt005">
  <h2 id="idOgmVrsatt005H2">size of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt005H2"></a></h2>
  <p id="idOgmVrsatt005nam">name::
    <br>* McsEngl.ogmVirus'att005-size,
    <br>* McsEngl.ogmVirus'size-att005,
    <a class="clsHide" href="#idOgmVrsatt005nam"></a></p>
  <p id="idOgmVrsatt005dsn">description::
    <br>"Viruses display a wide diversity of shapes and sizes, called 'morphologies'. In general, viruses are much smaller than bacteria. Most viruses that have been studied have a diameter between 20 and 300 nanometres. Some filoviruses have a total length of up to 1400 nm; their diameters are only about 80 nm.[69] Most viruses cannot be seen with an optical microscope, so scanning and transmission electron microscopes are used to visualise them.[70] To increase the contrast between viruses and the background, electron-dense "stains" are used. These are solutions of salts of heavy metals, such as tungsten, that scatter the electrons from regions covered with the stain. When virions are coated with stain (positive staining), fine detail is obscured. Negative staining overcomes this problem by staining the background only.[71]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Virus#Structure]
    <a class="clsHide" href="#idOgmVrsatt005dsn"></a></p>
  </section>

  <section id="idOgmVrsatt023">
  <h2 id="idOgmVrsatt023H2">weapon of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt023H2"></a></h2>
  <p id="idOgmVrsatt023nam">name::
    <br>* McsEngl.ogmVirus'att023-weapon,
    <br>* McsEngl.ogmVirus'weapon-att023,
    <a class="clsHide" href="#idOgmVrsatt023nam"></a></p>
  <p id="idOgmVrsatt023dsn">description::
    <br>"The ability of viruses to cause devastating epidemics in human societies has led to the concern that viruses could be weaponised for biological warfare. Further concern was raised by the successful recreation of the infamous 1918 influenza virus in a laboratory.[270]
    <br>Smallpox virus devastated numerous societies throughout history before its eradication. There are only two centres in the world authorised by the WHO to keep stocks of smallpox virus: the State Research Center of Virology and Biotechnology VECTOR in Russia and the Centers for Disease Control and Prevention in the United States.[271] It may be used as a weapon,[271] as the vaccine for smallpox sometimes had severe side-effects, it is no longer used routinely in any country. Thus, much of the modern human population has almost no established resistance to smallpox and would be vulnerable to the virus.[271]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Weapons]
    <a class="clsHide" href="#idOgmVrsatt023dsn"></a></p>
  </section>

  <section id="idOgmVrsrsc">
  <h2 id="idOgmVrsrscH2">Inforesource of ogmVirus
    <a class="clsHide" href="#idOgmVrsrscH2"></a></h2>
  <p id="idOgmVrsrscnam">name::
    <br>* McsEngl.ogmVirus'Infrsc,
    <a class="clsHide" href="#idOgmVrsrscnam"></a></p>
  <p id="idOgmVrsrscdsn">description::
    <br>·
    <a class="clsHide" href="#idOgmVrsrscdsn"></a></p>

  <section id="idOgmVrssci">
  <h3 id="idOgmVrssciH3">science of ogmVirus
    <a class="clsHide" href="#idOgmVrssciH3"></a></h3>
  <p id="idOgmVrsscinam">name::
    <br>* McsEngl.ogmVirus'science,
    <br>* McsEngl.science'ogmVirus,
    <br>* McsEngl.virology,
    <a class="clsHide" href="#idOgmVrsscinam"></a></p>
  <p id="idOgmVrsscidsn">description::
    <br>"The study of viruses is known as virology, a subspeciality of microbiology."
    <br>[{2020-04-12} https://en.wikipedia.org/wiki/Virus]
    <a class="clsHide" href="#idOgmVrsscidsn"></a></p>
  </section>
  </section>

  <section id="idOgmVrssrtr">
  <h2 id="idOgmVrssrtrH2">structure of ogmVirus
    <a class="clsHide" href="#idOgmVrssrtrH2"></a></h2>
  <p id="idOgmVrssrtrnam">name::
    <br>* McsEngl.ogmVirus'structure,
    <a class="clsHide" href="#idOgmVrssrtrnam"></a></p>
  <p id="idOgmVrssrtrdsn">description::
    <br>"Some viruses that infect Archaea have complex structures unrelated to any other form of virus, with a wide variety of unusual shapes, ranging from spindle-shaped structures, to viruses that resemble hooked rods, teardrops or even bottles. Other archaeal viruses resemble the tailed bacteriophages, and can have multiple tail structures."
    <br>[{2020-04-14} https://en.wikipedia.org/wiki/Virus#Giant_viruses]
    <a class="clsHide" href="#idOgmVrssrtrdsn"></a></p>
  </section>

  <section id="idOgmVrsdng">
  <h2 id="idOgmVrsdngH2">DOING of ogmVirus
    <a class="clsHide" href="#idOgmVrsdngH2"></a></h2>
  <p id="idOgmVrsdngnam">name::
    <br>* McsEngl.ogmVirus'doing,
    <a class="clsHide" href="#idOgmVrsdngnam"></a></p>
  <p id="idOgmVrsdngdsn">description::
    <br>·
    <a class="clsHide" href="#idOgmVrsdngdsn"></a></p>

  <section id="idOgmVrsatt003">
  <h3 id="idOgmVrsatt003H3">transmission of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt003H3"></a></h3>
  <p id="idOgmVrsatt003nam">name::
    <br>* McsEngl.ogmVirus'att003-transmission,
    <br>* McsEngl.ogmVirus'transmission-att003,
    <a class="clsHide" href="#idOgmVrsatt003nam"></a></p>
  <p id="idOgmVrsatt003dsn">description::
    <br>"The virus can find a new host through:[29]
    <br>* Droplet transmission- passed on through body fluids (sneezing on someone)
    <br>- An example is the influenza virus[30]
    <br>* Airborne transmission- passed on through the air (brought in by breathing)
    <br>- An example would be how viral meningitis is passed on[31]
    <br>* Vector transmission- picked up by a carrier and brought to a new host
    <br>- An example is viral encephalitis[32]
    <br>* Waterborne transmission- leaving a host, infecting the water, and being consumed in a new host
    <br>- Poliovirus is an example for this[33]
    <br>* Sit-and-wait-transmission- the virus is living outside a host for long periods of time
    <br>- The smallpox virus is also an example for this[33]"
    <br>[{2020-04-14} https://en.wikipedia.org/wiki/Viral_evolution#Transmission]
    <a class="clsHide" href="#idOgmVrsatt003dsn"></a></p>
  </section>

  <section id="idOgmVrsatt018">
  <h3 id="idOgmVrsatt018H3">culture of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt018H3"></a></h3>
  <p id="idOgmVrsatt018nam">name::
    <br>* McsEngl.ogmVirus'att018-culture,
    <br>* McsEngl.ogmVirus'culture-att018,
    <br>* McsEngl.viral-culture-att018,
    <a class="clsHide" href="#idOgmVrsatt018nam"></a></p>
  <p id="idOgmVrsatt018dsn">description::
    <br>"Viral culture is a laboratory test in which samples are placed with a cell type that the virus being tested for is able to infect. If the cells show changes, known as cytopathic effects, then the culture is positive.[1]
    <br>Traditional viral culture has been generally superseded by shell vial culture, in which the sample is centrifuged onto a single layer of cells and viral growth is measured by antigen detection methods. This greatly reduces the time to detection for slow growing viruses such as cytomegalovirus, for which the method was developed.[2] In addition, the centrifugation step in shell vial culture enhances the sensitivity of this method because after centrifugation, the viral particles of the sample are in close proximity to the cells.
    <br>Human and monkey cells are used in both traditional viral culture and shell vial culture.
    <br>Human virus types that can be identified by viral culture include adenovirus, cytomegalovirus, enteroviruses, herpes simplex virus, influenza virus, parainfluenza virus, rhinovirus, respiratory syncytial virus, varicella zoster virus, measles and mumps.[3] For these, the final identification method is generally by immunofluorescence, with exception of cytomegalovirus and rhinovirus, whose identification in a viral culture are determined by cytopathic effects.[3]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Viral_culture]
    <a class="clsHide" href="#idOgmVrsatt018dsn"></a></p>
  </section>
  </section>

  <section id="idOgmVrsevg">
  <h2 id="idOgmVrsevgH2">evoluting of ogmVirus
    <a class="clsHide" href="#idOgmVrsevgH2"></a></h2>
  <p id="idOgmVrsevgnam">name::
    <br>* McsEngl.ogmVirus'evoluting,
    <a class="clsHide" href="#idOgmVrsevgnam"></a></p>
  <p id="idOgmVrsevgdsn">description::
    <br>·
    <a class="clsHide" href="#idOgmVrsevgdsn"></a></p>

  <section id="idOgmVrsatt001">
  <h3 id="idOgmVrsatt001H3">life-cicle of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt001H3"></a></h3>
  <p id="idOgmVrsatt001nam">name::
    <br>* McsEngl.ogmVirus'att001-life-cicle,
    <br>* McsEngl.ogmVirus'life-cicle-att001,
    <a class="clsHide" href="#idOgmVrsatt001nam"></a></p>
  <p id="idOgmVrsatt001dsn">description::
    <br>"Viral populations do not grow through cell division, because they are acellular. Instead, they use the machinery and metabolism of a host cell to produce multiple copies of themselves, and they assemble in the cell.[107]
    <br>Their life cycle differs greatly between species, but there are six basic stages in their life cycle:
    <br>* 1.attachment,
    <br>* 2.penetration,
    <br>* 3.uncoating,
    <br>* 4.replication,
    <br>* 5.assembly,
    <br>* 6.release,"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Replication_cycle]
    <a class="clsHide" href="#idOgmVrsatt001dsn"></a></p>
  </section>

  <section id="idOgmVrsatt012">
  <h3 id="idOgmVrsatt012H3">1_attachment of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt012H3"></a></h3>
  <p id="idOgmVrsatt012nam">name::
    <br>* McsEngl.ogmVirus'att012-attachment,
    <br>* McsEngl.ogmVirus'attachment-att012,
    <br>* McsEngl.ogmVirus'life-cycle'1_attachment,
    <a class="clsHide" href="#idOgmVrsatt012nam"></a></p>
  <p id="idOgmVrsatt012dsn">description::
    <br>"Attachment is a specific binding between viral capsid proteins and specific receptors on the host cellular surface. This specificity determines the host range of a virus. For example, HIV infects a limited range of human leucocytes. This is because its surface protein, gp120, specifically interacts with the CD4 molecule—a chemokine receptor—which is most commonly found on the surface of CD4+ T-Cells. This mechanism has evolved to favour those viruses that infect only cells in which they are capable of replication. Attachment to the receptor can induce the viral envelope protein to undergo changes that result in the fusion of viral and cellular membranes, or changes of non-enveloped virus surface proteins that allow the virus to enter.[109]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Replication_cycle]
    <a class="clsHide" href="#idOgmVrsatt012dsn"></a></p>
  </section>

  <section id="idOgmVrsatt013">
  <h3 id="idOgmVrsatt013H3">2_penetration of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt013H3"></a></h3>
  <p id="idOgmVrsatt013nam">name::
    <br>* McsEngl.ogmVirus'att013-penetration,
    <br>* McsEngl.ogmVirus'penetration-att013,
    <br>* McsEngl.ogmVirus'life-cycle'2_penetration,
    <a class="clsHide" href="#idOgmVrsatt013nam"></a></p>
  <p id="idOgmVrsatt013dsn">description::
    <br>"Penetration follows attachment: Virions enter the host cell through receptor-mediated endocytosis or membrane fusion in a process often known as viral entry. The infection of plant and fungal cells is different from that of animal cells. Plants have a rigid cell wall made of cellulose, and fungi one of chitin, so most viruses can get inside these cells only after trauma to the cell wall.[110] Nearly all plant viruses (such as tobacco mosaic virus) can also move directly from cell to cell, in the form of single-stranded nucleoprotein complexes, through pores called plasmodesmata.[111] Bacteria, like plants, have strong cell walls that a virus must breach to infect the cell. Given that bacterial cell walls are much thinner than plant cell walls due to their much smaller size, some viruses have evolved mechanisms that inject their genome into the bacterial cell across the cell wall, while the viral capsid remains outside.[112]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Replication_cycle]
    <a class="clsHide" href="#idOgmVrsatt013dsn"></a></p>
  </section>

  <section id="idOgmVrsatt014">
  <h3 id="idOgmVrsatt014H3">3_uncoating of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt014H3"></a></h3>
  <p id="idOgmVrsatt014nam">name::
    <br>* McsEngl.ogmVirus'att014-uncoating,
    <br>* McsEngl.ogmVirus'uncoating-att014,
    <br>* McsEngl.ogmVirus'life-cycle'3_uncoating,
    <a class="clsHide" href="#idOgmVrsatt014nam"></a></p>
  <p id="idOgmVrsatt014dsn">description::
    <br>"Uncoating is a process in which the viral capsid is removed: This may be by degradation by viral enzymes or host enzymes or by simple dissociation; the end-result is the releasing of the viral genomic nucleic acid.[113]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Replication_cycle]
    <a class="clsHide" href="#idOgmVrsatt014dsn"></a></p>
  </section>

  <section id="idOgmVrsatt007">
  <h3 id="idOgmVrsatt007H3">4_replication of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt007H3"></a></h3>
  <p id="idOgmVrsatt007nam">name::
    <br>* McsEngl.ogmVirus'att007-replication,
    <br>* McsEngl.ogmVirus'replication-att007,
    <br>* McsEngl.ogmVirus'life-cycle'4_replication-att007,
    <a class="clsHide" href="#idOgmVrsatt007nam"></a></p>
  <p id="idOgmVrsatt007dsn">description::
    <br>"Replication of viruses involves primarily multiplication of the genome. Replication involves synthesis of viral messenger RNA (mRNA) from "early" genes (with exceptions for positive sense RNA viruses), viral protein synthesis, possible assembly of viral proteins, then viral genome replication mediated by early or regulatory protein expression. This may be followed, for complex viruses with larger genomes, by one or more further rounds of mRNA synthesis: "late" gene expression is, in general, of structural or virion proteins.[114]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Replication_cycle]
    <a class="clsHide" href="#idOgmVrsatt007dsn"></a></p>
  </section>

  <section id="idOgmVrsatt015">
  <h3 id="idOgmVrsatt015H3">5_assembly of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt015H3"></a></h3>
  <p id="idOgmVrsatt015nam">name::
    <br>* McsEngl.ogmVirus'att015-assembly,
    <br>* McsEngl.ogmVirus'assembly-att015,
    <br>* McsEngl.ogmVirus'life-cycle'5_assembly,
    <a class="clsHide" href="#idOgmVrsatt015nam"></a></p>
  <p id="idOgmVrsatt015dsn">description::
    <br>"Assembly – Following the structure-mediated self-assembly of the virus particles, some modification of the proteins often occurs. In viruses such as HIV, this modification (sometimes called maturation) occurs after the virus has been released from the host cell.[115]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Replication_cycle]
    <a class="clsHide" href="#idOgmVrsatt015dsn"></a></p>
  </section>

  <section id="idOgmVrsatt016">
  <h3 id="idOgmVrsatt016H3">6_release of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt016H3"></a></h3>
  <p id="idOgmVrsatt016nam">name::
    <br>* McsEngl.ogmVirus'att016-release,
    <br>* McsEngl.ogmVirus'release-att016,
    <br>* McsEngl.ogmVirus'life-cycle'6_release,
    <a class="clsHide" href="#idOgmVrsatt016nam"></a></p>
  <p id="idOgmVrsatt016dsn">description::
    <br>"Release – Viruses can be released from the host cell by lysis, a process that kills the cell by bursting its membrane and cell wall if present: this is a feature of many bacterial and some animal viruses. Some viruses undergo a lysogenic cycle where the viral genome is incorporated by genetic recombination into a specific place in the host's chromosome. The viral genome is then known as a "provirus" or, in the case of bacteriophages a "prophage".[116] Whenever the host divides, the viral genome is also replicated. The viral genome is mostly silent within the host. At some point, the provirus or prophage may give rise to active virus, which may lyse the host cells.[117] Enveloped viruses (e.g., HIV) typically are released from the host cell by budding. During this process the virus acquires its envelope, which is a modified piece of the host's plasma or other, internal membrane.[118]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Replication_cycle]
    <a class="clsHide" href="#idOgmVrsatt016dsn"></a></p>
  </section>

  <section id="idOgmVrsatt019">
  <h3 id="idOgmVrsatt019H3">latency of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt019H3"></a></h3>
  <p id="idOgmVrsatt019nam">name::
    <br>* McsEngl.ogmVirus'att019-latency,
    <br>* McsEngl.ogmVirus'latency-att019,
    <a class="clsHide" href="#idOgmVrsatt019nam"></a></p>
  <p id="idOgmVrsatt019dsn">description::
    <br>"Virus latency (or viral latency) is the ability of a pathogenic virus to lie dormant (latent) within a cell, denoted as the lysogenic part of the viral life cycle.[1] A latent viral infection is a type of persistent viral infection which is distinguished from a chronic viral infection. <strong>Latency is the phase in certain viruses' life cycles</strong> in which, after initial infection, proliferation of virus particles ceases. However, the viral genome is not fully eradicated. The result of this is that the virus can reactivate and begin producing large amounts of viral progeny (the lytic part of the viral life cycle) without the host becoming reinfected by new outside virus, and stays within the host indefinitely.[2]
    <br>Virus latency is not to be confused with clinical latency during the incubation period when a virus is not dormant."
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus_latency]
    <a class="clsHide" href="#idOgmVrsatt019dsn"></a></p>
  </section>
  </section>

  <section id="idOgmVrsgst">
  <h2 id="idOgmVrsgstH2">GENERIC-SPECIFIC-TREE of ogmVirus
    <a class="clsHide" href="#idOgmVrsgstH2"></a></h2>
  <p id="idOgmVrsgstnam">name::
    <br>* McsEngl.ogmVirus'generic-specific-tree,
    <a class="clsHide" href="#idOgmVrsgstnam"></a></p>

  <section id="idOgmVrsgtr">
  <h3 id="idOgmVrsgtrH3">GENERIC-TREE of ogmVirus
    <a class="clsHide" href="#idOgmVrsgtrH3"></a></h3>
  <p id="idOgmVrsgnc">generic-of-ogmVirus::
    <br>* ogmCellNo,
    <br>===
    <br>* pathogen,
    <a class="clsHide" href="#idOgmVrsgnc"></a></p>
  <p id="idOgmVrsatttr">attribute-tree-of-ogmVirus::
    <br>* ,
    <a class="clsHide" href="#idOgmVrsatttr"></a></p>
  <p id="idOgmVrsattihf">att-tree-inherited-from::
    <br>· :
    <br>* ,
    <a class="clsHide" href="#idOgmVrsattihf"></a></p>
  <p id="idOgmVrsattown">att-tree-own-of-ogmVirus::
    <br>* ,
    <a class="clsHide" href="#idOgmVrsattown"></a></p>
  </section>

  <section id="idOgmVrsstr">
  <h3 id="idOgmVrsstrH3">SPECIFIC-TREE of ogmVirus
    <a class="clsHide" href="#idOgmVrsstrH3"></a></h3>
  <p id="idOgmVrsspc">specific-of-ogmVirus::
    <br>* ,
    <a class="clsHide" href="#idOgmVrsspc"></a></p>
  </section>

  <section id="idOgmVrsatt020">
  <h3 id="idOgmVrsatt020H3">ICTV of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt020H3"></a></h3>
  <p id="idOgmVrsatt020nam">name::
    <br>* McsEngl.ogmVirus'att020-Ictv,
    <br>* McsEngl.ogmVirus'Ictv-att020,
    <br>* McsEngl.ICTV-international-committee-on-taxonomy-of-viruses,
    <br>* McsEngl.international-committee-on-taxonomy-of-viruses-ICTV,
    <a class="clsHide" href="#idOgmVrsatt020nam"></a></p>
  <p id="idOgmVrsatt020dsn">description::
    <br>"The International Committee on Taxonomy of Viruses (ICTV) authorizes and organizes the taxonomic classification of and the nomenclatures for viruses.[1][2][3] The ICTV has developed a universal taxonomic scheme for viruses, and thus has the means to appropriately describe, name, and classify every virus that affects living organisms. The members of the International Committee on Taxonomy of Viruses are considered expert virologists.[4] The ICTV was formed from and is governed by the Virology Division of the International Union of Microbiological Societies.[5] Detailed work, such as delimiting the boundaries of species within a family, typically is performed by study groups of experts in the families.[2]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/International_Committee_on_Taxonomy_of_Viruses]
    <a class="clsHide" href="#idOgmVrsatt020dsn"></a></p>
  </section>

  <section id="idOgmVrsatt021">
  <h3 id="idOgmVrsatt021H3">Ictv-classification of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt021H3"></a></h3>
  <p id="idOgmVrsatt021nam">name::
    <br>* McsEngl.ogmVirus'att021-Ictv-classification,
    <br>* McsEngl.ogmVirus'Ictv-classification-att021,
    <a class="clsHide" href="#idOgmVrsatt021nam"></a></p>
  <p id="idOgmVrsatt021dsn">description::
    <br>"The International Committee on Taxonomy of Viruses (ICTV) developed the current classification system and wrote guidelines that put a greater weight on certain virus properties to maintain family uniformity. A unified taxonomy (a universal system for classifying viruses) has been established. Only a small part of the total diversity of viruses has been studied.[140]
    <br>The general taxonomic structure of taxon ranges actually used (as of 2018) is as follows:
    <br>* Realm (-viria)
    <br>** Subrealm (-vira)
    <br>*** Phylum (-viricota)
    <br>**** Subphylum (-viricotina)
    <br>***** Class (-viricetes)
    <br>****** Order (-virales)
    <br>******* Suborder (-virineae)
    <br>******** Family (-viridae)
    <br>********* Subfamily (-virinae)
    <br>********** Genus (-virus)
    <br>*********** Subgenus (-virus)
    <br>************ Species
    <br>As of 2018, 1 realm, 1 phylum, 2 subphyla, 6 classes, 14 orders, 7 suborders, 150 families, 79 subfamilies, 1,019 genera, 59 subgenera, and 5,560 species of viruses have been defined by the ICTV.[141] The realm Riboviria was recognized as the first realm in 2018, and the existing phylum, subphyla, and classes each belong to Riboviria.[142] Riboviria includes all RNA viruses and viroids that replicate by means of RNA-dependent RNA polymerases, which are Baltimore Groups III (dsRNA), IV ((+)ssRNA), and V ((-)ssRNA).[143][144]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#ICTV_classification]
    <a class="clsHide" href="#idOgmVrsatt021dsn"></a></p>
  </section>

  <section id="idOgmVrsatt022">
  <h3 id="idOgmVrsatt022H3">Baltimore-classification of ogmVirus
    <a class="clsHide" href="#idOgmVrsatt022H3"></a></h3>
  <p id="idOgmVrsatt022nam">name::
    <br>* McsEngl.ogmVirus'att022-Baltimore-classification,
    <br>* McsEngl.ogmVirus'Baltimore-classification-att022,
    <br>* McsEngl.Baltimore-classification-of-ogmVirusatt022,
    <a class="clsHide" href="#idOgmVrsatt022nam"></a></p>
  <p id="idOgmVrsatt022dsn">description::
    <br>"The Nobel Prize-winning biologist David Baltimore devised the Baltimore classification system.[40][145] The ICTV classification system is used in conjunction with the Baltimore classification system in modern virus classification.[146][147][148]
    <br>The Baltimore classification of viruses is based on the mechanism of mRNA production. Viruses must generate mRNAs from their genomes to produce proteins and replicate themselves, but different mechanisms are used to achieve this in each virus family. Viral genomes may be single-stranded (ss) or double-stranded (ds), RNA or DNA, and may or may not use reverse transcriptase (RT). In addition, ssRNA viruses may be either sense (+) or antisense (−). This classification places viruses into seven groups:
    <br>* I: dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses)
    <br>* II: ssDNA viruses (+ strand or "sense") DNA (e.g. Parvoviruses)
    <br>* III: dsRNA viruses (e.g. Reoviruses)
    <br>* IV: (+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses)
    <br>* V: (−)ssRNA viruses (− strand or antisense) RNA (e.g. Orthomyxoviruses, Rhabdoviruses)
    <br>* VI: ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses)
    <br>* VII: dsDNA-RT viruses DNA with RNA intermediate in life-cycle (e.g. Hepadnaviruses)
    <br>As an example of viral classification, the chicken pox virus, varicella zoster (VZV), belongs to the order Herpesvirales, family Herpesviridae, subfamily Alphaherpesvirinae, and genus Varicellovirus. VZV is in Group I of the Baltimore Classification because it is a dsDNA virus that does not use reverse transcriptase.
    <br>The complete set of viruses in an organism or habitat is called the virome; for example, all human viruses constitute the human virome.[149]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Baltimore_classification]
    <a class="clsHide" href="#idOgmVrsatt022dsn"></a></p>
  </section>
  </section>

  <section id="idOgmVrs001">
  <h2 id="idOgmVrs001H2">ogmVirus.aggregate-001
    <a class="clsHide" href="#idOgmVrs001H2"></a></h2>
  <p id="idOgmVrs001nam">name::
    <br>* McsEngl.ogmVirus.001-aggregate,
    <br>* McsEngl.ogmVirus.aggregate-001,
    <a class="clsHide" href="#idOgmVrs001nam"></a></p>
  <p id="idOgmVrs001dsn">description::
    <br>"Viruses are by far the most abundant biological entities on Earth and they outnumber all the others put together."
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Host_range]
    <br>"Since Dmitri Ivanovsky's 1892 article describing a non-bacterial pathogen infecting tobacco plants, and the discovery of the tobacco mosaic virus by Martinus Beijerinck in 1898,[2] about 5,000 virus species have been described in detail,[3] of the millions of types of viruses in the environment. Viruses are found in almost every ecosystem on Earth and are the most numerous type of biological entity."
    <br>[{2020-04-12} https://en.wikipedia.org/wiki/Virus]
    <a class="clsHide" href="#idOgmVrs001dsn"></a></p>
  </section>

  <section id="idOgmVrs024">
  <h2 id="idOgmVrs024H2">ogmVirus.virome-024
    <a class="clsHide" href="#idOgmVrs024H2"></a></h2>
  <p id="idOgmVrs024nam">name::
    <br>* McsEngl.ogmVirus.024-virome!⇒vrsVirome,
    <br>* McsEngl.ogmVirus.virome-024!⇒vrsVirome,
    <br>* McsEngl.vrsVirome,
    <br>* McsEngl.virome!⇒vrsVirome,
    <a class="clsHide" href="#idOgmVrs024nam"></a></p>
  <p id="idOgmVrs024dsn">description::
    <br>The complete set of viruses in an organism or habitat is called the virome; for example, all human viruses constitute the human virome.[149]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Baltimore_classification]
    <a class="clsHide" href="#idOgmVrs024dsn"></a></p>
  </section>

  <section id="idOgmVrs018">
  <h2 id="idOgmVrs018H2">ogmVirus.pathogenic-018
    <a class="clsHide" href="#idOgmVrs018H2"></a></h2>
  <p id="idOgmVrs018nam">name::
    <br>* McsEngl.ogmVirus.018-pathogenic!⇒vrsPathogenic,
    <br>* McsEngl.ogmVirus.pathogenic-018!⇒vrsPathogenic,
    <br>* McsEngl.pathogenic-ogmVirus-018!⇒vrsPathogenic,
    <br>* McsEngl.vrsPathogenic,
    <a class="clsHide" href="#idOgmVrs018nam"></a></p>
  <p id="idOgmVrs018dsn">description::
    <br>"Pathogenic viruses are mainly from the families: Adenoviridae, Picornaviridae, Herpesviridae, Hepadnaviridae, Flaviviridae, Retroviridae, Orthomyxoviridae, Paramyxoviridae, Papovaviridae, Polyomavirus, Rhabdoviridae, and Togaviridae. HIV is a notable member of the family Retroviridae which affected 37.9 million people across the world in 2018."
    <br>[{2020-04-14} https://en.wikipedia.org/wiki/Pathogen#Viruses]
    <a class="clsHide" href="#idOgmVrs018dsn"></a></p>

  <section id="idOgmVrs018dsdr">
  <h3 id="idOgmVrs018dsdrH3">disorder-caused (link) of vrsPathogenic
    <a class="clsHide" href="#idOgmVrs018dsdrH3"></a></h3>
  </section>
  </section>

  <section id="idOgmVrs019">
  <h2 id="idOgmVrs019H2">ogmVirus.pathogenicNo-019
    <a class="clsHide" href="#idOgmVrs019H2"></a></h2>
  <p id="idOgmVrs019nam">name::
    <br>* McsEngl.ogmVirus.019-pathogenicNo!⇒vrsPathogenicNo,
    <br>* McsEngl.ogmVirus.pathogenicNo-019!⇒vrsPathogenicNo,
    <br>* McsEngl.pathogenicNo-ogmVirus-019!⇒vrsPathogenicNo,
    <br>* McsEngl.vrsPathogenicNo,
    <a class="clsHide" href="#idOgmVrs019nam"></a></p>
  <p id="idOgmVrs019dsn">description::
    <br>·
    <a class="clsHide" href="#idOgmVrs019dsn"></a></p>
  </section>

  <section id="idOgmVrs015">
  <h2 id="idOgmVrs015H2">ogmVirus.generic-015
    <a class="clsHide" href="#idOgmVrs015H2"></a></h2>
  <p id="idOgmVrs015nam">name::
    <br>* McsEngl.ogmVirus.015-generic,
    <br>* McsEngl.ogmVirus.generic-015,
    <a class="clsHide" href="#idOgmVrs015nam"></a></p>
  <p id="idOgmVrs015dsn">description::
    <br>· generic-virus is a-not-instance-virus.
    <a class="clsHide" href="#idOgmVrs015dsn"></a></p>
  </section>

  <section id="idOgmVrs014">
  <h2 id="idOgmVrs014H2">ogmVirus.virion-014
    <a class="clsHide" href="#idOgmVrs014H2"></a></h2>
  <p id="idOgmVrs014nam">name::
    <br>* McsEngl.ogmVirus.014-virion!⇒vrsVirion,
    <br>* McsEngl.ogmVirus.virion-014!⇒vrsVirion,
    <br>* McsEngl.virus-particle!⇒vrsVirion,
    <br>* McsEngl.vrsVirion,
    <a class="clsHide" href="#idOgmVrs014nam"></a></p>
  <p id="idOgmVrs014dsn">description::
    <br>· virion is an-instance of virus.
    <br>===
    <br>"A complete virus particle, known as a virion, consists of nucleic acid surrounded by a protective coat of protein called a capsid."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Virus]
    <a class="clsHide" href="#idOgmVrs014dsn"></a></p>
  </section>

  <section id="idOgmVrsHmn">
  <h2 id="idOgmVrsHmnH2">ogmVirus.human-002
    <a class="clsHide" href="#idOgmVrsHmnH2"></a></h2>
  <p id="idOgmVrsHmnnam">name::
    <br>* McsEngl.ogmVirus.002-human!⇒virusHmn,
    <br>* McsEngl.ogmVirus.human-002!⇒virusHmn,
    <br>* McsEngl.bodyHmn'att018-virus!⇒virusHmn,
    <br>* McsEngl.bodyHmn'virus-att018!⇒virusHmn,
    <br>* McsEngl.human-virus!⇒virusHmn,
    <br>* McsEngl.ogmHstHmn.virus!⇒virusHmn,
    <br>* McsEngl.virusHmn,
    <a class="clsHide" href="#idOgmVrsHmnnam"></a></p>
  <p id="idOgmVrsHmndsn">description::
    <br>"Viruses consist of a genome and sometimes a few enzymes stored in a capsule made of protein (called a capsid), and sometimes covered with a lipid layer (sometimes called an 'envelope').
    <br>Viruses cannot reproduce on their own, and instead propagate by subjugating a host cell to produce copies of themselves, thus producing the next generation."
    <br>[{2020-03-09} https://en.wikipedia.org/wiki/Antiviral_drug#Virus_life_cycle]
    <br>===
    <br>"Viruses, especially bacterial viruses (bacteriophages), colonize various body sites. These colonized sites include the skin,[37] gut,[38] lungs,[39] and oral cavity.[40] Virus communities have been associated with some diseases, and do not simply reflect the bacterial communities.[41][42][43]"
    <br>[{2020-02-16} https://en.wikipedia.org/wiki/Human_microbiome#Viruses]
    <a class="clsHide" href="#idOgmVrsHmndsn"></a></p>

  <section id="idOgmVrsHmnsrtr">
  <h3 id="idOgmVrsHmnsrtrH3">structure of virusHmn
    <a class="clsHide" href="#idOgmVrsHmnsrtrH3"></a></h3>
  <p id="idOgmVrsHmnsrtrnam">name::
    <br>* McsEngl.virusHmn'structure,
    <a class="clsHide" href="#idOgmVrsHmnsrtrnam"></a></p>
  <p id="idOgmVrsHmnsrtrdsn">description::
    <br>"Viruses consist of a genome and sometimes a few enzymes stored in a capsule made of protein (called a capsid), and sometimes covered with a lipid layer (sometimes called an 'envelope')."
    <br>[{2020-03-09} https://en.wikipedia.org/wiki/Antiviral_drug#Virus_life_cycle]
    <a class="clsHide" href="#idOgmVrsHmnsrtrdsn"></a></p>
  </section>

  <section id="idOgmVrsHmnevg">
  <h3 id="idOgmVrsHmnevgH3">evoluting of virusHmn
    <a class="clsHide" href="#idOgmVrsHmnevgH3"></a></h3>
  <p id="idOgmVrsHmnevgnam">name::
    <br>* McsEngl.virusHmn'evoluting,
    <a class="clsHide" href="#idOgmVrsHmnevgnam"></a></p>
  <p id="idOgmVrsHmn1901">{time.1901}::
    <br>"Viruses were confirmed as human pathogens in 1901, with the discovery of the yellow fever virus by Walter Reed."
    <br>[{2020-02-22} https://en.wikipedia.org/wiki/Immune_system]
    <a class="clsHide" href="#idOgmVrsHmn1901"></a></p>
  </section>
  </section>

  <section id="idOgmVrs016">
  <h2 id="idOgmVrs016H2">ogmVirus.enveloped-016
    <a class="clsHide" href="#idOgmVrs016H2"></a></h2>
  <p id="idOgmVrs016nam">name::
    <br>* McsEngl.ogmVirus.0160enveloped!⇒vrsEnveloped,
    <br>* McsEngl.ogmVirus.enveloped-016!⇒vrsEnveloped,
    <br>* McsEngl.vrsEnveloped,
    <a class="clsHide" href="#idOgmVrs016nam"></a></p>
  <p id="idOgmVrs016dsn">description::
    <br>Enveloped examples
    <br>Classes of enveloped viruses that contain human pathogens:
    <br>DNA viruses
    <br>* Herpesvirus
    <br>* Poxviruses
    <br>* Hepadnaviruses
    <br>* Asfarviridae
    <br>RNA viruses
    <br>* Flavivirus
    <br>* Alphavirus
    <br>* Togavirus
    <br>* Coronavirus
    <br>* Hepatitis D
    <br>* Orthomyxovirus
    <br>* Paramyxovirus
    <br>* Rhabdovirus[2]
    <br>* Bunyavirus
    <br>* Filovirus
    <br>Retroviruses
    <br>* Retroviruses - env
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Viral_envelope]
    <a class="clsHide" href="#idOgmVrs016dsn"></a></p>

  <section id="idOgmVrs016att001">
  <h3 id="idOgmVrs016att001H3">envelope of vrsEnveloped
    <a class="clsHide" href="#idOgmVrs016att001H3"></a></h3>
  <p id="idOgmVrs016att001nam">name::
    <br>* McsEngl.vrsEnveloped'envelope,
    <a class="clsHide" href="#idOgmVrs016att001nam"></a></p>
  <p id="idOgmVrs016att001dsn">description::
    <br>"Most viruses (e.g. HIV and many animal viruses) have viral envelopes as their outer layer[1] at the stage of their life-cycle when they are between host cells. The envelopes are typically derived from portions of the host cell membranes (phospholipids and proteins), but include some viral glycoproteins. They may help viruses avoid the host immune system. Glycoproteins on the surface of the envelope serve to identify and bind to receptor sites on the host's membrane. The viral envelope then fuses with the host's membrane, allowing the capsid and viral genome to enter and infect the host.
    <br>The cell from which the virus itself buds will often die or be weakened and shed more viral particles for an extended period. The lipid bilayer envelope of these viruses is relatively sensitive to desiccation, heat, and detergents, therefore these viruses are easier to sterilize than non-enveloped viruses, have limited survival outside host environments, and typically transfer directly from host to host. Enveloped viruses possess great adaptability and can change in a short time in order to evade the immune system. Enveloped viruses can cause persistent infections."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Viral_envelope]
    <a class="clsHide" href="#idOgmVrs016att001dsn"></a></p>
  </section>
  </section>

  <section id="idOgmVrs017">
  <h2 id="idOgmVrs017H2">ogmVirus.envelopedNo-017
    <a class="clsHide" href="#idOgmVrs017H2"></a></h2>
  <p id="idOgmVrs017nam">name::
    <br>* McsEngl.ogmVirus.017-envelopedNo!⇒vrsEnvelopedNo,
    <br>* McsEngl.ogmVirus.envelopedNo-017!⇒vrsEnvelopedNo,
    <br>* McsEngl.vrsEnvelopedNo,
    <a class="clsHide" href="#idOgmVrs017nam"></a></p>
  <p id="idOgmVrs017dsn">description::
    <br>"Nonenveloped examples
    <br>Classes of nonenveloped viruses that contain human pathogens:
    <br>DNA viruses
    <br>* Adenoviridae
    <br>* Papillomaviridae
    <br>RNA viruses
    <br>* Picornaviridae
    <br>* Caliciviridae"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Viral_envelope]
    <a class="clsHide" href="#idOgmVrs017dsn"></a></p>
  </section>

  <section id="idOgmVrs003">
  <h2 id="idOgmVrs003H2">ogmVirus.SARS-CoV-2-003
    <a class="clsHide" href="#idOgmVrs003H2"></a></h2>
  <p id="idOgmVrs003nam">name::
    <br>* McsEngl.ogmVirus.003-SARS-CoV-2!⇒vrsSarsc2,
    <br>* McsEngl.ogmVirus.SARS-CoV-2-003!⇒vrsSarsc2,
    <br>* McsEngl.SARS-CoV-2!⇒vrsSarsc2,
    <br>* McsEngl.virus.SARS-CoV-2!⇒vrsSarsc2,
    <br>* McsEngl.vrsSarsc2,
    <a class="clsHide" href="#idOgmVrs003nam"></a></p>
  <p id="idOgmVrs003dsn">description::
    <br>"Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)[2][3] is the virus strain that causes coronavirus disease 2019 (COVID-19), a respiratory illness. It is colloquially known as the coronavirus, and was previously referred to by its provisional name 2019 novel coronavirus (2019-nCoV).[4][5][6] SARS-CoV-2 is a positive-sense single-stranded RNA virus.[7] It is contagious in humans, and the World Health Organization (WHO) has designated the ongoing pandemic of COVID-19 a Public Health Emergency of International Concern.[8][9][10] Because the strain was first discovered in Wuhan, China, it is sometimes referred to as the "Wuhan virus"[11][12] or "Wuhan coronavirus".[13][14][15] Since the WHO discourages the use of names based upon locations[16][17] and to avoid confusion with the disease SARS,[18] it sometimes refers to SARS-CoV-2 as "the COVID-19 virus" in public health communications.[19] The general public frequently calls both SARS-CoV-2 and the disease it causes "coronavirus", but scientists typically use more precise terminology.[20]
    <br>Taxonomically, SARS-CoV-2 is a strain of Severe acute respiratory syndrome-related coronavirus (SARSr-CoV).[2] It is believed to have zoonotic origins and has close genetic similarity to bat coronaviruses, suggesting it emerged from a bat-borne virus.[21][22][23] An intermediate animal reservoir such as a pangolin is also thought to be involved in its introduction to humans.[16][24] The virus shows little genetic diversity, indicating that the spillover event introducing SARS-CoV-2 to humans is likely to have occurred in late 2019.[25]
    <br>Epidemiological studies estimate each infection results in 1.4 to 3.9 new ones when no members of the community are immune and no preventive measures taken. The virus is primarily spread between people through close contact and via respiratory droplets produced from coughs or sneezes.[26][27] It mainly enters human cells by binding to the receptor angiotensin converting enzyme 2 (ACE2).[21][28][29]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2]
    <a class="clsHide" href="#idOgmVrs003dsn"></a></p>
  <p id="idOgmVrs003gtr">generic-tree-of-::
    <br>* coronavirus,
    <a class="clsHide" href="#idOgmVrs003gtr"></a></p>

  <section id="idOgmVrs003evg">
  <h3 id="idOgmVrs003evgH3">evoluting of vrsSarsc2
    <a class="clsHide" href="#idOgmVrs003evgH3"></a></h3>
  <p id="idOgmVrs003evgnam">name::
    <br>* McsEngl.vrsSarsc2'evoluting,
    <a class="clsHide" href="#idOgmVrs003evgnam"></a></p>

  <section id="idOgmVrs003lftm">
  <h4 id="idOgmVrs003lftmH4">lifetime of vrsSarsc2
    <a class="clsHide" href="#idOgmVrs003lftmH4"></a></h4>
  <p id="idOgmVrs003lftmnam">name::
    <br>* McsEngl.vrsSarsc2'lifetime,
    <a class="clsHide" href="#idOgmVrs003lftmnam"></a></p>
  <p id="idOgmVrs003lftmdsn">description::
    <br>"Preliminary research indicates that the virus may remain viable on plastic and steel for up to three days, but does not survive on cardboard for more than one day or on copper for more than four hours;[32] the virus is inactivated by soap, which destabilises its lipid bilayer."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2]
    <a class="clsHide" href="#idOgmVrs003lftmdsn"></a></p>
  </section>
  </section>
  </section>

  <section id="idOgmVrs004">
  <h2 id="idOgmVrs004H2">ogmVirus.coronavirus-004
    <a class="clsHide" href="#idOgmVrs004H2"></a></h2>
  <p id="idOgmVrs004nam">name::
    <br>* McsEngl.ogmVirus.004-coronavirus!⇒vrsCorona,
    <br>* McsEngl.ogmVirus.coronavirus-004!⇒vrsCorona,
    <br>* McsEngl.vrsCorona,
    <a class="clsHide" href="#idOgmVrs004nam"></a></p>
  <p id="idOgmVrs004dsn">description::
    <br>"Coronaviruses are a group of related viruses that cause diseases in mammals and birds. In humans, coronaviruses cause respiratory tract infections that can range from mild to lethal. Mild illnesses include some cases of the common cold (which has other possible causes, predominantly rhinoviruses), while more lethal varieties can cause SARS, MERS, and COVID-19. Symptoms in other species vary: in chickens, they cause an upper respiratory tract disease, while in cows and pigs they cause diarrhea. There are yet to be vaccines or antiviral drugs to prevent or treat human coronavirus infections.
    <br>Coronaviruses constitute the subfamily Orthocoronavirinae, in the family Coronaviridae, order Nidovirales, and realm Riboviria.[5][6] They are enveloped viruses with a positive-sense single-stranded RNA genome and a nucleocapsid of helical symmetry. The genome size of coronaviruses ranges from approximately 26 to 32 kilobases, one of the largest among RNA viruses.[7] They have characteristic club-shaped spikes that project from their surface, which in electron micrographs create an image reminiscent of the solar corona from which their name derives.[8]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Coronavirus]
    <a class="clsHide" href="#idOgmVrs004dsn"></a></p>

  <section id="idOgmVrs004gst">
  <h3 id="idOgmVrs004gstH3">GENERIC-SPECIFIC-TREE of vrsCorona
    <a class="clsHide" href="#idOgmVrs004gstH3"></a></h3>
  <p id="idOgmVrs004gstnam">name::
    <br>* McsEngl.vrsCorona'generic-specific-tree,
    <a class="clsHide" href="#idOgmVrs004gstnam"></a></p>
  <p id="idOgmVrs004gtr">generic-tree-of-vrsCorona::
    <br>*
    <br>* ... <a class="clsPreview" href="../dirCor/McsCor000003.last.html#idOverview">entity</a>,
    <a class="clsHide" href="#idOgmVrs004gtr"></a></p>
  <p id="idOgmVrs004attihf">att-inherited-from::
    <br>· :
    <br>* ,
    <a class="clsHide" href="#idOgmVrs004attihf"></a></p>
  <p id="idOgmVrs004attown">att-own-of-vrsCorona::
    <br>* ,
    <a class="clsHide" href="#idOgmVrs004attown"></a></p>
  <p id="idOgmVrs004str">specific-tree-of-vrsCorona::
    <br>"It [SARS-CoV-2] is the seventh known coronavirus to infect people, after 229E, NL63, OC43, HKU1, MERS-CoV, and the original SARS-CoV."
    <br>[https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2]
    <a class="clsHide" href="#idOgmVrs004str"></a></p>
  </section>
  </section>

  <section id="idOgmVrs005">
  <h2 id="idOgmVrs005H2">ogmVirus.Dna-005
    <a class="clsHide" href="#idOgmVrs005H2"></a></h2>
  <p id="idOgmVrs005nam">name::
    <br>* McsEngl.ogmVirus.005-Dna!⇒vrsDna,
    <br>* McsEngl.ogmVirus.Dna-005!⇒vrsDna,
    <br>* McsEngl.vrsDna,
    <a class="clsHide" href="#idOgmVrs005nam"></a></p>
  <p id="idOgmVrs005dsn">description::
    <br>"The genome replication of most DNA viruses takes place in the cell's nucleus. If the cell has the appropriate receptor on its surface, these viruses enter the cell either by direct fusion with the cell membrane (e.g., herpesviruses) or—more usually—by receptor-mediated endocytosis. Most DNA viruses are entirely dependent on the host cell's DNA and RNA synthesising machinery, and RNA processing machinery. Viruses with larger genomes may encode much of this machinery themselves. In eukaryotes the viral genome must cross the cell's nuclear membrane to access this machinery, while in bacteria it need only enter the cell.[119]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Virus#Genome_replication]
    <a class="clsHide" href="#idOgmVrs005dsn"></a></p>
  </section>

  <section id="idOgmVrs006">
  <h2 id="idOgmVrs006H2">ogmVirus.Rna-006
    <a class="clsHide" href="#idOgmVrs006H2"></a></h2>
  <p id="idOgmVrs006nam">name::
    <br>* McsEngl.ogmVirus.006-Rna!⇒vrsRna,
    <br>* McsEngl.ogmVirus.Rna-006!⇒vrsRna,
    <br>* McsEngl.vrsRna,
    <a class="clsHide" href="#idOgmVrs006nam"></a></p>
  <p id="idOgmVrs006dsn">description::
    <br>"Replication of RNA viruses usually takes place in the cytoplasm. RNA viruses can be placed into four different groups depending on their modes of replication. The polarity (whether or not it can be used directly by ribosomes to make proteins) of single-stranded RNA viruses largely determines the replicative mechanism; the other major criterion is whether the genetic material is single-stranded or double-stranded. All RNA viruses use their own RNA replicase enzymes to create copies of their genomes."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Virus#Genome_replication]
    <a class="clsHide" href="#idOgmVrs006dsn"></a></p>
  </section>

  <section id="idOgmVrs007">
  <h2 id="idOgmVrs007H2">ogmVirus.animal-007
    <a class="clsHide" href="#idOgmVrs007H2"></a></h2>
  <p id="idOgmVrs007nam">name::
    <br>* McsEngl.ogmVirus.007-animal!⇒vrsAnimal,
    <br>* McsEngl.ogmVirus.animal-007!⇒vrsAnimal,
    <br>* McsEngl.vrsAnimal,
    <a class="clsHide" href="#idOgmVrs007nam"></a></p>
  <p id="idOgmVrs007dsn">description::
    <br>"Animal viruses are viruses that infect animals. Viruses infect all cellular life and although viruses infect every animal, plant, fungus and protist species, each has its own specific range of viruses that often infect only that species."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Animal_virus]
    <a class="clsHide" href="#idOgmVrs007dsn"></a></p>
  </section>

  <section id="idOgmVrs008">
  <h2 id="idOgmVrs008H2">ogmVirus.plant-008
    <a class="clsHide" href="#idOgmVrs008H2"></a></h2>
  <p id="idOgmVrs008nam">name::
    <br>* McsEngl.ogmVirus.008-plant!⇒vrsPlant,
    <br>* McsEngl.ogmVirus.plant-008!⇒vrsPlant,
    <br>* McsEngl.vrsPlant,
    <a class="clsHide" href="#idOgmVrs008nam"></a></p>
  <p id="idOgmVrs008dsn">description::
    <br>"Plant viruses are viruses that affect plants. Like all other viruses, plant viruses are obligate intracellular parasites that do not have the molecular machinery to replicate without a host. Plant viruses can be pathogenic to higher plants.
    <br>Most plant viruses are rod-shaped, with protein discs forming a tube surrounding the viral genome; isometric particles are another common structure. They rarely have an envelope. The great majority have an RNA genome, which is usually small and single stranded (ss), but some viruses have double-stranded (ds) RNA, ssDNA or dsDNA genomes. Although plant viruses are not as well understood as their animal counterparts, one plant virus has become iconic: tobacco mosaic virus (TMV), the first virus to be discovered. This and other viruses cause an estimated US $60 billion loss in crop yields worldwide each year. Plant viruses are grouped into 73 genera and 49 families. However, these figures relate only to cultivated plants, which represent only a tiny fraction of the total number of plant species. Viruses in wild plants have been relatively little studied, but the interactions between wild plants and their viruses often do not appear to cause disease in the host plants.[1]
    <br>To transmit from one plant to another and from one plant cell to another, plant viruses must use strategies that are usually different from animal viruses. Plants do not move, and so plant-to-plant transmission usually involves vectors (such as insects). Plant cells are surrounded by solid cell walls, therefore transport through plasmodesmata is the preferred path for virions to move between plant cells. Plants have specialized mechanisms for transporting mRNAs through plasmodesmata, and these mechanisms are thought to be used by RNA viruses to spread from one cell to another.[2] Plant defenses against viral infection include, among other measures, the use of siRNA in response to dsRNA.[3] Most plant viruses encode a protein to suppress this response.[4] Plants also reduce transport through plasmodesmata in response to injury.[2]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Plant_virus]
    <a class="clsHide" href="#idOgmVrs008dsn"></a></p>
  </section>

  <section id="idOgmVrs009">
  <h2 id="idOgmVrs009H2">ogmVirus.bacteriophage-009
    <a class="clsHide" href="#idOgmVrs009H2"></a></h2>
  <p id="idOgmVrs009nam">name::
    <br>* McsEngl.ogmVirus.009-bacteriophage!⇒vrsBacteriophage,
    <br>* McsEngl.ogmVirus.bacteriophage-009!⇒vrsBacteriophage,
    <br>* McsEngl.ogmVirus.phage-009!⇒vrsBacteriophage,
    <br>* McsEngl.phage!⇒vrsBacteriophage,
    <br>* McsEngl.vrsBacteriophage, /baktírioféijj/,
    <a class="clsHide" href="#idOgmVrs009nam"></a></p>
  <p id="idOgmVrs009dsn">description::
    <br>"A bacteriophage (/bækˈtɪərioʊfeɪdʒ/), also known informally as a phage (/feɪdʒ/), is a virus that infects and replicates within bacteria and archaea. The term was derived from "bacteria" and the Greek φαγεῖν (phagein), meaning "to devour". Bacteriophages are composed of proteins that encapsulate a DNA or RNA genome, and may have structures that are either simple or elaborate. Their genomes may encode as few as four genes (e.g. MS2) and as many as hundreds of genes. Phages replicate within the bacterium following the injection of their genome into its cytoplasm.
    <br>Bacteriophages are among the most common and diverse entities in the biosphere.[1] Bacteriophages are ubiquitous viruses, found wherever bacteria exist. It is estimated there are more than 1031 bacteriophages on the planet, more than every other organism on Earth, including bacteria, combined.[2] One of the densest natural sources for phages and other viruses is seawater, where up to 9x108 virions per millilitre have been found in microbial mats at the surface,[3] and up to 70% of marine bacteria may be infected by phages.[4]
    <br>Phages have been used since the late 19th century as an alternative to antibiotics in the former Soviet Union and Central Europe, as well as in France.[5][6] They are seen as a possible therapy against multi-drug-resistant strains of many bacteria (see phage therapy).[7] Phages of Inoviridae have been shown to complicate biofilms involved in pneumonia and cystic fibrosis and to shelter the bacteria from drugs meant to eradicate disease, thus promoting persistent infection.[8]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Bacteriophage]
    <a class="clsHide" href="#idOgmVrs009dsn"></a></p>
  </section>

  <section id="idOgmVrs010">
  <h2 id="idOgmVrs010H2">ogmVirus.archaeal-010
    <a class="clsHide" href="#idOgmVrs010H2"></a></h2>
  <p id="idOgmVrs010nam">name::
    <br>* McsEngl.ogmVirus.010-archaeal!⇒vrsArchaeal,
    <br>* McsEngl.ogmVirus.archaeal-010!⇒vrsArchaeal,
    <br>* McsEngl.vrsArchaeal,
    <a class="clsHide" href="#idOgmVrs010nam"></a></p>
  <p id="idOgmVrs010dsn">description::
    <br>"Archaeal viruses
    <br>Some viruses replicate within archaea: these are double-stranded DNA viruses with unusual and sometimes unique shapes.[5][88] These viruses have been studied in most detail in the thermophilic archaea, particularly the orders Sulfolobales and Thermoproteales.[239] Defences against these viruses involve RNA interference from repetitive DNA sequences within archaean genomes that are related to the genes of the viruses.[240][241] Most archaea have CRISPR–Cas systems as an adaptive defence against viruses. These enable archaea to retain sections of viral DNA, which are then used to target and eliminate subsequent infections by the virus using a process similar to RNA interference."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Virus#Archaeal_viruses]
    <a class="clsHide" href="#idOgmVrs010dsn"></a></p>
  </section>

  <section id="idOgmVrs011">
  <h2 id="idOgmVrs011H2">ogmVirus.tobacco-mosaic-virus-011
    <a class="clsHide" href="#idOgmVrs011H2"></a></h2>
  <p id="idOgmVrs011nam">name::
    <br>* McsEngl.ogmVirus.011-tobacco-mosaic-virus!⇒vrsTmv,
    <br>* McsEngl.ogmVirus.tobacco-mosaic-virus-011!⇒vrsTmv,
    <br>* McsEngl.TMV-tobacco-mosaic-virus!⇒vrsTmv,
    <br>* McsEngl.tobacco-mosaic-virus-Tmv!⇒vrsTmv,
    <br>* McsEngl.vrsTmv,
    <a class="clsHide" href="#idOgmVrs011nam"></a></p>
  <p id="idOgmVrs011dsn">description::
    <br>"Tobacco mosaic virus (TMV) is a positive-sense single stranded RNA virus in the genus Tobamovirus that infects a wide range of plants, especially tobacco and other members of the family Solanaceae. The infection causes characteristic patterns, such as "mosaic"-like mottling and discoloration on the leaves (hence the name). TMV was the first virus to be discovered. Although it was known from the late 19th century that a non-bacterial infectious disease was damaging tobacco crops, it was not until 1930 that the infectious agent was determined to be a virus. It is the first pathogen identified as a virus."
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Tobacco_mosaic_virus]
    <a class="clsHide" href="#idOgmVrs011dsn"></a></p>
  </section>

  <section id="idOgmVrs012">
  <h2 id="idOgmVrs012H2">ogmVirus.retrovirus-012
    <a class="clsHide" href="#idOgmVrs012H2"></a></h2>
  <p id="idOgmVrs012nam">name::
    <br>* McsEngl.ogmVirus.012-retrovirus!⇒vrsRetro,
    <br>* McsEngl.ogmVirus.retrovirus-012!⇒vrsRetro,
    <br>* McsEngl.vrsRetro,
    <a class="clsHide" href="#idOgmVrs012nam"></a></p>
  <p id="idOgmVrs012dsn">description::
    <br>"A retrovirus is a type of RNA virus[a] that inserts a copy of its genome into the DNA of a host cell that it invades, thus changing the genome of that cell.[3] Once inside the host cell's cytoplasm, the virus uses its own reverse transcriptase enzyme to produce DNA from its RNA genome, the reverse of the usual pattern, thus retro (backwards). The new DNA is then incorporated into the host cell genome by an integrase enzyme, at which point the retroviral DNA is referred to as a provirus. The host cell then treats the viral DNA as part of its own genome, transcribing and translating the viral genes along with the cell's own genes, producing the proteins required to assemble new copies of the virus. It is difficult to detect the virus until it has infected the host. At that point, the infection will persist indefinitely.[citation needed]
    <br>Human retroviruses include HIV-1 and HIV-2, the cause of the disease AIDS. Retroviruses are also a valuable research tools in molecular biology, and they have been used successfully in gene delivery systems.[4]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Retrovirus]
    <a class="clsHide" href="#idOgmVrs012dsn"></a></p>
  </section>

  <section id="idOgmVrs013">
  <h2 id="idOgmVrs013H2">ogmVirus.cytomegalovirus-013
    <a class="clsHide" href="#idOgmVrs013H2"></a></h2>
  <p id="idOgmVrs013nam">name::
    <br>* McsEngl.ogmVirus.013-cytomegalovirus!⇒vrsCytomega,
    <br>* McsEngl.ogmVirus.cytomegalovirus-013!⇒vrsCytomega,
    <br>* McsEngl.ogmVirus.vrsCytomega,
    <a class="clsHide" href="#idOgmVrs013nam"></a></p>
  <p id="idOgmVrs013dsn">description::
    <br>"Cytomegalovirus (CMV) (from the Greek cyto-, "cell," and megalo-, "large") is a genus of viruses in the order Herpesvirales, in the family Herpesviridae, in the subfamily Betaherpesvirinae. Humans and monkeys serve as natural hosts. The eight species in this genus include the type species, Human betaherpesvirus 5 (HCMV, human cytomegalovirus, HHV-5), which is the species that infects humans. Diseases associated with HHV-5 include mononucleosis, and pneumonia.[3][4] In the medical literature, most mentions of CMV without further specification refer implicitly to human CMV. Human CMV is the most studied of all cytomegaloviruses.[5]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Cytomegalovirus]
    <a class="clsHide" href="#idOgmVrs013dsn"></a></p>
  </section>

  <section id="idOgmVrs020">
  <h2 id="idOgmVrs020H2">ogmVirus.giant-020
    <a class="clsHide" href="#idOgmVrs020H2"></a></h2>
  <p id="idOgmVrs020nam">name::
    <br>* McsEngl.ogmVirus.020-giant!⇒vrsGiant,
    <br>* McsEngl.ogmVirus.giant-020!⇒vrsGiant,
    <br>* McsEngl.giant-ogmVirus-020!⇒vrsGiant,
    <br>* McsEngl.vrsGiant,
    <a class="clsHide" href="#idOgmVrs020nam"></a></p>
  <p id="idOgmVrs020dsn">description::
    <br>"Mimivirus is one of the largest characterised viruses, with a capsid diameter of 400 nm. Protein filaments measuring 100 nm project from the surface. The capsid appears hexagonal under an electron microscope, therefore the capsid is probably icosahedral.[84] In 2011, researchers discovered the largest then known virus in samples of water collected from the ocean floor off the coast of Las Cruces, Chile. Provisionally named Megavirus chilensis, it can be seen with a basic optical microscope.[85] In 2013, the Pandoravirus genus was discovered in Chile and Australia, and has genomes about twice as large as Megavirus and Mimivirus.[86] All giant viruses have dsDNA genomes and they are classified into several families: Mimiviridae, Pithoviridae, Pandoraviridae, Phycodnaviridae, and the Mollivirus genus.[87]"
    <br>[{2020-04-14} https://en.wikipedia.org/wiki/Virus#Giant_viruses]
    <a class="clsHide" href="#idOgmVrs020dsn"></a></p>
  </section>

  <section id="idOgmVrs021">
  <h2 id="idOgmVrs021H2">ogmVirus.descendant-021
    <a class="clsHide" href="#idOgmVrs021H2"></a></h2>
  <p id="idOgmVrs021nam">name::
    <br>* McsEngl.ogmVirus.021-descendant!⇒vrsDescendant,
    <br>* McsEngl.ogmVirus.descendant-021!⇒vrsDescendant,
    <br>* McsEngl.descendant-ogmVirus-021!⇒vrsDescendant,
    <br>* McsEngl.vrsDescendant,
    <a class="clsHide" href="#idOgmVrs021nam"></a></p>
  <p id="idOgmVrs021dsn">description::
    <br>"One virus particle can produce millions of progeny viruses in just one cycle of replication, therefore the production of a few "dud" viruses is not a problem."
    <br>[{2020-04-14} https://en.wikipedia.org/wiki/Viral_evolution]
    <a class="clsHide" href="#idOgmVrs021dsn"></a></p>
  </section>

  <section id="idOgmVrs022">
  <h2 id="idOgmVrs022H2">ogmVirus.antecedent-022
    <a class="clsHide" href="#idOgmVrs022H2"></a></h2>
  <p id="idOgmVrs022nam">name::
    <br>* McsEngl.ogmVirus.022-antecedent!⇒vrsAntecedent,
    <br>* McsEngl.ogmVirus.antecedent-022!⇒vrsAntecedent,
    <br>* McsEngl.antecedent-ogmVirus-022!⇒vrsAntecedent,
    <br>* McsEngl.vrsAntecedent,
    <a class="clsHide" href="#idOgmVrs022nam"></a></p>
  <p id="idOgmVrs022dsn">description::
    <br>·
    <a class="clsHide" href="#idOgmVrs022dsn"></a></p>
  </section>

  <section id="idOgmVrs023">
  <h2 id="idOgmVrs023H2">ogmVirus.rota-023
    <a class="clsHide" href="#idOgmVrs023H2"></a></h2>
  <p id="idOgmVrs023nam">name::
    <br>* McsEngl.ogmVirus.023-rota,
    <br>* McsEngl.ogmVirus.rota-023,
    <br>* McsEngl.rotavirus-023,
    <br>* McsEngl.vrsRota,
    <a class="clsHide" href="#idOgmVrs023nam"></a></p>
  <p id="idOgmVrs023dsn">description::
    <br>"Rotavirus is a genus of double-stranded RNA viruses in the family Reoviridae. Rotaviruses are the most common cause of diarrhoeal disease among infants and young children.[1] Nearly every child in the world is infected with a rotavirus at least once by the age of five.[2] Immunity develops with each infection, so subsequent infections are less severe; adults are rarely affected.[3] There are ten species of the genus, referred to as A, B, C, D, E, F, G, H, I and J. Rotavirus A, the most common species, causes more than 90% of rotavirus infections in humans.
    <br>The virus is transmitted by the faecal-oral route. It infects and damages the cells that line the small intestine and causes gastroenteritis (which is often called "stomach flu" despite having no relation to influenza). Although Rotavirus was discovered in 1973 by Ruth Bishop and her colleagues by electron micrograph images[4] and accounts for approximately one third of hospitalisations for severe diarrhoea in infants and children,[5] its importance has historically been underestimated within the public health community, particularly in developing countries.[6] In addition to its impact on human health, rotavirus also infects other animals, and is a pathogen of livestock.[7]
    <br>Rotaviral enteritis is usually an easily managed disease of childhood, but in 2013, rotaviruses caused 37 percent of deaths of children from diarrhoea and 215,000 deaths worldwide,[8] and almost two million more became severely ill.[6] Most of these deaths occurred in developing countries.[9] In the United States, before initiation of the rotavirus vaccination programme in the 2000s, rotavirus caused about 2.7 million cases of severe gastroenteritis in children, almost 60,000 hospitalisations, and around 37 deaths each year.[10] Following rotavirus vaccine introduction in the United States, hospitalisation rates have fallen significantly.[11][12] Public health campaigns to combat rotavirus focus on providing oral rehydration therapy for infected children and vaccination to prevent the disease.[13] The incidence and severity of rotavirus infections has declined significantly in countries that have added rotavirus vaccine to their routine childhood immunisation policies.[14][15][16]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Rotavirus]
    <a class="clsHide" href="#idOgmVrs023dsn"></a></p>
  </section>

  <section id="idOgmVrs024">
  <h2 id="idOgmVrs024H2">ogmVirus.oncovirus-024
    <a class="clsHide" href="#idOgmVrs024H2"></a></h2>
  <p id="idOgmVrs024nam">name::
    <br>* McsEngl.ogmVirus.024oncovirus!⇒vrsOnco,
    <br>* McsEngl.ogmVirus.oncovirus-024!⇒vrsOnco,
    <br>* McsEngl.oncovirus-024!⇒vrsOnco,
    <br>* McsEngl.tumor-virus-024!⇒vrsOnco,
    <br>* McsEngl.vrsOnco,
    <a class="clsHide" href="#idOgmVrs024nam"></a></p>
  <p id="idOgmVrs024dsn">description::
    <br>"An oncovirus is a virus that can cause cancer. This term originated from studies of acutely transforming retroviruses in the 1950–60s,[1] when the term "oncornaviruses" was used to denote their RNA virus origin.[2] With the letters "RNA" removed, it now refers to any virus with a DNA or RNA genome causing cancer and is synonymous with "tumor virus" or "cancer virus". The vast majority of human and animal viruses do not cause cancer, probably because of longstanding co-evolution between the virus and its host. Oncoviruses have been important not only in epidemiology, but also in investigations of cell cycle control mechanisms such as the retinoblastoma protein.
    <br>The World Health Organization's International Agency for Research on Cancer estimated that in 2002, infection caused 17.8% of human cancers, with 11.9% caused by one of seven viruses.[3] A 2020 study of 2,658 samples from 38 different types of cancer found that 16% were associated with a virus.[4] These cancers might be easily prevented through vaccination (e.g., papillomavirus vaccines), diagnosed with simple blood tests, and treated with less-toxic antiviral compounds."
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Oncovirus]
    <a class="clsHide" href="#idOgmVrs024dsn"></a></p>
  </section>

  <section id="idOgmVrs025">
  <h2 id="idOgmVrs025H2">ogmVirus.synthetic-025
    <a class="clsHide" href="#idOgmVrs025H2"></a></h2>
  <p id="idOgmVrs025nam">name::
    <br>* McsEngl.ogmVirus.025-synthetic!⇒vrsSynthetic,
    <br>* McsEngl.ogmVirus.synthetic-025!⇒vrsSynthetic,
    <a class="clsHide" href="#idOgmVrs025nam"></a></p>
  <p id="idOgmVrs025dsn">description::
    <br>"Many viruses can be synthesised de novo ("from scratch") and the first synthetic virus was created in 2002.[267] Although somewhat of a misconception, it is not the actual virus that is synthesised, but rather its DNA genome (in case of a DNA virus), or a cDNA copy of its genome (in case of RNA viruses). For many virus families the naked synthetic DNA or RNA (once enzymatically converted back from the synthetic cDNA) is infectious when introduced into a cell. That is, they contain all the necessary information to produce new viruses. This technology is now being used to investigate novel vaccine strategies.[268] The ability to synthesise viruses has far-reaching consequences, since viruses can no longer be regarded as extinct, as long as the information of their genome sequence is known and permissive cells are available.
    <br>As of November 2017, the full-length genome sequences of 7454 different viruses, including smallpox, are publicly available in an online database maintained by the National Institutes of Health.[269]"
    <br>[{2020-04-15} https://en.wikipedia.org/wiki/Virus#Synthetic_viruses]
    <a class="clsHide" href="#idOgmVrs025dsn"></a></p>
  </section>
</section>

<section id="idOgmVrd">
  <h1 id="idOgmVrdH1">ogmCellNo.viroid-002
    <a class="clsHide" href="#idOgmVrdH1"></a></h1>
  <p id="idOgmVrdnam">name::
    <br>* McsEngl.ogmCellNo.002-viroid!⇒ogmViroid,
    <br>* McsEngl.ogmCellNo.viroid-002!⇒ogmViroid,
    <br>* McsEngl.ogmViroid,
    <a class="clsHide" href="#idOgmVrdnam"></a></p>
  <p id="idOgmVrddsn">description::
    <br>"Viroids are the smallest infectious pathogens known. They are composed solely of a short strand of circular, single-stranded RNA that has no protein coating. All known viroids are inhabitants of higher plants, and most cause diseases, whose respective economic importance on humans vary widely.
    <br>The first discoveries of viroids triggered the historically third major extension of the biosphere—to include smaller lifelike entities —after the discoveries, in 1675 by Antonie van Leeuwenhoek (of the "subvisible" microorganisms) and in 1892 by Dmitri Iosifovich Ivanovsky (of the "submicroscopic" viruses). The unique properties of viroids have been recognized by the International Committee on Taxonomy of Viruses, in creating a new order of subviral agents.[1]
    <br>The first recognized viroid, the pathogenic agent of the potato spindle tuber disease, was discovered, initially molecularly characterized, and named by Theodor Otto Diener, plant pathologist at the U.S Department of Agriculture's Research Center in Beltsville, Maryland, in 1971.[2][3] This viroid is now called Potato spindle tuber viroid, abbreviated PSTVd.
    <br>Although viroids are composed of nucleic acid, they do not code for any protein.[4][5] The viroid's replication mechanism uses RNA polymerase II, a host cell enzyme normally associated with synthesis of messenger RNA from DNA, which instead catalyzes "rolling circle" synthesis of new RNA using the viroid's RNA as a template. Some viroids are ribozymes, having catalytic properties that allow self-cleavage and ligation of unit-size genomes from larger replication intermediates.[6]
    <br>With Diener's 1989 hypothesis[7] that viroids may represent "living relics" from the widely assumed, ancient, and non-cellular RNA world—extant before the evolution of DNA or proteins—viroids have assumed significance beyond plant pathology to evolutionary science, by representing the most plausible RNAs capable of performing crucial steps in abiogenesis, the evolution of life from inanimate matter.
    <br>The human pathogen hepatitis D virus is a "defective" RNA virus similar to a viroid.[8]"
    <br>[{2020-04-13} https://en.wikipedia.org/wiki/Viroid]
    <a class="clsHide" href="#idOgmVrddsn"></a></p>
</section>

<section id="idMeta">
  <h1 id="idMetaH1">meta-info
    <a class="clsHide" href="#idMetaH1"></a></h1>
  <p id="idMetaCounter" class="clsCenter">this page was-visited
    <span class="clsColorRed">
    <script src="../../dirPgm/dirCntr/counter.php?page=McsOgm000008"></script>
    </span>
    times since {2020-04-02}</p>
  <!-- the content of page-path paragraph is displayed as it is on top of toc -->
  <p id="idMetaWebpage_path"><span class="clsB clsColorGreen">page-wholepath</span>:
    <a class="clsPreview" href="../../#idOverview">synagonism.net</a> /
    <a class="clsPreview" href="../Mcs000000.last.html#idOverview">worldviewSngo</a> /
    <a class="clsPreview" href="McsOgm000000.last.html#idOverview">dirOgm</a> /
    ogmCellNo
    </p>
  <p id="idMetaP1">SEARCH::
    <br>· this page uses '<span class="clsColorRed">locator-names</span>', names that when you find them, you find the-LOCATION of the-concept they denote.
    <br>⊛ <strong>GLOBAL-SEARCH</strong>:
    <br>· clicking on <span class="clsColorGreenBg">the-green-BAR of a-page</span> you have access to the-global--locator-names of my-site.
    <br>· use the-prefix '<span class="clsColorRed">ogmCellNo</span>' for <a class="clsPreview" href="../dirCor/McsCor000002.last.html#idOverview">sensorial-concepts</a> related to current concept 'cellNo-organism'.
    <br>⊛ <strong>LOCAL-SEARCH</strong>:
    <br>· TYPE <span class="clsColorRed">CTRL+F "McsLang.words-of-concept's-name"</span>, to go to the-LOCATION of the-concept.
    <br>· a-preview of the-description of a-global-name makes reading fast.
    <a class="clsHide" href="#idMetaP1"></a></p>
  <p id="idFooterP1">footer::
    <br>• author: <a class="clsPreview" href="../dirHmn/McsHmn000003.last.html#idOverview">Kaseluris.Nikos.1959</a>
    <br>• email:
    <br> &nbsp;<img src="../../dirRsc/dirImg/mail.png">
    <br>• edit on github: https://github.com/synagonism/McsWorld/blob/master/dirOgm/McsOgm000008.last.html,
    <br>• comments on <a class="clsPreview" href="McsOgm000000.last.html#idComment">Disqus</a>,
    <br>• twitter: <a href="https://twitter.com/synagonism">@synagonism</a>,
    <a class="clsHide" href="#idFooterP1"></a></p>
  <!--                              -->
  <p id="idMetaVersion">webpage-versions::
    <br>• version.last.dynamic: <a class="clsPreview" href="McsOgm000008.last.html">McsOgm000008.last.html</a>,
    <br>• version.1-0-0.2021-04-15: (0-5) <a href="../../dirMiwMcs/dirOgm/filMcsOgmCllN.1-0-0.2021-04-15.html">../../dirMiwMcs/dirOgm/filMcsOgmCllN.1-0-0.2021-04-15.html</a>,
    <br>• filMcsOgmCllN.0-1-0.2020-04-02.last.html: draft creation,
    <a class="clsHide" href="#idMetaVersion"></a></p>
</section>

<section id="idSupport">
  <h1 id="idSupportH1">support (<a class="clsPreview" href="../../#idSupport">link</a>)</h1>
  <p></p>
</section>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-19285371-5', 'synagonism.net');
  ga('send', 'pageview');
</script>

<script src="../Mcsmgr/hitp.js"></script>
</body>
</html>